<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.8.24">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Jannik Buhr">
<meta name="dcterms.date" content="2026-02-12">
<meta name="keywords" content="Collagen, Mechanochemistry, Hydrolysis, Quantum Mechanics / Molecular Mechanics, Molecular Dynamics, Research Software Engineering">

<title>Dissecting cleavage mechanisms in tensed collagen fibrils using reactive molecular dynamics simulations</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
html { -webkit-text-size-adjust: 100%; }
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
  margin-bottom: 0em;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="index_files/libs/clipboard/clipboard.min.js"></script>
<script src="index_files/libs/quarto-html/quarto.js" type="module"></script>
<script src="index_files/libs/quarto-html/tabsets/tabsets.js" type="module"></script>
<script src="index_files/libs/quarto-html/axe/axe-check.js" type="module"></script>
<script src="index_files/libs/quarto-html/popper.min.js"></script>
<script src="index_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="index_files/libs/quarto-html/anchor.min.js"></script>
<link href="index_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="index_files/libs/quarto-html/quarto-syntax-highlighting-dc55a5b9e770e841cd82e46aadbfb9b0.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-text-highlighting-styles">
<link href="index_files/libs/quarto-html/quarto-syntax-highlighting-dark-b651517ce65839d647a86e2780455cfb.css" rel="stylesheet" class="quarto-color-scheme quarto-color-alternate" id="quarto-text-highlighting-styles">
<link href="index_files/libs/quarto-html/quarto-syntax-highlighting-dc55a5b9e770e841cd82e46aadbfb9b0.css" rel="stylesheet" class="quarto-color-scheme-extra" id="quarto-text-highlighting-styles">
<script src="index_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="index_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="index_files/libs/bootstrap/bootstrap-347c7beb9f3f6df44ff8f742ab6b71c1.min.css" rel="stylesheet" append-hash="true" class="quarto-color-scheme" id="quarto-bootstrap" data-mode="light">
<link href="index_files/libs/bootstrap/bootstrap-dark-ac773e4cb3c72cc83dc4c11a6c85c077.min.css" rel="stylesheet" append-hash="true" class="quarto-color-scheme quarto-color-alternate" id="quarto-bootstrap" data-mode="dark">
<link href="index_files/libs/bootstrap/bootstrap-347c7beb9f3f6df44ff8f742ab6b71c1.min.css" rel="stylesheet" append-hash="true" class="quarto-color-scheme-extra" id="quarto-bootstrap" data-mode="light">
<script src="index_files/libs/quarto-contrib/molstar-v3.13.0/molstar.js"></script>
<link href="index_files/libs/quarto-contrib/molstar-v3.13.0/molstar.css" rel="stylesheet">
<link href="index_files/libs/quarto-contrib/molstar-v3.13.0/app-container.css" rel="stylesheet">
<script src="index_files/libs/quarto-contrib/glightbox/glightbox.min.js"></script>
<link href="index_files/libs/quarto-contrib/glightbox/glightbox.min.css" rel="stylesheet">
<link href="index_files/libs/quarto-contrib/glightbox/lightbox.css" rel="stylesheet">

  <script src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6"></script>
  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

<script type="text/javascript">
const typesetMath = (el) => {
  if (window.MathJax) {
    // MathJax Typeset
    window.MathJax.typeset([el]);
  } else if (window.katex) {
    // KaTeX Render
    var mathElements = el.getElementsByClassName("math");
    var macros = [];
    for (var i = 0; i < mathElements.length; i++) {
      var texText = mathElements[i].firstChild;
      if (mathElements[i].tagName == "SPAN" && texText && texText.data) {
        window.katex.render(texText.data, mathElements[i], {
          displayMode: mathElements[i].classList.contains('display'),
          throwOnError: false,
          macros: macros,
          fleqn: false
        });
      }
    }
  }
}
window.Quarto = {
  typesetMath
};
</script>

<link rel="stylesheet" href="style.css">
<meta name="citation_title" content="Dissecting cleavage mechanisms\
in tensed collagen fibrils\
using reactive molecular dynamics simulations
">
<meta name="citation_keywords" content="Collagen,Mechanochemistry,Hydrolysis,Quantum Mechanics / Molecular Mechanics,Molecular Dynamics,Research Software Engineering">
<meta name="citation_author" content="Jannik Buhr">
<meta name="citation_publication_date" content="2026-02-12">
<meta name="citation_cover_date" content="2026-02-12">
<meta name="citation_year" content="2026">
<meta name="citation_online_date" content="2026-02-12">
<meta name="citation_fulltext_html_url" content="https://jmbuhr.de/phd-thesis">
<meta name="citation_pdf_url" content="https://jmbuhr.de/phd-thesis/index.pdf">
<meta name="citation_language" content="en">
<meta name="citation_dissertation_institution" content="Heidelberg University">
<meta name="citation_reference" content="citation_title=GROMACS: High performance molecular simulations through multi-level parallelism from laptops to supercomputers;,citation_abstract=GROMACS is one of the most widely used open-source and free software codes in chemistry, used primarily for dynamical simulations of biomolecules. It provides a rich set of calculation types, preparation and analysis tools. Several advanced techniques for free-energy calculations are supported. In version 5, it reaches new performance heights, through several new and enhanced parallelization algorithms. These work on every level; SIMD registers inside cores, multithreading, heterogeneous CPU–GPU acceleration, state-of-the-art 3D domain decomposition, and ensemble-level parallelization through built-in replica exchange and the separate Copernicus framework. The latest best-in-class compressed trajectory storage format is supported.;,citation_author=Mark James Abraham;,citation_author=Teemu Murtola;,citation_author=Roland Schulz;,citation_author=Szilárd Páll;,citation_author=Jeremy C. Smith;,citation_author=Berk Hess;,citation_author=Erik Lindahl;,citation_publication_date=2015-09;,citation_cover_date=2015-09;,citation_year=2015;,citation_fulltext_html_url=https://www.sciencedirect.com/science/article/pii/S2352711015000059;,citation_doi=10.1016/j.softx.2015.06.001;,citation_issn=2352-7110;,citation_volume=1–2;,citation_language=en-US;,citation_journal_title=SoftwareX;">
<meta name="citation_reference" content="citation_title=Toward reliable density functional methods without adjustable parameters: The PBE0 model;,citation_abstract=We present an analysis of the performances of a parameter free density functional model (PBE0) obtained combining the so called PBE generalized gradient functional with a predefined amount of exact exchange. The results obtained for structural, thermodynamic, kinetic and spectroscopic (magnetic, infrared and electronic) properties are satisfactory and not far from those delivered by the most reliable functionals including heavy parameterization. The way in which the functional is derived and the lack of empirical parameters fitted to specific properties make the PBE0 model a widely applicable method for both quantum chemistry and condensed matter physics.;,citation_author=Carlo Adamo;,citation_author=Vincenzo Barone;,citation_publication_date=1999-04;,citation_cover_date=1999-04;,citation_year=1999;,citation_fulltext_html_url=https://doi.org/10.1063/1.478522;,citation_issue=13;,citation_doi=10.1063/1.478522;,citation_issn=0021-9606;,citation_volume=110;,citation_journal_title=The Journal of Chemical Physics;">
<meta name="citation_reference" content="citation_title=Scalable Constant pH Molecular Dynamics in GROMACS;,citation_abstract=Molecular dynamics (MD) computer simulations are used routinely to compute atomistic trajectories of complex systems. Systems are simulated in various ensembles, depending on the experimental conditions one aims to mimic. While constant energy, temperature, volume, and pressure are rather straightforward to model, pH, which is an equally important parameter in experiments, is more difficult to account for in simulations. Although a constant pH algorithm based on the λ-dynamics approach by Brooks and co-workers [Kong, X.; Brooks III, C. L. J. Chem. Phys.1996, 105, 2414–2423] was implemented in a fork of the GROMACS molecular dynamics program, uptake has been rather limited, presumably due to the poor scaling of that code with respect to the number of titratable sites. To overcome this limitation, we implemented an alternative scheme for interpolating the Hamiltonians of the protonation states that makes the constant pH molecular dynamics simulations almost as fast as a normal MD simulation with GROMACS. In addition, we implemented a simpler scheme, called multisite representation, for modeling side chains with multiple titratable sites, such as imidazole rings. This scheme, which is based on constraining the sum of the λ-coordinates, not only reduces the complexity associated with parametrizing the intramolecular interactions between the sites but also is easily extendable to other molecules with multiple titratable sites. With the combination of a more efficient interpolation scheme and multisite representation of titratable groups, we anticipate a rapid uptake of constant pH molecular dynamics simulations within the GROMACS user community.;,citation_author=Noora Aho;,citation_author=Pavel Buslaev;,citation_author=Anton Jansen;,citation_author=Paul Bauer;,citation_author=Gerrit Groenhof;,citation_author=Berk Hess;,citation_publication_date=2022-10;,citation_cover_date=2022-10;,citation_year=2022;,citation_fulltext_html_url=https://doi.org/10.1021/acs.jctc.2c00516;,citation_issue=10;,citation_doi=10.1021/acs.jctc.2c00516;,citation_issn=1549-9618;,citation_volume=18;,citation_journal_title=Journal of Chemical Theory and Computation;,citation_publisher=American Chemical Society;">
<meta name="citation_reference" content="citation_title=Motional timescale predictions by molecular dynamics simulations: Case study using proline and hydroxyproline sidechain dynamics: Proline Force Field Parameters;,citation_abstract=We propose a new approach for force field optimizations which aims at reproducing dynamics characteristics using biomolecular MD simulations, in addition to improved prediction of motionally averaged structural properties available from experiment. As the source of experimental data for dynamics fittings, we use 13C NMR spin‐lattice relaxation times T1 of backbone and sidechain carbons, which allow to determine correlation times of both overall molecular and intramolecular motions. For structural fittings, we use motionally averaged experimental values of NMR J couplings. The proline residue and its derivative 4‐hydroxyproline with relatively simple cyclic structure and sidechain dynamics were chosen for the assessment of the new approach in this work. Initially, grid search and simplexed MD simulations identified large number of parameter sets which fit equally well experimental J couplings. Using the Arrhenius‐type relationship between the force constant and the correlation time, the available MD data for a series of parameter sets were analyzed to predict the value of the force constant that best reproduces experimental timescale of the sidechain dynamics. Verification of the new force‐field (termed as AMBER99SB‐ILDNP) against NMR J couplings and correlation times showed consistent and significant improvements compared to the original force field in reproducing both structural and dynamics properties. The results suggest that matching experimental timescales of motions together with motionally averaged characteristics is the valid approach for force field parameter optimization. Such a comprehensive approach is not restricted to cyclic residues and can be extended to other amino acid residues, as well as to the backbone. Proteins 2014; 82:195–215. © 2013 Wiley Periodicals, Inc.;,citation_author=Abil E. Aliev;,citation_author=Martin Kulke;,citation_author=Harmeet S. Khaneja;,citation_author=Vijay Chudasama;,citation_author=Tom D. Sheppard;,citation_author=Rachel M. Lanigan;,citation_publication_date=2014-02;,citation_cover_date=2014-02;,citation_year=2014;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/10.1002/prot.24350;,citation_issue=2;,citation_doi=10.1002/prot.24350;,citation_issn=08873585;,citation_volume=82;,citation_language=en-US;,citation_journal_title=Proteins: Structure, Function, and Bioinformatics;">
<meta name="citation_reference" content="citation_title=Quarto;,citation_author=J. J. Allaire;,citation_author=Charles Teague;,citation_author=Carlos Scheidegger;,citation_author=Yihui Xie;,citation_author=Christophe Dervieux;,citation_publication_date=2022-01;,citation_cover_date=2022-01;,citation_year=2022;,citation_fulltext_html_url=https://github.com/quarto-dev/quarto-cli;,citation_doi=10.5281/zenodo.5960048;">
<meta name="citation_reference" content="citation_title=Über die Reaktionsgeschwindigkeit bei der Inversion von Rohrzucker durch Säuren;,citation_author=Svante Arrhenius;,citation_publication_date=1889-07;,citation_cover_date=1889-07;,citation_year=1889;,citation_fulltext_html_url=https://www.degruyter.com/document/doi/10.1515/zpch-1889-0416/html;,citation_issue=1;,citation_doi=10.1515/zpch-1889-0416;,citation_issn=2196-7156, 0942-9352;,citation_volume=4U;,citation_language=en-US;,citation_journal_title=Zeitschrift für Physikalische Chemie;">
<meta name="citation_reference" content="citation_title=Sailors’ scurvy before and after James Lind – a reassessment;,citation_abstract=Scurvy is a thousand-year-old stereotypical disease characterized by apathy, weakness, easy bruising with tiny or large skin hemorrhages, friable bleeding gums, and swollen legs. Untreated patients may die. In the last five centuries sailors and some ships’ doctors used oranges and lemons to cure and prevent scurvy, yet university-trained European physicians with no experience of either the disease or its cure by citrus fruits persisted in reviews of the extensive but conflicting literature. In the 20th century scurvy was shown to be due to a deficiency of the essential food factor ascorbic acid. This vitamin C was synthesized, and in adequate quantities it completely prevents and completely cures the disease, which is now rare. The protagonist of this medical history was James Lind. His report of a prospective controlled therapeutic trial in 1747 preceded by a half-century the British Navy’s prevention and cure of scurvy by citrus fruits. After lime-juice was unwittingly substituted for lemon juice in about 1860, the disease returned, especially among sailors on polar explorations. In recent decades revisionist historians have challenged normative accounts, including that of scurvy, and the historicity of Lind’s trial. It is therefore timely to reassess systematically the strengths and weaknesses of the canonical saga.;,citation_author=Jeremy Hugh Baron;,citation_publication_date=2009-06;,citation_cover_date=2009-06;,citation_year=2009;,citation_fulltext_html_url=https://doi.org/10.1111/j.1753-4887.2009.00205.x;,citation_issue=6;,citation_doi=10.1111/j.1753-4887.2009.00205.x;,citation_issn=0029-6643;,citation_volume=67;,citation_journal_title=Nutrition Reviews;">
<meta name="citation_reference" content="citation_title=A well-behaved electrostatic potential based method using charge restraints for deriving atomic charges: The RESP model;,citation_author=Christopher I. Bayly;,citation_author=Piotr Cieplak;,citation_author=Wendy Cornell;,citation_author=Peter A. Kollman;,citation_publication_date=2002-05;,citation_cover_date=2002-05;,citation_year=2002;,citation_fulltext_html_url=https://pubs.acs.org/doi/pdf/10.1021/j100142a004;,citation_doi=10.1021/j100142a004;,citation_language=en-US;,citation_journal_title=ACS Publications;,citation_publisher=American Chemical Society;">
<meta name="citation_reference" content="citation_title=Density‐functional thermochemistry. III. The role of exact exchange;,citation_abstract=Despite the remarkable thermochemical accuracy of Kohn–Sham density‐functional theories with gradient corrections for exchange‐correlation [see, for example, A. D. Becke, J. Chem. Phys. 96, 2155 (1992)], we believe that further improvements are unlikely unless exact‐exchange information is considered. Arguments to support this view are presented, and a semiempirical exchange‐correlation functional containing local‐spin‐density, gradient, and exact‐exchange terms is tested on 56 atomization energies, 42 ionization potentials, 8 proton affinities, and 10 total atomic energies of first‐ and second‐row systems. This functional performs significantly better than previous functionals with gradient corrections only, and fits experimental atomization energies with an impressively small average absolute deviation of 2.4 kcal/mol.;,citation_author=Axel D. Becke;,citation_publication_date=1993-04;,citation_cover_date=1993-04;,citation_year=1993;,citation_fulltext_html_url=https://doi.org/10.1063/1.464913;,citation_issue=7;,citation_doi=10.1063/1.464913;,citation_issn=0021-9606;,citation_volume=98;,citation_journal_title=The Journal of Chemical Physics;">
<meta name="citation_reference" content="citation_title=Generalized Neural-Network Representation of High-Dimensional Potential-Energy Surfaces;,citation_abstract=The accurate description of chemical processes often requires the use of computationally demanding methods like density-functional theory (DFT), making long simulations of large systems unfeasible. In this Letter we introduce a new kind of neural-network representation of DFT potential-energy surfaces, which provides the energy and forces as a function of all atomic positions in systems of arbitrary size and is several orders of magnitude faster than DFT. The high accuracy of the method is demonstrated for bulk silicon and compared with empirical potentials and DFT. The method is general and can be applied to all types of periodic and nonperiodic systems.;,citation_author=Jörg Behler;,citation_author=Michele Parrinello;,citation_publication_date=2007-04;,citation_cover_date=2007-04;,citation_year=2007;,citation_fulltext_html_url=https://link.aps.org/doi/10.1103/PhysRevLett.98.146401;,citation_issue=14;,citation_doi=10.1103/PhysRevLett.98.146401;,citation_volume=98;,citation_journal_title=Physical Review Letters;,citation_publisher=American Physical Society;">
<meta name="citation_reference" content="citation_title=Models for the Specific Adhesion of Cells to Cells;,citation_abstract=A theoretical framework is proposed for the analysis of adhesion between cells or of cells to surfaces when the adhesion is mediated by reversible bonds between specific molecules such as antigen and antibody, lectin and carbohydrate, or enzyme and substrate. From a knowledge of the reaction rates for reactants in solution and of their diffusion constants both in solution and on membranes, it is possible to estimate reaction rates for membrane-bound reactants. Two models are developed for predicting the rate of bond formation between cells and are compared with experiments. The force required to separate two cells is shown to be greater than the expected electrical forces between cells, and of the same order of magnitude as the forces required to pull gangliosides and perhaps some integral membrane proteins out of the cell membrane.;,citation_author=George I. Bell;,citation_publication_date=1978-05;,citation_cover_date=1978-05;,citation_year=1978;,citation_fulltext_html_url=https://www.science.org/doi/abs/10.1126/science.347575;,citation_issue=4342;,citation_doi=10.1126/science.347575;,citation_volume=200;,citation_journal_title=Science;,citation_publisher=American Association for the Advancement of Science;">
<meta name="citation_reference" content="citation_title=GROMACS: A message-passing parallel molecular dynamics implementation;,citation_abstract=A parallel message-passing implementation of a molecular dynamics (MD) program that is useful for bio(macro)molecules in aqueous environment is described. The software has been developed for a custom-designed 32-processor ring GROMACS (GROningen MAchine for Chemical Simulation) with communication to and from left and right neighbours, but can run on any parallel system onto which a a ring of processors can be mapped and which supports PVM-like block send and receive calls. The GROMACS software consists of a preprocessor, a parallel MD and energy minimization program that can use an arbitrary number of processors (including one), an optional monitor, and several analysis tools. The programs are written in ANSI C and available by ftp (information: gromacs@chem.rug.nl). The functionality is based on the GROMOS (GROningen MOlecular Simulation) package (van Gunsteren and Berendsen, 1987; BIOMOS B.V., Nijenborgh 4, 9747 AG Groningen). Conversion programs between GROMOS and GROMACS formats are included. The MD program can handle rectangular periodic boundary conditions with temperature and pressure scaling. The interactions that can be handled without modification are variable non-bonded pair interactions with Coulomb and Lennard-Jones or Buckingham potentials, using a twin-range cut-off based on charge groups, and fixed bonded interactions of either harmonic or constraint type for bonds and bond angles and either periodic or cosine power series interactions for dihedral angles. Special forces can be added to groups of particles (for non-equilibrium dynamics or for position restraining) or between particles (for distance restraints). The parallelism is based on particle decomposition. Interprocessor communication is largely limited to position and force distribution over the ring once per time step.;,citation_author=H. J. C. Berendsen;,citation_author=D. Spoel;,citation_author=R. Drunen;,citation_publication_date=1995-09;,citation_cover_date=1995-09;,citation_year=1995;,citation_fulltext_html_url=http://www.sciencedirect.com/science/article/pii/001046559500042E;,citation_issue=1;,citation_doi=10.1016/0010-4655(95)00042-E;,citation_issn=0010-4655;,citation_volume=91;,citation_language=en-US;,citation_journal_title=Computer Physics Communications;">
<meta name="citation_reference" content="citation_title=Molecular dynamics with coupling to an external bath;,citation_abstract=In molecular dynamics (MD) simulations the need often arises to maintain such parameters as temperature or pressure rather than energy and volume, or to impose gradients for studying transport properties in nonequilibrium MD. A method is described to realize coupling to an external bath with constant temperature or pressure with adjustable time constants for the coupling. The method is easily extendable to other variables and to gradients, and can be applied also to polyatomic molecules involving internal constraints. The influence of coupling time constants on dynamical variables is evaluated. A leap‐frog algorithm is presented for the general case involving constraints with coupling to both a constant temperature and a constant pressure bath.;,citation_author=H. J. C. Berendsen;,citation_author=J. P. M. Postma;,citation_author=W. F. Gunsteren;,citation_author=A. DiNola;,citation_author=J. R. Haak;,citation_publication_date=1984-10;,citation_cover_date=1984-10;,citation_year=1984;,citation_fulltext_html_url=https://doi.org/10.1063/1.448118;,citation_issue=8;,citation_doi=10.1063/1.448118;,citation_issn=0021-9606;,citation_volume=81;,citation_journal_title=The Journal of Chemical Physics;">
<meta name="citation_reference" content="citation_title=The Protein Data Bank archive as an open data resource;,citation_abstract=The Protein Data Bank archive was established in 1971, and recently celebrated its 40th anniversary (Berman et al. in Structure&nbsp;20:391, 2012). An analysis of interrelationships of the science, technology and community leads to further insights into how this resource evolved into one of the oldest and most widely used open-access data resources in biology.;,citation_author=Helen M. Berman;,citation_author=Gerard J. Kleywegt;,citation_author=Haruki Nakamura;,citation_author=John L. Markley;,citation_publication_date=2014-10;,citation_cover_date=2014-10;,citation_year=2014;,citation_fulltext_html_url=https://doi.org/10.1007/s10822-014-9770-y;,citation_issue=10;,citation_doi=10.1007/s10822-014-9770-y;,citation_issn=1573-4951;,citation_volume=28;,citation_language=en-US;,citation_journal_title=Journal of Computer-Aided Molecular Design;">
<meta name="citation_reference" content="citation_title=Sur le mélange des gaz;,citation_author=Daniel Berthelot;,citation_publication_date=1898;,citation_cover_date=1898;,citation_year=1898;,citation_issue=3;,citation_volume=126;,citation_journal_title=Compt. Rendus;">
<meta name="citation_reference" content="citation_title=Knee muscle forces during walking and running in patellofemoral pain patients and pain-free controls;,citation_abstract=One proposed mechanism of patellofemoral pain, increased stress in the joint, is dependent on forces generated by the quadriceps muscles. Describing causal relationships between muscle forces, tissue stresses, and pain is difficult due to the inability to directly measure these variables in vivo. The purpose of this study was to estimate quadriceps forces during walking and running in a group of male and female patients with patellofemoral pain (n=27, 16 female; 11 male) and compare these to pain-free controls (n=16, 8 female; 8 male). Subjects walked and ran at self-selected speeds in a gait laboratory. Lower limb kinematics and electromyography (EMG) data were input to an EMG-driven musculoskeletal model of the knee, which was scaled and calibrated to each individual to estimate forces in 10 muscles surrounding the joint. Compared to controls, the patellofemoral pain group had greater co-contraction of quadriceps and hamstrings (p=0.025) and greater normalized muscle forces during walking, even though the net knee moment was similar between groups. Muscle forces during running were similar between groups, but the net knee extension moment was less in the patellofemoral pain group compared to controls. Females displayed 30–50% greater normalized hamstring and gastrocnemius muscle forces during both walking and running compared to males (p$&amp;amp;amp;lt;$0.05). These results suggest that some patellofemoral pain patients might experience greater joint contact forces and joint stresses than pain-free subjects. The muscle force data are available as supplementary material.;,citation_author=Thor F. Besier;,citation_author=Michael Fredericson;,citation_author=Garry E. Gold;,citation_author=Gary S. Beaupré;,citation_author=Scott L. Delp;,citation_publication_date=2009-05;,citation_cover_date=2009-05;,citation_year=2009;,citation_fulltext_html_url=https://www.sciencedirect.com/science/article/pii/S0021929009000396;,citation_issue=7;,citation_doi=10.1016/j.jbiomech.2009.01.032;,citation_issn=0021-9290;,citation_volume=42;,citation_journal_title=Journal of Biomechanics;">
<meta name="citation_reference" content="citation_title=Optimized Molecular Dynamics Force Fields Applied to the Helix−Coil Transition of Polypeptides;,citation_abstract=Obtaining the correct balance of secondary structure propensities is a central priority in protein force-field development. Given that current force fields differ significantly in their α-helical propensities, a correction to match experimental results would be highly desirable. We have determined simple backbone energy corrections for two force fields to reproduce the fraction of helix measured in short peptides at 300 K. As validation, we show that the optimized force fields produce results in excellent agreement with nuclear magnetic resonance experiments for folded proteins and short peptides not used in the optimization. However, despite the agreement at ambient conditions, the dependence of the helix content on temperature is too weak, a problem shared with other force fields. A fit of the Lifson−Roig helix−coil theory shows that both the enthalpy and entropy of helix formation are too small: the helix extension parameter w agrees well with experiment, but its entropic and enthalpic components are both only about half the respective experimental estimates. Our structural and thermodynamic analyses point toward the physical origins of these shortcomings in current force fields, and suggest ways to address them in future force-field development.;,citation_author=Robert B. Best;,citation_author=Gerhard Hummer;,citation_publication_date=2009-07;,citation_cover_date=2009-07;,citation_year=2009;,citation_fulltext_html_url=https://doi.org/10.1021/jp901540t;,citation_issue=26;,citation_doi=10.1021/jp901540t;,citation_issn=1520-6106;,citation_volume=113;,citation_journal_title=The Journal of Physical Chemistry B;,citation_publisher=American Chemical Society;">
<meta name="citation_reference" content="citation_title=Mechanochemistry:  The Mechanical Activation of Covalent Bonds;,citation_author=Martin K. Beyer;,citation_author=Hauke Clausen-Schaumann;,citation_publication_date=2005-08;,citation_cover_date=2005-08;,citation_year=2005;,citation_fulltext_html_url=https://doi.org/10.1021/cr030697h;,citation_issue=8;,citation_doi=10.1021/cr030697h;,citation_issn=0009-2665;,citation_volume=105;,citation_journal_title=Chemical Reviews;,citation_publisher=American Chemical Society;">
<meta name="citation_reference" content="citation_title=Mechanical properties, physiological behavior, and function of aponeurosis and tendon;,citation_abstract=During human movement, the muscle and tendinous structures interact as a mechanical system in which forces are generated and transmitted to the bone and energy is stored and released to optimize function and economy of movement and/or to reduce risk of injury. The present review addresses certain aspects of how the anatomical design and mechanical and material properties of the force-transmitting tissues contribute to the function of the muscle-tendon unit and thus overall human function. The force-bearing tissues are examined from a structural macroscopic point of view down to the nanoscale level of the collagen fibril. In recent years, the understanding of in vivo mechanical function of the force-bearing tissues has increased, and it has become clear that these tissues adapt to loading and unloading and furthermore that force transmission mechanics is more complex than previously thought. Future investigations of the force-transmitting tissues in three dimensions will enable a greater understanding of the complex functional interplay between muscle and tendon, with relevance for performance, injury mechanisms, and rehabilitation strategies.;,citation_author=Jens Bojsen-Møller;,citation_author=S. Peter Magnusson;,citation_publication_date=2019-06;,citation_cover_date=2019-06;,citation_year=2019;,citation_fulltext_html_url=https://journals.physiology.org/doi/full/10.1152/japplphysiol.00671.2018;,citation_issue=6;,citation_doi=10.1152/japplphysiol.00671.2018;,citation_issn=8750-7587;,citation_volume=126;,citation_journal_title=Journal of Applied Physiology;,citation_publisher=American Physiological Society;">
<meta name="citation_reference" content="citation_title=Einige allgemeine sätze über Wärmegleichgewicht;,citation_author=Ludwig Eduard Boltzmann;,citation_publication_date=1871;,citation_cover_date=1871;,citation_year=1871;">
<meta name="citation_reference" content="citation_title=Vorlesungen über Gastheorie;,citation_author=Ludwig Boltzmann;,citation_publication_date=1896;,citation_cover_date=1896;,citation_year=1896;,citation_language=de-DE;">
<meta name="citation_reference" content="citation_title=Zur Quantenmechanik der Stoßvorgänge;,citation_abstract=Durch eine Untersuchung der Stoßvorgänge wird die Auffassung entwickelt, daß die Quantenmechanik in der Schrödingerschen Form nicht nur die stationären Zustände, sondern auch die Quantensprünge zu beschreiben gestattet.;,citation_author=Max Born;,citation_publication_date=1926-12;,citation_cover_date=1926-12;,citation_year=1926;,citation_fulltext_html_url=https://doi.org/10.1007/BF01397477;,citation_issue=12;,citation_doi=10.1007/BF01397477;,citation_issn=0044-3328;,citation_volume=37;,citation_language=de-DE;,citation_journal_title=Zeitschrift für Physik;">
<meta name="citation_reference" content="citation_title=Zur Quantentheorie der Molekeln;,citation_author=M. Born;,citation_author=R. Oppenheimer;,citation_publication_date=1927;,citation_cover_date=1927;,citation_year=1927;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1002/andp.19273892002;,citation_issue=20;,citation_doi=10.1002/andp.19273892002;,citation_issn=1521-3889;,citation_volume=389;,citation_language=en-US;,citation_journal_title=Annalen der Physik;">
<meta name="citation_reference" content="citation_title=A new algorithm for Monte Carlo simulation of Ising spin systems;,citation_author=A. B. Bortz;,citation_author=M. H. Kalos;,citation_author=J. L. Lebowitz;,citation_publication_date=1975-01;,citation_cover_date=1975-01;,citation_year=1975;,citation_fulltext_html_url=https://linkinghub.elsevier.com/retrieve/pii/0021999175900601;,citation_issue=1;,citation_doi=10.1016/0021-9991(75)90060-1;,citation_issn=00219991;,citation_volume=17;,citation_language=en-US;,citation_journal_title=Journal of Computational Physics;">
<meta name="citation_reference" content="citation_title=Molecular Structure of the Collagen Triple Helix;,citation_abstract=The molecular conformation of the collagen triple helix confers strict amino acid sequence constraints, requiring a (Gly-X-Y)n repeating pattern and a high content of imino acids. The increasing family of collagens and proteins with collagenous domains shows the collagen triple helix to be a basic motif adaptable to a range of proteins and functions. Its rodlike domain has the potential for various modes of self-association and the capacity to bind receptors, other proteins, GAGs, and nucleic acids. High-resolution crystal structures obtained for collagen model peptides confirm the supercoiled triple helix conformation, and provide new information on hydrogen bonding patterns, hydration, sidechain interactions, and ligand binding. For several peptides, the helix twist was found to be sequence dependent, and such variation in helix twist may serve as recognition features or to orient the triple helix for binding. Mutations in the collagen triple-helix domain lead to a variety of human disorders. The most common mutations are single-base substitutions that lead to the replacement of one Gly residue, breaking the Gly-X-Y repeating pattern. A single Gly substitution destabilizes the triple helix through a local disruption in hydrogen bonding and produces a discontinuity in the register of the helix. Molecular information about the collagen triple helix and the effect of mutations will lead to a better understanding of function and pathology.;,citation_author=Barbara Brodsky;,citation_author=Anton V. Persikov;,citation_publication_date=2005-01;,citation_cover_date=2005-01;,citation_year=2005;,citation_fulltext_html_url=https://www.sciencedirect.com/science/article/pii/S0065323305700097;,citation_doi=10.1016/S0065-3233(05)70009-7;,citation_volume=70;,citation_inbook_title=Advances in Protein Chemistry;,citation_series_title=Fibrous Proteins: Coiled-Coils, Collagen and Elastomers;">
<meta name="citation_reference" content="citation_title=CHARMM: The biomolecular simulation program;,citation_abstract=CHARMM (Chemistry at HARvard Molecular Mechanics) is a highly versatile and widely used molecular simulation program. It has been developed over the last three decades with a primary focus on molecules of biological interest, including proteins, peptides, lipids, nucleic acids, carbohydrates, and small molecule ligands, as they occur in solution, crystals, and membrane environments. For the study of such systems, the program provides a large suite of computational tools that include numerous conformational and path sampling methods, free energy estimators, molecular minimization, dynamics, and analysis techniques, and model-building capabilities. The CHARMM program is applicable to problems involving a much broader class of many-particle systems. Calculations with CHARMM can be performed using a number of different energy functions and models, from mixed quantum mechanical-molecular mechanical force fields, to all-atom classical potential energy functions with explicit solvent and various boundary conditions, to implicit solvent and membrane models. The program has been ported to numerous platforms in both serial and parallel architectures. This article provides an overview of the program as it exists today with an emphasis on developments since the publication of the original CHARMM article in 1983. © 2009 Wiley Periodicals, Inc.J Comput Chem, 2009.;,citation_author=B. R. Brooks;,citation_author=C. L. Brooks;,citation_author=A. D. Mackerell;,citation_author=L. Nilsson;,citation_author=R. J. Petrella;,citation_author=B. Roux;,citation_author=Y. Won;,citation_author=G. Archontis;,citation_author=C. Bartels;,citation_author=S. Boresch;,citation_author=A. Caflisch;,citation_author=L. Caves;,citation_author=Q. Cui;,citation_author=A. R. Dinner;,citation_author=M. Feig;,citation_author=S. Fischer;,citation_author=J. Gao;,citation_author=M. Hodoscek;,citation_author=W. Im;,citation_author=K. Kuczera;,citation_author=T. Lazaridis;,citation_author=J. Ma;,citation_author=V. Ovchinnikov;,citation_author=E. Paci;,citation_author=R. W. Pastor;,citation_author=C. B. Post;,citation_author=J. Z. Pu;,citation_author=M. Schaefer;,citation_author=B. Tidor;,citation_author=R. M. Venable;,citation_author=H. L. Woodcock;,citation_author=X. Wu;,citation_author=W. Yang;,citation_author=D. M. York;,citation_author=M. Karplus;,citation_publication_date=2009;,citation_cover_date=2009;,citation_year=2009;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1002/jcc.21287;,citation_issue=10;,citation_doi=10.1002/jcc.21287;,citation_issn=1096-987X;,citation_volume=30;,citation_language=en-US;,citation_journal_title=Journal of Computational Chemistry;">
<meta name="citation_reference" content="citation_title=CHARMM: The biomolecular simulation program;,citation_abstract=CHARMM (Chemistry at HARvard Molecular Mechanics) is a highly versatile and widely used molecular simulation program. It has been developed over the last three decades with a primary focus on molecules of biological interest, including proteins, peptides, lipids, nucleic acids, carbohydrates, and small molecule ligands, as they occur in solution, crystals, and membrane environments. For the study of such systems, the program provides a large suite of computational tools that include numerous conformational and path sampling methods, free energy estimators, molecular minimization, dynamics, and analysis techniques, and model-building capabilities. The CHARMM program is applicable to problems involving a much broader class of many-particle systems. Calculations with CHARMM can be performed using a number of different energy functions and models, from mixed quantum mechanical-molecular mechanical force fields, to all-atom classical potential energy functions with explicit solvent and various boundary conditions, to implicit solvent and membrane models. The program has been ported to numerous platforms in both serial and parallel architectures. This article provides an overview of the program as it exists today with an emphasis on developments since the publication of the original CHARMM article in 1983. © 2009 Wiley Periodicals, Inc.J Comput Chem, 2009.;,citation_author=B. R. Brooks;,citation_author=C. L. Brooks III;,citation_author=A. D. Mackerell Jr.;,citation_author=L. Nilsson;,citation_author=R. J. Petrella;,citation_author=B. Roux;,citation_author=Y. Won;,citation_author=G. Archontis;,citation_author=C. Bartels;,citation_author=S. Boresch;,citation_author=A. Caflisch;,citation_author=L. Caves;,citation_author=Q. Cui;,citation_author=A. R. Dinner;,citation_author=M. Feig;,citation_author=S. Fischer;,citation_author=J. Gao;,citation_author=M. Hodoscek;,citation_author=W. Im;,citation_author=K. Kuczera;,citation_author=T. Lazaridis;,citation_author=J. Ma;,citation_author=V. Ovchinnikov;,citation_author=E. Paci;,citation_author=R. W. Pastor;,citation_author=C. B. Post;,citation_author=J. Z. Pu;,citation_author=M. Schaefer;,citation_author=B. Tidor;,citation_author=R. M. Venable;,citation_author=H. L. Woodcock;,citation_author=X. Wu;,citation_author=W. Yang;,citation_author=D. M. York;,citation_author=M. Karplus;,citation_publication_date=2009;,citation_cover_date=2009;,citation_year=2009;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1002/jcc.21287;,citation_issue=10;,citation_doi=10.1002/jcc.21287;,citation_issn=1096-987X;,citation_volume=30;,citation_language=en-US;,citation_journal_title=Journal of Computational Chemistry;">
<meta name="citation_reference" content="citation_title=Stereochemistry of reaction paths at carbonyl centres;,citation_abstract=Results of recent experimental and theoretical studies of nucleophilic addition to carbonyl groups are described. The reaction paths found by different methods for different nucleophiles show some striking similarities that appear to be characteristic for the reaction type.;,citation_author=H. B. Bürgi;,citation_author=J. D. Dunitz;,citation_author=J. M. Lehn;,citation_author=G. Wipff;,citation_publication_date=1974-01;,citation_cover_date=1974-01;,citation_year=1974;,citation_fulltext_html_url=https://www.sciencedirect.com/science/article/pii/S0040402001906787;,citation_issue=12;,citation_doi=10.1016/S0040-4020(01)90678-7;,citation_issn=0040-4020;,citation_volume=30;,citation_language=en-US;,citation_journal_title=Tetrahedron;">
<meta name="citation_reference" content="citation_title=Canonical sampling through velocity rescaling;,citation_abstract=The authors present a new molecular dynamics algorithm for sampling the canonical distribution. In this approach the velocities of all the particles are rescaled by a properly chosen random factor. The algorithm is formally justified and it is shown that, in spite of its stochastic nature, a quantity can still be defined that remains constant during the evolution. In numerical applications this quantity can be used to measure the accuracy of the sampling. The authors illustrate the properties of this new method on Lennard-Jones and TIP4P water models in the solid and liquid phases. Its performance is excellent and largely independent of the thermostat parameter also with regard to the dynamic properties.;,citation_author=Giovanni Bussi;,citation_author=Davide Donadio;,citation_author=Michele Parrinello;,citation_publication_date=2007-01;,citation_cover_date=2007-01;,citation_year=2007;,citation_fulltext_html_url=https://doi.org/10.1063/1.2408420;,citation_issue=1;,citation_doi=10.1063/1.2408420;,citation_issn=0021-9606;,citation_volume=126;,citation_journal_title=The Journal of Chemical Physics;">
<meta name="citation_reference" content="citation_title=AmberTools;,citation_abstract=AmberTools is a free and open-source collection of programs used to set up, run, and analyze molecular simulations. The newer features contained within AmberTools23 are briefly described in this Application note.;,citation_author=David A. Case;,citation_author=Hasan Metin Aktulga;,citation_author=Kellon Belfon;,citation_author=David S. Cerutti;,citation_author=G. Andrés Cisneros;,citation_author=Vinícius Wilian D. Cruzeiro;,citation_author=Negin Forouzesh;,citation_author=Timothy J. Giese;,citation_author=Andreas W. Götz;,citation_author=Holger Gohlke;,citation_author=Saeed Izadi;,citation_author=Koushik Kasavajhala;,citation_author=Mehmet C. Kaymak;,citation_author=Edward King;,citation_author=Tom Kurtzman;,citation_author=Tai-Sung Lee;,citation_author=Pengfei Li;,citation_author=Jian Liu;,citation_author=Tyler Luchko;,citation_author=Ray Luo;,citation_author=Madushanka Manathunga;,citation_author=Matias R. Machado;,citation_author=Hai Minh Nguyen;,citation_author=Kurt A. O’Hearn;,citation_author=Alexey V. Onufriev;,citation_author=Feng Pan;,citation_author=Sergio Pantano;,citation_author=Ruxi Qi;,citation_author=Ali Rahnamoun;,citation_author=Ali Risheh;,citation_author=Stephan Schott-Verdugo;,citation_author=Akhil Shajan;,citation_author=Jason Swails;,citation_author=Junmei Wang;,citation_author=Haixin Wei;,citation_author=Xiongwu Wu;,citation_author=Yongxian Wu;,citation_author=Shi Zhang;,citation_author=Shiji Zhao;,citation_author=Qiang Zhu;,citation_author=Thomas E. III Cheatham;,citation_author=Daniel R. Roe;,citation_author=Adrian Roitberg;,citation_author=Carlos Simmerling;,citation_author=Darrin M. York;,citation_author=Maria C. Nagan;,citation_author=Kenneth M. Jr. Merz;,citation_publication_date=2023-10;,citation_cover_date=2023-10;,citation_year=2023;,citation_fulltext_html_url=https://doi.org/10.1021/acs.jcim.3c01153;,citation_issue=20;,citation_doi=10.1021/acs.jcim.3c01153;,citation_issn=1549-9596;,citation_volume=63;,citation_journal_title=Journal of Chemical Information and Modeling;,citation_publisher=American Chemical Society;">
<meta name="citation_reference" content="citation_title=2′,7′-Dichlorodihydrofluorescein as a fluorescent probe for reactive oxygen species measurement: Forty years of application and controversy;,citation_abstract=Reactive oxygen species (ROS) are critically important chemical intermediates in biological studies, due to their multiple physiologically essential functions and their often pathologically deleterious effects. Consequently, it is vital that their presence in biological samples has to be quantifiable. However, their high activity, very short life span and extremely low concentrations make ROS measurement a scientifically challenging subject for researchers. One of the widespread methods for ROS detection, based on the oxidation of the non-fluorescent probe 2′,7′-dichlorodihydrofluorescein (DCFH2) to yield the highly fluorescent 2′,7′-dichlorofluorescein (DCF), was developed more than 40 years ago. However, from its initial application, argumentative questions have arisen regarding its action mechanisms, reaction principles and especially its specificity. Herein, the authors attempt to undertake a comprehensive review: to describe the basic characteristics of DCFH2; to discuss the present views of the mechanisms of its fluorescence formation; to summarize the fluorescence formation interferents; to outline its application in biological research; and to underline its advantages and disadvantages in ROS detection as well as for the methodological considerations that arise during analysis.;,citation_author=Xiuping Chen;,citation_author=Zhangfeng Zhong;,citation_author=Zengtao Xu;,citation_author=Lidian Chen;,citation_author=Yitao Wang;,citation_publication_date=2010-01;,citation_cover_date=2010-01;,citation_year=2010;,citation_fulltext_html_url=https://doi.org/10.3109/10715761003709802;,citation_issue=6;,citation_doi=10.3109/10715761003709802;,citation_issn=1071-5762;,citation_pmid=20370560;,citation_volume=44;,citation_journal_title=Free Radical Research;,citation_publisher=Taylor &amp;amp;amp; Francis;">
<meta name="citation_reference" content="citation_title=Water content alters viscoelastic behaviour of the normal adolescent rabbit medial collateral ligament;,citation_abstract=Testing environment is an important factor in the outcome of mechanical tests on connective tissue. The purpose of this investigation was to determine the effect of ligament water content on ligament mechanical behaviour by altering the test environment. Water content of medial collateral ligament (MCLs) from 19 three-month-old New Zealand White rabbits was varied in subsets of ligaments pairs by means of immersion in 2, 10 or 25% sucrose or 0.9% phosphate-buffered saline (PBS) solutions for 1 h. One knee joint was cycled 50 times in the designated solution (experimental), while the contralateral knee (uncycled control) was simultaneously soaked in the same tank. Following cycling, the water contents of both test and control ligaments were determined. Water contents of 22 normal MCLs were determined immediately post-sacrifice and served as “normal water content” controls. Normalized peak cyclic load changes were used as a measure of the viscoelastic behaviour of each MCL. Results demonstrated that only ligaments soaked (but not cycled) in a 10% sucrose solution had water contents (60.5±2.5%) which were statistically similar to the 22 fresh normal MCLs (63.9±6.0%). Ligaments soaked in PBS (74.0±1.3%) or 2% sucrose (69.2±2.3%) had significantly higher water contents compared to fresh normal MCLs. Ligaments with higher water contents (e.g. soaked in PBS or 2% sucrose) demonstrated greater cyclic load relaxation compared to ligaments with lower contents (e.g. soaked in 25 or 10% sucrose). Different fluid test environments can significantly alter ligament water content and, in turn, significantly affect ligament viscoelastic behaviour.;,citation_author=D. Chimich;,citation_author=N. Shrive;,citation_author=C. Frank;,citation_author=L. Marchuk;,citation_author=R. Bray;,citation_publication_date=1992-08;,citation_cover_date=1992-08;,citation_year=1992;,citation_fulltext_html_url=https://www.sciencedirect.com/science/article/pii/002192909290223N;,citation_issue=8;,citation_doi=10.1016/0021-9290(92)90223-N;,citation_issn=0021-9290;,citation_volume=25;,citation_journal_title=Journal of Biomechanics;">
<meta name="citation_reference" content="citation_title=Collagenase unwinds triple‐helical collagen prior to peptide bond hydrolysis;,citation_abstract=Breakdown of triple‐helical interstitial collagens is essential in embryonic development, organ morphogenesis and tissue remodelling and repair. Aberrant collagenolysis may result in diseases such as arthritis, cancer, atherosclerosis, aneurysm and fibrosis. In vertebrates, it is initiated by collagenases belonging to the matrix metalloproteinase (MMP) family. The three‐dimensional structure of a prototypic collagenase, MMP‐1, indicates that the substrate‐binding site of the enzyme is too narrow to accommodate triple‐helical collagen. Here we report that collagenases bind and locally unwind the triple‐helical structure before hydrolyzing the peptide bonds. Mutation of the catalytically essential residue Glu200 of MMP‐1 to Ala resulted in a catalytically inactive enzyme, but in its presence noncollagenolytic proteinases digested collagen into typical 3/4 and 1/4 fragments, indicating that the MMP‐1(E200A) mutant unwinds the triple‐helical collagen. The study also shows that MMP‐1 preferentially interacts with the α2(I) chain of type I collagen and cleaves the three α chains in succession. Our results throw light on the basic mechanisms that control a wide range of biological and pathological processes associated with tissue remodelling.;,citation_author=Linda Chung;,citation_author=Deendayal Dinakarpandian;,citation_author=Naoto Yoshida;,citation_author=Janelle L Lauer‐Fields;,citation_author=Gregg B Fields;,citation_author=Robert Visse;,citation_author=Hideaki Nagase;,citation_publication_date=2004-08;,citation_cover_date=2004-08;,citation_year=2004;,citation_fulltext_html_url=https://www.embopress.org/doi/full/10.1038/sj.emboj.7600318;,citation_issue=15;,citation_doi=10.1038/sj.emboj.7600318;,citation_issn=0261-4189;,citation_volume=23;,citation_journal_title=The EMBO Journal;,citation_publisher=John Wiley &amp;amp;amp; Sons, Ltd;">
<meta name="citation_reference" content="citation_title=XVI. On a mechanical theorem applicable to heat;,citation_abstract=(1870). XVI. On a mechanical theorem applicable to heat . The London, Edinburgh, and Dublin Philosophical Magazine and Journal of Science: Vol. 40, No. 265, pp. 122-127.;,citation_author=R. Clausius;,citation_publication_date=1870-08;,citation_cover_date=1870-08;,citation_year=1870;,citation_fulltext_html_url=https://www.tandfonline.com/doi/abs/10.1080/14786447008640370;,citation_doi=10.1080/14786447008640370;,citation_language=en-US;,citation_journal_title=The London, Edinburgh, and Dublin Philosophical Magazine and Journal of Science;,citation_publisher=Taylor &amp;amp;amp; Francis Group;">
<meta name="citation_reference" content="citation_title=The ultrastructure of mouse articular cartilage: Collagen orientation and implications for tissue functionality. A polarised light and scanning electron microscope study and review.;,citation_abstract=Adult mouse articular cartilage (AC) has not been thoroughly described using high resolution imaging techniques, despite the fact that the availability of knockout mice with specific extracellular matrix (ECM) mutations have renewed interest in using the mouse as a model for a variety of different human conditions. With osteoarthritis affecting millions of people worldwide, investigations into the structure and, therefore, the ability of AC to act as a load-bearing tissue, are crucial for developing treatments and prevention techniques to limit the degree of severity in this condition. Cryofixation and formaldehyde fixation as well as chemical digestion of the uncalcified regions of AC were used in combination with bright field light, polarised light and scanning electron microscopy to image the structure of adult mouse AC. Chemical digestion of the tissue revealed unique insights into the structure of mouse AC and the high cellular density of the tissue. Tightly packed sheets of collagen fibrils formed the territorial matrix (TM) of the deep zone. These were observed closely surrounding the chondrons, after applying both chemical and cryofixation techniques. The interterritorial matrix (IM), in contrast, was more isotropically arranged. The results of the study have implications for the interpretation of biomechanical functionality of mouse AC with probable applications to other species.;,citation_author=The University of Wales Bioimaging laboratory, Institute of Biological Sciences, The University of Wales, Aberystwyth, Wales, UK;,citation_author=Lc Hughes;,citation_author=Cw Archer;,citation_author=I Ap Gwynn;,citation_publication_date=2005-06;,citation_cover_date=2005-06;,citation_year=2005;,citation_fulltext_html_url=https://www.ecmjournal.org/papers/vol009/vol009a09.php;,citation_doi=10.22203/eCM.v009a09;,citation_volume=9;,citation_language=en-US;,citation_journal_title=European Cells and Materials;">
<meta name="citation_reference" content="citation_title=Conventional Commits;,citation_abstract=A specification for adding human and machine readable meaning to commit messages;,citation_fulltext_html_url=https://www.conventionalcommits.org/en/v1.0.0/;,citation_journal_title=Conventional Commits;">
<meta name="citation_reference" content="citation_title=A Second Generation Force Field for the Simulation of Proteins, Nucleic Acids, and Organic Molecules;,citation_author=Wendy D. Cornell;,citation_author=Piotr Cieplak;,citation_author=Christopher I. Bayly;,citation_author=Ian R. Gould;,citation_author=Kenneth M. Merz;,citation_author=David M. Ferguson;,citation_author=David C. Spellmeyer;,citation_author=Thomas Fox;,citation_author=James W. Caldwell;,citation_author=Peter A. Kollman;,citation_publication_date=1995-05;,citation_cover_date=1995-05;,citation_year=1995;,citation_fulltext_html_url=https://doi.org/10.1021/ja00124a002;,citation_issue=19;,citation_doi=10.1021/ja00124a002;,citation_issn=0002-7863;,citation_volume=117;,citation_journal_title=Journal of the American Chemical Society;,citation_publisher=American Chemical Society;">
<meta name="citation_reference" content="citation_title=First memoir on electricity and magnetism;,citation_author=A. Coulomb;,citation_publication_date=1785;,citation_cover_date=1785;,citation_year=1785;,citation_fulltext_html_url=http://www.ampere.cnrs.fr/i-corpuspic/tab/Sources/coulomb/Coulomb_El_1785.pdf;,citation_journal_title=A Source Book in Physics;">
<meta name="citation_reference" content="citation_title=Variational methods for the solution of problems of equilibrium and vibrations;,citation_author=R. Courant;,citation_publication_date=1943;,citation_cover_date=1943;,citation_year=1943;,citation_fulltext_html_url=https://www.ams.org/bull/1943-49-01/S0002-9904-1943-07818-4/;,citation_issue=1;,citation_doi=10.1090/S0002-9904-1943-07818-4;,citation_issn=0002-9904, 1936-881X;,citation_volume=49;,citation_language=en-US;,citation_journal_title=Bulletin of the American Mathematical Society;">
<meta name="citation_reference" content="citation_title=Particle mesh Ewald: An N log (N) method for Ewald sums in large systems;,citation_author=Tom Darden;,citation_author=Darrin York;,citation_author=Lee Pedersen;,citation_publication_date=1993;,citation_cover_date=1993;,citation_year=1993;,citation_fulltext_html_url=https://www.academia.edu/download/93518571/article.pdf;,citation_volume=98;,citation_journal_title=Journal of chemical physics;,citation_publisher=American Institute of Physics;">
<meta name="citation_reference" content="citation_title=Recherches sur la théorie des quanta;,citation_author=Louis De Broglie;,citation_publication_date=1924;,citation_cover_date=1924;,citation_year=1924;,citation_fulltext_html_url=https://theses.hal.science/tel-00006807/document;,citation_dissertation_institution=Migration-université en cours d’affectation;">
<meta name="citation_reference" content="citation_title=Dynamic strength of molecular adhesion bonds;,citation_abstract=In biology, molecular linkages at, within, and beneath cell interfaces arise mainly from weak noncovalent interactions. These bonds will fail under any level of pulling force if held for sufficient time. Thus, when tested with ultrasensitive force probes, we expect cohesive material strength and strength of adhesion at interfaces to be time- and loading rate-dependent properties. To examine what can be learned from measurements of bond strength, we have extended Kramers’ theory for reaction kinetics in liquids to bond dissociation under force and tested the predictions by smart Monte Carlo (Brownian dynamics) simulations of bond rupture. By definition, bond strength is the force that produces the most frequent failure in repeated tests of breakage, i.e., the peak in the distribution of rupture forces. As verified by the simulations, theory shows that bond strength progresses through three dynamic regimes of loading rate. First, bond strength emerges at a critical rate of loading ($&amp;amp;amp;gt;$ or = 0) at which spontaneous dissociation is just frequent enough to keep the distribution peak at zero force. In the slow-loading regime immediately above the critical rate, strength grows as a weak power of loading rate and reflects initial coupling of force to the bonding potential. At higher rates, there is crossover to a fast regime in which strength continues to increase as the logarithm of the loading rate over many decades independent of the type of attraction. Finally, at ultrafast loading rates approaching the domain of molecular dynamics simulations, the bonding potential is quickly overwhelmed by the rapidly increasing force, so that only naked frictional drag on the structure remains to retard separation. Hence, to expose the energy landscape that governs bond strength, molecular adhesion forces must be examined over an enormous span of time scales. However, a significant gap exists between the time domain of force measurements in the laboratory and the extremely fast scale of molecular motions. Using results from a simulation of biotin-avidin bonds (Izrailev, S., S. Stepaniants, M. Balsera, Y. Oono, and K. Schulten. 1997. Molecular dynamics study of unbinding of the avidin-biotin complex. Biophys. J., this issue), we describe how Brownian dynamics can help bridge the gap between molecular dynamics and probe tests.;,citation_author=E. Evans;,citation_author=K. Ritchie;,citation_publication_date=1997-04;,citation_cover_date=1997-04;,citation_year=1997;,citation_fulltext_html_url=https://www.sciencedirect.com/science/article/pii/S0006349597788027;,citation_issue=4;,citation_doi=10.1016/S0006-3495(97)78802-7;,citation_issn=0006-3495;,citation_volume=72;,citation_journal_title=Biophysical Journal;">
<meta name="citation_reference" content="citation_title=Die Berechnung optischer und elektrostatischer Gitterpotentiale;,citation_author=P. P. Ewald;,citation_publication_date=1921;,citation_cover_date=1921;,citation_year=1921;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1002/andp.19213690304;,citation_issue=3;,citation_doi=10.1002/andp.19213690304;,citation_issn=1521-3889;,citation_volume=369;,citation_language=en-US;,citation_journal_title=Annalen der Physik;">
<meta name="citation_reference" content="citation_title=The Activated Complex in Chemical Reactions;,citation_abstract=The calculation of absolute reaction rates is formulated in terms of quantities which are available from the potential surfaces which can be constructed at the present time. The probability of the activated state is calculated using ordinary statistical mechanics. This probability multiplied by the rate of decomposition gives the specific rate of reaction. The occurrence of quantized vibrations in the activated complex, in degrees of freedom which are unquantized in the original molecules, leads to relative reaction rates for isotopes quite different from the rates predicted using simple kinetic theory. The necessary conditions for the general statistical treatment to reduce to the usual kinetic treatment are given.;,citation_author=Henry Eyring;,citation_publication_date=1935-02;,citation_cover_date=1935-02;,citation_year=1935;,citation_fulltext_html_url=https://pubs.aip.org/jcp/article/3/2/107/203352/The-Activated-Complex-in-Chemical-Reactions;,citation_issue=2;,citation_doi=10.1063/1.1749604;,citation_issn=0021-9606, 1089-7690;,citation_volume=3;,citation_language=en-US;,citation_journal_title=The Journal of Chemical Physics;">
<meta name="citation_reference" content="citation_title=Experimental evaluation of multiscale tendon mechanics;,citation_abstract=Tendon’s primary function is a mechanical link between muscle and bone. The hierarchical structure of tendon and specific compositional constituents are believed to be critical for proper mechanical function. With increased appreciation for tendon importance and the development of various technological advances, this review paper summarizes recent experimental approaches that have been used to study multiscale tendon mechanics, includes an overview of studies that have evaluated the role of specific tissue constituents, and also proposes challenges/opportunities facing tendon study. Tendon has been demonstrated to have specific structural characteristics (e.g., multi-level hierarchy, crimp pattern, helix) and complex mechanical properties (e.g., non-linearity, anisotropy, viscoelasticity). Physical mechanisms including uncrimping, fiber sliding, and collagen reorganization have been shown to govern tendon mechanical responses under both static and dynamic loading. Several tendon constituents with relatively small quantities have been suggested to play a role in its mechanics, although some results are conflicting. Further research should be performed to understand the interplay and communication of tendon mechanical properties across levels of the hierarchical structure, and further show how each of these components contribute to tendon mechanics. The studies summarized and discussed in this review have helped elucidate important aspects of multiscale tendon mechanics, which is a prerequisite for analyzing stress/strain transfer between multiple scales and identifying key principles of mechanotransduction. This information could further facilitate interpreting the functional diversity of tendons from different species, different locations, and even different developmental stages, and then better understand and identify fundamental concepts related to tendon degeneration, disease, and healing. © 2017 Orthopaedic Research Society. Published by Wiley Periodicals, Inc. J Orthop Res 35:1353–1365, 2017.;,citation_author=Fei Fang;,citation_author=Spencer P. Lake;,citation_publication_date=2017;,citation_cover_date=2017;,citation_year=2017;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1002/jor.23488;,citation_issue=7;,citation_doi=10.1002/jor.23488;,citation_issn=1554-527X;,citation_volume=35;,citation_language=en-US;,citation_journal_title=Journal of Orthopaedic Research;">
<meta name="citation_reference" content="citation_title=File formats - GROMACS 2025.3 documentation;,citation_fulltext_html_url=https://manual.gromacs.org/current/reference-manual/topologies/topology-file-formats.html#topology-file;">
<meta name="citation_reference" content="citation_title=In vivo human triceps surae and quadriceps femoris muscle function in a squat jump and counter movement jump;,citation_abstract=An optic fibre method was used to measure in humans in vivo Achilles (ATF) and patellar tendon forces (PTF) during submaximal squat jumps (SJ) and counter movement jumps (CMJ). Normal two-legged jumps on a force plate and one-legged jumps on a sledge apparatus were made by four volunteers. Kinetics, kinematics, and muscle activity from seven muscles were recorded. The loading patterns of the tendomuscular system differed among the jumping conditions, but were similar when the jumping height was varied. Peak PTF were greater than ATF in each condition. In contrast to earlier simulation studies it was observed that tendomuscular force could continue to increase during the shortening of muscle-tendon unit in CMJ. The concentric tendomuscular output was related to the force at the end of the stretching phase while the enhancement of the output in CMJ compared to SJ could not be explained by increases in muscle activity. The stretching phase in CMJ was characterised by little or no electromyogram activity. Therefore, the role of active stretch in creating beneficial conditions for the utilisation of elastic energy in muscle was only minor in these submaximal performances. The modelling, as used in the present study, showed, however, that tendon underwent a stretch-shortening cycle, thus having potential for elastic energy storage and utilisation. In general, the interaction between muscle and tendon components may be organised in a manner that takes advantage of the basic properties of muscle at given submaximal and variable activity levels of normal human locomotion.;,citation_author=Taija Finni;,citation_author=Paavo V. Komi;,citation_author=Vesa Lepola;,citation_publication_date=2000-11;,citation_cover_date=2000-11;,citation_year=2000;,citation_fulltext_html_url=https://doi.org/10.1007/s004210000289;,citation_issue=4;,citation_doi=10.1007/s004210000289;,citation_issn=1439-6327;,citation_volume=83;,citation_language=en-US;,citation_journal_title=European Journal of Applied Physiology;">
<meta name="citation_reference" content="citation_title=Mechanochemical Reaction Cascade for Sensitive Detection of Covalent Bond Breakage in Hydrogels;,citation_abstract=A novel strategy is reported for sensing chemical bond breakage in hydrogels at low levels of mechanical stress using a mechanochemical reaction cascade to generate fluorescence. Hydrogels are promising substrates and frameworks for cell growth and tissue engineering, particularly for cardiovascular repair and cartilage replacement. For these applications, it is important to maintain careful control over gel mechanical properties so that these hydrogels not only match the properties of the desired tissue for replacement but also retain their integrity for extended periods. Since the failure of hydrogels begins with the breakage of cross-links within the structure, methods are needed to sense these initial events for monitoring the performance of implants. In this work, it was hypothesized that nonspecific covalent bond breakage would produce radicals that would react with water to produce reactive oxygen species, which in turn could activate fluorophores sensitive to these. A series of multiarm poly(ethylene glycol) hydrogels were synthesized with a variety of cross-links of different bond dissociation energies. It was found that gels loaded with the masked fluorophore 3′-(p-aminophenyl) fluorescein became fluorescent during compression, even with as little as 5 kPa of pressure. The effect of compression on fluorescence activation was found to depend primarily on the strength of the cross-linking functional group. Future studies include utilizing this system to image mechanical variability in heterogeneous gel structures.;,citation_author=Kirsten R. Fitch;,citation_author=Andrew P. Goodwin;,citation_publication_date=2014-12;,citation_cover_date=2014-12;,citation_year=2014;,citation_fulltext_html_url=https://doi.org/10.1021/cm503253n;,citation_issue=23;,citation_doi=10.1021/cm503253n;,citation_issn=0897-4756;,citation_volume=26;,citation_journal_title=Chemistry of Materials;,citation_publisher=American Chemical Society;">
<meta name="citation_reference" content="citation_title=ACYCLIC STEREOSELECTION. XVI: HIGH DIASTEREOFACIAL SELECTIVITY IN LEWIS ACID MEDIATED ADDITIONS OF ENOLSILANES TO CHIRAL ALDEHYDES;,citation_author=L. A. Flippin;,citation_author=C. H. Heathcock;,citation_publication_date=1983;,citation_cover_date=1983;,citation_year=1983;,citation_journal_title=ACYCLIC STEREOSELECTION. XVI: HIGH DIASTEREOFACIAL SELECTIVITY IN LEWIS ACID MEDIATED ADDITIONS OF ENOLSILANES TO CHIRAL ALDEHYDES;">
<meta name="citation_reference" content="citation_title=Näherungsmethode zur Lösung des quantenmechanischen Mehrkörperproblems;,citation_abstract=Das Variationsprinzip ∫δ¯Ψ(L − E) Ψdτ=0 (L=Energieoperator) liefert, bekanntlich die Wellengleichung im Konfigurationsraum. Es wird gezeigt, daß der Ansatz Ψ=ψ1 (χ1) ψ2 (χ1)... ψN(χN) (N=Anzahl der Elektronen) zu den Gleichungen der Hartreeschen Theorie des „selfconsistent field“ führt. Dieser Ansatz hat aber nicht die richtige Symmetrie. In dem wichtigen Spezialfall der „völligen Entartung des Termsystems“ kann aber Ψ durch ėin Produkt zweier Determinanten [Formel (50) des Textes] approximiert werden. Die entsprechende Rechnung wird durchgeführt. Die Gleichungen, die sich für ψi(χ) ergeben, enthalten „Austauschglieder“ und können als Eulersche Gleichungen eines dreidimensionalen Variationsproblems mit der Energie als Wirkungsintegral [Formel (93)] aufgefaßt werden. Die Gleichungen sind nicht wesentlich komplizierter als die von Hartree, dürften aber viel genauere Resultate ergeben. Zum Schluß wird eine Formel für die Intensitäten angegeben, die- Glieder enthält, welche einer „Umgruppierung“ der inneren Elektronen bei einem Quantensprung entsprechen.;,citation_author=V. Fock;,citation_publication_date=1930-01;,citation_cover_date=1930-01;,citation_year=1930;,citation_fulltext_html_url=https://doi.org/10.1007/BF01340294;,citation_issue=1;,citation_doi=10.1007/BF01340294;,citation_issn=0044-3328;,citation_volume=61;,citation_language=de-DE;,citation_journal_title=Zeitschrift für Physik;">
<meta name="citation_reference" content="citation_title=Mémoire sur la propagation de la chaleur dans les corps solides;,citation_editor=Jean Baptiste Joseph Fourier;,citation_editor=Jean Gaston Darboux;,citation_fulltext_html_url=https://www.cambridge.org/core/books/oeuvres-de-fourier/memoire-sur-la-propagation-de-la-chaleur-dans-les-corps-solides/C362182852542BAFCF34C323DD4B76D0;,citation_doi=10.1017/CBO9781139568159.009;,citation_isbn=978-1-108-05939-8;,citation_volume=2;,citation_inbook_title=Oeuvres de Fourier: Publiées par les soins de Gaston Darboux;,citation_series_title=Cambridge Library Collection - Mathematics;">
<meta name="citation_reference" content="citation_title=Finite element modeling of soft tissues: Material models, tissue interaction and challenges;,citation_abstract=Background Musculoskeletal soft tissues, such as articular cartilage, ligaments, knee meniscus and intervertebral disk, have a complex structure, which provides elasticity and capability to support and distribute the body loads. Soft tissues describe an inhomogeneous and multiphasic structure, and exhibit a nonlinear, time-dependent behavior. Their mechanical response is governed by a substance composed of protein fiber-rich and proteoglycan-rich extracellular matrix and interstitial fluid. Protein fibers (e.g. collagen) give the tissue direction dependent stiffness and strength. To investigate these complex biological systems, the use of mathematical tools is well established, alone or in combination with experimental in vitro and in vivo tests. However, the development of these models poses many challenges due to the complex structure and mechanical response of soft tissues. Methods Non-systematic literature review. Findings This paper provides a summary of different modeling strategies with associated material properties, contact interactions between articulating tissues, validation and sensitivity of soft tissues with special focus on knee joint soft tissues and intervertebral disk. Furthermore, it reviews and discusses some salient clinical findings of reported finite element simulations. Interpretation Model studies extensively contributed to the understanding of functional biomechanics of soft tissues. Models can be effectively used to elucidate clinically relevant questions. However, users should be aware of the complexity of such tissues and of the capabilities and limitations of these approaches to adequately simulate a specific in vivo or in vitro phenomenon.;,citation_author=Maren Freutel;,citation_author=Hendrik Schmidt;,citation_author=Lutz Dürselen;,citation_author=Anita Ignatius;,citation_author=Fabio Galbusera;,citation_publication_date=2014-04;,citation_cover_date=2014-04;,citation_year=2014;,citation_fulltext_html_url=https://www.sciencedirect.com/science/article/pii/S0268003314000102;,citation_issue=4;,citation_doi=10.1016/j.clinbiomech.2014.01.006;,citation_issn=0268-0033;,citation_volume=29;,citation_journal_title=Clinical Biomechanics;">
<meta name="citation_reference" content="citation_title=Comparison between the directly measured achilles tendon force and the tendon force calculated from the ankle joint moment during vertical jumps;,citation_abstract=The purpose of this study was to compare the relationship between the achilles tendon force and the tendon force estimated from the ankle joint moment (EATF) during vertical jump. A healthy male subject performed the following jumps on the force platform: (a) maximal vertical jump from a squat position without counter-movement; (b) maximal vertical jump from an erect standing position with a preliminary counter-movement; (c) repetitive submaximal hopping on the spot with preferred frequency. The achilles tendon force was measured directly with an implanted tendon transducer in all jumping conditions. In addition the joint moment obtained from the film and ground reaction force was used to estimate the tendon force. Measurements revealed that the achilles tendon sustained high loads during the vertical jumps. The peak values of the real achilles tendon forces were 2234 N, 1896 N, and 3787 N in the squatting, the counter-movement, and the hopping jumps respectively. The achilles tendon forces were about twice the ground reaction force. The estimated achilles tendon force showed almost the same impulses with actual force, especially during hopping. However, estimated force overestimated the actual force during the early plantarflexion phase of jumping and the maximal error of the estimation was relatively large. Relevance The direct measurement of the achilles tendon forces during human movement may open ways to better understanding of the mechanisms of muscle function. Its clinical importance can be seen in its possibility to reveal the true loading characteristics of the muscle—tendon complex in almost any movement situation, including those which are expected to cause injury. The findings can be used for estimation of the true tendon forces from the method of inverse dynamics.;,citation_author=S Fukashiro;,citation_author=P. V. Komi;,citation_author=M Järvinen;,citation_author=M Miyashita;,citation_publication_date=1993-01;,citation_cover_date=1993-01;,citation_year=1993;,citation_fulltext_html_url=https://www.sciencedirect.com/science/article/pii/S0268003305800063;,citation_issue=1;,citation_doi=10.1016/S0268-0033(05)80006-3;,citation_issn=0268-0033;,citation_volume=8;,citation_journal_title=Clinical Biomechanics;">
<meta name="citation_reference" content="citation_title=A general method for numerically simulating the stochastic time evolution of coupled chemical reactions;,citation_author=Daniel T Gillespie;,citation_publication_date=1976-12;,citation_cover_date=1976-12;,citation_year=1976;,citation_fulltext_html_url=https://linkinghub.elsevier.com/retrieve/pii/0021999176900413;,citation_issue=4;,citation_doi=10.1016/0021-9991(76)90041-3;,citation_issn=00219991;,citation_volume=22;,citation_language=en-US;,citation_journal_title=Journal of Computational Physics;">
<meta name="citation_reference" content="citation_title=MDAnalysis: A Python Package for the Rapid Analysis of Molecular Dynamics Simulations;,citation_abstract=MDAnalysis (http://mdanalysis.org) is a library for structural and temporal analysis of molecular dynamics (MD) simulation trajectories and individual protein structures. MD simulations of biological molecules have become an important tool to elucidate the relationship between molecular structure and physiological function.;,citation_author=Richard J. Gowers;,citation_author=Max Linke;,citation_author=Jonathan Barnoud;,citation_author=Tyler J. E. Reddy;,citation_author=Manuel N. Melo;,citation_author=Sean L. Seyler;,citation_author=Jan Domański;,citation_author=David L. Dotson;,citation_author=Sébastien Buchoux;,citation_author=Ian M. Kenney;,citation_author=Oliver Beckstein;,citation_publication_date=2016-06;,citation_cover_date=2016-06;,citation_year=2016;,citation_fulltext_html_url=https://proceedings.scipy.org/articles/Majora-629e541a-00e;,citation_doi=10.25080/Majora-629e541a-00e;,citation_language=en-US;,citation_journal_title=scipy;">
<meta name="citation_reference" content="citation_title=GROMACS 2025.3 documentation;,citation_fulltext_html_url=https://manual.gromacs.org/current/index.html;">
<meta name="citation_reference" content="citation_title=WHAM: The weighted histogram analysis method;,citation_author=Alan Grossfield;,citation_fulltext_html_url=http://membrane.urmc.rochester.edu/?page_id=126;">
<meta name="citation_reference" content="citation_title=Array programming with NumPy;,citation_author=Charles R. Harris;,citation_author=K. Jarrod Millman;,citation_author=Stéfan J. Walt;,citation_author=Ralf Gommers;,citation_author=Pauli Virtanen;,citation_author=David Cournapeau;,citation_author=Eric Wieser;,citation_author=Julian Taylor;,citation_author=Sebastian Berg;,citation_author=Nathaniel J. Smith;,citation_author=Robert Kern;,citation_author=Matti Picus;,citation_author=Stephan Hoyer;,citation_author=Marten H. Kerkwijk;,citation_author=Matthew Brett;,citation_author=Allan Haldane;,citation_author=Jaime Fernández Río;,citation_author=Mark Wiebe;,citation_author=Pearu Peterson;,citation_author=Pierre Gérard-Marchant;,citation_author=Kevin Sheppard;,citation_author=Tyler Reddy;,citation_author=Warren Weckesser;,citation_author=Hameer Abbasi;,citation_author=Christoph Gohlke;,citation_author=Travis E. Oliphant;,citation_publication_date=2020-09;,citation_cover_date=2020-09;,citation_year=2020;,citation_fulltext_html_url=https://doi.org/10.1038/s41586-020-2649-2;,citation_issue=7825;,citation_doi=10.1038/s41586-020-2649-2;,citation_volume=585;,citation_journal_title=Nature;,citation_publisher=Springer Science and Business Media LLC;">
<meta name="citation_reference" content="citation_title=Mechanism of collagen folding propagation studied by Molecular Dynamics simulations;,citation_abstract=Collagen forms a characteristic triple helical structure and plays a central role for stabilizing the extra-cellular matrix. After a C-terminal nucleus formation folding proceeds to form long triple-helical fibers. The molecular details of triple helix folding process is of central importance for an understanding of several human diseases associated with misfolded or unstable collagen fibrils. However, the folding propagation is too rapid to be studied by experimental high resolution techniques. We employed multiple Molecular Dynamics simulations starting from unfolded peptides with an already formed nucleus to successfully follow the folding propagation in atomic detail. The triple helix folding was found to propagate involving first two chains forming a short transient template. Secondly, three residues of the third chain fold on this template with an overall mean propagation of ~75 ns per unit. The formation of loops with multiples of the repeating unit was found as a characteristic misfolding event especially when starting from an unstable nucleus. Central Gly→Ala or Gly→Thr substitutions resulted in reduced stability and folding rates due to structural deformations interfering with folding propagation.;,citation_author=Julian Hartmann;,citation_author=Martin Zacharias;,citation_publication_date=2021-06;,citation_cover_date=2021-06;,citation_year=2021;,citation_fulltext_html_url=https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1009079;,citation_issue=6;,citation_doi=10.1371/journal.pcbi.1009079;,citation_issn=1553-7358;,citation_volume=17;,citation_language=en-US;,citation_journal_title=PLOS Computational Biology;,citation_publisher=Public Library of Science;">
<meta name="citation_reference" content="citation_title=The calculation of atomic structures;,citation_author=Douglas Rayner Hartree;,citation_publication_date=1957;,citation_cover_date=1957;,citation_year=1957;">
<meta name="citation_reference" content="citation_title=MP2 energy evaluation by direct methods;,citation_abstract=An efficient algorithm is presented for evaluating the second-order Møller-Plesset (MP2) energy directly from two-electron integrals in the atomic orbital basis, i.e. the integrals are not stored. The floating point operation count and memory requirements are analyzed, and illustrative calculations are presented. This direct MP2 method should be useful for large molecule calculations, where, due to storage limitations, conventional disk-based MP2 procedures are often not feasible.;,citation_author=Martin Head-Gordon;,citation_author=John A. Pople;,citation_author=Michael J. Frisch;,citation_publication_date=1988-12;,citation_cover_date=1988-12;,citation_year=1988;,citation_fulltext_html_url=https://www.sciencedirect.com/science/article/pii/0009261488852503;,citation_issue=6;,citation_doi=10.1016/0009-2614(88)85250-3;,citation_issn=0009-2614;,citation_volume=153;,citation_journal_title=Chemical Physics Letters;">
<meta name="citation_reference" content="citation_title=Long time scale kinetic Monte Carlo simulations without lattice approximation and predefined event table;,citation_abstract=We present a method for carrying out long time scale dynamics simulations within the harmonic transition state theory approximation. For each state of the system, characterized by a local minimum on the potential energy surface, multiple searches for saddle points are carried out using random initial directions. The dimer method is used for the saddle point searches and the rate for each transition mechanism is estimated using harmonic transition state theory. Transitions are selected and the clock advanced according to the kinetic Monte Carlo algorithm. Unlike traditional applications of kinetic Monte Carlo, the atoms are not assumed to sit on lattice sites and a list of all possible transitions need not be specified beforehand. Rather, the relevant transitions are found on the fly during the simulation. A multiple time scale simulation of Al(100) crystal growth is presented where the deposition event, occurring on the time scale of picoseconds, is simulated by ordinary classical dynamics, but the time interval in between deposition events, on the order of milliseconds, is simulated by the long time scale algorithm. The Al(100) surface is found to grow remarkably smooth, even at 30 K because of concerted displacements of multiple atoms with significantly lower activation energy than adatom diffusion on the flat terrace.;,citation_author=Graeme Henkelman;,citation_author=Hannes Jónsson;,citation_publication_date=2001-12;,citation_cover_date=2001-12;,citation_year=2001;,citation_fulltext_html_url=https://pubs.aip.org/jcp/article/115/21/9657/453229/Long-time-scale-kinetic-Monte-Carlo-simulations;,citation_issue=21;,citation_doi=10.1063/1.1415500;,citation_issn=0021-9606, 1089-7690;,citation_volume=115;,citation_language=en-US;,citation_journal_title=The Journal of Chemical Physics;">
<meta name="citation_reference" content="citation_title=LINCS: A linear constraint solver for molecular simulations;,citation_abstract=In this article, we present a new LINear Constraint Solver (LINCS) for molecular simulations with bond constraints. The algorithm is inherently stable, as the constraints themselves are reset instead of derivatives of the constraints, thereby eliminating drift. Although the derivation of the algorithm is presented in terms of matrices, no matrix matrix multiplications are needed and only the nonzero matrix elements have to be stored, making the method useful for very large molecules. At the same accuracy, the LINCS algorithm is three to four times faster than the SHAKE algorithm. Parallelization of the algorithm is straightforward. © 1997 John Wiley &amp;amp;amp; Sons, Inc. J Comput Chem 18: 1463–1472, 1997;,citation_author=Berk Hess;,citation_author=Henk Bekker;,citation_author=Herman J. C. Berendsen;,citation_author=Johannes G. E. M. Fraaije;,citation_publication_date=1997;,citation_cover_date=1997;,citation_year=1997;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1002/%28SICI%291096-987X%28199709%2918%3A12%3C1463%3A%3AAID-JCC4%3E3.0.CO%3B2-H;,citation_issue=12;,citation_doi=10.1002/(SICI)1096-987X(199709)18:12&amp;amp;lt;1463::AID-JCC4&amp;gt;3.0.CO;2-H;,citation_issn=1096-987X;,citation_volume=18;,citation_language=en-US;,citation_journal_title=Journal of Computational Chemistry;">
<meta name="citation_reference" content="citation_title=Quiet high-resolution computer models of a plasma;,citation_abstract=We introduce two new computer models in which the influence of the finite potential mesh is, for most practical purposes, eliminated. The Quiet Particle-Mesh model (QPM) uses a Gaussian-shaped charge cloud and careful shaping of the potential solution in either real or transform space. The QPM model is measured to be 150 times less noisy than the conventional CIC model and is suitable for large collisionless plasma simulations with 105 or more particles. This is achieved at little or no cost in computing time. The hybrid Particle-Particle/Particle-Mesh model (PPPM), on the other hand, adds to the above mesh calculation contributions from nearby particles by direct summation. This model has a spatial resolution which is independent of the mesh spacing and may be a small fraction of a mesh cell. It is more time-consuming and is suitable for high-resolution sub-Debye length investigations with a smaller number of particles (∼104). Applied to molecular dynamics, the PPPM model can simulate systems with approximately 100 times the number of particles than conventional methods.;,citation_author=R. W Hockney;,citation_author=S. P Goel;,citation_author=J. W Eastwood;,citation_publication_date=1974-02;,citation_cover_date=1974-02;,citation_year=1974;,citation_fulltext_html_url=https://www.sciencedirect.com/science/article/pii/0021999174900102;,citation_issue=2;,citation_doi=10.1016/0021-9991(74)90010-2;,citation_issn=0021-9991;,citation_volume=14;,citation_journal_title=Journal of Computational Physics;">
<meta name="citation_reference" content="citation_title=Inhomogeneous Electron Gas;,citation_abstract=This paper deals with the ground state of an interacting electron gas in an external potential 𝑣⁡(r). It is proved that there exists a universal functional of the density, 𝐹⁡[𝑛⁡(r)], independent of 𝑣⁡(r), such that the expression 𝐸≡∫𝑣⁡(r)⁢𝑛⁡(r)𝑑r +𝐹⁡[𝑛⁡(r)] has as its minimum value the correct ground-state energy associated with 𝑣⁡(r). The functional 𝐹⁡[𝑛⁡(r)] is then discussed for two situations: (1) 𝑛⁡(r) =𝑛0 +̃𝑛⁡(r), ̃𝑛𝑛0 ≪1, and (2) 𝑛⁡(r) =𝜙⁡(r𝑟0) with 𝜙 arbitrary and 𝑟0 →∞. In both cases 𝐹 can be expressed entirely in terms of the correlation energy and linear and higher order electronic polarizabilities of a uniform electron gas. This approach also sheds some light on generalized Thomas-Fermi methods and their limitations. Some new extensions of these methods are presented.;,citation_author=P. Hohenberg;,citation_author=W. Kohn;,citation_publication_date=1964-11;,citation_cover_date=1964-11;,citation_year=1964;,citation_fulltext_html_url=https://link.aps.org/doi/10.1103/PhysRev.136.B864;,citation_issue=3B;,citation_doi=10.1103/PhysRev.136.B864;,citation_volume=136;,citation_journal_title=Physical Review;,citation_publisher=American Physical Society;">
<meta name="citation_reference" content="citation_title=Solution of Problems of Elasticity by the Framework Method;,citation_abstract=Because of mathematical difficulties which make the solution of differential equations of the theory of elasticity impossible in many cases, the author has been impelled to seek some other method of approach than one of pure mathematical analysis. The method outlined in this paper is of this character and may with some qualifications be applied to problems of two-dimensional stress, bending of plates, bending of cylindrical shells, the general case of three-dimensional stress, and a great variety of others. Essentially, the method consists in replacing the continuous material of the elastic body being studied by a framework of bars arranged according to a definite pattern, the elements of which are endowed with elastic properties suitable to the type of problem. This framework is then analyzed, according to the procedure outlined in the paper for various types of elastic problems. Examples of the application of the principles involved are also given.;,citation_author=A. Hrennikoff;,citation_publication_date=2021-03;,citation_cover_date=2021-03;,citation_year=2021;,citation_fulltext_html_url=https://doi.org/10.1115/1.4009129;,citation_issue=4;,citation_doi=10.1115/1.4009129;,citation_issn=0021-8936;,citation_volume=8;,citation_journal_title=Journal of Applied Mechanics;">
<meta name="citation_reference" content="citation_title=G_wham—A Free Weighted Histogram Analysis Implementation Including Robust Error and Autocorrelation Estimates;,citation_abstract=The Weighted Histogram Analysis Method (WHAM) is a standard technique used to compute potentials of mean force (PMFs) from a set of umbrella sampling simulations. Here, we present a new WHAM implementation, termed g_wham, which is distributed freely with the GROMACS molecular simulation suite. g_wham estimates statistical errors using the technique of bootstrap analysis. Three bootstrap methods are supported: (i) bootstrapping new trajectories based on the umbrella histograms, (ii) bootstrapping of complete histograms, and (iii) Bayesian bootstrapping of complete histograms, that is, bootstrapping via the assignment of random weights to the histograms. Because methods ii and iii consider only complete histograms as independent data points, these methods do not require the accurate calculation of autocorrelation times. We demonstrate that, given sufficient sampling, bootstrapping new trajectories allows for an accurate error estimate. In the presence of long autocorrelations, however, (Bayesian) bootstrapping of complete histograms yields a more reliable error estimate, whereas bootstrapping of new trajectories may underestimate the error. In addition, we emphasize that the incorporation of autocorrelations into WHAM reduces the bias from limited sampling, in particular, when computing periodic PMFs in inhomogeneous systems such as solvated lipid membranes or protein channels.;,citation_author=Jochen S. Hub;,citation_author=Bert L. Groot;,citation_author=David Spoel;,citation_publication_date=2010-12;,citation_cover_date=2010-12;,citation_year=2010;,citation_fulltext_html_url=https://doi.org/10.1021/ct100494z;,citation_issue=12;,citation_doi=10.1021/ct100494z;,citation_issn=1549-9618;,citation_volume=6;,citation_journal_title=Journal of Chemical Theory and Computation;,citation_publisher=American Chemical Society;">
<meta name="citation_reference" content="citation_title=Matplotlib: A 2D graphics environment;,citation_abstract=Matplotlib is a 2D graphics package used for Python for application development, interactive scripting, and publication-quality image generation across user interfaces and operating systems.;,citation_author=J. D. Hunter;,citation_publication_date=2007;,citation_cover_date=2007;,citation_year=2007;,citation_issue=3;,citation_doi=10.1109/MCSE.2007.55;,citation_volume=9;,citation_journal_title=Computing in Science &amp;amp;amp; Engineering;,citation_publisher=IEEE COMPUTER SOC;">
<meta name="citation_reference" content="citation_title=Hybrid Quantum-Classical simulations (QM/MM) with CP2K interface - GROMACS 2025.3 documentation;,citation_fulltext_html_url=https://manual.gromacs.org/current/reference-manual/special/qmmm.html;">
<meta name="citation_reference" content="citation_title=Invoke-ai/InvokeAI;,citation_abstract=Invoke is a leading creative engine for Stable Diffusion models, empowering professionals, artists, and enthusiasts to generate and create visual media using the latest AI-driven technologies. The solution offers an industry leading WebUI, and serves as the foundation for multiple commercial products.;,citation_publication_date=2025-09;,citation_cover_date=2025-09;,citation_year=2025;,citation_fulltext_html_url=https://github.com/invoke-ai/InvokeAI;,citation_publisher=InvokeAI;">
<meta name="citation_reference" content="citation_title=Architecture of&nbsp;the&nbsp;Slurm Workload Manager;,citation_abstract=Slurm is an open source, fault-tolerant, and highly scalable workload manager used on many of the world’s supercomputers and computer clusters. As a cluster workload manager, Slurm has three key functions. First, it allocates exclusive and/or non-exclusive access to resources for some duration of time. Second, it provides a framework for starting, executing, and monitoring work on the allocated resources. Finally, it arbitrates contention for resources by managing queues of pending work and enforcing administrative policies. This paper describes the current design and capabilities of Slurm.;,citation_author=Morris A. Jette;,citation_author=Tim Wickberg;,citation_editor=Dalibor Klusáček;,citation_editor=Julita Corbalán;,citation_editor=Gonzalo P. Rodrigo;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_doi=10.1007/978-3-031-43943-8_1;,citation_isbn=978-3-031-43943-8;,citation_language=en-US;,citation_conference_title=Job Scheduling Strategies for Parallel Processing;,citation_conference=Springer Nature Switzerland;">
<meta name="citation_reference" content="citation_title=Prediction of organic homolytic bond dissociation enthalpies at near chemical accuracy with sub-second computational cost;,citation_abstract=Bond dissociation enthalpies (BDEs) of organic molecules play a fundamental role in determining chemical reactivity and selectivity. However, BDE computations at sufficiently high levels of quantum mechanical theory require substantial computing resources. In this paper, we develop a machine learning model capable of accurately predicting BDEs for organic molecules in a fraction of a second. We perform automated density functional theory (DFT)&nbsp;calculations at the M06-2X/def2-TZVP level of theory for 42,577 small organic molecules, resulting in 290,664 BDEs. A graph neural network trained on a subset of these results achieves a mean absolute error of 0.58 kcal mol−1 (vs DFT) for BDEs of unseen molecules. We further demonstrate the model on two applications: first, we rapidly and accurately predict major sites of hydrogen abstraction in the metabolism of drug-like molecules, and second, we determine the dominant molecular fragmentation pathways during soot formation.;,citation_author=Peter C. St John;,citation_author=Yanfei Guan;,citation_author=Yeonjoon Kim;,citation_author=Seonah Kim;,citation_author=Robert S. Paton;,citation_publication_date=2020-05;,citation_cover_date=2020-05;,citation_year=2020;,citation_fulltext_html_url=https://www.nature.com/articles/s41467-020-16201-z;,citation_issue=1;,citation_doi=10.1038/s41467-020-16201-z;,citation_issn=2041-1723;,citation_volume=11;,citation_language=en-US;,citation_journal_title=Nature Communications;,citation_publisher=Nature Publishing Group;">
<meta name="citation_reference" content="citation_title=Highly accurate protein structure prediction with AlphaFold;,citation_abstract=Proteins are essential to life, and understanding their structure can facilitate a mechanistic understanding of their function. Through an enormous experimental effort1–4, the structures of around 100,000 unique proteins have been determined5, but this represents a small fraction of the billions of known protein sequences6,7. Structural coverage is bottlenecked by the months to years of painstaking effort required to determine a single protein structure. Accurate computational approaches are needed to address this gap and to enable large-scale structural bioinformatics. Predicting the three-dimensional structure that a protein will adopt based solely on its amino acid sequence—the structure prediction component of the “protein folding problem”8—has been an important open research problem for more than 50&nbsp;years9. Despite recent progress10–14, existing methods fall far&nbsp;short of atomic accuracy, especially when no homologous structure is available. Here we provide the first computational method that can regularly predict protein structures with atomic accuracy even in cases in which no similar structure is known. We validated an entirely redesigned version of our neural network-based model, AlphaFold, in the challenging 14th Critical Assessment of protein Structure Prediction (CASP14)15, demonstrating accuracy competitive with experimental structures in a majority of cases and greatly outperforming other methods. Underpinning the latest version of AlphaFold is a novel machine learning approach that incorporates physical and biological knowledge about protein structure, leveraging multi-sequence alignments, into the design of the deep learning algorithm.;,citation_author=John Jumper;,citation_author=Richard Evans;,citation_author=Alexander Pritzel;,citation_author=Tim Green;,citation_author=Michael Figurnov;,citation_author=Olaf Ronneberger;,citation_author=Kathryn Tunyasuvunakool;,citation_author=Russ Bates;,citation_author=Augustin Žídek;,citation_author=Anna Potapenko;,citation_author=Alex Bridgland;,citation_author=Clemens Meyer;,citation_author=Simon A. A. Kohl;,citation_author=Andrew J. Ballard;,citation_author=Andrew Cowie;,citation_author=Bernardino Romera-Paredes;,citation_author=Stanislav Nikolov;,citation_author=Rishub Jain;,citation_author=Jonas Adler;,citation_author=Trevor Back;,citation_author=Stig Petersen;,citation_author=David Reiman;,citation_author=Ellen Clancy;,citation_author=Michal Zielinski;,citation_author=Martin Steinegger;,citation_author=Michalina Pacholska;,citation_author=Tamas Berghammer;,citation_author=Sebastian Bodenstein;,citation_author=David Silver;,citation_author=Oriol Vinyals;,citation_author=Andrew W. Senior;,citation_author=Koray Kavukcuoglu;,citation_author=Pushmeet Kohli;,citation_author=Demis Hassabis;,citation_publication_date=2021-08;,citation_cover_date=2021-08;,citation_year=2021;,citation_fulltext_html_url=https://www.nature.com/articles/s41586-021-03819-2;,citation_issue=7873;,citation_doi=10.1038/s41586-021-03819-2;,citation_issn=1476-4687;,citation_volume=596;,citation_language=en-US;,citation_journal_title=Nature;,citation_publisher=Nature Publishing Group;">
<meta name="citation_reference" content="citation_title=The Viscous Flow of Large Molecules;,citation_author=Walter Kauzmann;,citation_author=Henry Eyring;,citation_publication_date=1940-11;,citation_cover_date=1940-11;,citation_year=1940;,citation_fulltext_html_url=https://doi.org/10.1021/ja01868a059;,citation_issue=11;,citation_doi=10.1021/ja01868a059;,citation_issn=0002-7863;,citation_volume=62;,citation_journal_title=Journal of the American Chemical Society;,citation_publisher=American Chemical Society;">
<meta name="citation_reference" content="citation_title=KIMMDY: A biomolecular reaction emulator;,citation_abstract=Molecular simulations have become indispensable in biological research. Their accuracy continues to improve, but directly modelling biochemical reactions – central to all life processes – remains computationally challenging. Here, we present a biomolecular reaction emulator that models reactions across conformational ensembles using kinetic Monte Carlo. Our method, KIMMDY, is capable of handling dynamic, large-scale systems with successive, competing reactions, even on the second timescale or slower. It leverages graph neural networks for large-scale prediction of reaction rates, while also being capable of using simpler physics-based or heuristic models. We validate our approach against experimental data and showcase its power and versatility through a series of applications, including radical reactions, nucleophilic substitutions, and photodimerization. Example systems span proteins and DNA. KIMMDY aids the understanding of biochemical reaction cascades in complex systems, helps to re-interpret experimental data, and can inspire future wet-lab experiments.;,citation_author=Jannik Buhr*;,citation_author=Eric Hartmann*;,citation_author=Kai Riedmiller*;,citation_author=Evgeni Ulanov;,citation_author=Boris N. Schüpp;,citation_author=Denis Kiesewetter;,citation_author=Daniel Sucerquia;,citation_author=Camilo Aponte-Santamaría;,citation_author=Frauke Gräter;,citation_publication_date=2025-07;,citation_cover_date=2025-07;,citation_year=2025;,citation_fulltext_html_url=https://www.biorxiv.org/content/10.1101/2025.07.02.662624v1;,citation_doi=10.1101/2025.07.02.662624;,citation_language=en-US;,citation_publisher=bioRxiv;">
<meta name="citation_reference" content="citation_title=Self-Consistent Equations Including Exchange and Correlation Effects;,citation_abstract=From a theory of Hohenberg and Kohn, approximation methods for treating an inhomogeneous system of interacting electrons are developed. These methods are exact for systems of slowly varying or high density. For the ground state, they lead to self-consistent equations analogous to the Hartree and Hartree-Fock equations, respectively. In these equations the exchange and correlation portions of the chemical potential of a uniform electron gas appear as additional effective potentials. (The exchange portion of our effective potential differs from that due to Slater by a factor of 23.) Electronic systems at finite temperatures and in magnetic fields are also treated by similar methods. An appendix deals with a further correction for systems with short-wavelength density oscillations.;,citation_author=W. Kohn;,citation_author=L. J. Sham;,citation_publication_date=1965-11;,citation_cover_date=1965-11;,citation_year=1965;,citation_fulltext_html_url=https://link.aps.org/doi/10.1103/PhysRev.140.A1133;,citation_issue=4A;,citation_doi=10.1103/PhysRev.140.A1133;,citation_volume=140;,citation_journal_title=Physical Review;,citation_publisher=American Physical Society;">
<meta name="citation_reference" content="citation_title=Benchmark DFT studies on C–CN homolytic cleavage and screening the substitution effect on bond dissociation energy;,citation_abstract=Nitriles are important chemical species in organic transformations, material chemistry, and environmental sciences. Nitriles are used as cyanating reagents in many organic reactions, where the C–CN bond dissociation has an important role. The reactivity of nitriles can be better understood by studying the bond dissociation energy (BDE) of the C–CN bond. In this benchmark study, homolytic cleavage of the C–CN bond in 12 nitrile compounds is studied. Thirty-one functionals from eight different DFT classes along with three types of basis sets are employed. Theoretical results are compared with the available experimental data. Based on statistical outcomes, the CAM-B3LYP functional of the range separated hybrid GGA class with Pople 6-311G(d,p) basis set provides the most accurate results for calculating the BDE of the C–CN bond. The mean absolute error (MAE) value is 0.06&nbsp;kcal&nbsp;mol−1, whereas standard deviation (SD) and Pearson’s correlation (R) are 2.79&nbsp;kcal&nbsp;mol−1 and 0.96, respectively, when compared with experimental data. The substitutional effect on the homolytic cleavage (BDE) of respective bonds in differently substituted nitriles is also investigated. The BDE results indicate that electron withdrawing groups (EWGs) lower the BDE, while electron donating groups (EDGs) increase the BDE of the C–CN bond. The NBO and HOMO–LUMO orbitals analyses are also performed to further elaborate the variational BDE patterns of C–CN bond cleavage.;,citation_author=Naveen Kosar;,citation_author=Khurshid Ayub;,citation_author=Mazhar Amjad Gilani;,citation_author=Tariq Mahmood;,citation_publication_date=2019-01;,citation_cover_date=2019-01;,citation_year=2019;,citation_fulltext_html_url=https://doi.org/10.1007/s00894-019-3930-x;,citation_issue=2;,citation_doi=10.1007/s00894-019-3930-x;,citation_issn=0948-5023;,citation_volume=25;,citation_language=en-US;,citation_journal_title=Journal of Molecular Modeling;">
<meta name="citation_reference" content="citation_title=CP2K: An electronic structure and molecular dynamics software package - Quickstep: Efficient and accurate electronic structure calculations;,citation_abstract=CP2K is an open source electronic structure and molecular dynamics software package to perform atomistic simulations of solid-state, liquid, molecular, and biological systems. It is especially aimed at massively parallel and linear-scaling electronic structure methods and state-of-the-art ab initio molecular dynamics simulations. Excellent performance for electronic structure calculations is achieved using novel algorithms implemented for modern high-performance computing systems. This review revisits the main capabilities of CP2K to perform efficient and accurate electronic structure simulations. The emphasis is put on density functional theory and multiple post–Hartree–Fock methods using the Gaussian and plane wave approach and its augmented all-electron extension.;,citation_author=Thomas D. Kühne;,citation_author=Marcella Iannuzzi;,citation_author=Mauro Del Ben;,citation_author=Vladimir V. Rybkin;,citation_author=Patrick Seewald;,citation_author=Frederick Stein;,citation_author=Teodoro Laino;,citation_author=Rustam Z. Khaliullin;,citation_author=Ole Schütt;,citation_author=Florian Schiffmann;,citation_author=Dorothea Golze;,citation_author=Jan Wilhelm;,citation_author=Sergey Chulkov;,citation_author=Mohammad Hossein Bani-Hashemian;,citation_author=Valéry Weber;,citation_author=Urban Borštnik;,citation_author=Mathieu Taillefumier;,citation_author=Alice Shoshana Jakobovits;,citation_author=Alfio Lazzaro;,citation_author=Hans Pabst;,citation_author=Tiziano Müller;,citation_author=Robert Schade;,citation_author=Manuel Guidon;,citation_author=Samuel Andermatt;,citation_author=Nico Holmberg;,citation_author=Gregory K. Schenter;,citation_author=Anna Hehn;,citation_author=Augustin Bussy;,citation_author=Fabian Belleflamme;,citation_author=Gloria Tabacchi;,citation_author=Andreas Glöß;,citation_author=Michael Lass;,citation_author=Iain Bethune;,citation_author=Christopher J. Mundy;,citation_author=Christian Plessl;,citation_author=Matt Watkins;,citation_author=Joost VandeVondele;,citation_author=Matthias Krack;,citation_author=Jürg Hutter;,citation_publication_date=2020-05;,citation_cover_date=2020-05;,citation_year=2020;,citation_fulltext_html_url=https://aip.scitation.org/doi/10.1063/5.0007045;,citation_issue=19;,citation_doi=10.1063/5.0007045;,citation_issn=0021-9606;,citation_volume=152;,citation_journal_title=The Journal of Chemical Physics;,citation_publisher=American Institute of Physics;">
<meta name="citation_reference" content="citation_title=THE weighted histogram analysis method for free-energy calculations on biomolecules. I. The method;,citation_abstract=The Weighted Histogram Analysis Method (WHAM), an extension of Ferrenberg and Swendsen’s Multiple Histogram Technique, has been applied for the first time on complex biomolecular Hamiltonians. The method is presented here as an extension of the Umbrella Sampling method for free-energy and Potential of Mean Force calculations. This algorithm possesses the following advantages over methods that are currently employed: (1) It provides a built-in estimate of sampling errors thereby yielding objective estimates of the optimal location and length of additional simulations needed to achieve a desired level of precision; (2) it yields the “best” value of free energies by taking into account all the simulations so as to minimize the statistical errors; (3) in addition to optimizing the links between simulations, it also allows multiple overlaps of probability distributions for obtaining better estimates of the free-energy differences. By recasting the Ferrenberg–Swendsen Multiple Histogram equations in a form suitable for molecular mechanics type Hamiltonians, we have demonstrated the feasibility and robustness of this method by applying it to a test problem of the generation of the Potential of Mean Force profile of the pseudorotation phase angle of the sugar ring in deoxyadenosine. © 1992 by John Wiley &amp;amp;amp; Sons, Inc.;,citation_author=Shankar Kumar;,citation_author=John M. Rosenberg;,citation_author=Djamal Bouzida;,citation_author=Robert H. Swendsen;,citation_author=Peter A. Kollman;,citation_publication_date=1992;,citation_cover_date=1992;,citation_year=1992;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1002/jcc.540130812;,citation_issue=8;,citation_doi=10.1002/jcc.540130812;,citation_issn=1096-987X;,citation_volume=13;,citation_language=en-US;,citation_journal_title=Journal of Computational Chemistry;">
<meta name="citation_reference" content="citation_title=An Efficient Real Space Multigrid QM/MM Electrostatic Coupling;,citation_abstract=A popular strategy for simulating large systems where quantum chemical effects are important is the use of mixed quantum mechanical/molecular mechanics methods (QM/MM). While the cost of solving the Schrödinger equation in the QM part is the bottleneck of these calculations, evaluating the Coulomb interaction between the QM and the MM part is surprisingly expensive. In fact it can be just as time-consuming as solving the QM part. We present here a novel real space multigrid approach that handles Coulomb interactions very effectively and implement it in the CP2K code. This novel scheme cuts the cost of this part of the calculation by 2 orders of magnitude. The method does not need very fine-tuning or adjustable parameters, and it is quite accurate, leading to a dynamics with very good energy conservation. We exemplify the validity of our algorithms with simulations of water and of a zwitterionic dipeptide solvated in water.;,citation_author=Teodoro Laino;,citation_author=Fawzi Mohamed;,citation_author=Alessandro Laio;,citation_author=Michele Parrinello;,citation_publication_date=2005-11;,citation_cover_date=2005-11;,citation_year=2005;,citation_fulltext_html_url=https://doi.org/10.1021/ct050123f;,citation_issue=6;,citation_doi=10.1021/ct050123f;,citation_issn=1549-9618;,citation_volume=1;,citation_journal_title=Journal of Chemical Theory and Computation;,citation_publisher=American Chemical Society;">
<meta name="citation_reference" content="citation_title=Development of the Colle-Salvetti correlation-energy formula into a functional of the electron density;,citation_abstract=A correlation-energy formula due to Colle and Salvetti [Theor. Chim. Acta 37, 329 (1975)], in which the correlation energy density is expressed in terms of the electron density and a Laplacian of the second-order Hartree-Fock density matrix, is restated as a formula involving the density and local kinetic-energy density. On insertion of gradient expansions for the local kinetic-energy density, density-functional formulas for the correlation energy and correlation potential are then obtained. Through numerical calculations on a number of atoms, positive ions, and molecules, of both open- and closed-shell type, it is demonstrated that these formulas, like the original Colle-Salvetti formulas, give correlation energies within a few percent.;,citation_author=Chengteh Lee;,citation_author=Weitao Yang;,citation_author=Robert G. Parr;,citation_publication_date=1988-01;,citation_cover_date=1988-01;,citation_year=1988;,citation_fulltext_html_url=https://link.aps.org/doi/10.1103/PhysRevB.37.785;,citation_issue=2;,citation_doi=10.1103/PhysRevB.37.785;,citation_volume=37;,citation_journal_title=Physical Review B;,citation_publisher=American Physical Society;">
<meta name="citation_reference" content="citation_title=The interpretation of protein structures: Estimation of static accessibility;,citation_author=B. Lee;,citation_author=F. M. Richards;,citation_publication_date=1971-02;,citation_cover_date=1971-02;,citation_year=1971;,citation_issue=3;,citation_doi=10.1016/0022-2836(71)90324-x;,citation_issn=0022-2836;,citation_pmid=5551392;,citation_volume=55;,citation_language=en-US;,citation_journal_title=Journal of Molecular Biology;">
<meta name="citation_reference" content="citation_title=Cohesion;,citation_author=J. E. Lennard-Jones;,citation_publication_date=1931-09;,citation_cover_date=1931-09;,citation_year=1931;,citation_fulltext_html_url=https://dx.doi.org/10.1088/0959-5309/43/5/301;,citation_issue=5;,citation_doi=10.1088/0959-5309/43/5/301;,citation_issn=0959-5309;,citation_volume=43;,citation_language=en-US;,citation_journal_title=Proceedings of the Physical Society;">
<meta name="citation_reference" content="citation_title=A quantum chemistry study on C–H homolytic bond dissociation enthalpies of five-membered and six-membered heterocyclic compounds;,citation_abstract=The C–H homolytic bond dissociation enthalpies (BDEs) of 27 heterocyclic compounds of small systems (less than 8 non-hydrogen atoms) were evaluated by the composite ab initio methods of G4 and CBS-Q. In addition, the C–H BDEs of an extended database including 60 heterocyclic compounds were assessed by 16 DFT functionals. The correlation between the theoretical and experimental values reveals that the BMK functional provided the lowest root mean square error (RMSE) of 10.2&nbsp;kJ/mol, and the correlation coefficient (R2) was 0.955. The mean deviation (MD), mean absolute deviation (MAD) of BMK are 0.1&nbsp;kJ/mol and 7.9&nbsp;kJ/mol separately. Therefore, we utilized BMK to research C–H BDEs together with the substituent effects of five-membered and six-membered heterocyclic compounds including large systems. The nature of C–H BDE change pattern was analyzed by the natural bond orbital (NBO).;,citation_author=Yingning Liang;,citation_author=Wenrui Zheng;,citation_author=Hongyun Xie;,citation_author=Xinyan Zha;,citation_author=Ting Wang;,citation_publication_date=2022-07;,citation_cover_date=2022-07;,citation_year=2022;,citation_fulltext_html_url=https://www.sciencedirect.com/science/article/pii/S0019452222001893;,citation_issue=7;,citation_doi=10.1016/j.jics.2022.100527;,citation_issn=0019-4522;,citation_volume=99;,citation_language=en-US;,citation_journal_title=Journal of the Indian Chemical Society;">
<meta name="citation_reference" content="citation_title=GROMACS 2020 Source code;,citation_abstract=Source code archive of the 2020 version of GROMACS.;,citation_author=undefined Lindahl;,citation_author=undefined Abraham;,citation_author=undefined Hess;,citation_author=van der Spoel;,citation_publication_date=2020-01;,citation_cover_date=2020-01;,citation_year=2020;,citation_fulltext_html_url=https://zenodo.org/record/3562495;,citation_doi=10.5281/zenodo.3562495;,citation_publisher=Zenodo;">
<meta name="citation_reference" content="citation_title=Improved side‐chain torsion potentials for the Amber ff99SB protein force field;,citation_abstract=Abstract Recent advances in hardware and software have enabled increasingly long molecular dynamics (MD) simulations of biomolecules, exposing certain limitations in the accuracy of the force fields used for such simulations and spurring efforts to refine these force fields. Recent modifications to the Amber and CHARMM protein force fields, for example, have improved the backbone torsion potentials, remedying deficiencies in earlier versions. Here, we further advance simulation accuracy by improving the amino acid side‐chain torsion potentials of the Amber ff99SB force field. First, we used simulations of model alpha‐helical systems to identify the four residue types whose rotamer distribution differed the most from expectations based on Protein Data Bank statistics. Second, we optimized the side‐chain torsion potentials of these residues to match new, high‐level quantum‐mechanical calculations. Finally, we used microsecond‐timescale MD simulations in explicit solvent to validate the resulting force field against a large set of experimental NMR measurements that directly probe side‐chain conformations. The new force field, which we have termed Amber ff99SB‐ILDN, exhibits considerably better agreement with the NMR data. Proteins 2010. © 2010 Wiley‐Liss, Inc.;,citation_author=Kresten Lindorff‐Larsen;,citation_author=Stefano Piana;,citation_author=Kim Palmo;,citation_author=Paul Maragakis;,citation_author=John L. Klepeis;,citation_author=Ron O. Dror;,citation_author=David E. Shaw;,citation_publication_date=2010-06;,citation_cover_date=2010-06;,citation_year=2010;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/10.1002/prot.22711;,citation_issue=8;,citation_doi=10.1002/prot.22711;,citation_issn=0887-3585, 1097-0134;,citation_volume=78;,citation_language=en-US;,citation_journal_title=Proteins: Structure, Function, and Bioinformatics;">
<meta name="citation_reference" content="citation_title=A Treatise on the Scurvy: In Three Parts. Containing an Inquiry Into the Nature, Causes, and Cure, of that Disease. Together with a Critical and Chronological View of what Has Been Published on the Subject;,citation_author=James Lind;,citation_publication_date=1772;,citation_cover_date=1772;,citation_year=1772;,citation_language=en-US;">
<meta name="citation_reference" content="citation_title=Steric effects, as well as σ*-orbital energies, are important in diastereoface differentiation in additions to chiral aldehydes;,citation_author=E. P Lodge;,citation_author=C. H. Heathcock;,citation_publication_date=1987;,citation_cover_date=1987;,citation_year=1987;,citation_issue=11;,citation_issn=0002-7863;,citation_volume=109;,citation_journal_title=Steric effects, as well as σ*-orbital energies, are important in diastereoface differentiation in additions to chiral aldehydes;,citation_publisher=American Chemical Society;">
<meta name="citation_reference" content="citation_title=Ueber die Anwendung des Satzes vom Virial in der kinetischen Theorie der Gase;,citation_abstract=n/a;,citation_author=H. A. Lorentz;,citation_publication_date=1881-01;,citation_cover_date=1881-01;,citation_year=1881;,citation_fulltext_html_url=https://zenodo.org/records/1423778;,citation_doi=10.1002/andp.18812480110;">
<meta name="citation_reference" content="citation_title=PET-MAD, a universal interatomic potential for advanced materials modeling;,citation_abstract=Machine-learning interatomic potentials (MLIPs) have greatly extended the reach of atomic-scale simulations, offering the accuracy of first-principles calculations at a fraction of the effort. Leveraging large quantum mechanical databases and expressive architectures, recent &amp;amp;amp;quot;universal&amp;quot; models deliver qualitative accuracy across the periodic table but are often biased toward low-energy configurations. We introduce PET-MAD, a generally applicable MLIP trained on a dataset combining stable inorganic and organic solids, systematically modified to enhance atomic diversity. Using a moderate but highly-consistent level of electronic-structure theory, we assess PET-MAD’s accuracy on established benchmarks and advanced simulations of six materials. PET-MAD rivals state-of-the-art MLIPs for inorganic solids, while also being reliable for molecules, organic materials, and surfaces. It is stable and fast, enabling, out-of-the-box, the near-quantitative study of thermal and quantum mechanical fluctuations, functional properties, and phase transitions. It can be efficiently fine-tuned to deliver full quantum mechanical accuracy with a minimal number of targeted calculations.;,citation_author=Arslan Mazitov;,citation_author=Filippo Bigi;,citation_author=Matthias Kellner;,citation_author=Paolo Pegolo;,citation_author=Davide Tisi;,citation_author=Guillaume Fraux;,citation_author=Sergey Pozdnyakov;,citation_author=Philip Loche;,citation_author=Michele Ceriotti;,citation_publication_date=2025-03;,citation_cover_date=2025-03;,citation_year=2025;,citation_fulltext_html_url=http://arxiv.org/abs/2503.14118;,citation_doi=10.48550/arXiv.2503.14118;,citation_publisher=arXiv;">
<meta name="citation_reference" content="citation_title=Data Structures for Statistical Computing in Python;,citation_author=Wes McKinney;,citation_editor=Stéfan Walt;,citation_editor=Jarrod Millman;,citation_publication_date=2010;,citation_cover_date=2010;,citation_year=2010;,citation_doi=10.25080/Majora-92bf1922-00a;,citation_conference_title=Proceedings of the 9th Python in Science Conference;">
<meta name="citation_reference" content="citation_title=MDAnalysis: A toolkit for the analysis of molecular dynamics simulations;,citation_abstract=MDAnalysis is an object-oriented library for structural and temporal analysis of molecular dynamics (MD) simulation trajectories and individual protein structures. It is written in the Python language with some performance-critical code in C. It uses the powerful NumPy package to expose trajectory data as fast and efficient NumPy arrays. It has been tested on systems of millions of particles. Many common file formats of simulation packages including CHARMM, Gromacs, Amber, and NAMD and the Protein Data Bank format can be read and written. Atoms can be selected with a syntax similar to CHARMM’s powerful selection commands. MDAnalysis enables both novice and experienced programmers to rapidly write their own analytical tools and access data stored in trajectories in an easily accessible manner that facilitates interactive explorative analysis. MDAnalysis has been tested on and works for most Unix-based platforms such as Linux and Mac OS X. It is freely available under the GNU General Public License from http://mdanalysis.googlecode.com. © 2011 Wiley Periodicals, Inc. J Comput Chem 2011;,citation_author=Naveen Michaud-Agrawal;,citation_author=Elizabeth J. Denning;,citation_author=Thomas B. Woolf;,citation_author=Oliver Beckstein;,citation_publication_date=2011;,citation_cover_date=2011;,citation_year=2011;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1002/jcc.21787;,citation_issue=10;,citation_doi=10.1002/jcc.21787;,citation_issn=1096-987X;,citation_volume=32;,citation_language=en-US;,citation_journal_title=Journal of Computational Chemistry;">
<meta name="citation_reference" content="citation_title=FreeSASA: An open source C library for solvent accessible surface area calculations;,citation_abstract=Calculating solvent accessible surface areas (SASA) is a run-of-the-mill calculation in structural biology. Although there are many programs available for this calculation, there are no free-standing, open-source tools designed for easy tool-chain integration. FreeSASA is an open source C library for SASA calculations that provides both command-line and Python interfaces in addition to its C API. The library implements both Lee and Richards’ and Shrake and Rupley’s approximations, and is highly configurable to allow the user to control molecular parameters, accuracy and output granularity. It only depends on standard C libraries and should therefore be easy to compile and install on any platform. The library is well-documented, stable and efficient. The command-line interface can easily replace closed source legacy programs, with comparable or better accuracy and speed, and with some added functionality.;,citation_author=Simon Mitternacht;,citation_publication_date=2016-02;,citation_cover_date=2016-02;,citation_year=2016;,citation_fulltext_html_url=https://f1000research.com/articles/5-189;,citation_doi=10.12688/f1000research.7931.1;,citation_language=en-US;,citation_publisher=F1000Research;">
<meta name="citation_reference" content="citation_title=Settle: An analytical version of the SHAKE and RATTLE algorithm for rigid water models;,citation_abstract=An analytical algorithm, called SETTLE, for resetting the positions and velocities to satisfy the holonomic constraints on the rigid water model is presented. This method is still based on the Cartesian coordinate system and can be used in place of SHAKE and RATTLE. We implemented this algorithm in the SPASMS package of molecular mechanics and dynamics. Several series of molecular dynamics simulations were carried out to examine the performance of the new algorithm in comparison with the original RATTLE method. It was found that SETTLE is of higher accuracy and is faster than RATTLE with reasonable tolerances by three to nine times on a scalar machine. Furthermore, the performance improvement ranged from factors of 26 to 98 on a vector machine since the method presented is not iterative. © 1992 by John Wiley &amp;amp;amp; Sons, Inc.;,citation_author=Shuichi Miyamoto;,citation_author=Peter A. Kollman;,citation_publication_date=1992;,citation_cover_date=1992;,citation_year=1992;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1002/jcc.540130805;,citation_issue=8;,citation_doi=10.1002/jcc.540130805;,citation_issn=1096-987X;,citation_volume=13;,citation_language=en-US;,citation_journal_title=Journal of Computational Chemistry;">
<meta name="citation_reference" content="citation_title=Note on an Approximation Treatment for Many-Electron Systems;,citation_abstract=A perturbation theory is developed for treating a system of 𝑛 electrons in which the Hartree-Fock solution appears as the zero-order approximation. It is shown by this development that the first order correction for the energy and the charge density of the system is zero. The expression for the second-order correction for the energy greatly simplifies because of the special property of the zero-order solution. It is pointed out that the development of the higher approximation involves only calculations based on a definite one-body problem.;,citation_author=Chr. Møller;,citation_author=M. S. Plesset;,citation_publication_date=1934-10;,citation_cover_date=1934-10;,citation_year=1934;,citation_fulltext_html_url=https://link.aps.org/doi/10.1103/PhysRev.46.618;,citation_issue=7;,citation_doi=10.1103/PhysRev.46.618;,citation_volume=46;,citation_journal_title=Physical Review;,citation_publisher=American Physical Society;">
<meta name="citation_reference" content="citation_title=Webinar: Multiscale QM/MM simulations: Exploring chemical reactions using novel GROMACS/CP2K interface (2020-12-08);,citation_abstract=View slides on Zenodo BioExcel’s webinar series continues with a presentation by Dmitry Morozov Register Abstract Nowadays, computer simulations play an;,citation_author=Dmitry Morozov;,citation_publication_date=2020-11;,citation_cover_date=2020-11;,citation_year=2020;,citation_fulltext_html_url=https://bioexcel.eu/webinar-multiscale-qm-mm-simulations-exploring-chemical-reactions-using-novel-gromacs-cp2k-interface-2020-12-08/;,citation_language=en-US;,citation_journal_title=BioExcel - Centre of Excellence for Computation Biomolecular Research;">
<meta name="citation_reference" content="citation_title=Diatomic Molecules According to the Wave Mechanics. II. Vibrational Levels;,citation_abstract=An exact solution is obtained for the Schroedinger equation representing the motions of the nuclei in a diatomic molecule, when the potential energy function is assumed to be of a form similar to those required by Heitler and London and others. The allowed vibrational energy levels are found to be given by the formula 𝐸⁡(𝑛) =𝐸𝑒 +ℎ⁡𝜔0⁢(𝑛 +12) −ℎ⁡𝜔0⁢𝑥⁢(𝑛+12)2, which is known to express the experimental values quite accurately. The empirical law relating the normal molecular separation 𝑟0 and the classical vibration frequency 𝜔0 is shown to be 𝑟03⁢𝜔0 =𝐾 to within a probable error of 4 percent, where 𝐾 is the same constant for all diatomic molecules and for all electronic levels. By means of this law, and by means of the solution above, the experimental data for many of the electronic levels of various molecules are analyzed and a table of constants is obtained from which the potential energy curves can be plotted. The changes in the above mentioned vibrational levels due to molecular rotation are found to agree with the Kratzer formula to the first approximation.;,citation_author=Philip M. Morse;,citation_publication_date=1929-07;,citation_cover_date=1929-07;,citation_year=1929;,citation_fulltext_html_url=https://link.aps.org/doi/10.1103/PhysRev.34.57;,citation_issue=1;,citation_doi=10.1103/PhysRev.34.57;,citation_volume=34;,citation_journal_title=Physical Review;,citation_publisher=American Physical Society;">
<meta name="citation_reference" content="citation_title=Learning local equivariant representations for large-scale atomistic dynamics;,citation_abstract=A simultaneously accurate and computationally efficient parametrization of the potential energy surface of molecules and materials is a long-standing goal in the natural sciences. While atom-centered message passing neural networks (MPNNs) have shown remarkable accuracy, their information propagation has limited the accessible length-scales. Local methods, conversely, scale to large simulations but have suffered from inferior accuracy. This work introduces Allegro, a strictly local equivariant deep neural network interatomic potential architecture that simultaneously exhibits excellent accuracy and scalability. Allegro represents a many-body potential using iterated tensor products of learned equivariant representations without atom-centered message passing. Allegro obtains improvements over state-of-the-art methods on QM9 and revMD17. A single tensor product layer outperforms existing deep MPNNs and transformers on QM9. Furthermore, Allegro displays remarkable generalization to out-of-distribution data. Molecular simulations using Allegro recover structural and kinetic properties of an amorphous electrolyte in excellent agreement with ab-initio simulations. Finally, we demonstrate parallelization with a simulation of 100 million atoms.;,citation_author=Albert Musaelian;,citation_author=Simon Batzner;,citation_author=Anders Johansson;,citation_author=Lixin Sun;,citation_author=Cameron J. Owen;,citation_author=Mordechai Kornbluth;,citation_author=Boris Kozinsky;,citation_publication_date=2023-02;,citation_cover_date=2023-02;,citation_year=2023;,citation_fulltext_html_url=https://www.nature.com/articles/s41467-023-36329-y;,citation_issue=1;,citation_doi=10.1038/s41467-023-36329-y;,citation_issn=2041-1723;,citation_volume=14;,citation_language=en-US;,citation_journal_title=Nature Communications;,citation_publisher=Nature Publishing Group;">
<meta name="citation_reference" content="citation_title=MACE: Higher order equivariant message passing neural networks for fast and accurate force fields;,citation_author=Ilyes Batatia;,citation_author=David P Kovacs;,citation_author=Gregor Simm;,citation_author=Christoph Ortner;,citation_author=Gabor Csanyi;,citation_editor=S. Koyejo;,citation_editor=S. Mohamed;,citation_editor=A. Agarwal;,citation_editor=D. Belgrave;,citation_editor=K. Cho;,citation_editor=A. Oh;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_fulltext_html_url=https://proceedings.neurips.cc/paper_files/paper/2022/file/4a36c3c51af11ed9f34615b81edb5bbc-Paper-Conference.pdf;,citation_volume=35;,citation_conference_title=Advances in neural information processing systems;,citation_conference=Curran Associates, Inc.;">
<meta name="citation_reference" content="citation_title=ColBuilder: A server to build collagen fibril models;,citation_abstract=Type I collagen is the main structural component of many tissues in the human body. It provides excellent mechanical properties to connective tissue and acts as a protein interaction hub. There is thus a wide interest in understanding the properties and diverse functions of type I collagen at the molecular level. A precondition is an atomistic collagen I structure as it occurs in native tissue. To this end, we built full-atom models of crosslinked collagen fibrils by integrating the low-resolution structure of collagen fibril available from X-ray fiber diffraction with high-resolution structures of short collagen-like peptides from X-ray crystallography and mass spectrometry data. We created a web resource of collagen models for 20 different species with a large variety of crosslink types and localization within the fibril, to facilitate structure-based analyses and simulations of type I collagen in health and disease. To easily enable simulations, we provide parameters of the modeled crosslinks for an Amber force field. The repository of collagen models is available at https://colbuilder.h-its.org.;,citation_author=Agnieszka Obarska-Kosinska;,citation_author=Benedikt Rennekamp;,citation_author=Aysecan Ünal;,citation_author=Frauke Gräter;,citation_publication_date=2021-07;,citation_cover_date=2021-07;,citation_year=2021;,citation_fulltext_html_url=https://doi.org/10.1016/j.bpj.2021.07.009;,citation_doi=10.1016/j.bpj.2021.07.009;,citation_issn=1542-0086;,citation_pmid=34265261;,citation_language=en-US;,citation_journal_title=Biophysical journal;">
<meta name="citation_reference" content="citation_title=A flexible algorithm for calculating pair interactions on SIMD architectures;,citation_abstract=Calculating interactions or correlations between pairs of particles is typically the most time-consuming task in particle simulation or correlation analysis. Straightforward implementations using a double loop over particle pairs have traditionally worked well, especially since compilers usually do a good job of unrolling the inner loop. In order to reach high performance on modern CPU and accelerator architectures, single-instruction multiple-data (SIMD) parallelization has become essential. Avoiding memory bottlenecks is also increasingly important and requires reducing the ratio of memory to arithmetic operations. Moreover, when pairs only interact within a certain cut-off distance, good SIMD utilization can only be achieved by reordering input and output data, which quickly becomes a limiting factor. Here we present an algorithm for SIMD parallelization based on grouping a fixed number of particles, e.g. 2, 4, or 8, into spatial clusters. Calculating all interactions between particles in a pair of such clusters improves data reuse compared to the traditional scheme and results in a more efficient SIMD parallelization. Adjusting the cluster size allows the algorithm to map to SIMD units of various widths. This flexibility not only enables fast and efficient implementation on current CPUs and accelerator architectures like GPUs or Intel MIC, but it also makes the algorithm future-proof. We present the algorithm with an application to molecular dynamics simulations, where we can also make use of the effective buffering the method introduces.;,citation_author=Szilárd Páll;,citation_author=Berk Hess;,citation_publication_date=2013-12;,citation_cover_date=2013-12;,citation_year=2013;,citation_fulltext_html_url=https://www.sciencedirect.com/science/article/pii/S0010465513001975;,citation_issue=12;,citation_doi=10.1016/j.cpc.2013.06.003;,citation_issn=0010-4655;,citation_volume=184;,citation_journal_title=Computer Physics Communications;">
<meta name="citation_reference" content="citation_title=Polymorphic transitions in single crystals: A new molecular dynamics method;,citation_author=M. Parrinello;,citation_author=A. Rahman;,citation_publication_date=1981-12;,citation_cover_date=1981-12;,citation_year=1981;,citation_fulltext_html_url=https://aip.scitation.org/doi/10.1063/1.328693;,citation_issue=12;,citation_doi=10.1063/1.328693;,citation_issn=0021-8979;,citation_volume=52;,citation_journal_title=Journal of Applied Physics;,citation_publisher=American Institute of Physics;">
<meta name="citation_reference" content="citation_title=Generalized Gradient Approximation Made Simple;,citation_abstract=Generalized gradient approximations (GGA’s) for the exchange-correlation energy improve upon the local spin density (LSD) description of atoms, molecules, and solids. We present a simple derivation of a simple GGA, in which all parameters (other than those in LSD) are fundamental constants. Only general features of the detailed construction underlying the Perdew-Wang 1991 (PW91) GGA are invoked. Improvements over PW91 include an accurate description of the linear response of the uniform electron gas, correct behavior under uniform scaling, and a smoother potential.;,citation_author=John P. Perdew;,citation_author=Kieron Burke;,citation_author=Matthias Ernzerhof;,citation_publication_date=1996-10;,citation_cover_date=1996-10;,citation_year=1996;,citation_fulltext_html_url=https://link.aps.org/doi/10.1103/PhysRevLett.77.3865;,citation_issue=18;,citation_doi=10.1103/PhysRevLett.77.3865;,citation_volume=77;,citation_journal_title=Physical Review Letters;,citation_publisher=American Physical Society;">
<meta name="citation_reference" content="citation_title=Rationale for mixing exact exchange with density functional approximations;,citation_abstract=Density functional approximations for the exchange‐correlation energy EDFAxc of an electronic system are often improved by admixing some exact exchange Ex: Exc≊EDFAxc+(1/n)(Ex−EDFAx). This procedure is justified when the error in EDFAxc arises from the λ=0 or exchange end of the coupling‐constant integral ∫10dλ EDFAxc,λ. We argue that the optimum integer n is approximately the lowest order of Görling–Levy perturbation theory which provides a realistic description of the coupling‐constant dependence Exc,λ in the range 0≤λ≤1, whence n≊4 for atomization energies of typical molecules. We also propose a continuous generalization of n as an index of correlation strength, and a possible mixing of second‐order perturbation theory with the generalized gradient approximation.;,citation_author=John P. Perdew;,citation_author=Matthias Ernzerhof;,citation_author=Kieron Burke;,citation_publication_date=1996-12;,citation_cover_date=1996-12;,citation_year=1996;,citation_fulltext_html_url=https://doi.org/10.1063/1.472933;,citation_issue=22;,citation_doi=10.1063/1.472933;,citation_issn=0021-9606;,citation_volume=105;,citation_journal_title=The Journal of Chemical Physics;">
<meta name="citation_reference" content="citation_title=Electrostatic interactions involving lysine make major contributions to collagen triple-helix stability;,citation_abstract=Important stabilizing features for the collagen triple helix include the presence of Gly as every third residue, a high content of imino acids, and interchain hydrogen bonds. Host-guest peptides have been used previously to characterize triple-helix propensities of individual residues and Gly-X-Y triplets. Here, comparison of the thermal stabilities of host-guest peptides of the form (Gly-Pro-Hyp)3-Gly-X-Y-Gly-X’-Y’-(Gly-Pro-Hyp)3 extends the study to adjacent tripeptide sequences, to encompass the major classes of potential direct intramolecular interactions. Favorable hydrophobic interactions were observed, as well as stabilizing intrachain interactions between residues of opposite charge in the i and i + 3 positions. However, the greatest gain in triple-helix stability was achieved in the presence of Gly-Pro-Lys-Gly-Asp/Glu-Hyp sequences, leading to a T(m) value equal to that seen for a Gly-Pro-Hyp-Gly-Pro-Hyp sequence. This stabilization is seen for Lys but not for Arg and can be assigned to interchain ion pairs, as shown by molecular modeling. Computational analysis shows that Lys-Gly-Asp/Glu sequences are present at a frequency much greater than expected in collagen, suggesting this interaction is biologically important. These results add significantly to the understanding of which surface ion pairs can contribute to protein stability.;,citation_author=Anton V. Persikov;,citation_author=John A. M. Ramshaw;,citation_author=Alan Kirkpatrick;,citation_author=Barbara Brodsky;,citation_publication_date=2005-02;,citation_cover_date=2005-02;,citation_year=2005;,citation_issue=5;,citation_doi=10.1021/bi048216r;,citation_issn=0006-2960;,citation_pmid=15683226;,citation_volume=44;,citation_language=en-US;,citation_journal_title=Biochemistry;">
<meta name="citation_reference" content="citation_title=Mechanical Activation Drastically Accelerates Amide Bond Hydrolysis, Matching Enzyme Activity;,citation_abstract=Abstract Amide bonds, which include peptide bonds connecting amino acids in proteins and polypeptides, give proteins and synthetic polyamides their enormous strength. Although proteins and polyamides sustain mechanical force in nature and technology, how forces affect amide and peptide bond stability is still unknown. Using single‐molecule force spectroscopy, we discover that forces of only a few hundred pN accelerate amide hydrolysis 10 9 ‐fold, an acceleration hitherto only known from proteolytic enzymes. The drastic acceleration at low force precedes a moderate additional acceleration at nN forces. Quantum mechanochemical ab initio calculations explain these experimental results mechanistically and kinetically. Our findings reveal that, in contrast to previous belief, amide stability is strongly force dependent. These calculations provide a fundamental understanding of the role of mechanical activation in amide hydrolysis and point the way to potential applications from the recycling of macromolecular waste to the design of bioengineered proteolytic enzymes.;,citation_author=Michael F. Pill;,citation_author=Allan L. L. East;,citation_author=Dominik Marx;,citation_author=Martin K. Beyer;,citation_author=Hauke Clausen‐Schaumann;,citation_publication_date=2019-07;,citation_cover_date=2019-07;,citation_year=2019;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/10.1002/anie.201902752;,citation_issue=29;,citation_doi=10.1002/anie.201902752;,citation_issn=1433-7851, 1521-3773;,citation_volume=58;,citation_language=en-US;,citation_journal_title=Angewandte Chemie International Edition;">
<meta name="citation_reference" content="citation_title=Regulation of collagen biosynthesis by ascorbic acid: A review.;,citation_abstract=L-ascorbic acid is an essential cofactor for lysyl hydroxylase and prolyl hydroxylase, enzymes essential for collagen biosynthesis. In addition, L-ascorbic acid preferentially stimulates collagen synthesis in a manner which appears unrelated to the effect of L-ascorbic acid on hydroxylation reactions. This reaction is stereospecific and unrelated to intracellular degradation of collagen. The effect apparently occurs at a transcriptional or translational level, since L-ascorbic acid preferentially stimulates collagen-specific mRNA. In addition, it stimulates lysyl hydroxylase activity but inhibits prolyl hydroxylase activity in human skin fibroblasts in culture.;,citation_author=S. R. Pinnell;,citation_publication_date=1985;,citation_cover_date=1985;,citation_year=1985;,citation_fulltext_html_url=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2589959/;,citation_issue=6;,citation_issn=0044-0086;,citation_pmid=3008449;,citation_volume=58;,citation_journal_title=The Yale Journal of Biology and Medicine;">
<meta name="citation_reference" content="citation_title=Plotnine: A grammar of graphics for Python;,citation_author=The team;,citation_fulltext_html_url=https://github.com/has2k1/plotnine;,citation_doi=10.5281/zenodo.1325308;">
<meta name="citation_reference" content="citation_title=The Age of Scurvy;,citation_abstract=In a time of warring empires and transoceanic voyages, sailors dreaded scurvy more than any other disease.;,citation_author=Catherine Price;,citation_publication_date=2017-08;,citation_cover_date=2017-08;,citation_year=2017;,citation_fulltext_html_url=https://www.sciencehistory.org/stories/magazine/the-age-of-scurvy/;,citation_language=en-US;,citation_journal_title=Science History Institute;">
<meta name="citation_reference" content="citation_title=Diátaxis documentation framework;,citation_author=Daniele Procida;,citation_fulltext_html_url=https://diataxis.fr/;">
<meta name="citation_reference" content="citation_title=The PyMOL molecular graphics system, version 1.8;,citation_author=Schrödinger, LLC;,citation_publication_date=2015-11;,citation_cover_date=2015-11;,citation_year=2015;">
<meta name="citation_reference" content="citation_title=Stereochemistry of polypeptide chain configurations;,citation_author=G. N. Ramachandran;,citation_author=C. Ramakrishnan;,citation_author=V. Sasisekharan;,citation_publication_date=1963-07;,citation_cover_date=1963-07;,citation_year=1963;,citation_fulltext_html_url=https://www.sciencedirect.com/science/article/pii/S0022283663800236;,citation_issue=1;,citation_doi=10.1016/S0022-2836(63)80023-6;,citation_issn=0022-2836;,citation_volume=7;,citation_journal_title=Journal of Molecular Biology;">
<meta name="citation_reference" content="citation_title=The MEROPS database of proteolytic enzymes, their substrates and inhibitors in 2017 and a comparison with peptidases in the PANTHER database;,citation_author=Neil D Rawlings;,citation_author=Alan J Barrett;,citation_author=Paul D Thomas;,citation_author=Xiaosong Huang;,citation_author=Alex Bateman;,citation_author=Robert D Finn;,citation_publication_date=2018-01;,citation_cover_date=2018-01;,citation_year=2018;,citation_fulltext_html_url=http://academic.oup.com/nar/article/46/D1/D624/4626772;,citation_issue=D1;,citation_doi=10.1093/nar/gkx1134;,citation_issn=0305-1048, 1362-4962;,citation_volume=46;,citation_language=en-US;,citation_journal_title=Nucleic Acids Research;">
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;,citation_author=R Core Team;,citation_publication_date=2018;,citation_cover_date=2018;,citation_year=2018;,citation_fulltext_html_url=https://www.R-project.org/;">
<meta name="citation_reference" content="citation_title=Collagen breaks at weak sacrificial bonds taming its mechanoradicals;,citation_abstract=Collagen is a force-bearing, hierarchical structural protein important to all connective tissue. In tendon collagen, high load even below macroscopic failure level creates mechanoradicals by homolytic bond scission, similar to polymers. The location and type of initial rupture sites critically decide on both the mechanical and chemical impact of these micro-ruptures on the tissue, but are yet to be explored. We here use scale-bridging simulations supported by gel electrophoresis and mass spectrometry to determine breakage points in collagen. We find collagen crosslinks, as opposed to the backbone, to harbor the weakest bonds, with one particular bond in trivalent crosslinks as the most dominant rupture site. We identify this bond as sacrificial, rupturing prior to other bonds while maintaining the material’s integrity. Also, collagen’s weak bonds funnel ruptures such that the potentially harmful mechanoradicals are readily stabilized. Our results suggest this unique failure mode of collagen to be tailored towards combatting an early onset of macroscopic failure and material ageing.;,citation_author=Benedikt Rennekamp;,citation_author=Christoph Karfusehr;,citation_author=Markus Kurth;,citation_author=Aysecan Ünal;,citation_author=Debora Monego;,citation_author=Kai Riedmiller;,citation_author=Ganna Gryn’ova;,citation_author=David M. Hudson;,citation_author=Frauke Gräter;,citation_publication_date=2023-04;,citation_cover_date=2023-04;,citation_year=2023;,citation_fulltext_html_url=https://www.nature.com/articles/s41467-023-37726-z;,citation_issue=1;,citation_doi=10.1038/s41467-023-37726-z;,citation_issn=2041-1723;,citation_volume=14;,citation_language=en-US;,citation_journal_title=Nature Communications;,citation_publisher=Nature Publishing Group;">
<meta name="citation_reference" content="citation_title=Hybrid Kinetic Monte Carlo/Molecular Dynamics Simulations of Bond Scissions in Proteins;,citation_abstract=Proteins are exposed to various mechanical loads that can lead to covalent bond scissions even before macroscopic failure occurs. Knowledge of these molecular breakages is important to understand mechanical properties of the protein. In regular molecular dynamics (MD) simulations, covalent bonds are predefined, and reactions cannot occur. Furthermore, such events rarely take place on MD time scales. Existing approaches that tackle this limitation either rely on computationally expensive quantum calculations (e.g., QM/MM) or complex bond order formalisms in force fields (e.g., ReaxFF). To circumvent these limitations, we present a new reactive kinetic Monte Carlo/molecular dynamics (KIMMDY) scheme. Here, bond rupture rates are calculated based on the interatomic distances in the MD simulation and then serve as an input for a kinetic Monte Carlo step. This easily scalable hybrid approach drastically increases the accessible time scales. Using this new technique, we investigate bond ruptures in a multimillion atom system of tensed collagen, a structural protein found in skin, bones, and tendons. Our findings show a clear concentration of bond scissions near chemical cross-links in collagen. We also examine subsequent dynamic relaxation steps. Our method exhibits only a minor slowdown compared to classical MD and is straightforwardly applicable to other complex (bio)materials under load and related chemistries.;,citation_author=Benedikt Rennekamp;,citation_author=Fabian Kutzki;,citation_author=Agnieszka Obarska-Kosinska;,citation_author=Christopher Zapp;,citation_author=Frauke Gräter;,citation_publication_date=2020-01;,citation_cover_date=2020-01;,citation_year=2020;,citation_fulltext_html_url=https://doi.org/10.1021/acs.jctc.9b00786;,citation_issue=1;,citation_doi=10.1021/acs.jctc.9b00786;,citation_issn=1549-9618;,citation_volume=16;,citation_journal_title=Journal of Chemical Theory and Computation;,citation_publisher=American Chemical Society;">
<meta name="citation_reference" content="citation_title=The Collagen Family;,citation_abstract=Collagens are the most abundant proteins in mammals. The collagen family comprises 28 members that contain at least one triple-helical domain. Collagens are deposited in the extracellular matrix where most of them form supramolecular assemblies. Four collagens are type II membrane proteins that also exist in a soluble form released from the cell surface by shedding. Collagens play structural roles and contribute to mechanical properties, organization, and shape of tissues. They interact with cells via several receptor families and regulate their proliferation, migration, and differentiation. Some collagens have a restricted tissue distribution and hence specific biological functions., Collagens form supramolecular assemblies in the extracellular matrix that help organize and shape tissues. They also interact with cell surface receptors to regulate cell proliferation, differentiation, and migration.;,citation_author=Sylvie Ricard-Blum;,citation_publication_date=2011-01;,citation_cover_date=2011-01;,citation_year=2011;,citation_fulltext_html_url=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3003457/;,citation_issue=1;,citation_doi=10.1101/cshperspect.a004978;,citation_issn=1943-0264;,citation_pmid=21421911;,citation_volume=3;,citation_journal_title=Cold Spring Harbor Perspectives in Biology;">
<meta name="citation_reference" content="citation_title=Substituting density functional theory in reaction barrier calculations for hydrogen atom transfer in proteins;,citation_author=Kai Riedmiller;,citation_author=Patrick Reiser;,citation_author=Elizaveta Bobkova;,citation_author=Kiril Maltsev;,citation_author=Ganna Gryn’ova;,citation_author=Pascal Friederich;,citation_author=Frauke Gräter;,citation_publication_date=2024;,citation_cover_date=2024;,citation_year=2024;,citation_fulltext_html_url=https://pubs.rsc.org/en/content/articlelanding/2024/sc/d3sc03922f;,citation_issue=7;,citation_doi=10.1039/D3SC03922F;,citation_volume=15;,citation_language=en-US;,citation_journal_title=Chemical Science;,citation_publisher=Royal Society of Chemistry;">
<meta name="citation_reference" content="citation_title=Improved Peptide and Protein Torsional Energetics with the OPLS-AA Force Field;,citation_abstract=The development and validation of new peptide dihedral parameters are reported for the OPLS-AA force field. High accuracy quantum chemical methods were used to scan φ, ψ, χ1, and χ2 potential energy surfaces for blocked dipeptides. New Fourier coefficients for the dihedral angle terms of the OPLS-AA force field were fit to these surfaces, utilizing a Boltzmann-weighted error function and systematically examining the effects of weighting temperature. To prevent overfitting to the available data, a minimal number of new residue-specific and peptide-specific torsion terms were developed. Extensive experimental solution-phase and quantum chemical gas-phase benchmarks were used to assess the quality of the new parameters, named OPLS-AA/M, demonstrating significant improvement over previous OPLS-AA force fields. A Boltzmann weighting temperature of 2000 K was determined to be optimal for fitting the new Fourier coefficients for dihedral angle parameters. Conclusions are drawn from the results for best practices for developing new torsion parameters for protein force fields.;,citation_author=Michael J. Robertson;,citation_author=Julian Tirado-Rives;,citation_author=William L. Jorgensen;,citation_publication_date=2015-07;,citation_cover_date=2015-07;,citation_year=2015;,citation_fulltext_html_url=https://doi.org/10.1021/acs.jctc.5b00356;,citation_issue=7;,citation_doi=10.1021/acs.jctc.5b00356;,citation_issn=1549-9618;,citation_volume=11;,citation_journal_title=Journal of Chemical Theory and Computation;,citation_publisher=American Chemical Society;">
<meta name="citation_reference" content="citation_title=High-Resolution Image Synthesis with Latent Diffusion Models;,citation_abstract=By decomposing the image formation process into a sequential application of denoising autoencoders, diffusion models (DMs) achieve state-of-the-art synthesis results on image data and beyond. Additionally, their formulation allows for a guiding mechanism to control the image generation process without retraining. However, since these models typically operate directly in pixel space, optimization of powerful DMs often consumes hundreds of GPU days and inference is expensive due to sequential evaluations. To enable DM training on limited computational resources while retaining their quality and flexibility, we apply them in the latent space of powerful pretrained autoencoders. In contrast to previous work, training diffusion models on such a representation allows for the first time to reach a near-optimal point between complexity reduction and detail preservation, greatly boosting visual fidelity. By introducing cross-attention layers into the model architecture, we turn diffusion models into powerful and flexible generators for general conditioning inputs such as text or bounding boxes and high-resolution synthesis becomes possible in a convolutional manner. Our latent diffusion models (LDMs) achieve a new state of the art for image inpainting and highly competitive performance on various tasks, including unconditional image generation, semantic scene synthesis, and super-resolution, while significantly reducing computational requirements compared to pixel-based DMs. Code is available at https://github.com/CompVis/latent-diffusion .;,citation_author=Robin Rombach;,citation_author=Andreas Blattmann;,citation_author=Dominik Lorenz;,citation_author=Patrick Esser;,citation_author=Björn Ommer;,citation_publication_date=2022-04;,citation_cover_date=2022-04;,citation_year=2022;,citation_fulltext_html_url=http://arxiv.org/abs/2112.10752;,citation_doi=10.48550/arXiv.2112.10752;,citation_publisher=arXiv;">
<meta name="citation_reference" content="citation_title=Numerical integration of the cartesian equations of motion of a system with constraints: Molecular dynamics of &amp;amp;amp;lt;i&amp;gt;n&amp;lt;/i&amp;gt;-alkanes;,citation_abstract=A numerical algorithm integrating the 3N Cartesian equations of motion of a system of N points subject to holonomic constraints is formulated. The relations of constraint remain perfectly fulfilled at each step of the trajectory despite the approximate character of numerical integration. The method is applied to a molecular dynamics simulation of a liquid of 64 n-butane molecules and compared to a simulation using generalized coordinates. The method should be useful for molecular dynamics calculations on large molecules with internal degrees of freedom.;,citation_author=Jean-Paul Ryckaert;,citation_author=Giovanni Ciccotti;,citation_author=Herman J. C Berendsen;,citation_publication_date=1977-03;,citation_cover_date=1977-03;,citation_year=1977;,citation_fulltext_html_url=https://www.sciencedirect.com/science/article/pii/0021999177900985;,citation_issue=3;,citation_doi=10.1016/0021-9991(77)90098-5;,citation_issn=0021-9991;,citation_volume=23;,citation_journal_title=Journal of Computational Physics;">
<meta name="citation_reference" content="citation_title=Reliability of Distal Hamstring Tendon Length and Cross-sectional Area Using 3-D Freehand Ultrasound;,citation_author=Chrysostomos Sahinis;,citation_author=Eleftherios Kellis;,citation_author=Konstantinos Dafkou;,citation_author=Athanasios Ellinoudis;,citation_publication_date=2021-09;,citation_cover_date=2021-09;,citation_year=2021;,citation_fulltext_html_url=https://www.umbjournal.org/article/S0301-5629(21)00255-6/fulltext;,citation_issue=9;,citation_doi=10.1016/j.ultrasmedbio.2021.05.025;,citation_issn=0301-5629, 1879-291X;,citation_pmid=34246531;,citation_volume=47;,citation_language=en-US;,citation_journal_title=Ultrasound in Medicine and Biology;,citation_publisher=Elsevier;">
<meta name="citation_reference" content="citation_title=An Undulatory Theory of the Mechanics of Atoms and Molecules;,citation_abstract=The paper gives an account of the author’s work on a new form of quantum theory. §1. The Hamiltonian analogy between mechanics and optics. §2. The analogy is to be extended to include real &amp;amp;amp;quot;physical&amp;quot; or &quot;undulatory&quot; mechanics instead of mere geometrical mechanics. §3. The significance of wave-length; macro-mechanical and micro-mechanical problems. §4. The wave-equation and its application to the hydrogen atom. §5. The intrinsic reason for the appearance of discrete characteristic frequencies. §6. Other problems; intensity of emitted light. §7. The wave-equation derived from a Hamiltonian variation-principle; generalization to an arbitrary conservative system. §8. The wave-function physically means and determines a continuous distribution of electricity in space, the fluctuations of which determine the radiation by the laws of ordinary electrodynamics. §9. Non-conservative systems. Theory of dispersion and scattering and of the &quot;transitions&quot; between the &quot;stationary states.&quot; §10. The question of relativity and the action of a magnetic field. Incompleteness of that part of the theory.;,citation_author=E. Schrödinger;,citation_publication_date=1926-12;,citation_cover_date=1926-12;,citation_year=1926;,citation_fulltext_html_url=https://link.aps.org/doi/10.1103/PhysRev.28.1049;,citation_issue=6;,citation_doi=10.1103/PhysRev.28.1049;,citation_volume=28;,citation_journal_title=Physical Review;,citation_publisher=American Physical Society;">
<meta name="citation_reference" content="citation_title=An improved GROMOS96 force field for aliphatic hydrocarbons in the condensed phase;,citation_abstract=Over the past 4 years the GROMOS96 force field has been successfully used in biomolecular simulations, for example in peptide folding studies and detailed protein investigations, but no applications to lipid systems have been published yet. Here we provide a detailed investigation of aliphatic liquid systems. For liquids of larger aliphatic chains, n-heptane and longer, the standard GROMOS96 parameter sets 43A1 and 43A2 yield a too low pressure at the experimental density. Therefore, a reparametrization of the GROMOS96 force field regarding aliphatic carbons was initiated. The new force field parameter set 45A3 shows considerable improvements for n-alkanes, cyclo-, iso-, and neoalkanes and other branched aliphatics. Liquid densities and heat of vaporization are reproduced for almost all of these molecules. Excellent agreement is found with experiment for the free energy of hydration for alkanes. The GROMOS96 45A3 parameter set should, therefore, be suitable for application to lipid aggregates such as membranes and micelles, for mixed systems of aliphatics with or without water, for polymers, and other apolar systems that may interact with different biomolecules. © 2001 John Wiley &amp;amp;amp; Sons, Inc. J Comput Chem 22: 1205–1218, 2001;,citation_author=Lukas D. Schuler;,citation_author=Xavier Daura;,citation_author=Wilfred F. Gunsteren;,citation_publication_date=2001;,citation_cover_date=2001;,citation_year=2001;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1002/jcc.1078;,citation_issue=11;,citation_doi=10.1002/jcc.1078;,citation_issn=1096-987X;,citation_volume=22;,citation_language=en-US;,citation_journal_title=Journal of Computational Chemistry;">
<meta name="citation_reference" content="citation_title=SchNet – A deep learning architecture for molecules and materials;,citation_abstract=Deep learning has led to a paradigm shift in artificial intelligence, including web, text, and image search, speech recognition, as well as bioinformatics, with growing impact in chemical physics. Machine learning, in general, and deep learning, in particular, are ideally suitable for representing quantum-mechanical interactions, enabling us to model nonlinear potential-energy surfaces or enhancing the exploration of chemical compound space. Here we present the deep learning architecture SchNet that is specifically designed to model atomistic systems by making use of continuous-filter convolutional layers. We demonstrate the capabilities of SchNet by accurately predicting a range of properties across chemical space for molecules and materials, where our model learns chemically plausible embeddings of atom types across the periodic table. Finally, we employ SchNet to predict potential-energy surfaces and energy-conserving force fields for molecular dynamics simulations of small molecules and perform an exemplary study on the quantum-mechanical properties of C20-fullerene that would have been infeasible with regular ab initio molecular dynamics.;,citation_author=K. T. Schütt;,citation_author=H. E. Sauceda;,citation_author=P.-J. Kindermans;,citation_author=A. Tkatchenko;,citation_author=K.-R. Müller;,citation_publication_date=2018-03;,citation_cover_date=2018-03;,citation_year=2018;,citation_fulltext_html_url=https://doi.org/10.1063/1.5019779;,citation_issue=24;,citation_doi=10.1063/1.5019779;,citation_issn=0021-9606;,citation_volume=148;,citation_journal_title=The Journal of Chemical Physics;">
<meta name="citation_reference" content="citation_title=Mol* Viewer: Modern web app for 3D visualization and analysis of large biomolecular structures;,citation_author=David Sehnal;,citation_author=Sebastian Bittrich;,citation_author=Mandar Deshpande;,citation_author=Radka Svobodová;,citation_author=Karel Berka;,citation_author=Václav Bazgier;,citation_author=Sameer Velankar;,citation_author=Stephen K Burley;,citation_author=Jaroslav Koča;,citation_author=Alexander S Rose;,citation_publication_date=2021-07;,citation_cover_date=2021-07;,citation_year=2021;,citation_issue=W1;,citation_doi=10.1093/nar/gkab314;,citation_volume=49;,citation_journal_title=Nucleic Acids Research;">
<meta name="citation_reference" content="citation_title=The ReaxFF reactive force-field: Development, applications and future directions;,citation_abstract=The reactive force-field (ReaxFF) interatomic potential is a powerful computational tool for exploring, developing and optimizing material properties. Methods based on the principles of quantum mechanics (QM), while offering valuable theoretical guidance at the electronic level, are often too computationally intense for simulations that consider the full dynamic evolution of a system. Alternatively, empirical interatomic potentials that are based on classical principles require significantly fewer computational resources, which enables simulations to better describe dynamic processes over longer timeframes and on larger scales. Such methods, however, typically require a predefined connectivity between atoms, precluding simulations that involve reactive events. The ReaxFF method was developed to help bridge this gap. Approaching the gap from the classical side, ReaxFF casts the empirical interatomic potential within a bond-order formalism, thus implicitly describing chemical bonding without expensive QM calculations. This article provides an overview of the development, application, and future directions of the ReaxFF method.;,citation_author=Thomas P. Senftle;,citation_author=Sungwook Hong;,citation_author=Md Mahbubul Islam;,citation_author=Sudhir B. Kylasa;,citation_author=Yuanxia Zheng;,citation_author=Yun Kyung Shin;,citation_author=Chad Junkermeier;,citation_author=Roman Engel-Herbert;,citation_author=Michael J. Janik;,citation_author=Hasan Metin Aktulga;,citation_author=Toon Verstraelen;,citation_author=Ananth Grama;,citation_author=Adri C. T. Duin;,citation_publication_date=2016-03;,citation_cover_date=2016-03;,citation_year=2016;,citation_fulltext_html_url=https://www.nature.com/articles/npjcompumats201511;,citation_issue=1;,citation_doi=10.1038/npjcompumats.2015.11;,citation_issn=2057-3960;,citation_volume=2;,citation_language=en-US;,citation_journal_title=npj Computational Materials;,citation_publisher=Nature Publishing Group;">
<meta name="citation_reference" content="citation_title=Grappa – a machine learned molecular mechanics force field;,citation_abstract=Simulating large molecular systems over long timescales requires force fields that are both accurate and efficient. In recent years, E(3) equivariant neural networks have lifted the tension between computational efficiency and accuracy of force fields, but they are still several orders of magnitude more expensive than established molecular mechanics (MM) force fields. Here, we propose Grappa, a machine learning framework to predict MM parameters from the molecular graph, employing a graph attentional neural network and a transformer with symmetry-preserving positional encoding. The resulting Grappa force field outperforms tabulated and machine-learned MM force fields in terms of accuracy at the same computational efficiency and can be used in existing Molecular Dynamics (MD) engines like GROMACS and OpenMM. It predicts energies and forces of small molecules, peptides, and RNA at state-of-the-art MM accuracy, while also reproducing experimentally measured values for J-couplings. With its simple input features and high data-efficiency, Grappa is well suited for extensions to uncharted regions of chemical space, which we show on the example of peptide radicals. We demonstrate Grappa’s transferability to macromolecules in MD simulations from a small fast-folding protein up to a whole virus particle. Our force field sets the stage for biomolecular simulations closer to chemical accuracy, but with the same computational cost as established protein force fields.;,citation_author=Leif Seute;,citation_author=Eric Hartmann;,citation_author=Jan Stühmer;,citation_author=Frauke Gräter;,citation_publication_date=2025-02;,citation_cover_date=2025-02;,citation_year=2025;,citation_fulltext_html_url=https://pubs.rsc.org/en/content/articlelanding/2025/sc/d4sc05465b;,citation_issue=6;,citation_doi=10.1039/D4SC05465B;,citation_issn=2041-6539;,citation_volume=16;,citation_language=en-US;,citation_journal_title=Chemical Science;,citation_publisher=The Royal Society of Chemistry;">
<meta name="citation_reference" content="citation_title=The Theory of Complex Spectra;,citation_abstract=Atomic multiplets are treated by wave mechanics, without using group theory. In part 1 Hund’s scheme for multiplet classification is derived directly from theory. Part 2 is devoted to the computation of the energy distances between multiplets, and comparison of these distances with experiment in some typical examples. There is no treatment of the separations between the various terms of a multiplet, since that has been done elsewhere, but only between one multiplet and another. It is found that Hund’s rule, that terms of large L and S values lie lowest, has no general significance; the present theory leads to the same results as the rule when it is obeyed experimentally, but many cases which were exceptions to that rule are in agreement with the theory. The method of calculation of multiplet distances is described in sufficient detail, with the necessary tables of coefficients, etc., so that further checks with experiment could easily be made.;,citation_author=J. C. Slater;,citation_publication_date=1929-11;,citation_cover_date=1929-11;,citation_year=1929;,citation_fulltext_html_url=https://link.aps.org/doi/10.1103/PhysRev.34.1293;,citation_issue=10;,citation_doi=10.1103/PhysRev.34.1293;,citation_volume=34;,citation_journal_title=Physical Review;,citation_publisher=American Physical Society;">
<meta name="citation_reference" content="citation_title=ANI-1: An extensible neural network potential with DFT accuracy at force field computational cost;,citation_author=J. Smith;,citation_author=O. Isayev;,citation_author=A. E. Roitberg;,citation_publication_date=2017;,citation_cover_date=2017;,citation_year=2017;,citation_fulltext_html_url=https://pubs.rsc.org/en/content/articlelanding/2017/sc/c6sc05720a;,citation_issue=4;,citation_doi=10.1039/C6SC05720A;,citation_volume=8;,citation_language=en-US;,citation_journal_title=Chemical Science;,citation_publisher=Royal Society of Chemistry;">
<meta name="citation_reference" content="citation_title=Exploring the Helix-Coil Transition via All-Atom Equilibrium Ensemble Simulations;,citation_author=Eric J. Sorin;,citation_author=Vijay S. Pande;,citation_publication_date=2005-04;,citation_cover_date=2005-04;,citation_year=2005;,citation_fulltext_html_url=https://www.cell.com/biophysj/abstract/S0006-3495(05)73304-X;,citation_issue=4;,citation_doi=10.1529/biophysj.104.051938;,citation_issn=0006-3495, 1542-0086;,citation_pmid=15665128;,citation_volume=88;,citation_language=en-US;,citation_journal_title=Biophysical Journal;,citation_publisher=Elsevier;">
<meta name="citation_reference" content="citation_title=Extension to the weighted histogram analysis method: Combining umbrella sampling with free energy calculations;,citation_abstract=The Weighted Histogram Analysis Method (WHAM) of Kumar et al. (J. Comput. Chem. 13 (1992) 1011), is used to combine free energy perturbations with umbrella sampling calculations. The formulation is general and allows optimal calculation of the free energies from a set of molecular dynamics simulations generated in the presence of arbitrary biasing umbrella sampling window potentials. The method yields the free energy associated with a given simulation as well as the probability distribution of the molecular system configurations by extracting the information contained in all the biased simulations (the windows) in an optimal way. The method presents some advantages compared to the standard free energy perturbation (FEP) and thermodynamic integration (TI) methods, because the window potential can be used for restricting the conformational space to specific regions during free energy calculations.;,citation_author=Marc Souaille;,citation_author=Benoı̂t Roux;,citation_publication_date=2001-03;,citation_cover_date=2001-03;,citation_year=2001;,citation_fulltext_html_url=https://www.sciencedirect.com/science/article/pii/S0010465500002150;,citation_issue=1;,citation_doi=10.1016/S0010-4655(00)00215-0;,citation_issn=0010-4655;,citation_volume=135;,citation_journal_title=Computer Physics Communications;">
<meta name="citation_reference" content="citation_title=Martini 3: A general purpose force field for coarse-grained molecular dynamics;,citation_abstract=The coarse-grained Martini force field is widely used in biomolecular simulations. Here we present the refined model, Martini 3 (http://cgmartini.nl), with an improved interaction balance, new bead types and expanded ability to include specific interactions representing, for example, hydrogen bonding and electronic polarizability. The updated model allows more accurate predictions of molecular packing and interactions in general, which is exemplified with a vast and diverse set of applications, ranging from oil/water partitioning and miscibility data to complex molecular systems, involving protein–protein and protein–lipid interactions and material science applications as ionic liquids and aedamers.;,citation_author=Paulo C. T. Souza;,citation_author=Riccardo Alessandri;,citation_author=Jonathan Barnoud;,citation_author=Sebastian Thallmair;,citation_author=Ignacio Faustino;,citation_author=Fabian Grünewald;,citation_author=Ilias Patmanidis;,citation_author=Haleh Abdizadeh;,citation_author=Bart M. H. Bruininks;,citation_author=Tsjerk A. Wassenaar;,citation_author=Peter C. Kroon;,citation_author=Josef Melcr;,citation_author=Vincent Nieto;,citation_author=Valentina Corradi;,citation_author=Hanif M. Khan;,citation_author=Jan Domański;,citation_author=Matti Javanainen;,citation_author=Hector Martinez-Seara;,citation_author=Nathalie Reuter;,citation_author=Robert B. Best;,citation_author=Ilpo Vattulainen;,citation_author=Luca Monticelli;,citation_author=Xavier Periole;,citation_author=D. Peter Tieleman;,citation_author=Alex H. Vries;,citation_author=Siewert J. Marrink;,citation_publication_date=2021-04;,citation_cover_date=2021-04;,citation_year=2021;,citation_fulltext_html_url=https://www.nature.com/articles/s41592-021-01098-3;,citation_issue=4;,citation_doi=10.1038/s41592-021-01098-3;,citation_issn=1548-7105;,citation_volume=18;,citation_language=en-US;,citation_journal_title=Nature Methods;,citation_publisher=Nature Publishing Group;">
<meta name="citation_reference" content="citation_title=Achilles Tendon Cross Sectional Area Changes Over a Division I NCAA Cross Country Season;,citation_abstract=Tracking Achilles tendon cross sectional area (CSA) changes may help clinicians understand exercise adaptations and Achilles tendon injury mechanisms. To track adaptations in the Achilles tendon in response to a cross country season.;,citation_author=Joshua K. Sponbeck;,citation_author=Candace L. Perkins;,citation_author=Matthew J. Berg;,citation_author=Justin H. Rigby;,citation_publication_date=2017-12;,citation_cover_date=2017-12;,citation_year=2017;,citation_fulltext_html_url=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5786201/;,citation_issue=8;,citation_doi=10.70252/NOKT1275;,citation_issn=1939-795X;,citation_pmid=29399249;,citation_volume=10;,citation_journal_title=International Journal of Exercise Science;">
<meta name="citation_reference" content="citation_title=Interstitial pH in human skeletal muscle during and after dynamic graded exercise;,citation_abstract=In this study a new method has been used to measure interstitial pH continuously in human muscle during graded exercise. Human subjects performed 5 min of one-legged knee-extensor exercise at power outputs of 30, 50 and 70 W. Muscle interstitial pH ...;,citation_author=Darrin Street;,citation_author=Jens Bangsbo;,citation_author=Carsten Juel;,citation_publication_date=2001-12;,citation_cover_date=2001-12;,citation_year=2001;,citation_fulltext_html_url=https://pmc.ncbi.nlm.nih.gov/articles/PMC2279011/;,citation_issue=Pt 3;,citation_doi=10.1111/j.1469-7793.2001.00993.x;,citation_pmid=11744771;,citation_volume=537;,citation_language=en-US;,citation_journal_title=The Journal of Physiology;">
<meta name="citation_reference" content="citation_title=Mechanical properties of human patellar tendon at the hierarchical levels of tendon and fibril;,citation_abstract=Tendons are strong hierarchical structures, but how tensile forces are transmitted between different levels remains incompletely understood. Collagen fibrils are thought to be primary determinants of whole tendon properties, and therefore we hypothesized that the whole human patellar tendon and its distinct collagen fibrils would display similar mechanical properties. Human patellar tendons (n = 5) were mechanically tested in vivo by ultrasonography. Biopsies were obtained from each tendon, and individual collagen fibrils were dissected and tested mechanically by atomic force microscopy. The Young’s modulus was 2.0 ± 0.5 GPa, and the toe region reached 3.3 ± 1.9% strain in whole patellar tendons. Based on dry cross-sectional area, the Young’s modulus of isolated collagen fibrils was 2.8 ± 0.3 GPa, and the toe region reached 0.86 ± 0.08% strain. The measured fibril modulus was insufficient to account for the modulus of the tendon in vivo when fibril content in the tendon was accounted for. Thus, our original hypothesis was not supported, although the in vitro fibril modulus corresponded well with reported in vitro tendon values. This correspondence together with the fibril modulus not being greater than that of tendon supports that fibrillar rather than interfibrillar properties govern the subfailure tendon response, making the fibrillar level a meaningful target of intervention. The lower modulus found in vitro suggests a possible adverse effect of removing the tissue from its natural environment. In addition to the primary work comparing the two hierarchical levels, we also verified the existence of viscoelastic behavior in isolated human collagen fibrils.;,citation_author=René B. Svensson;,citation_author=Philip Hansen;,citation_author=Tue Hassenkam;,citation_author=Bjarki T. Haraldsson;,citation_author=Per Aagaard;,citation_author=Vuokko Kovanen;,citation_author=Michael Krogsgaard;,citation_author=Michael Kjaer;,citation_author=S. Peter Magnusson;,citation_publication_date=2012-02;,citation_cover_date=2012-02;,citation_year=2012;,citation_fulltext_html_url=https://journals.physiology.org/doi/full/10.1152/japplphysiol.01172.2011;,citation_issue=3;,citation_doi=10.1152/japplphysiol.01172.2011;,citation_issn=8750-7587;,citation_volume=112;,citation_journal_title=Journal of Applied Physiology;,citation_publisher=American Physiological Society;">
<meta name="citation_reference" content="citation_title=Methodus incrementorum directa &amp;amp;amp; inversa. Auctore Brook Taylor ...;,citation_abstract=Leg. in pelle; sul dorso tassello e fregi impressi in oro; tagli spruzzati di azzurro; cuffie e labbri decorati in oro; c. di guardia marmorizzate. Sulla controguardia anteriore etichetta di possesso: &amp;amp;quot;Thomae e com.bus Valpergiae et Maxini&amp;quot;; nota ms. sul front;,citation_author=Brook Taylor;,citation_publication_date=1715;,citation_cover_date=1715;,citation_year=1715;,citation_fulltext_html_url=http://archive.org/details/UFIE003454_TO0324_PNI-2529_000000;,citation_language=lat;">
<meta name="citation_reference" content="citation_title=Pandas-dev/pandas: Pandas;,citation_abstract=Pandas is a powerful data structures for data analysis, time series, and statistics.;,citation_author=The team;,citation_publication_date=2025-09;,citation_cover_date=2025-09;,citation_year=2025;,citation_fulltext_html_url=https://zenodo.org/records/17229934;,citation_doi=10.5281/zenodo.17229934;,citation_publisher=Zenodo;">
<meta name="citation_reference" content="citation_title=Bond dissociation energies of X–H bonds in proteins;,citation_abstract=A complete dataset of bond dissociation energies of X–H bonds in amino acids is reported, calculated using state of the art DFT methods. It shall aid interpreting experiments or predicting reactivity via machine learning. , Knowledge of reliable X–H bond dissociation energies (X = C, N, O, S) for amino acids in proteins is key for studying the radical chemistry of proteins. X–H bond dissociation energies of model dipeptides were computed using the isodesmic reaction method at the BMK/6-31+G(2df,p) and G4(MP2)-6X levels of theory. The density functional theory values agree well with the composite-level calculations. By this high level of theory, combined with a careful choice of reference compounds and peptide model systems, our work provides a highly valuable data set of bond dissociation energies with unprecedented accuracy and comprehensiveness. It will likely prove useful to predict protein biochemistry involving radicals, e.g. , by machine learning.;,citation_author=Wojtek Treyde;,citation_author=Kai Riedmiller;,citation_author=Frauke Gräter;,citation_publication_date=2022;,citation_cover_date=2022;,citation_year=2022;,citation_fulltext_html_url=https://xlink.rsc.org/?DOI=D2RA04002F;,citation_issue=53;,citation_doi=10.1039/D2RA04002F;,citation_issn=2046-2069;,citation_volume=12;,citation_language=en-US;,citation_journal_title=RSC Advances;">
<meta name="citation_reference" content="citation_title=PLUMED 2: New feathers for an old bird;,citation_abstract=Enhancing sampling and analyzing simulations are central issues in molecular simulation. Recently, we introduced PLUMED, an open-source plug-in that provides some of the most popular molecular dynamics (MD) codes with implementations of a variety of different enhanced sampling algorithms and collective variables (CVs). The rapid changes in this field, in particular new directions in enhanced sampling and dimensionality reduction together with new hardware, require a code that is more flexible and more efficient. We therefore present PLUMED 2 here—a complete rewrite of the code in an object-oriented programming language (C++). This new version introduces greater flexibility and greater modularity, which both extends its core capabilities and makes it far easier to add new methods and CVs. It also has a simpler interface with the MD engines and provides a single software library containing both tools and core facilities. Ultimately, the new code better serves the ever-growing community of users and contributors in coping with the new challenges arising in the field. Program summary Program title: PLUMED 2 Catalogue identifier: AEEE_v2_0 Program summary URL:http://cpc.cs.qub.ac.uk/summaries/AEEE_v2_0.html Program obtainable from: CPC Program Library, Queen’s University, Belfast, N. Ireland Licensing provisions: Yes No. of lines in distributed program, including test data, etc.: 700646 No. of bytes in distributed program, including test data, etc.: 6618136 Distribution format: tar.gz Programming language: ANSI-C++. Computer: Any computer capable of running an executable produced by a C++&nbsp;compiler. Operating system: Linux operating system, Unix OSs. Has the code been vectorized or parallelized?: Yes, parallelized using MPI. RAM: Depends on the number of atoms, the method chosen and the collective variables used. Classification: 3, 7.7, 23. Catalogue identifier of previous version: AEEE_v1_0. Journal reference of previous version: Comput. Phys. Comm. 180 (2009) 1961. External routines: GNU libmatheval, Lapack, Blas, MPI. Does the new version supersede the previous version?: This version supersedes the previous version for the most part. There are a small number of very specific situations where the previous version is better, due to performance or to non-ported features. We are actively working on porting these last few features into the new code. Nature of problem: Calculation of free-energy surfaces for molecular systems of interest in biology, chemistry and materials science, on the fly and a posteriori analysis of molecular dynamics trajectories using advanced collective variables. Solution method: Implementations of various collective variables and enhanced sampling techniques. Reasons for new version: The old version was difficult to maintain and its design was not as flexible as this new version. This lack of flexibility made it difficult to implement a number of novel methods that have emerged since the release of the original code. Summary of revisions: The new version of the code has a completely redesigned architecture, which allows for several important enhancements. This allows for a much simpler and robust input syntax and for improved performance. In addition, it provides several, more-complex collective variables which could not have been written using the previous implementation. Furthermore, the entire code is fully documented so it is easier to extend. Finally, the code is designed so that users can implement new variables directly in the input files and thus develop bespoke applications of these powerful algorithms. Unusual features: PLUMED 2 can be used either as a standalone program, e.g. for a posteriori analysis of trajectories, or as a library embedded in a molecular dynamics code (such as GROMACS, NAMD, Quantum ESPRESSO, and LAMMPS). Interfaces with these particular codes are provided in patches, which a simple script will insert into the underlying molecular dynamics codes source code files. For other molecular dynamics codes there is extensive documentation on how to add PLUMED in our manual. Additional comments: The distribution file contains a test suite, user and developer documentation and a collection of patches and utilities. Running time: Depends on the number of atoms, the method chosen and the collective variables used. The regression test suite provided takes approximately 1 min to run.;,citation_author=Gareth A. Tribello;,citation_author=Massimiliano Bonomi;,citation_author=Davide Branduardi;,citation_author=Carlo Camilloni;,citation_author=Giovanni Bussi;,citation_publication_date=2014-02;,citation_cover_date=2014-02;,citation_year=2014;,citation_fulltext_html_url=https://www.sciencedirect.com/science/article/pii/S0010465513003196;,citation_issue=2;,citation_doi=10.1016/j.cpc.2013.09.018;,citation_issn=0010-4655;,citation_volume=185;,citation_journal_title=Computer Physics Communications;">
<meta name="citation_reference" content="citation_title=Gaussian basis sets for accurate calculations on molecular systems in gas and condensed phases;,citation_abstract=We present a library of Gaussian basis sets that has been specifically optimized to perform accurate molecular calculations based on density functional theory. It targets a wide range of chemical environments, including the gas phase, interfaces, and the condensed phase. These generally contracted basis sets, which include diffuse primitives, are obtained minimizing a linear combination of the total energy and the condition number of the overlap matrix for a set of molecules with respect to the exponents and contraction coefficients of the full basis. Typically, for a given accuracy in the total energy, significantly fewer basis functions are needed in this scheme than in the usual split valence scheme, leading to a speedup for systems where the computational cost is dominated by diagonalization. More importantly, binding energies of hydrogen bonded complexes are of similar quality as the ones obtained with augmented basis sets, i.e., have a small (down to 0.2 kcal/mol) basis set superposition error, and the monomers have dipoles within 0.1 D of the basis set limit. However, contrary to typical augmented basis sets, there are no near linear dependencies in the basis, so that the overlap matrix is always well conditioned, also, in the condensed phase. The basis can therefore be used in first principles molecular dynamics simulations and is well suited for linear scaling calculations.;,citation_author=Joost VandeVondele;,citation_author=Jürg Hutter;,citation_publication_date=2007-09;,citation_cover_date=2007-09;,citation_year=2007;,citation_issue=11;,citation_doi=10.1063/1.2770708;,citation_issn=0021-9606;,citation_pmid=17887826;,citation_volume=127;,citation_language=en-US;,citation_journal_title=The Journal of Chemical Physics;">
<meta name="citation_reference" content="citation_title=ReaxFF:  A Reactive Force Field for Hydrocarbons;,citation_abstract=To make practical the molecular dynamics simulation of large scale reactive chemical systems (1000s of atoms), we developed ReaxFF, a force field for reactive systems. ReaxFF uses a general relationship between bond distance and bond order on one hand and between bond order and bond energy on the other hand that leads to proper dissociation of bonds to separated atoms. Other valence terms present in the force field (angle and torsion) are defined in terms of the same bond orders so that all these terms go to zero smoothly as bonds break. In addition, ReaxFF has Coulomb and Morse (van der Waals) potentials to describe nonbond interactions between all atoms (no exclusions). These nonbond interactions are shielded at short range so that the Coulomb and van der Waals interactions become constant as Rij → 0. We report here the ReaxFF for hydrocarbons. The parameters were derived from quantum chemical calculations on bond dissociation and reactions of small molecules plus heat of formation and geometry data for a number of stable hydrocarbon compounds. We find that the ReaxFF provides a good description of these data. Generally, the results are of an accuracy similar or better than PM3, while ReaxFF is about 100 times faster. In turn, the PM3 is about 100 times faster than the QC calculations. Thus, with ReaxFF we hope to be able to study complex reactions in hydrocarbons.;,citation_author=Adri C. T. Duin;,citation_author=Siddharth Dasgupta;,citation_author=Francois Lorant;,citation_author=William A. Goddard;,citation_publication_date=2001-10;,citation_cover_date=2001-10;,citation_year=2001;,citation_fulltext_html_url=https://doi.org/10.1021/jp004368u;,citation_issue=41;,citation_doi=10.1021/jp004368u;,citation_issn=1089-5639;,citation_volume=105;,citation_journal_title=The Journal of Physical Chemistry A;,citation_publisher=American Chemical Society;">
<meta name="citation_reference" content="citation_title=Computer &amp;amp;amp;quot;Experiments&amp;quot; on Classical Fluids. I. Thermodynamical Properties of Lennard-Jones Molecules;,citation_abstract=The equation of motion of a system of 864 particles interacting through a Lennard-Jones potential has been integrated for various values of the temperature and density, relative, generally, to a fluid state. The equilibrium properties have been calculated and are shown to agree very well with the corresponding properties of argon. It is concluded that, to a good approximation, the equilibrium state of argon can be described through a two-body potential.;,citation_author=Loup Verlet;,citation_publication_date=1967-07;,citation_cover_date=1967-07;,citation_year=1967;,citation_fulltext_html_url=https://link.aps.org/doi/10.1103/PhysRev.159.98;,citation_issue=1;,citation_doi=10.1103/PhysRev.159.98;,citation_volume=159;,citation_journal_title=Physical Review;,citation_publisher=American Physical Society;">
<meta name="citation_reference" content="citation_title=Aging of the Collagen Fiber;,citation_author=F. Verzár;,citation_publication_date=1964;,citation_cover_date=1964;,citation_year=1964;,citation_fulltext_html_url=https://linkinghub.elsevier.com/retrieve/pii/B9781483167510500124;,citation_doi=10.1016/B978-1-4831-6751-0.50012-4;,citation_isbn=978-1-4831-6751-0;,citation_volume=2;,citation_language=en-US;,citation_inbook_title=International Review of Connective Tissue Research;">
<meta name="citation_reference" content="citation_title=VMD – Visual Molecular Dynamics;,citation_author=William Humphrey;,citation_author=Andrew Dalke;,citation_author=Klaus Schulten;,citation_publication_date=1996;,citation_cover_date=1996;,citation_year=1996;,citation_volume=14;,citation_journal_title=Journal of Molecular Graphics;">
<meta name="citation_reference" content="citation_title=Stochastic Simulation of Biomolecular Networks in Dynamic Environments;,citation_abstract=Simulation of biomolecular networks is now indispensable for studying biological systems, from small reaction networks to large ensembles of cells. Here we present a novel approach for stochastic simulation of networks embedded in the dynamic environment of the cell and its surroundings. We thus sample trajectories of the stochastic process described by the chemical master equation with time-varying propensities. A comparative analysis shows that existing approaches can either fail dramatically, or else can impose impractical computational burdens due to numerical integration of reaction propensities, especially when cell ensembles are studied. Here we introduce the Extrande method which, given a simulated time course of dynamic network inputs, provides a conditionally exact and several orders-of-magnitude faster simulation solution. The new approach makes it feasible to demonstrate—using decision-making by a large population of quorum sensing bacteria—that robustness to fluctuations from upstream signaling places strong constraints on the design of networks determining cell fate. Our approach has the potential to significantly advance both understanding of molecular systems biology and design of synthetic circuits.;,citation_author=Margaritis Voliotis;,citation_author=Philipp Thomas;,citation_author=Ramon Grima;,citation_author=Clive G. Bowsher;,citation_publication_date=2016-06;,citation_cover_date=2016-06;,citation_year=2016;,citation_fulltext_html_url=https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004923;,citation_issue=6;,citation_doi=10.1371/journal.pcbi.1004923;,citation_issn=1553-7358;,citation_volume=12;,citation_language=en-US;,citation_journal_title=PLOS Computational Biology;,citation_publisher=Public Library of Science;">
<meta name="citation_reference" content="citation_title=How well does a restrained electrostatic potential (RESP) model perform in calculating conformational energies of organic and biological molecules?;,citation_abstract=In this study, we present conformational energies for a molecular mechanical model (Parm99) developed for organic and biological molecules using the restrained electrostatic potential (RESP) approach to derive the partial charges. This approach uses the simple “generic” force field model (Parm94), and attempts to add a minimal number of extra Fourier components to the torsional energies, but doing so only when there is a physical justification. The results are quite encouraging, not only for the 34-molecule set that has been studied by both the highest level ab initiomodel (GVB/LMP2) and experiment, but also for the 55-molecule set for which high-quality experimental data are available. Considering the 55 molecules studied by all the force field models for which there are experimental data, the average absolute errors (AAEs) are 0.28 (this model), 0.52 (MM3), 0.57 (CHARMm [MSI]), and 0.43 kcal/mol (MMFF). For the 34-molecule set, the AAEs of this model versus experiment and ab initio are 0.28 and 0.27 kcal/mol, respectively. This is a lower error than found with MM3 and CHARMm, and is comparable to that found with MMFF (0.31 and 0.22 kcal/mol). We also present two examples of how well the torsional parameters are transferred from the training set to the test set. The absolute errors of molecules in the test set are only slightly larger than in the training set (differences of $&amp;amp;amp;lt;$0.1 kcal/mol). Therefore, it can be concluded that a simple “generic” force field with a limited number of specific torsional parameters can describe intra- and intermolecular interactions, although all comparison molecules were selected from our 82-compound training set. We also show how this effective two-body model can be extended for use with a nonadditive force field (NAFF), both with and without lone pairs. Without changing the torsional parameters, the use of more accurate charges and polarization leads to an increase in average absolute error compared with experiment, but adjustment of the parameters restores the level of agreement found with the additive model. After reoptimizing the Ψ, Φ torsional parameters in peptides using alanine dipeptide (6 conformational pairs) and alanine tetrapeptide (11 conformational pairs), the new model gives better energies than the Cornell et al. ( J Am Chem Soc 1995, 117, 5179–5197) force field. The average absolute error of this model for high-level ab initio calculation is 0.82 kcal/mol for alanine dipeptide and tetrapeptide as compared with 1.80 kcal/mol for the Cornell et al. model. For nucleosides, the new model also gives improved energies compared with the Cornell et al. model. To optimize force field parameters, we developed a program called parmscan, which can iteratively scan the torsional parameters in a systematic manner and finally obtain the best torsional potentials. Besides the organic molecules in our test set, parmscan was also successful in optimizing the Ψ, Φ torsional parameters in peptides to significantly improve agreement between molecular mechanical and high-level ab initio energies. © 2000 John Wiley &amp; Sons, Inc. J Comput Chem 21: 1049–1074, 2000;,citation_author=Junmei Wang;,citation_author=Piotr Cieplak;,citation_author=Peter A. Kollman;,citation_publication_date=2000;,citation_cover_date=2000;,citation_year=2000;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/abs/10.1002/1096-987X%28200009%2921%3A12%3C1049%3A%3AAID-JCC3%3E3.0.CO%3B2-F;,citation_issue=12;,citation_doi=10.1002/1096-987X(200009)21:12&amp;amp;lt;1049::AID-JCC3&amp;gt;3.0.CO;2-F;,citation_issn=1096-987X;,citation_volume=21;,citation_language=fr-FR;,citation_journal_title=Journal of Computational Chemistry;">
<meta name="citation_reference" content="citation_title=Welcome to the tidyverse;,citation_author=Hadley Wickham;,citation_author=Mara Averick;,citation_author=Jennifer Bryan;,citation_author=Winston Chang;,citation_author=Lucy D’Agostino McGowan;,citation_author=Romain François;,citation_author=Garrett Grolemund;,citation_author=Alex Hayes;,citation_author=Lionel Henry;,citation_author=Jim Hester;,citation_author=Max Kuhn;,citation_author=Thomas Lin Pedersen;,citation_author=Evan Miller;,citation_author=Stephan Milton Bache;,citation_author=Kirill Müller;,citation_author=Jeroen Ooms;,citation_author=David Robinson;,citation_author=Dana Paige Seidel;,citation_author=Vitalie Spinu;,citation_author=Kohske Takahashi;,citation_author=Davis Vaughan;,citation_author=Claus Wilke;,citation_author=Kara Woo;,citation_author=Hiroaki Yutani;,citation_publication_date=2019;,citation_cover_date=2019;,citation_year=2019;,citation_issue=43;,citation_doi=10.21105/joss.01686;,citation_volume=4;,citation_journal_title=Journal of Open Source Software;">
<meta name="citation_reference" content="citation_title=Base-Catalyzed Peptide Hydrolysis Is Insensitive to Mechanical Stress;,citation_abstract=Biochemical reactions can be guided by mechanical stress. An external force has been previously shown both experimentally and theoretically to act as a catalyst for the scission of a disulfide bond in thiol/disulfide exchange reactions. How the dynamics of peptide hydrolysis, one of the most prevalent biochemical reactions, is influenced by a stretching force was investigated here using combined quantum and molecular mechanical (QM/MM) simulations together with transition path sampling. Our simulations predict mechanical force to only marginally enhance the reactivity of the rate-limiting step, the nucleophilic attack of hydroxide to the peptide moiety, and not to alter the reaction mechanism, even though the peptide bond and its π-electron conjugation is weakened by force. We describe a previously unidentified hydrogen bonded intermediate state, which is likely to play a role in general in base-catalyzed and analogous enzymatic reactions. Our predictions can be directly tested by single molecule stretching experiments.;,citation_author=Fei Xia;,citation_author=Agnieszka K. Bronowska;,citation_author=Shanmei Cheng;,citation_author=Frauke Gräter;,citation_publication_date=2011-08;,citation_cover_date=2011-08;,citation_year=2011;,citation_fulltext_html_url=https://doi.org/10.1021/jp202162r;,citation_issue=33;,citation_doi=10.1021/jp202162r;,citation_issn=1520-6106;,citation_volume=115;,citation_journal_title=The Journal of Physical Chemistry B;,citation_publisher=American Chemical Society;">
<meta name="citation_reference" content="citation_title=Mechanoradicals in tensed tendon collagen as a source of oxidative stress;,citation_abstract=As established nearly a century ago, mechanoradicals originate from homolytic bond scission in polymers. The existence, nature and biological relevance of mechanoradicals in proteins, instead, are unknown. We here show that mechanical stress on collagen produces radicals and subsequently reactive oxygen species, essential biological signaling molecules. Electron-paramagnetic resonance (EPR) spectroscopy of stretched rat tail tendon, atomistic molecular dynamics simulations and quantum-chemical calculations show that the radicals form by bond scission in the direct vicinity of crosslinks in collagen. Radicals migrate to adjacent clusters of aromatic residues and stabilize on oxidized tyrosyl radicals, giving rise to a distinct EPR spectrum consistent with a stable dihydroxyphenylalanine (DOPA) radical. The protein mechanoradicals, as a yet undiscovered source of oxidative stress, finally convert into hydrogen peroxide. Our study suggests collagen I to have evolved as a radical sponge against mechano-oxidative damage and proposes a mechanism for exercise-induced oxidative stress and redox-mediated pathophysiological processes.;,citation_author=Christopher Zapp;,citation_author=Agnieszka Obarska-Kosinska;,citation_author=Benedikt Rennekamp;,citation_author=Markus Kurth;,citation_author=David M. Hudson;,citation_author=Davide Mercadante;,citation_author=Uladzimir Barayeu;,citation_author=Tobias P. Dick;,citation_author=Vasyl Denysenkov;,citation_author=Thomas Prisner;,citation_author=Marina Bennati;,citation_author=Csaba Daday;,citation_author=Reinhard Kappl;,citation_author=Frauke Gräter;,citation_publication_date=2020-05;,citation_cover_date=2020-05;,citation_year=2020;,citation_fulltext_html_url=https://www.nature.com/articles/s41467-020-15567-4;,citation_issue=1;,citation_doi=10.1038/s41467-020-15567-4;,citation_issn=2041-1723;,citation_volume=11;,citation_language=en-US;,citation_journal_title=Nature Communications;,citation_publisher=Nature Publishing Group;">
</head>

<body class="quarto-light"><script id="quarto-html-before-body" type="application/javascript">
    const toggleBodyColorMode = (bsSheetEl) => {
      const mode = bsSheetEl.getAttribute("data-mode");
      const bodyEl = window.document.querySelector("body");
      if (mode === "dark") {
        bodyEl.classList.add("quarto-dark");
        bodyEl.classList.remove("quarto-light");
      } else {
        bodyEl.classList.add("quarto-light");
        bodyEl.classList.remove("quarto-dark");
      }
    }
    const toggleBodyColorPrimary = () => {
      const bsSheetEl = window.document.querySelector("link#quarto-bootstrap:not([rel=disabled-stylesheet])");
      if (bsSheetEl) {
        toggleBodyColorMode(bsSheetEl);
      }
    }
    const setColorSchemeToggle = (alternate) => {
      const toggles = window.document.querySelectorAll('.quarto-color-scheme-toggle');
      for (let i=0; i < toggles.length; i++) {
        const toggle = toggles[i];
        if (toggle) {
          if (alternate) {
            toggle.classList.add("alternate");
          } else {
            toggle.classList.remove("alternate");
          }
        }
      }
    };
    const toggleColorMode = (alternate) => {
      // Switch the stylesheets
      const primaryStylesheets = window.document.querySelectorAll('link.quarto-color-scheme:not(.quarto-color-alternate)');
      const alternateStylesheets = window.document.querySelectorAll('link.quarto-color-scheme.quarto-color-alternate');
      manageTransitions('#quarto-margin-sidebar .nav-link', false);
      if (alternate) {
        // note: dark is layered on light, we don't disable primary!
        enableStylesheet(alternateStylesheets);
        for (const sheetNode of alternateStylesheets) {
          if (sheetNode.id === "quarto-bootstrap") {
            toggleBodyColorMode(sheetNode);
          }
        }
      } else {
        disableStylesheet(alternateStylesheets);
        enableStylesheet(primaryStylesheets)
        toggleBodyColorPrimary();
      }
      manageTransitions('#quarto-margin-sidebar .nav-link', true);
      // Switch the toggles
      setColorSchemeToggle(alternate)
      // Hack to workaround the fact that safari doesn't
      // properly recolor the scrollbar when toggling (#1455)
      if (navigator.userAgent.indexOf('Safari') > 0 && navigator.userAgent.indexOf('Chrome') == -1) {
        manageTransitions("body", false);
        window.scrollTo(0, 1);
        setTimeout(() => {
          window.scrollTo(0, 0);
          manageTransitions("body", true);
        }, 40);
      }
    }
    const disableStylesheet = (stylesheets) => {
      for (let i=0; i < stylesheets.length; i++) {
        const stylesheet = stylesheets[i];
        stylesheet.rel = 'disabled-stylesheet';
      }
    }
    const enableStylesheet = (stylesheets) => {
      for (let i=0; i < stylesheets.length; i++) {
        const stylesheet = stylesheets[i];
        if(stylesheet.rel !== 'stylesheet') { // for Chrome, which will still FOUC without this check
          stylesheet.rel = 'stylesheet';
        }
      }
    }
    const manageTransitions = (selector, allowTransitions) => {
      const els = window.document.querySelectorAll(selector);
      for (let i=0; i < els.length; i++) {
        const el = els[i];
        if (allowTransitions) {
          el.classList.remove('notransition');
        } else {
          el.classList.add('notransition');
        }
      }
    }
    const isFileUrl = () => {
      return window.location.protocol === 'file:';
    }
    const hasAlternateSentinel = () => {
      let styleSentinel = getColorSchemeSentinel();
      if (styleSentinel !== null) {
        return styleSentinel === "alternate";
      } else {
        return false;
      }
    }
    const setStyleSentinel = (alternate) => {
      const value = alternate ? "alternate" : "default";
      if (!isFileUrl()) {
        window.localStorage.setItem("quarto-color-scheme", value);
      } else {
        localAlternateSentinel = value;
      }
    }
    const getColorSchemeSentinel = () => {
      if (!isFileUrl()) {
        const storageValue = window.localStorage.getItem("quarto-color-scheme");
        return storageValue != null ? storageValue : localAlternateSentinel;
      } else {
        return localAlternateSentinel;
      }
    }
    const toggleGiscusIfUsed = (isAlternate, darkModeDefault) => {
      const baseTheme = document.querySelector('#giscus-base-theme')?.value ?? 'light';
      const alternateTheme = document.querySelector('#giscus-alt-theme')?.value ?? 'dark';
      let newTheme = '';
      if(authorPrefersDark) {
        newTheme = isAlternate ? baseTheme : alternateTheme;
      } else {
        newTheme = isAlternate ? alternateTheme : baseTheme;
      }
      const changeGiscusTheme = () => {
        // From: https://github.com/giscus/giscus/issues/336
        const sendMessage = (message) => {
          const iframe = document.querySelector('iframe.giscus-frame');
          if (!iframe) return;
          iframe.contentWindow.postMessage({ giscus: message }, 'https://giscus.app');
        }
        sendMessage({
          setConfig: {
            theme: newTheme
          }
        });
      }
      const isGiscussLoaded = window.document.querySelector('iframe.giscus-frame') !== null;
      if (isGiscussLoaded) {
        changeGiscusTheme();
      }
    };
    const authorPrefersDark = false;
    const darkModeDefault = authorPrefersDark;
      document.querySelector('link#quarto-text-highlighting-styles.quarto-color-scheme-extra').rel = 'disabled-stylesheet';
      document.querySelector('link#quarto-bootstrap.quarto-color-scheme-extra').rel = 'disabled-stylesheet';
    let localAlternateSentinel = darkModeDefault ? 'alternate' : 'default';
    // Dark / light mode switch
    window.quartoToggleColorScheme = () => {
      // Read the current dark / light value
      let toAlternate = !hasAlternateSentinel();
      toggleColorMode(toAlternate);
      setStyleSentinel(toAlternate);
      toggleGiscusIfUsed(toAlternate, darkModeDefault);
      window.dispatchEvent(new Event('resize'));
    };
    // Switch to dark mode if need be
    if (hasAlternateSentinel()) {
      toggleColorMode(true);
    } else {
      toggleColorMode(false);
    }
  </script>

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article toc-left">
<div id="quarto-sidebar-toc-left" class="sidebar toc-left">
  <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#preamble" id="toc-preamble" class="nav-link active" data-scroll-target="#preamble">Preamble</a>
  <ul>
  <li><a href="#zusammenfassung" id="toc-zusammenfassung" class="nav-link" data-scroll-target="#zusammenfassung">Zusammenfassung</a></li>
  <li><a href="#abstract" id="toc-abstract" class="nav-link" data-scroll-target="#abstract">Abstract</a></li>
  <li><a href="#acknowledgments" id="toc-acknowledgments" class="nav-link" data-scroll-target="#acknowledgments">Acknowledgments</a></li>
  <li><a href="#ai-statement" id="toc-ai-statement" class="nav-link" data-scroll-target="#ai-statement">AI statement</a></li>
  </ul></li>
  <li><a href="#sec-intro" id="toc-sec-intro" class="nav-link" data-scroll-target="#sec-intro"><span class="header-section-number">1</span> Introduction</a>
  <ul>
  <li><a href="#sec-collagen" id="toc-sec-collagen" class="nav-link" data-scroll-target="#sec-collagen"><span class="header-section-number">1.1</span> Concerning collagen</a>
  <ul class="collapse">
  <li><a href="#sec-salisbury" id="toc-sec-salisbury" class="nav-link" data-scroll-target="#sec-salisbury"><span class="header-section-number">1.1.1</span> A journey on the HMS Salisbury</a></li>
  <li><a href="#sec-synth" id="toc-sec-synth" class="nav-link" data-scroll-target="#sec-synth"><span class="header-section-number">1.1.2</span> The biosynthesis of collagen</a></li>
  </ul></li>
  <li><a href="#sec-intro-mech" id="toc-sec-intro-mech" class="nav-link" data-scroll-target="#sec-intro-mech"><span class="header-section-number">1.2</span> A rather radical discovery: mechanoradicals in collagen</a></li>
  <li><a href="#aims-of-this-thesis" id="toc-aims-of-this-thesis" class="nav-link" data-scroll-target="#aims-of-this-thesis"><span class="header-section-number">1.3</span> Aims of this thesis</a></li>
  </ul></li>
  <li><a href="#sec-methods" id="toc-sec-methods" class="nav-link" data-scroll-target="#sec-methods"><span class="header-section-number">2</span> Methods and theory</a>
  <ul>
  <li><a href="#molecular-dynamics" id="toc-molecular-dynamics" class="nav-link" data-scroll-target="#molecular-dynamics"><span class="header-section-number">2.1</span> Molecular dynamics</a></li>
  <li><a href="#molecular-mechanics" id="toc-molecular-mechanics" class="nav-link" data-scroll-target="#molecular-mechanics"><span class="header-section-number">2.2</span> Molecular mechanics</a>
  <ul class="collapse">
  <li><a href="#energy-terms" id="toc-energy-terms" class="nav-link" data-scroll-target="#energy-terms"><span class="header-section-number">2.2.1</span> Energy terms</a></li>
  <li><a href="#force-fields-and-topologies" id="toc-force-fields-and-topologies" class="nav-link" data-scroll-target="#force-fields-and-topologies"><span class="header-section-number">2.2.2</span> Force fields and topologies</a></li>
  </ul></li>
  <li><a href="#periodic-boundary-conditions" id="toc-periodic-boundary-conditions" class="nav-link" data-scroll-target="#periodic-boundary-conditions"><span class="header-section-number">2.3</span> Periodic boundary conditions</a></li>
  <li><a href="#constraint-algorithms" id="toc-constraint-algorithms" class="nav-link" data-scroll-target="#constraint-algorithms"><span class="header-section-number">2.4</span> Constraint algorithms</a></li>
  <li><a href="#initial-conditions-and-ensembles" id="toc-initial-conditions-and-ensembles" class="nav-link" data-scroll-target="#initial-conditions-and-ensembles"><span class="header-section-number">2.5</span> Initial conditions and ensembles</a></li>
  <li><a href="#thermostats-and-barostats" id="toc-thermostats-and-barostats" class="nav-link" data-scroll-target="#thermostats-and-barostats"><span class="header-section-number">2.6</span> Thermostats and barostats</a></li>
  <li><a href="#long-range-interactions" id="toc-long-range-interactions" class="nav-link" data-scroll-target="#long-range-interactions"><span class="header-section-number">2.7</span> Long-range interactions</a></li>
  <li><a href="#external-biasing-potentials" id="toc-external-biasing-potentials" class="nav-link" data-scroll-target="#external-biasing-potentials"><span class="header-section-number">2.8</span> External biasing potentials</a></li>
  <li><a href="#sec-meth-us" id="toc-sec-meth-us" class="nav-link" data-scroll-target="#sec-meth-us"><span class="header-section-number">2.9</span> Umbrella sampling and the weighted histogram analysis method</a></li>
  <li><a href="#free-energy-perturbation-by-slow-growth" id="toc-free-energy-perturbation-by-slow-growth" class="nav-link" data-scroll-target="#free-energy-perturbation-by-slow-growth"><span class="header-section-number">2.10</span> Free energy perturbation by slow-growth</a></li>
  <li><a href="#quantum-mechanics-and-density-functional-theory" id="toc-quantum-mechanics-and-density-functional-theory" class="nav-link" data-scroll-target="#quantum-mechanics-and-density-functional-theory"><span class="header-section-number">2.11</span> Quantum mechanics and density functional theory</a>
  <ul class="collapse">
  <li><a href="#the-schrödinger-equation" id="toc-the-schrödinger-equation" class="nav-link" data-scroll-target="#the-schrödinger-equation"><span class="header-section-number">2.11.1</span> The Schrödinger equation</a></li>
  <li><a href="#the-electronic-schrödinger-equation" id="toc-the-electronic-schrödinger-equation" class="nav-link" data-scroll-target="#the-electronic-schrödinger-equation"><span class="header-section-number">2.11.2</span> The electronic Schrödinger equation</a></li>
  <li><a href="#hartree-product" id="toc-hartree-product" class="nav-link" data-scroll-target="#hartree-product"><span class="header-section-number">2.11.3</span> Hartree product</a></li>
  <li><a href="#slater-determinant" id="toc-slater-determinant" class="nav-link" data-scroll-target="#slater-determinant"><span class="header-section-number">2.11.4</span> Slater determinant</a></li>
  <li><a href="#the-hartree-fock-method" id="toc-the-hartree-fock-method" class="nav-link" data-scroll-target="#the-hartree-fock-method"><span class="header-section-number">2.11.5</span> The Hartree-Fock method</a></li>
  <li><a href="#hohenberg-kohn-theorems" id="toc-hohenberg-kohn-theorems" class="nav-link" data-scroll-target="#hohenberg-kohn-theorems"><span class="header-section-number">2.11.6</span> Hohenberg-Kohn theorems</a></li>
  <li><a href="#kohn-sham-density-functional-theory" id="toc-kohn-sham-density-functional-theory" class="nav-link" data-scroll-target="#kohn-sham-density-functional-theory"><span class="header-section-number">2.11.7</span> Kohn-Sham density functional theory</a></li>
  <li><a href="#basis-sets" id="toc-basis-sets" class="nav-link" data-scroll-target="#basis-sets"><span class="header-section-number">2.11.8</span> Basis sets</a></li>
  </ul></li>
  <li><a href="#sec-meth-qmmm" id="toc-sec-meth-qmmm" class="nav-link" data-scroll-target="#sec-meth-qmmm"><span class="header-section-number">2.12</span> Quantum mechanics/molecular mechanics</a></li>
  <li><a href="#computational-molecular-dynamics-engines" id="toc-computational-molecular-dynamics-engines" class="nav-link" data-scroll-target="#computational-molecular-dynamics-engines"><span class="header-section-number">2.13</span> Computational molecular dynamics engines</a></li>
  <li><a href="#sec-kmc" id="toc-sec-kmc" class="nav-link" data-scroll-target="#sec-kmc"><span class="header-section-number">2.14</span> Kinetic Monte Carlo</a></li>
  <li><a href="#sec-arrhenius" id="toc-sec-arrhenius" class="nav-link" data-scroll-target="#sec-arrhenius"><span class="header-section-number">2.15</span> Arrhenius equation and transition state theory</a></li>
  </ul></li>
  <li><a href="#sec-main-qm" id="toc-sec-main-qm" class="nav-link" data-scroll-target="#sec-main-qm"><span class="header-section-number">3</span> Collagen hydrolysis by quantum mechanics/molecular mechanics</a>
  <ul>
  <li><a href="#previous-work-on-base-catalyzed-hydrolysis" id="toc-previous-work-on-base-catalyzed-hydrolysis" class="nav-link" data-scroll-target="#previous-work-on-base-catalyzed-hydrolysis"><span class="header-section-number">3.1</span> Previous work on base-catalyzed hydrolysis</a></li>
  <li><a href="#goals-of-the-calculations" id="toc-goals-of-the-calculations" class="nav-link" data-scroll-target="#goals-of-the-calculations"><span class="header-section-number">3.2</span> Goals of the calculations</a></li>
  <li><a href="#system-setup-and-simulation-pipeline" id="toc-system-setup-and-simulation-pipeline" class="nav-link" data-scroll-target="#system-setup-and-simulation-pipeline"><span class="header-section-number">3.3</span> System setup and simulation pipeline</a>
  <ul class="collapse">
  <li><a href="#initial-structure-preparation" id="toc-initial-structure-preparation" class="nav-link" data-scroll-target="#initial-structure-preparation"><span class="header-section-number">3.3.1</span> Initial structure preparation</a></li>
  <li><a href="#equilibration-protocol" id="toc-equilibration-protocol" class="nav-link" data-scroll-target="#equilibration-protocol"><span class="header-section-number">3.3.2</span> Equilibration protocol</a></li>
  <li><a href="#quantum-mechanics-subsystem-selection-and-setup" id="toc-quantum-mechanics-subsystem-selection-and-setup" class="nav-link" data-scroll-target="#quantum-mechanics-subsystem-selection-and-setup"><span class="header-section-number">3.3.3</span> Quantum mechanics subsystem selection and setup</a></li>
  </ul></li>
  <li><a href="#results" id="toc-results" class="nav-link" data-scroll-target="#results"><span class="header-section-number">3.4</span> Results</a>
  <ul class="collapse">
  <li><a href="#simulations-reveal-mechanistic-details-of-the-tetrahedral-intermediate-formation" id="toc-simulations-reveal-mechanistic-details-of-the-tetrahedral-intermediate-formation" class="nav-link" data-scroll-target="#simulations-reveal-mechanistic-details-of-the-tetrahedral-intermediate-formation"><span class="header-section-number">3.4.1</span> Simulations reveal mechanistic details of the tetrahedral intermediate formation</a></li>
  <li><a href="#the-triple-helix-marginally-impedes-hydroxide-attack-thermodynamically" id="toc-the-triple-helix-marginally-impedes-hydroxide-attack-thermodynamically" class="nav-link" data-scroll-target="#the-triple-helix-marginally-impedes-hydroxide-attack-thermodynamically"><span class="header-section-number">3.4.2</span> The triple helix marginally impedes hydroxide attack thermodynamically</a></li>
  <li><a href="#proton-mobility-in-quantum-mechanically-described-solvent-molecules-allows-breaking-of-the-peptide-bond" id="toc-proton-mobility-in-quantum-mechanically-described-solvent-molecules-allows-breaking-of-the-peptide-bond" class="nav-link" data-scroll-target="#proton-mobility-in-quantum-mechanically-described-solvent-molecules-allows-breaking-of-the-peptide-bond"><span class="header-section-number">3.4.3</span> Proton mobility in quantum mechanically described solvent molecules allows breaking of the peptide bond</a></li>
  </ul></li>
  <li><a href="#discussion-and-outlook-of-the-hybrid-simulations" id="toc-discussion-and-outlook-of-the-hybrid-simulations" class="nav-link" data-scroll-target="#discussion-and-outlook-of-the-hybrid-simulations"><span class="header-section-number">3.5</span> Discussion and outlook of the hybrid simulations</a></li>
  </ul></li>
  <li><a href="#sec-main-kimmdy" id="toc-sec-main-kimmdy" class="nav-link" data-scroll-target="#sec-main-kimmdy"><span class="header-section-number">4</span> Kinetic Monte Carlo Molecular Dynamics: KIMMDY</a>
  <ul>
  <li><a href="#running-kimmdy-implementation-and-usage" id="toc-running-kimmdy-implementation-and-usage" class="nav-link" data-scroll-target="#running-kimmdy-implementation-and-usage"><span class="header-section-number">4.1</span> Running KIMMDY: implementation and usage</a>
  <ul class="collapse">
  <li><a href="#installation" id="toc-installation" class="nav-link" data-scroll-target="#installation"><span class="header-section-number">4.1.1</span> Installation</a></li>
  <li><a href="#configuration-and-inputoutput" id="toc-configuration-and-inputoutput" class="nav-link" data-scroll-target="#configuration-and-inputoutput"><span class="header-section-number">4.1.2</span> Configuration and input/output</a></li>
  <li><a href="#ensemble-generation" id="toc-ensemble-generation" class="nav-link" data-scroll-target="#ensemble-generation"><span class="header-section-number">4.1.3</span> Ensemble generation</a></li>
  <li><a href="#reactions-and-recipes" id="toc-reactions-and-recipes" class="nav-link" data-scroll-target="#reactions-and-recipes"><span class="header-section-number">4.1.4</span> Reactions and recipes</a></li>
  <li><a href="#kinetic-monte-carlo-algorithms" id="toc-kinetic-monte-carlo-algorithms" class="nav-link" data-scroll-target="#kinetic-monte-carlo-algorithms"><span class="header-section-number">4.1.5</span> Kinetic Monte Carlo algorithms</a></li>
  <li><a href="#sec-top" id="toc-sec-top" class="nav-link" data-scroll-target="#sec-top"><span class="header-section-number">4.1.6</span> Topology module</a></li>
  <li><a href="#sec-grappa" id="toc-sec-grappa" class="nav-link" data-scroll-target="#sec-grappa"><span class="header-section-number">4.1.7</span> Parametrization interface</a></li>
  <li><a href="#slow-growth" id="toc-slow-growth" class="nav-link" data-scroll-target="#slow-growth"><span class="header-section-number">4.1.8</span> Slow-growth</a></li>
  <li><a href="#checkpoints-and-restarting" id="toc-checkpoints-and-restarting" class="nav-link" data-scroll-target="#checkpoints-and-restarting"><span class="header-section-number">4.1.9</span> Checkpoints and restarting</a></li>
  <li><a href="#high-performance-computing-interface" id="toc-high-performance-computing-interface" class="nav-link" data-scroll-target="#high-performance-computing-interface"><span class="header-section-number">4.1.10</span> High performance computing interface</a></li>
  <li><a href="#analysis" id="toc-analysis" class="nav-link" data-scroll-target="#analysis"><span class="header-section-number">4.1.11</span> Analysis</a></li>
  <li><a href="#code-quality-and-testing" id="toc-code-quality-and-testing" class="nav-link" data-scroll-target="#code-quality-and-testing"><span class="header-section-number">4.1.12</span> Code quality and testing</a></li>
  </ul></li>
  <li><a href="#applying-kimmdy-to-collagen-hydrolysis-and-homolysis" id="toc-applying-kimmdy-to-collagen-hydrolysis-and-homolysis" class="nav-link" data-scroll-target="#applying-kimmdy-to-collagen-hydrolysis-and-homolysis"><span class="header-section-number">4.2</span> Applying KIMMDY to collagen hydrolysis and homolysis</a>
  <ul class="collapse">
  <li><a href="#sec-hyd-hom-plugins" id="toc-sec-hyd-hom-plugins" class="nav-link" data-scroll-target="#sec-hyd-hom-plugins"><span class="header-section-number">4.2.1</span> The hydrolysis and homolysis plugins</a></li>
  <li><a href="#system-setup" id="toc-system-setup" class="nav-link" data-scroll-target="#system-setup"><span class="header-section-number">4.2.2</span> System setup</a></li>
  </ul></li>
  <li><a href="#results-1" id="toc-results-1" class="nav-link" data-scroll-target="#results-1"><span class="header-section-number">4.3</span> Results</a>
  <ul class="collapse">
  <li><a href="#physiologically-expected-ph-changes-have-only-a-minor-contribution-to-hydrolysis-rates" id="toc-physiologically-expected-ph-changes-have-only-a-minor-contribution-to-hydrolysis-rates" class="nav-link" data-scroll-target="#physiologically-expected-ph-changes-have-only-a-minor-contribution-to-hydrolysis-rates"><span class="header-section-number">4.3.1</span> Physiologically expected pH changes have only a minor contribution to hydrolysis rates</a></li>
  <li><a href="#sec-sasa" id="toc-sec-sasa" class="nav-link" data-scroll-target="#sec-sasa"><span class="header-section-number">4.3.2</span> Solvent accessibility of peptide bonds in the fibril is diverse</a></li>
  <li><a href="#sec-prolines" id="toc-sec-prolines" class="nav-link" data-scroll-target="#sec-prolines"><span class="header-section-number">4.3.3</span> Prolines in collagen may offer protection against hydrolysis</a></li>
  <li><a href="#sec-kmdy-rates" id="toc-sec-kmdy-rates" class="nav-link" data-scroll-target="#sec-kmdy-rates"><span class="header-section-number">4.3.4</span> Homolysis becomes competitive with hydrolysis by force concentration</a></li>
  <li><a href="#discussion-and-outlook-of-applying-kimmdy-to-hydrolysis-and-homolysis" id="toc-discussion-and-outlook-of-applying-kimmdy-to-hydrolysis-and-homolysis" class="nav-link" data-scroll-target="#discussion-and-outlook-of-applying-kimmdy-to-hydrolysis-and-homolysis"><span class="header-section-number">4.3.5</span> Discussion and outlook of applying KIMMDY to hydrolysis and homolysis</a></li>
  </ul></li>
  <li><a href="#beyond-hydrolysis-kimmdy-can-be-applied-to-a-variety-of-systems-and-reactions" id="toc-beyond-hydrolysis-kimmdy-can-be-applied-to-a-variety-of-systems-and-reactions" class="nav-link" data-scroll-target="#beyond-hydrolysis-kimmdy-can-be-applied-to-a-variety-of-systems-and-reactions"><span class="header-section-number">4.4</span> Beyond hydrolysis: KIMMDY can be applied to a variety of systems and reactions</a>
  <ul class="collapse">
  <li><a href="#kimmdy-finds-a-new-radical-scavenging-candidate-in-collagen" id="toc-kimmdy-finds-a-new-radical-scavenging-candidate-in-collagen" class="nav-link" data-scroll-target="#kimmdy-finds-a-new-radical-scavenging-candidate-in-collagen"><span class="header-section-number">4.4.1</span> KIMMDY finds a new radical scavenging candidate in collagen</a></li>
  </ul></li>
  </ul></li>
  <li><a href="#concluding-thoughts-and-next-steps" id="toc-concluding-thoughts-and-next-steps" class="nav-link" data-scroll-target="#concluding-thoughts-and-next-steps"><span class="header-section-number">5</span> Concluding thoughts and next steps</a></li>
  <li><a href="#appendix" id="toc-appendix" class="nav-link" data-scroll-target="#appendix">Appendix</a>
  <ul>
  <li><a href="#extended-data-figures" id="toc-extended-data-figures" class="nav-link" data-scroll-target="#extended-data-figures">Extended data figures</a></li>
  <li><a href="#extended-data-listings" id="toc-extended-data-listings" class="nav-link" data-scroll-target="#extended-data-listings">Extended data listings</a></li>
  <li><a href="#additional-software-and-resources" id="toc-additional-software-and-resources" class="nav-link" data-scroll-target="#additional-software-and-resources">Additional software and resources</a>
  <ul class="collapse">
  <li><a href="#knowledge-resources" id="toc-knowledge-resources" class="nav-link" data-scroll-target="#knowledge-resources">Knowledge resources</a></li>
  <li><a href="#software" id="toc-software" class="nav-link" data-scroll-target="#software">Software</a></li>
  </ul></li>
  <li><a href="#list-of-abbreviations-and-constants" id="toc-list-of-abbreviations-and-constants" class="nav-link" data-scroll-target="#list-of-abbreviations-and-constants">List of abbreviations and constants</a></li>
  </ul></li>
  <li><a href="#references" id="toc-references" class="nav-link" data-scroll-target="#references">References</a></li>
  </ul>
<div class="quarto-alternate-formats"><h2>Other Formats</h2><ul><li><a href="index.pdf"><i class="bi bi-file-pdf"></i>PDF</a></li><li><a href="presentation.html"><i class="bi bi-easel"></i>Presentation</a></li></ul></div></nav>
</div>
<div id="quarto-margin-sidebar" class="sidebar margin-sidebar zindex-bottom">
</div>
<main class="content page-columns page-full" id="quarto-document-content">


<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title"><p>Dissecting cleavage mechanisms<br>
in tensed collagen fibrils<br>
using reactive molecular dynamics simulations</p></h1>
</div>


<div class="quarto-title-meta-author">
  <div class="quarto-title-meta-heading">Author</div>
  <div class="quarto-title-meta-heading">Affiliations</div>
  
    <div class="quarto-title-meta-contents">
    <p class="author">Jannik Buhr <a href="https://orcid.org/0000-0002-9350-2629" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
  </div>
  <div class="quarto-title-meta-contents">
        <p class="affiliation">
            <a href="https://www.uni-heidelberg.de/en">
            Heidelberg University
            </a>
          </p>
        <p class="affiliation">
            <a href="https://www.iwr.uni-heidelberg.de/en">
            Interdisciplinary Center for Scientific Computing (IWR)
            </a>
          </p>
        <p class="affiliation">
            <a href="https://h-its.org/">
            Heidelberg Institute for Theoretical Studies (HITS)
            </a>
          </p>
        <p class="affiliation">
            <a href="https://www.mpip-mainz.mpg.de">
            Max-Planck-Institute for Polymer Studies (MPI-P)
            </a>
          </p>
      </div>
  </div>

<div class="quarto-title-meta">

      
    <div>
    <div class="quarto-title-meta-heading">Date of the defense</div>
    <div class="quarto-title-meta-contents">
      <p class="date">February 12, 2026</p>
    </div>
  </div>
  
    
  </div>
  

<div>
  <div class="keywords">
    <div class="block-title">Keywords</div>
    <p>Collagen, Mechanochemistry, Hydrolysis, Quantum Mechanics / Molecular Mechanics, Molecular Dynamics, Research Software Engineering</p>
  </div>
</div>

</header>


<section id="preamble" class="level1 unnumbered">
<h1 class="unnumbered">Preamble</h1>
<div class="callout callout-style-default callout-tip callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Tip
</div>
</div>
<div class="callout-body-container callout-body">
<p>You are reading the web version of this thesis. In the left sidebar, you also find the <a href="./index.pdf">print version</a> and the <a href="./presentation.html">presentation</a> for the defense in the sidebar or via these links.</p>
</div>
</div>
<div class="callout callout-style-default callout-tip callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Tip
</div>
</div>
<div class="callout-body-container callout-body">
<p>If you are reading this notice, this thesis is not yet defended.</p>
</div>
</div>
<div style="page-break-after: always;"></div>
<section id="zusammenfassung" class="level2 unnumbered">
<h2 class="unnumbered anchored" data-anchor-id="zusammenfassung">Zusammenfassung</h2>
<details>
<summary>German summary</summary>
<p>Kollagen ist unser wichtigstes strukturelles Protein. Als Hauptkomponente von Sehnen, Bändern und anderem Bindegewebe muss es extremen Kräften standhalten und weist eine interessante Chemie unter externer Last auf. Frühere Studien fanden Mechanoradikale in gespannten Kollagenfasern mittels Elektronenspinresonanz. Wie Homolyse, die Rektion, die eben diese Radikale produziert, mit anderen, nicht-radikalischen und im wässrigen Medium erwartbaren Reaktionen in Konkurrenz steht, war noch unklar. Vor kurzem wurde gezeigt, dass basenkatalysierte Hydrolyse durch selbst kleine Kräfte beschleunigt wird. In dieser Arbeit bringe ich diese augenscheinlich gegenteiligen Beobachtungen mit zwei ergänzenden, computerbasierten Methoden in Einklang. Hybride quantenmechanische/molekularmechanische Simulationen enthüllen den komplexen Mechanismus der Hydrolysereaktion und zeigen, wie die chemische Umgebung des Lösungsmittels essentiell ist und dass die einzigartige Struktur von Kollagen die Reaktionsbarriere der basenkatalysierten Hydrolyse erhöht. Dann präsentiere ich eine neue Methode, die in Zusammenarbeit mit zwei Kollegen entstanden ist: KIMMDY (Kinetic Monte Carlo Molecular Dynamics) ermöglicht Reaktionen in molekulardynamischen Simulationen auf eine Weise, die Zeitskalen überspannt und ist ein flexibles Framework, das einfach um neue Chemie erweiterbar ist. Mittels KIMMDY zeige ich, wie Kraftkonzentration in komplexen, quervernetzten Kollagenfibrillen Kräfte in Regime verschieben kann, in denen Homolyseraten die der Hydrolyse übertreffen.</p>
</details>
<div style="page-break-after: always;"></div>
</section>
<section id="abstract" class="level2 unnumbered">
<h2 class="unnumbered anchored" data-anchor-id="abstract">Abstract</h2>
<p>Collagen is our most important structural protein. As the chief component of tendons, ligaments, and other connective tissue, it experiences extreme forces and exhibits interesting chemistry under external stress. Earlier investigations found mechanoradicals in tensed collagen fibers through the use of electron paramagnetic resonance. However, it was still unclear how homolytic cleavage, the reaction that produces these radicals, competes with other, non-radical reactions that would be more expected in an aqueous medium such as the body. Base-catalyzed hydrolysis was recently shown to be greatly accelerated even by small forces, which would favor it in the competition. In this thesis, I resolve these seemingly contradictory findings with two complementary computational methods. Hybrid quantum mechanics/molecular mechanics simulations unveil the complexity of the hydrolysis reaction and show how the chemical environment of the solvent is crucial and that the unique collagen structure increases the reaction barrier of base-catalyzed hydrolysis. Then, I showcase a new method I developed in collaboration with two colleagues to tackle the comparison at scale: KIMMDY (Kinetic Monte Carlo Molecular Dynamics) allows simulating reactions in molecular dynamics simulations in a way that bridges time scales and is a flexible framework that can easily be extended to novel chemistries. With KIMMDY, I reveal how force concentration in complex cross-linked collagen fibrils can push forces to regimes where homolysis outcompetes hydrolysis.</p>
<div style="page-break-after: always;"></div>
</section>
<section id="acknowledgments" class="level2 unnumbered">
<h2 class="unnumbered anchored" data-anchor-id="acknowledgments">Acknowledgments</h2>
<p>Firstly, I want to thank Frauke and the past and present members of our research group that were part of my journey. I couldn’t have chosen a better environment for my PhD and am grateful for the perfect mix of advice, support, and freedom for my research. Special thanks go to my colleagues Eric Hartmann and Kai Riedmiller for being a great team when developing KIMMDY and many insightful scientific discussions.</p>
<p>I would like to thank the additional members of my Thesis Advisory Committee, Ada Cavalcanti-Adam and Dirk-Peter Herten, for helping to keep me on track, and especially Fabio Lolicato for jumping in as a last-minute member. I would like to thank the members of my defense committee, Eva Blasco, Robert Russell, and Ursula Kummer, for giving their time and expertise to evaluate my thesis, and especially Rob for agreeing to be my second referee. I am grateful to the Heidelberg Institute for Theoretical Studies (HITS) and the Max-Planck-Institute for Polymer Research (MPI-P) for providing the resources and environment to conduct my research. I would like to thank Dmitry Morozov and Gerrit Groenhof for hosting me in Jyväskylä and teaching me the ins and outs of QM/MM simulations in GROMACS.</p>
<p>Finally, I wouldn’t be close to where I am today without the support of my family and friends, and the greatest thanks goes to them. To my friends, who provide the much-needed work-life balance with board games (both on- and offline), bouldering, or acrobatics. To my parents and my brother, who put me into the position where I never even had to question whether something is possible, because I know we would always have each other’s backs. <em>“Das macht man so in einer Herde.”</em> And to my partner Franzi for accepting me into her pack and making me so inconceivably happy by just existing.</p>
<div style="page-break-after: always;"></div>
</section>
<section id="ai-statement" class="level2 unnumbered">
<h2 class="unnumbered anchored" data-anchor-id="ai-statement">AI statement</h2>
<p>While there exist many valid use cases for machine learning (ML) and large language models (LLMs) in particular, creating genuine human communication and, in doing so, respecting you, the reader, by crafting a text that is worth reading, are not among them. This is why, even though machine learning techniques are featured in my research and even play a central role in the software that came out of it, the text of this thesis was not generated by an LLM. The only exceptions to this are the list of abbreviations in the appendix, which was completed by a script written by an LLM, as well as spell-checking. So, dear reader, I thank you for your time in advance and sincerely hope you will not regret investing it.</p>
<div style="page-break-after: always;"></div>
</section>
</section>
<section id="sec-intro" class="level1 page-columns page-full" data-number="1">
<h1 data-number="1"><span class="header-section-number">1</span> Introduction</h1>
<section id="sec-collagen" class="level2 page-columns page-full" data-number="1.1">
<h2 data-number="1.1" class="anchored" data-anchor-id="sec-collagen"><span class="header-section-number">1.1</span> Concerning collagen</h2>
<p>I would like to open with a small historical story.</p>

<div class="no-row-height column-margin column-container"><div id="fig-salisbury" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="The Capture of Chandernagore.">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-salisbury-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="img/salisbury.png" class="lightbox" data-gallery="quarto-lightbox-gallery-1" title="Figure&nbsp;1.1: The Capture of Chandernagore, March 1757, by Dominic Serres the Elder in 1771, oil on canvas. The HMS Salisbury can be seen on the left."><img src="img/salisbury.png" class="img-fluid figure-img" data-fig-scap="The Capture of Chandernagore."></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-salisbury-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;1.1: The Capture of Chandernagore, March 1757, by Dominic Serres the Elder in 1771, oil on canvas. The HMS Salisbury can be seen on the left.
</figcaption>
</figure>
</div></div><section id="sec-salisbury" class="level3 page-columns page-full" data-number="1.1.1">
<h3 data-number="1.1.1" class="anchored" data-anchor-id="sec-salisbury"><span class="header-section-number">1.1.1</span> A journey on the HMS Salisbury</h3>
<p>It is the 20th of May, 1747. You are a sailor in the British Royal Navy. As you stand on the deck of the HMS Salisbury (<a href="#fig-salisbury" class="quarto-xref">Figure&nbsp;1.1</a>), you overlook the English Channel you have been patrolling for months now. The creaking of the rigging on the three wooden masts mixes with the moaning of your crewmates who have fallen sick with that all too common ailment: scurvy. At first they had felt lethargic, short of breath and couldn’t keep up with the hustle and bustle aboard the vessel. They felt a pain in their bones, bruised easily, and the usual smaller injuries sustained from working a physically demanding job just would not heal. As you bring fresh water down into their cabin, your crewmates muster a faint smile, exposing their reddened gums. Some are already losing teeth and you know they are not long for this world. They would eventually die from internal bleeding. This was not unusual. During this time it was common to assume that up to 50% of sailors would die from scurvy during a major voyage <span class="citation" data-cites="priceAgeScurvy2017">[<a href="#ref-priceAgeScurvy2017" role="doc-biblioref">1</a>]</span>.</p>
<p>But something was different that day. You observe your ship surgeon James Lind take 12 of the scurvy-ridden sailors and divide them into groups of two. He then administers each group a different remedy that had been suggested as a cure. The first got cider, the second vitriol, the third vinegar. The fourth got citrus fruits such as oranges and lemons, and again the next just a pint of sea water, while the last group got a paste made of balsam of Peru, garlic, myrrh, mustard seed, and radish root <span class="citation" data-cites="lindTreatiseScurvyThree1772">[<a href="#ref-lindTreatiseScurvyThree1772" role="doc-biblioref">2</a>]</span>. As the HMS Salisbury makes its way back to Plymouth at the end of May, the citrus fruits have run out, but the two sailors that had been assigned them as a cure have already recovered.</p>

<div class="no-row-height column-margin column-container"><div id="fig-remove-scurvy" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="Remove scurvy.">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-remove-scurvy-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="img/orange.png" class="lightbox" data-gallery="quarto-lightbox-gallery-2" title="Figure&nbsp;1.2: The ability icon for “REMOVE SCURVY”, ©Riot&nbsp;Games."><img src="img/orange.png" class="img-fluid figure-img" style="width:30.0%" data-fig-scap="Remove scurvy."></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-remove-scurvy-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;1.2: The ability icon for “REMOVE SCURVY”, ©Riot&nbsp;Games.
</figcaption>
</figure>
</div></div><p>James Lind was neither the first to use citrus fruits as a cure nor the first to conduct such an experiment <span class="citation" data-cites="baronSailorsScurvyJames2009">[<a href="#ref-baronSailorsScurvyJames2009" role="doc-biblioref">3</a>]</span>; however, he is widely regarded as the first to include control groups. This arguably makes it the first clinical trial in history. I leave the details for historians to debate.</p>

<div class="no-row-height column-margin column-container"><div id="fig-rifbound" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-rifbound-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./img/riftbound-buccaneer.png" class="lightbox" data-gallery="quarto-lightbox-gallery-3" title="Figure&nbsp;1.3: The Brazen Buccaneer card from Riftbound, the League of Legends trading card game, ©Riot&nbsp;Games."><img src="./img/riftbound-buccaneer.png" class="img-fluid figure-img" style="width:80.0%"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-rifbound-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;1.3: The Brazen Buccaneer card from Riftbound, the League of Legends trading card game, ©Riot&nbsp;Games.
</figcaption>
</figure>
</div></div><p>At that time, nobody knew the true cause of scurvy and even Lind himself had various incorrect theories as to the cause and why specifically citrus fruits helped. But they did help, and their ability to treat and prevent scurvy has even made it into popular culture to the extent that there are video game characters that include citrus fruits as part of their appearance to signify them being seafaring. Some of those even include a reference to the disease as well. The League of Legends champion “Gangplank”, a pirate character, comes with an ability to “remove scurvy”, whereby he eats an orange to be healed of negative status effects (<a href="#fig-remove-scurvy" class="quarto-xref">Figure&nbsp;1.2</a>) and the “Brazen Buccaneer” (<a href="#fig-rifbound" class="quarto-xref">Figure&nbsp;1.3</a>) in the card game “Riftbound”, also by Riot Games, is seen in his artwork handing out oranges.</p>
<p>As we now know, scurvy is a result of vitamin C deficiency, which in turn leads to issues with the biosynthesis of collagen. This thesis is not about scurvy directly, but it is about collagen.</p>
<p>Collagen is the most abundant protein in mammals (going by mass, that is; serum albumin fans may disagree on the counting method) <span class="citation" data-cites="verzarAgingCollagenFiber1964">[<a href="#ref-verzarAgingCollagenFiber1964" role="doc-biblioref">4</a>]</span>. As our primary structural protein, it makes up connective tissue, cartilage, skin, and tendons. Fascinatingly, it is both a component in structures that need stability, such as bones, as well as in those that need flexibility, such as blood vessels. The prevalence and necessity of properly formed collagen explains many of the symptoms of scurvy and also why I chose this obscure 18th-century ailment as a small introductory story from history. But to understand the relationship at a molecular level, we need to look into how collagen is made.</p>
</section>
<section id="sec-synth" class="level3 page-columns page-full" data-number="1.1.2">
<h3 data-number="1.1.2" class="anchored" data-anchor-id="sec-synth"><span class="header-section-number">1.1.2</span> The biosynthesis of collagen</h3>

<div class="no-row-height column-margin column-container"><div id="fig-glycine" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-glycine-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="img/glycine.svg" class="lightbox" data-gallery="quarto-lightbox-gallery-4" title="Figure&nbsp;1.4: Glycine."><img src="img/glycine.svg" class="img-fluid figure-img" style="width:50.0%"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-glycine-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;1.4: Glycine.
</figcaption>
</figure>
</div><div id="fig-proline" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-proline-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="img/proline.svg" class="lightbox" data-gallery="quarto-lightbox-gallery-5" title="Figure&nbsp;1.5: Proline."><img src="img/proline.svg" class="img-fluid figure-img" style="width:50.0%"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-proline-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;1.5: Proline.
</figcaption>
</figure>
</div><div id="fig-hydroxyproline" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-hydroxyproline-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="img/hydroxyproline.svg" class="lightbox" data-gallery="quarto-lightbox-gallery-6" title="Figure&nbsp;1.6: Hydroxyproline."><img src="img/hydroxyproline.svg" class="img-fluid figure-img" style="width:50.0%"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-hydroxyproline-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;1.6: Hydroxyproline.
</figcaption>
</figure>
</div></div>

<p>The biosynthesis of collagen starts with the pre-pro-peptide<a href="#fn1" class="footnote-ref" id="fnref1" role="doc-noteref"><sup>1</sup></a> at ribosomes in the cytosol. Collagen is characterized by the rather unusual motif Glycine-X-Y (Gly-X-Y), where X and Y are frequently proline (Pro) or 4-hydroxyproline (Hyp) (see <a href="#fig-glycine" class="quarto-xref">Figure&nbsp;1.4</a>, <a href="#fig-proline" class="quarto-xref">Figure&nbsp;1.5</a>, <a href="#fig-hydroxyproline" class="quarto-xref">Figure&nbsp;1.6</a>) and have a slightly higher propensity for alanine (Ala) <span class="citation" data-cites="ricard-blumCollagenFamily2011">[<a href="#ref-ricard-blumCollagenFamily2011" role="doc-biblioref">5</a>]</span>. At this stage though, what will be hydroxyproline is still proline. The pre-pro-collagen peptide makes its way to the endoplasmic reticulum (ER) through an N-terminal signal sequence, which gets cleaved off at its destination. We now have the collagen pro-peptide (pro-collagen).</p>

<div class="no-row-height column-margin column-container"><div id="fig-lysine" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-lysine-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="img/lysine.svg" class="lightbox" data-gallery="quarto-lightbox-gallery-7" title="Figure&nbsp;1.7: Lysine."><img src="img/lysine.svg" class="img-fluid figure-img" style="width:50.0%"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-lysine-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;1.7: Lysine.
</figcaption>
</figure>
</div><div id="fig-hydroxylysine" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-hydroxylysine-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="img/hydroxylysine.svg" class="lightbox" data-gallery="quarto-lightbox-gallery-8" title="Figure&nbsp;1.8: Hydroxylysine."><img src="img/hydroxylysine.svg" class="img-fluid figure-img" style="width:50.0%"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-hydroxylysine-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;1.8: Hydroxylysine.
</figcaption>
</figure>
</div></div>
<p>At the ER, two enzymes, prolyl and lysyl hydroxylase, do their work and turn some of the lysines (<a href="#fig-lysine" class="quarto-xref">Figure&nbsp;1.7</a>) and prolines into hydroxylysines (<a href="#fig-hydroxylysine" class="quarto-xref">Figure&nbsp;1.8</a>) and hydroxyprolines, respectively. Both of these enzymes need vitamin C as a cofactor, so remember to eat your oranges. Pro-collagen consists of a triple helix made up of the individual processed pro-peptides, where two strands share the same composition (alpha 1) and a third has a different composition (alpha 2). Pro-collagen is processed once more in the Golgi apparatus with the addition of oligosaccharides until finally being secreted to the outside of the cell.</p>
<p>Due to the high content of glycine (<a href="#fig-glycine" class="quarto-xref">Figure&nbsp;1.4</a>), proline (<a href="#fig-proline" class="quarto-xref">Figure&nbsp;1.5</a>) and hydroxyproline (<a href="#fig-hydroxyproline" class="quarto-xref">Figure&nbsp;1.6</a>), collagen peptides, and by extension also the tropo-collagen triple helices, can be wound very tightly. This is because glycine is the smallest amino acid and proline brings unusual stiffness due to its <span class="math inline">\(\Phi\)</span> dihedral angle being locked at -65° by the peptide bond nitrogen (<span class="math inline">\(\ce{N}\)</span>) being part of a ring structure. This makes collagen an excellent candidate as a structural protein.</p>
<p>At this point it is worth noting that there are currently 28 known members of the collagen family. All of which contain at least one such triple helix, but what happens after varies. For this work the most important members are of the fibrillar type (type I to VI, excluding IV), where understandably the early numbers also constitute the most common forms. More specifically, we will be looking at collagen type I, which is the chief component of tendons, bones, skin, and blood vessels.</p>
<p>Collagen peptidases (collagenases) on the outer membrane of the cell cleave off the ends, leaving us with tropo-collagen. Many tropo-collagen triple helices assemble into fibrillar structures, in which the triple helices are cross-linked to various degrees depending on type and age. The enzyme lysyl oxidase is responsible for creating many of those cross-links. It turns lysine into allysine (its aldehyde), which can then react in an aldol condensation with the hydroxylysines created earlier by lysyl hydroxylase, netting pyridinoline cross-links (PYD). This is where the connection to scurvy lies. A lack of vitamin C leads to a lack of hydroxylated residues, which leads to a lack of collagen cross-links. Thus, scurvy serves to show how crucial the cross-linked, fibrillar structure of collagen is by painting a grim picture of what happens when it is improperly formed.</p>
<div id="fig-collagen-structure" class="quarto-layout-panel">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-collagen-structure-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="quarto-layout-row">
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-collagen-structure" style="flex-basis: 33.3%;justify-content: flex-start;">
<div id="fig-collagen-chain" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-collagen-chain-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./figures/render-chain-c-tubes-cpk-cropped.png" class="lightbox" data-gallery="fig-collagen-structure" title="Figure&nbsp;1.9&nbsp;(a): "><img src="./figures/render-chain-c-tubes-cpk-cropped.png" id="fig-collagen-chain" class="img-fluid figure-img" data-ref-parent="fig-collagen-structure"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-collagen-chain-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(a)
</figcaption>
</figure>
</div>
</div>
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-collagen-structure" style="flex-basis: 33.3%;justify-content: flex-start;">
<div id="fig-collagen-triple-cpk" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-collagen-triple-cpk-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./figures/render-triple-tubes-cpk-cropped.png" class="lightbox" data-gallery="fig-collagen-structure" title="Figure&nbsp;1.9&nbsp;(b): "><img src="./figures/render-triple-tubes-cpk-cropped.png" id="fig-collagen-triple-cpk" class="img-fluid figure-img" data-ref-parent="fig-collagen-structure"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-collagen-triple-cpk-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(b)
</figcaption>
</figure>
</div>
</div>
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-collagen-structure" style="flex-basis: 33.3%;justify-content: flex-start;">
<div id="fig-collagen-triple-tubes" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-collagen-triple-tubes-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./figures/render-triple-tubes-black-cropped.png" class="lightbox" data-gallery="fig-collagen-structure" title="Figure&nbsp;1.9&nbsp;(c): "><img src="./figures/render-triple-tubes-black-cropped.png" id="fig-collagen-triple-tubes" class="img-fluid figure-img" data-ref-parent="fig-collagen-structure"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-collagen-triple-tubes-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(c)
</figcaption>
</figure>
</div>
</div>
</div>
<div class="quarto-layout-row">
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-collagen-structure" style="flex-basis: 66.7%;justify-content: flex-start;">
<div id="fig-collagen-fibril" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-collagen-fibril-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./figures/render-fibril-black-cropped.png" class="lightbox" data-gallery="fig-collagen-structure" title="Figure&nbsp;1.9&nbsp;(d): "><img src="./figures/render-fibril-black-cropped.png" id="fig-collagen-fibril" class="img-fluid figure-img" data-ref-parent="fig-collagen-structure"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-collagen-fibril-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(d)
</figcaption>
</figure>
</div>
</div>
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-collagen-structure" style="flex-basis: 33.3%;justify-content: flex-start;">
<div id="fig-collagen-crosslinks" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-collagen-crosslinks-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./figures/hyd/crosslinks-closer.png" class="lightbox" data-gallery="fig-collagen-structure" title="Figure&nbsp;1.9&nbsp;(e): "><img src="./figures/hyd/crosslinks-closer.png" id="fig-collagen-crosslinks" class="img-fluid figure-img" data-ref-parent="fig-collagen-structure"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-collagen-crosslinks-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(e)
</figcaption>
</figure>
</div>
</div>
</div>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-collagen-structure-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;1.9: Schematic of the structure of collagen type I. Three collagen alpha peptides (a) form a triple helix (b, c). Many triple helices are packed into a collagen fibril (d), in which the triple helices can be cross-linked (e). Only excerpts of the complete chains and fibril are shown.
</figcaption>
</figure>
</div>
<p>On a structural level, as seen in <a href="#fig-collagen-structure" class="quarto-xref">Figure&nbsp;1.9</a>, we then end up with: Three left-handed helical alpha peptides (pre-pro-collagen, <a href="#fig-collagen-chain" class="quarto-xref">Figure&nbsp;1.9 (a)</a>), wound in a right-handed triple helix 300&nbsp;nm in length and 1.5&nbsp;nm in diameter (pro-collagen, tropo-collagen, <a href="#fig-collagen-triple-cpk" class="quarto-xref">Figure&nbsp;1.9 (b)</a>), which is again supercoiled (right-handed) and interwoven and cross-linked with other supercoils to form a collagen fibril (<a href="#fig-collagen-fibril" class="quarto-xref">Figure&nbsp;1.9 (d)</a>, <a href="#fig-collagen-crosslinks" class="quarto-xref">Figure&nbsp;1.9 (e)</a>). The tropo-collagen molecules in a fibril are not all aligned, but instead exhibit a staggered formation with an offset of around 67&nbsp;nm (also called the D-Band). This creates a microfibrillar assembly where one part, the overlap region, contains five helices and another, the gap region, contains only 4. This propagates up to the larger fibrils and is thus visible in electron microscopy (<a href="#fig-collagen-em" class="quarto-xref">Figure&nbsp;1.10</a>).</p>
<div id="fig-collagen-em" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="SEM of cartilage collagen.">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-collagen-em-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="img/collagen-sem.png" class="lightbox" data-gallery="quarto-lightbox-gallery-14" title="Figure&nbsp;1.10: Scanning Electron Microscopy (SEM) image of adult mouse articular cartilage collagen by Hughes et al.&nbsp;from Figure 5d of @collagenSEM2005."><img src="img/collagen-sem.png" class="img-fluid figure-img" style="width:50.0%" data-fig-scap="SEM of cartilage collagen."></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-collagen-em-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;1.10: Scanning Electron Microscopy (SEM) image of adult mouse articular cartilage collagen by Hughes et al.&nbsp;from Figure 5d of <span class="citation" data-cites="collagenSEM2005">[<a href="#ref-collagenSEM2005" role="doc-biblioref">6</a>]</span>.
</figcaption>
</figure>
</div>
</section>
</section>
<section id="sec-intro-mech" class="level2 page-columns page-full" data-number="1.2">
<h2 data-number="1.2" class="anchored" data-anchor-id="sec-intro-mech"><span class="header-section-number">1.2</span> A rather radical discovery: mechanoradicals in collagen</h2>
<p>As a structural protein, collagen can be under a lot of stress, especially in tendons <span class="citation" data-cites="fangExperimentalEvaluationMultiscale2017">[<a href="#ref-fangExperimentalEvaluationMultiscale2017" role="doc-biblioref">7</a>]</span> (<a href="#fig-tendon" class="quarto-xref">Figure&nbsp;1.11</a>). Take, for example, the semimembranosus muscle, the medial of the three hamstring muscles in the thigh. Its namesake tendon has a cross-sectional area (CSA) of around 100&nbsp;mm<sup>2</sup> <span class="citation" data-cites="sahinisReliabilityDistalHamstring2021">[<a href="#ref-sahinisReliabilityDistalHamstring2021" role="doc-biblioref">8</a>]</span> and has to transfer forces up to 1&nbsp;kN <span class="citation" data-cites="besierKneeMuscleForces2009">[<a href="#ref-besierKneeMuscleForces2009" role="doc-biblioref">9</a>]</span>, which leads to a stress of 10&nbsp;MPa. More demanding or rapid exercises can lead to even higher forces, such as 4&nbsp;kN on the Achilles tendon during vertical jumping <span class="citation" data-cites="fukashiroComparisonDirectlyMeasured1993">[<a href="#ref-fukashiroComparisonDirectlyMeasured1993" role="doc-biblioref">10</a>]</span>. In this case it also has a higher CSA of around 450&nbsp;mm<sup>2</sup> <span class="citation" data-cites="sponbeckAchillesTendonCross2017">[<a href="#ref-sponbeckAchillesTendonCross2017" role="doc-biblioref">11</a>]</span>, so nature is coming prepared, but this still leads to stresses of around 9&nbsp;MPa. Or the patellar tendon, which in countermovement jumps has to withstand peak forces of 4&nbsp;kN <span class="citation" data-cites="finniVivoHumanTriceps2000">[<a href="#ref-finniVivoHumanTriceps2000" role="doc-biblioref">12</a>]</span>, but at a CSA of only around 100&nbsp;mm<sup>2</sup> <span class="citation" data-cites="svenssonMechanicalPropertiesHuman2012">[<a href="#ref-svenssonMechanicalPropertiesHuman2012" role="doc-biblioref">13</a>]</span>, leading to peak stresses of around 40&nbsp;MPa. These stresses can become even higher when you take into account the effective CSA, that is the collagen content of the tendon (the part that actually transduces forces), excluding the water content of around 60% <span class="citation" data-cites="chimichWaterContentAlters1992">[<a href="#ref-chimichWaterContentAlters1992" role="doc-biblioref">14</a>]</span>. One just has to wonder: What do these forces do to the molecular structure of collagen?</p>

<div class="no-row-height column-margin column-container"><div id="fig-tendon" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="Achilles tendon render.">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-tendon-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./img/Magnified_view_of_a_Tendon_cropped.png" class="lightbox" data-gallery="quarto-lightbox-gallery-15" title="Figure&nbsp;1.11: Simplified view of the bones, tendons, and ligaments of the foot. The Achilles tendon can be seen attaching the calf muscles to the heel bone. Image Source: cropped version of “Magnified view of a tendon” by Scientfic Animations, https://scientificanimations.com/wiki-images/, CC BY-SA 4.0."><img src="./img/Magnified_view_of_a_Tendon_cropped.png" class="img-fluid figure-img" data-fig-scap="Achilles tendon render."></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-tendon-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;1.11: Simplified view of the bones, tendons, and ligaments of the foot. The Achilles tendon can be seen attaching the calf muscles to the heel bone. Image Source: cropped version of “Magnified view of a tendon” by Scientfic Animations, <a href="https://scientificanimations.com/wiki-images/" class="uri">https://scientificanimations.com/wiki-images/</a>, CC BY-SA 4.0.
</figcaption>
</figure>
</div></div><p>It has long been known that mechanical forces can lead to homolytic bond scission and thus so-called mechanoradicals in polymers <span class="citation" data-cites="kauzmannViscousFlowLarge1940">[<a href="#ref-kauzmannViscousFlowLarge1940" role="doc-biblioref">15</a>]</span> and the field of mechanochemistry is well established by now <span class="citation" data-cites="beyerMechanochemistryMechanicalActivation2005">[<a href="#ref-beyerMechanochemistryMechanicalActivation2005" role="doc-biblioref">16</a>]</span>. For decades this thinking had only been applied to synthetic polymers. Biopolymers such as proteins seemed off the table until mechanoradicals in tensed collagen were finally measured through the use of electron paramagnetic resonance (EPR) <span class="citation" data-cites="zappMechanoradicalsTensedTendon2020">[<a href="#ref-zappMechanoradicalsTensedTendon2020" role="doc-biblioref">17</a>]</span>. Those radicals can then react further into reactive oxygen species (ROS) <span class="citation" data-cites="fitchMechanochemicalReactionCascade2014">[<a href="#ref-fitchMechanochemicalReactionCascade2014" role="doc-biblioref">18</a>]</span>, which can act as signalling molecules in cells or be detrimental at higher levels.</p>
<p>Overlooking biopolymers as candidates for mechanoradicals seems natural still in hindsight. After all, they tend to be solvated in aqueous medium, where other reactions should be more prevalent. One such reaction is hydrolysis, specifically base-catalyzed hydrolysis of the peptide bond of proteins. It was first assumed that such a reaction would be insensitive to external force due to the rate-limiting step being the attack of the hydroxide <span class="citation" data-cites="xiaBaseCatalyzedPeptideHydrolysis2011">[<a href="#ref-xiaBaseCatalyzedPeptideHydrolysis2011" role="doc-biblioref">19</a>]</span>. This made intuitive sense as an explanation as to why we are able to measure radicals in collagen under force, when a protein could also hydrolyze instead. However, a later study looked deeper into peptide hydrolysis and was able to experimentally show its force-dependence through the use of atomic force microscopy (AFM), supplemented with quantum mechanical (QM) calculations of their own <span class="citation" data-cites="pillMechanicalActivationDrastically2019">[<a href="#ref-pillMechanicalActivationDrastically2019" role="doc-biblioref">20</a>]</span>.</p>
<p>How can we reconcile seeing radicals in tensed collagen with a peptide bond hydrolysis reaction that is also highly force-dependent? How do the two reactions (<a href="#fig-hyd-vs-hom-reaction" class="quarto-xref">Figure&nbsp;1.12</a>) compete under varying levels of external force?</p>
<div id="fig-hyd-vs-hom-reaction" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="Hydrolysis vs. homolysis reaction schema.">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-hyd-vs-hom-reaction-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./img/hyd-vs-hom-reaction.svg" class="lightbox" data-gallery="quarto-lightbox-gallery-16" title="Figure&nbsp;1.12: Reaction schema of the two competing reactions, hydrolysis (upper pathway) and homolysis (lower pathway)."><img src="./img/hyd-vs-hom-reaction.svg" class="img-fluid figure-img" data-fig-scap="Hydrolysis vs. homolysis reaction schema."></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-hyd-vs-hom-reaction-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;1.12: Reaction schema of the two competing reactions, hydrolysis (upper pathway) and homolysis (lower pathway).
</figcaption>
</figure>
</div>
<p>During base-catalyzed hydrolysis, a hydroxide ion (<span class="math inline">\(\ce{OH-}\)</span>) acting as a nucleophile attacks the electrophilic carbon center of the peptide bond. The reaction goes through a tetrahedral intermediate and eventually leads to the rupture of the (<span class="math inline">\(\ce{C-N}\)</span>) peptide bond, resulting in carboxy- and amino-termini. It is explained in greater detail in <a href="#sec-main-qm" class="quarto-xref">Chapter&nbsp;3</a>. Force-induced homolysis, on the other hand, requires no additional molecules. Bonds along the vector of the applied force are weakened by the energy put into them by the external force, which puts them closer to their bond dissociation energy (BDE). This increases the probability of rupture, whereby the electrons of the scissile bond are split equally between the two binding partners, resulting in unpaired electrons, i.e., radicals. The reaction probability for a particular bond depends on the BDE and the distribution of force through the bonds. How one emulates both reactions with a physical model is explained in more detail in <a href="#sec-hyd-hom-plugins" class="quarto-xref">Section&nbsp;4.2.1</a>.</p>
</section>
<section id="aims-of-this-thesis" class="level2" data-number="1.3">
<h2 data-number="1.3" class="anchored" data-anchor-id="aims-of-this-thesis"><span class="header-section-number">1.3</span> Aims of this thesis</h2>
<p>While homolysis on the one hand leaves behind radicals as relatively straightforward to detect products, hydrolysis on the other hand just leaves us with unreactive peptides. Furthermore, the wet conditions required for peptide bond hydrolysis are not conducive to measuring radicals in the same sample. EPR measurements require the sample to be dried, because residual water would otherwise be heated up by the microwaves used during measurement and evaporate the sample. I wanted to look deeply at the two competing reactions at a molecular level and thus turned to molecular simulations, the types of which are explained in detail in the <a href="#sec-methods">Methods and theory</a> section.</p>
<p>The goals of this thesis are twofold. Firstly, there is the biological question at hand. How do the reaction rates of homolysis and hydrolysis compare to each other in collagen fibrils? What factors influence their competition? Secondly, there is the more general question of how chemical reactions can be simulated with computational models, even if the time scales of the reactions are disparate from the time scales that can be achieved with molecular simulations.</p>
<p>The <a href="#sec-main-qm">Collagen hydrolysis by quantum mechanics/molecular mechanics</a> chapter, <a href="#sec-main-qm" class="quarto-xref">Chapter&nbsp;3</a>, explores the base-catalyzed hydrolysis reaction in detail. Instead of static calculations, it aims to provide a view into the dynamic nature of the reaction and its complex energy landscape. This is used to directly see how the unique structure of collagen, its tightly wound triple helix, influences the energy barrier of hydrolysis.</p>
<p>But computationally expensive methods such as quantum mechanics/molecular mechanics simulations are inherently limited in the scope of their sampling. The <a href="#sec-main-kimmdy">Kinetic Monte Carlo Molecular Dynamics: KIMMDY</a> chapter, <a href="#sec-main-kimmdy" class="quarto-xref">Chapter&nbsp;4</a>, tackles the question of reactive simulations at scale. I will introduce a new method that is the result of a joint project with my colleagues Eric Hartmann and Kai Riedmiller. KIMMDY provides a generalized framework for bridging timescales in reactive molecular dynamics (MD) simulations. As such, the goal of this thesis is not just to answer this one particular question, but also to provide a new tool to the scientific community. In the second half of the chapter, our new tool will be deployed to directly compare the two competing reactions at the scale of a complete collagen fibril.</p>
<p>Lastly, there is one additional goal that can only be read between the lines. I hope for this thesis to be more than a mere enumeration of achievements and instead also provide some joy to the reader. This is why the historical anecdote about scurvy is included and why the upcoming <a href="#sec-methods">Methods and theory</a> chapter takes on a more educational tone.</p>
<div style="page-break-after: always;"></div>
</section>
</section>
<section id="sec-methods" class="level1 page-columns page-full" data-number="2">
<h1 data-number="2"><span class="header-section-number">2</span> Methods and theory</h1>
<p>In the following I will introduce the relevant methods and underlying theory used in the simulations performed for this thesis. The goal is to introduce each topic at least to the extent that the choices made for setting up the simulations and implementing solutions make sense in the context of the theory.</p>
<section id="molecular-dynamics" class="level2 page-columns page-full" data-number="2.1">
<h2 data-number="2.1" class="anchored" data-anchor-id="molecular-dynamics"><span class="header-section-number">2.1</span> Molecular dynamics</h2>

<div class="no-row-height column-margin column-container"><div id="fig-propionaldehyde" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="Propionaldehyde, chemical formula.">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-propionaldehyde-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./img/propionaldehyde.svg" class="lightbox" data-gallery="quarto-lightbox-gallery-17" title="Figure&nbsp;2.1: Chemical structure formula of propionaldehyde."><img src="./img/propionaldehyde.svg" class="img-fluid figure-img" style="width:60.0%" data-fig-scap="Propionaldehyde, chemical formula."></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-propionaldehyde-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2.1: Chemical structure formula of propionaldehyde.
</figcaption>
</figure>
</div><div id="fig-propionaldehyde-sticks" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="Propionaldehyde, ball-and-stick render.">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-propionaldehyde-sticks-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./img/molecule-cpk.png" class="lightbox" data-gallery="quarto-lightbox-gallery-18" title="Figure&nbsp;2.2: Propionaldehyde as a classic ball-and-stick render."><img src="./img/molecule-cpk.png" class="img-fluid figure-img" data-fig-scap="Propionaldehyde, ball-and-stick render."></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-propionaldehyde-sticks-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2.2: Propionaldehyde as a classic ball-and-stick render.
</figcaption>
</figure>
</div><div id="fig-propionaldehyde-density" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="Propionaldehyde, nuclei with electron density.">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-propionaldehyde-density-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./img/molecule-density.png" class="lightbox" data-gallery="quarto-lightbox-gallery-19" title="Figure&nbsp;2.3: Propionaldehyde represented with balls for atomic nuclei and the electron density around them."><img src="./img/molecule-density.png" class="img-fluid figure-img" data-fig-scap="Propionaldehyde, nuclei with electron density."></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-propionaldehyde-density-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2.3: Propionaldehyde represented with balls for atomic nuclei and the electron density around them.
</figcaption>
</figure>
</div></div>

<p>So how do we simulate molecular systems, especially large ones relevant for biomolecular research? Let’s start with a simple small molecule such as this propionaldehyde in <a href="#fig-propionaldehyde" class="quarto-xref">Figure&nbsp;2.1</a>. In the chemical formula, implicit hydrogens are often omitted, so let us also look at a classic ball-and-stick render of the same molecule in <a href="#fig-propionaldehyde-sticks" class="quarto-xref">Figure&nbsp;2.2</a>. Ultimately, this representation combines the nuclei and electrons of the atoms involved in the molecule into one representation, but we might also want to show a representation of the electron density like in <a href="#fig-propionaldehyde-density" class="quarto-xref">Figure&nbsp;2.3</a>. The challenge is now to find a representation in which it becomes feasible to describe the motion of atoms through time in order to build a system that we can observe and use to calculate properties of by sampling.</p>
<p>As the phrase, usually attributed to statistician George Box, goes: “All models are wrong, but some are useful.” So let us start with one of the most simplified ones that will allow us to simulate large systems. Afterwards, we will add more detail and then finally combine both methods.</p>
<p>We can describe our system as a collection of all the atomic positions <span class="math inline">\(\vec{r}\)</span> in 3-dimensional space and their momenta <span class="math inline">\(\vec{p}\)</span>, where both vectors are of length 3N with N being the number of atoms. If we can propagate this system through time, we get a path through the 6N-dimensional phase space that we call a trajectory. The overall method is called molecular dynamics (MD).</p>
<p>Let us imagine we already have a way to describe the total energy (kinetic plus potential energy) of the system and its derivative. We can use either Hamiltonian mechanics <a href="#eq-hamilton" class="quarto-xref">Equation&nbsp;2.1</a></p>
<p><span id="eq-hamilton"><span class="math display">\[
\begin{array}{rcl}
\frac{dr_i}{dt} &amp;=&amp; \frac{\delta H}{\delta p_i} \\
\frac{dp_i}{dt} &amp;=&amp; -\frac{\delta H}{\delta r_i},
\end{array}
\tag{2.1}\]</span></span></p>
<p>where the Hamiltonian operator <span class="math inline">\(H\)</span> is the total energy operator and <span class="math inline">\(r_i\)</span> is one value of <span class="math inline">\(\vec{r}\)</span>, i.e., the position of one atom along one axis and likewise for <span class="math inline">\(p_i\)</span>.</p>
<p>Or we can use Newton’s equations of motion <a href="#eq-newton" class="quarto-xref">Equation&nbsp;2.2</a></p>
<p><span id="eq-newton"><span class="math display">\[
\begin{array}{rcl}
F  &amp;=&amp; ma \\
-\frac{dV}{dr} &amp;=&amp; m\frac{d^2r}{dt^2},
\end{array}
\tag{2.2}\]</span></span></p>
<p>where <span class="math inline">\(V\)</span> is the potential, whose derivative with respect to the positions corresponds to the force <span class="math inline">\(F\)</span> and the acceleration <span class="math inline">\(a\)</span> is the second derivative of the positions with respect to time <span class="math inline">\(t\)</span>. The mass <span class="math inline">\(m\)</span> is assumed constant for each particle.</p>
<p>Both of these differential equations can be integrated over time to trace out a trajectory given initial conditions for the positions and momenta <span class="math inline">\(\vec{r}_0\)</span> and <span class="math inline">\(\vec{p}_0\)</span>. Most implementations use Newton’s equations for computations. The integration is performed stepwise over small intervals <span class="math inline">\(\Delta t\)</span> (or <span class="math inline">\(dt\)</span>). From the initial state, energies and forces are computed for each atom. Under the assumption of those being constant for a small timestep <span class="math inline">\(dt\)</span>, typically in the range of femtoseconds, the new positions are calculated from the forces acting on the atoms. The <em>leapfrog</em> algorithm <span class="citation" data-cites="hockneyQuietHighresolutionComputer1974">[<a href="#ref-hockneyQuietHighresolutionComputer1974" role="doc-biblioref">21</a>]</span> achieves this by interleaving position and velocity updates as shown in <a href="#eq-frog" class="quarto-xref">Equation&nbsp;2.3</a>. This gets repeated until sufficient sampling of phase space is achieved or computational resources, limited by time and money, run out.</p>
<p><span id="eq-frog"><span class="math display">\[
\begin{split}\begin{aligned}
\mathbf{v}(t+{{\frac{1}{2}}{{\Delta t}}})  &amp;~=~&amp;   \mathbf{v}(t-{{\frac{1}{2}}{{\Delta t}}})+\frac{{{\Delta t}}}{m}\mathbf{F}(t)   \\
\mathbf{r}(t+{{\Delta t}})   &amp;~=~&amp;   \mathbf{r}(t)+{{\Delta t}}\mathbf{v}(t+{{\frac{1}{2}}{{\Delta t}}})\end{aligned}\end{split}
\tag{2.3}\]</span></span></p>
<p>Because different initial conditions lead to different trajectories, one often runs multiple such computational runs in parallel. For ergodic properties, i.e., those for which the ergodic hypothesis holds, stating that the time average is equal to the ensemble average <span class="citation" data-cites="boltzmannVorlesungenUeberGastheorie1896">[<a href="#ref-boltzmannVorlesungenUeberGastheorie1896" role="doc-biblioref">22</a>]</span>, this can drastically improve sampling.</p>
</section>
<section id="molecular-mechanics" class="level2 page-columns page-full" data-number="2.2">
<h2 data-number="2.2" class="anchored" data-anchor-id="molecular-mechanics"><span class="header-section-number">2.2</span> Molecular mechanics</h2>
<p>How do we obtain a feasible equation for the total energy of our system that scales well to a large number of atoms? This is where we think back to our simplified stick representation in <a href="#fig-propionaldehyde-sticks" class="quarto-xref">Figure&nbsp;2.2</a> and arrive at molecular mechanics (MM), as in classical Newtonian mechanics. Let us assume our system is made up of different kinds of balls connected with springs.</p>
<section id="energy-terms" class="level3 page-columns page-full" data-number="2.2.1">
<h3 data-number="2.2.1" class="anchored" data-anchor-id="energy-terms"><span class="header-section-number">2.2.1</span> Energy terms</h3>
<p>The total potential energy of our system is described as the sum of bonded and non-bonded terms <a href="#eq-energy-total" class="quarto-xref">Equation&nbsp;2.4</a>, which we can further decompose with <a href="#eq-energy-bonded" class="quarto-xref">Equation&nbsp;2.5</a> and <a href="#eq-energy-nonbonded" class="quarto-xref">Equation&nbsp;2.6</a>.</p>
<p><span id="eq-energy-total"><span class="math display">\[V_{total} = V_{bonded} + V_{nonbonded} \tag{2.4}\]</span></span></p>
<p><span id="eq-energy-bonded"><span class="math display">\[V_{bonded} = V_{bonds}+V_{angles}+V_{dihedrals} \tag{2.5}\]</span></span></p>
<p><span id="eq-energy-nonbonded"><span class="math display">\[V_{nonbonded} = V_{electrostatic} + V_{van der Waals} \tag{2.6}\]</span></span></p>
<div id="fig-mm" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="Bonded energy terms.">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-mm-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./figures/gly-mm-annotated.png" class="lightbox" data-gallery="quarto-lightbox-gallery-20" title="Figure&nbsp;2.4: Excerpt from a glycine dipeptide with representative MM bonded terms, a bond \mathrm{r}, an angle \alpha, a dihedral \mathrm{\phi} as well as non-bonded terms \mathrm{nb} and 1–4."><img src="./figures/gly-mm-annotated.png" class="img-fluid figure-img" style="width:60.0%" data-fig-scap="Bonded energy terms."></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-mm-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2.4: Excerpt from a glycine dipeptide with representative MM bonded terms, a bond <span class="math inline">\(\mathrm{r}\)</span>, an angle <span class="math inline">\(\alpha\)</span>, a dihedral <span class="math inline">\(\mathrm{\phi}\)</span> as well as non-bonded terms <span class="math inline">\(\mathrm{nb}\)</span> and 1–4.
</figcaption>
</figure>
</div>
<p>The bonded and non-bonded terms are illustrated by examples in <a href="#fig-mm" class="quarto-xref">Figure&nbsp;2.4</a>. For each of these terms we find approximations that are easy to compute on modern hardware, which typically means that exponentials are avoided in favor of polynomials obtained through Taylor expansion <span class="citation" data-cites="taylorMethodusIncrementorumDirecta1715">[<a href="#ref-taylorMethodusIncrementorumDirecta1715" role="doc-biblioref">23</a>]</span>. For example, while we can describe the potential energy in a bond between two atoms with a Morse potential <span class="citation" data-cites="morseDiatomicMoleculesAccording1929">[<a href="#ref-morseDiatomicMoleculesAccording1929" role="doc-biblioref">24</a>]</span> (<a href="#eq-energy-morse" class="quarto-xref">Equation&nbsp;2.7</a>, <a href="#fig-morse-potential" class="quarto-xref">Figure&nbsp;2.5</a>):</p>

<div class="no-row-height column-margin column-container"><div id="fig-morse-potential" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="Morse potential.">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-morse-potential-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./img/morse-potential.png" class="lightbox" data-gallery="quarto-lightbox-gallery-21" title="Figure&nbsp;2.5: A Morse potential."><img src="./img/morse-potential.png" class="img-fluid figure-img" data-fig-scap="Morse potential."></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-morse-potential-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2.5: A Morse potential.
</figcaption>
</figure>
</div><div id="fig-harmonic-potential" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="Harmonic potential.">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-harmonic-potential-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./img/harmonic-potential.png" class="lightbox" data-gallery="quarto-lightbox-gallery-22" title="Figure&nbsp;2.6: A harmonic potential."><img src="./img/harmonic-potential.png" class="img-fluid figure-img" data-fig-scap="Harmonic potential."></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-harmonic-potential-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2.6: A harmonic potential.
</figcaption>
</figure>
</div></div>
<p><span id="eq-energy-morse"><span class="math display">\[V_{morse}(r) = D \left(1 - e^{-\beta(r - r_0)}\right)^2, \tag{2.7}\]</span></span></p>
<p>with the well depth <span class="math inline">\(D\)</span>, the equilibrium bond distance <span class="math inline">\(r_0\)</span> and <span class="math inline">\(\beta\)</span> relating to the force constant <span class="math inline">\(k\)</span> via <span class="math inline">\(\beta=\sqrt{\frac{k}{2D}}\)</span>, in practice this is often simplified to a harmonic potential (<a href="#eq-energy-harmonic" class="quarto-xref">Equation&nbsp;2.8</a>, <a href="#fig-harmonic-potential" class="quarto-xref">Figure&nbsp;2.6</a>):</p>
<p><span id="eq-energy-harmonic"><span class="math display">\[V_{harmonic}(r) = \frac{1}{2} k (r - r_0)^2 \tag{2.8}\]</span></span></p>
<p>This is a good approximation in the low-energy regime and has the added bonus that our molecule is unlikely to fall apart due to the infinitely scaling potential instead of the plateau in the Morse potential. Likewise, angles between three atoms can also be approximated with harmonic potentials. Finally, bonded connections between four atoms give rise to dihedral angles that are periodic in nature and are thus represented with cosines (<a href="#eq-energy-cosine" class="quarto-xref">Equation&nbsp;2.9</a>) as the result of a Fourier series <span class="citation" data-cites="fourierMemoirePropagationChaleur2013">[<a href="#ref-fourierMemoirePropagationChaleur2013" role="doc-biblioref">25</a>]</span>.</p>
<p><span id="eq-energy-cosine"><span class="math display">\[V_{cosine}(\phi) = k( 1 + \cos(n \phi - \phi_0)) \tag{2.9}\]</span></span></p>
<p>Sometimes additional bonded terms labelled “improper dihedrals” are used to, e.g., enforce planarity of certain molecules by adding potential energy for out-of-plane bending.</p>
<p>In addition to those bonded terms, we have the non-bonded terms split into electrostatic interactions and Van der Waals interactions.</p>
<p>Van der Waals forces (or consequently the potential energy from which they arise) are the result of a mixture of repulsion through overlapping electron density at very close distances and attraction at intermediate distances due to spontaneous and induced dipole-dipole interactions between the electron clouds of two atoms. While the long-range attraction scales with <span class="math inline">\(1/R^6\)</span>, the short-range repulsion features an exponential term. This is why in practice the Lennard-Jones potential <span class="citation" data-cites="lennard-jonesCohesion1931">[<a href="#ref-lennard-jonesCohesion1931" role="doc-biblioref">26</a>]</span> is used instead of a more accurate formulation, simplifying computations (<a href="#eq-lennard-jones" class="quarto-xref">Equation&nbsp;2.10</a>, <a href="#fig-lj-potential" class="quarto-xref">Figure&nbsp;2.7</a>).</p>
<p><span id="eq-lennard-jones"><span class="math display">\[
V_{vdw}(r_{ab})=4\epsilon_{ij} \left[(\frac{\sigma_{ij}}{r_{ij}})^{12}-(\frac{\sigma_{ij}}{r_{ij}})^6 \right],
\tag{2.10}\]</span></span></p>
<p>where <span class="math inline">\(r\)</span> is the distance between two atoms <span class="math inline">\(i\)</span> and <span class="math inline">\(j\)</span> and <span class="math inline">\(\sigma\)</span> and <span class="math inline">\(\epsilon\)</span> are parameters that depend on the pair of atoms.</p>

<div class="no-row-height column-margin column-container"><div id="fig-lj-potential" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="Lennard-Jones potential.">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-lj-potential-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./img/lennard-jones-potential.png" class="lightbox" data-gallery="quarto-lightbox-gallery-23" title="Figure&nbsp;2.7: A Lennard-Jones potential."><img src="./img/lennard-jones-potential.png" class="img-fluid figure-img" data-fig-scap="Lennard-Jones potential."></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-lj-potential-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2.7: A Lennard-Jones potential.
</figcaption>
</figure>
</div></div><p>The electrostatic or Coulomb <span class="citation" data-cites="coulombFirstMemoirElectricity1785">[<a href="#ref-coulombFirstMemoirElectricity1785" role="doc-biblioref">27</a>]</span> term (<a href="#eq-coulomb" class="quarto-xref">Equation&nbsp;2.11</a> and <a href="#fig-coulomb-potential" class="quarto-xref">Figure&nbsp;2.8</a>) is also summed for all pairs of atoms.</p>
<p><span id="eq-coulomb"><span class="math display">\[
V_{Coulomb}(r_{ij})=\frac{Q_a Q_b}{\epsilon r^2},
\tag{2.11}\]</span></span></p>
<p>with the effective dielectric constant <span class="math inline">\(\epsilon\)</span>.</p>

<div class="no-row-height column-margin column-container"><div id="fig-coulomb-potential" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="Coulomb potential.">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-coulomb-potential-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./img/coulomb-potential.png" class="lightbox" data-gallery="quarto-lightbox-gallery-24" title="Figure&nbsp;2.8: A Coulomb potential."><img src="./img/coulomb-potential.png" class="img-fluid figure-img" data-fig-scap="Coulomb potential."></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-coulomb-potential-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2.8: A Coulomb potential.
</figcaption>
</figure>
</div></div><p>Here, the charge <span class="math inline">\(Q\)</span> of an atom is not just the result of ionization. Instead, it includes so-called partial charges, where the total electron density of a molecule has been (artificially) assigned beforehand to individual atoms based on quantum mechanical calculation procedures like the Restrained Electrostatic Potential (RESP) <span class="citation" data-cites="baylyWellbehavedElectrostaticPotential2002">[<a href="#ref-baylyWellbehavedElectrostaticPotential2002" role="doc-biblioref">28</a>]</span>.</p>
<p>Those non-bonded interactions are sometimes scaled down when atoms are, albeit not directly, connected by being part of the same dihedral, which is then called a 1–4 interaction and are usually omitted when atoms are direct or second neighbors. The contributions are then already integrated into the bonded terms.</p>
</section>
<section id="force-fields-and-topologies" class="level3" data-number="2.2.2">
<h3 data-number="2.2.2" class="anchored" data-anchor-id="force-fields-and-topologies"><span class="header-section-number">2.2.2</span> Force fields and topologies</h3>
<p>Now that we have equations for the energy terms of our system that are also easily derivable, we need to feed some values into them. How do we get, store, and recall the parameters for all those equations? The answer comes in the form of force fields.</p>
<p>Many PhD students before me and other clever people have performed more detailed quantum mechanical <em>ab initio</em> calculations to figure out what those parameters should be, like, for example, the force constant <span class="math inline">\(k\)</span> (in <a href="#eq-energy-harmonic" class="quarto-xref">Equation&nbsp;2.8</a>) for a bond between two particular atoms. This will vary by molecule and chemical environment; not every carbon (<span class="math inline">\(\ce{C}\)</span>) to carbon bond is the same, but we can group similar atoms into “atomtypes”. In protein simulations a popular choice are the Amber force fields <span class="citation" data-cites="caseAmberTools2023">[<a href="#ref-caseAmberTools2023" role="doc-biblioref">29</a>]</span>. Since the early version developed in 1994, Amber94 <span class="citation" data-cites="cornellSecondGenerationForce1995">[<a href="#ref-cornellSecondGenerationForce1995" role="doc-biblioref">30</a>]</span> has been continuously refined. Amber99 improved upon it with RESP partial charges <span class="citation" data-cites="wangHowWellDoes2000">[<a href="#ref-wangHowWellDoes2000" role="doc-biblioref">31</a>]</span> and provided a basis for further modifications. The parameters are not only fit to <em>ab initio</em> QM calculations. They are also adjusted to match observables from simulations to experimental values, such as the dihedral angles of a peptide in a Ramachandran diagram <span class="citation" data-cites="ramachandranStereochemistryPolypeptideChain1963">[<a href="#ref-ramachandranStereochemistryPolypeptideChain1963" role="doc-biblioref">32</a>]</span>. The version used for the simulations in this thesis is <code>amber99sb*ildnp</code>, where <code>sb</code> stands for Stony Brook University, <code>*</code> stands for protein backbone modifications, <code>ildn</code> improved side-chain torsion potentials, and the <code>p</code> stands for improved proline and hydroxyproline parameters <span class="citation" data-cites="bestOptimizedMolecularDynamics2009 lindahlGROMACS2020Source2020 alievMotionalTimescalePredictions2014">[<a href="#ref-bestOptimizedMolecularDynamics2009" role="doc-biblioref">33</a>–<a href="#ref-alievMotionalTimescalePredictions2014" role="doc-biblioref">35</a>]</span>. Other well-known force fields include CHARMM <span class="citation" data-cites="brooksCHARMMBiomolecularSimulation2009">[<a href="#ref-brooksCHARMMBiomolecularSimulation2009" role="doc-biblioref">36</a>]</span>, GROMOS <span class="citation" data-cites="schulerImprovedGROMOS96Force2001">[<a href="#ref-schulerImprovedGROMOS96Force2001" role="doc-biblioref">37</a>]</span> and OPLS-AA <span class="citation" data-cites="robertsonImprovedPeptideProtein2015">[<a href="#ref-robertsonImprovedPeptideProtein2015" role="doc-biblioref">38</a>]</span>. The choice of a force field is one of the many crucial choices when setting up a molecular dynamics simulation, as different force fields have been optimized for different types of molecules or observables. Amber was chosen as it is specifically geared towards protein simulations. We will be going through examples from this force field, because how a force field is structured will be relevant to <a href="#sec-main-kimmdy" class="quarto-xref">Chapter&nbsp;4</a> in particular. Specifically, the lines shown are from a version of the files <span class="citation" data-cites="sorinExploringHelixCoilTransition2005">[<a href="#ref-sorinExploringHelixCoilTransition2005" role="doc-biblioref">39</a>]</span> ported to the molecular dynamics engine GROMACS <span class="citation" data-cites="abrahamGROMACSHighPerformance2015">[<a href="#ref-abrahamGROMACSHighPerformance2015" role="doc-biblioref">40</a>]</span> that I have been using during the course of my PhD.</p>
<p>Just like the total energy function <a href="#eq-energy-total" class="quarto-xref">Equation&nbsp;2.4</a>, the main entry point for the force field (<code>forcefield.itp</code>, <a href="#lst-ff" class="quarto-xref">Listing&nbsp;2.1</a>) includes references to the non-bonded and bonded parameters. In addition to that, it also holds more general parameters, such as the combination rule used to compute Lennard-Jones (LJ) parameters for a pair of atoms and the fudge parameters, which are factors for scaling down LJ and Coulomb (QQ) parameters for 1–4 interactions. <code>itp</code> stands for “include topology” and we will talk more about the topology concept later.</p>
<div id="lst-ff" class="ini listing quarto-float quarto-figure quarto-figure-left anchored">
<figure class="quarto-float quarto-float-lst figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-lst" id="lst-ff-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Listing&nbsp;2.1: Excerpt from forcefield.itp
</figcaption>
<div aria-describedby="lst-ff-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="code-copy-outer-scaffold"><div class="sourceCode" id="cb1"><pre class="sourceCode numberSource ini number-lines code-with-copy"><code class="sourceCode ini"><span id="cb1-1"><a href="#cb1-1"></a><span class="kw">[ defaults ]</span></span>
<span id="cb1-2"><a href="#cb1-2"></a><span class="co">; nbfunc  comb-rule  gen-pairs  fudgeLJ  fudgeQQ</span></span>
<span id="cb1-3"><a href="#cb1-3"></a><span class="dt">1         2          yes        0.5      0.8333</span></span>
<span id="cb1-4"><a href="#cb1-4"></a></span>
<span id="cb1-5"><a href="#cb1-5"></a><span class="co">#include "ffnonbonded.itp"</span></span>
<span id="cb1-6"><a href="#cb1-6"></a><span class="co">#include "ffbonded.itp"</span></span></code></pre></div><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></div>
</div>
</figure>
</div>
<div id="lst-nonbonded" class="ini listing quarto-float quarto-figure quarto-figure-left anchored">
<figure class="quarto-float quarto-float-lst figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-lst" id="lst-nonbonded-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Listing&nbsp;2.2: Select lines from ffnonbonded.itp
</figcaption>
<div aria-describedby="lst-nonbonded-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="code-copy-outer-scaffold"><div class="sourceCode" id="cb2"><pre class="sourceCode numberSource ini number-lines code-with-copy"><code class="sourceCode ini"><span id="cb2-1"><a href="#cb2-1"></a><span class="kw">[ atomtypes ]</span></span>
<span id="cb2-2"><a href="#cb2-2"></a><span class="co">; name at.num  mass    charge ptype  sigma      epsilon</span></span>
<span id="cb2-3"><a href="#cb2-3"></a><span class="dt">C       6      12.01   0.0000  A   3.39967e-01  3.59824e-01</span></span>
<span id="cb2-4"><a href="#cb2-4"></a><span class="dt">CA      6      12.01   0.0000  A   3.39967e-01  3.59824e-01</span></span>
<span id="cb2-5"><a href="#cb2-5"></a><span class="dt">H       1       1.008  0.0000  A   1.06908e-01  6.56888e-02</span></span>
<span id="cb2-6"><a href="#cb2-6"></a><span class="dt">HC      1       1.008  0.0000  A   2.64953e-01  6.56888e-02</span></span></code></pre></div><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></div>
</div>
</figure>
</div>
<p>For any given pair of atoms that have been assigned atom types and are not bonded, we can fill in the parameters of the LJ equation (<a href="#eq-lennard-jones" class="quarto-xref">Equation&nbsp;2.10</a>). The combination rule as mentioned in the general parameters decides how the parameters <span class="math inline">\(\epsilon\)</span> and <span class="math inline">\(\sigma\)</span> from both atoms are combined into one parameter each. Type 1 would indicate a geometric average, while type 2, as in this example, used the Lorentz-Berthelot <span class="citation" data-cites="lorentzUeberAnwendungSatzes1881 berthelotMelangeGaz1898">[<a href="#ref-lorentzUeberAnwendungSatzes1881" role="doc-biblioref">41</a>, <a href="#ref-berthelotMelangeGaz1898" role="doc-biblioref">42</a>]</span> rules, where <span class="math inline">\(\sigma\)</span> used an arithmetic average and <span class="math inline">\(\epsilon\)</span> uses a geometric average (<a href="#eq-lbrule" class="quarto-xref">Equation&nbsp;2.12</a>).</p>
<p><span id="eq-lbrule"><span class="math display">\[
\begin{array}{rcl}
\sigma_{ij}   &amp;=&amp; \left({\sigma_{ii} \, \sigma_{jj}}\right)^{1/2}        \\
\epsilon_{ij} &amp;=&amp; \left({\epsilon_{ii} \, \epsilon_{jj}}\right)^{1/2}
\end{array}
\tag{2.12}\]</span></span></p>
<p>Likewise, the MD engine can use the charges from <a href="#lst-nonbonded" class="quarto-xref">Listing&nbsp;2.2</a> to calculate the electrostatic potential. However, you might have noticed that all the charges in the default <code>atomtypes</code> are <code>0.0</code>. This is correct, as, for example, a carbon when saturated is not charged by default, but based on its chemical environment it should have a partial charge. How large should the partial charge be? This is defined in the <code>topology</code>.</p>
<p>The topology of a system lists all molecules and atoms of a system and how they are connected. So while a force field lists all potential interactions between types of atoms that it knows about, a topology is a concrete instance of the interactions and their parameters. The topology files shown are in the format of my MD engine of choice, GROMACS. A topology file (<a href="#lst-top" class="quarto-xref">Listing&nbsp;2.3</a>) may include the force field and then list a number of moleculetypes that can then be used to describe the system with the number of each type listed in the molecules section. Note though that a topology is only concerned with connectivity and types, but not with coordinates. Coordinates are specified in a separate file such as a <code>.gro</code> or <code>.pdb</code> <span class="citation" data-cites="bermanProteinDataBank2014">[<a href="#ref-bermanProteinDataBank2014" role="doc-biblioref">43</a>]</span> file, excerpts of which can be seen in <a href="#lst-gro" class="quarto-xref">Listing&nbsp;5.1</a> and <a href="#lst-pdb" class="quarto-xref">Listing&nbsp;5.2</a>. When GROMACS creates a topology from coordinates, it uses information such as the “residuetype” to fill out and assign interactions. For example, in <a href="#lst-top" class="quarto-xref">Listing&nbsp;2.3</a> the atom with identifier (ID) 1 is a carbon of type CT that belongs to the acyl cap of a glycine dipeptide, so it has residuetype ACE. <code>.rtp</code> files in the force fields contain specialized residue topologies. In this case, given that we are dealing with a peptide, GROMACS will look in <code>aminoacids.rtp</code> (<a href="#lst-rtp" class="quarto-xref">Listing&nbsp;2.5</a>), find the partial charges for the atoms of the residue, and write their values directly into the topology as part of the <code>atoms</code> section. Additionally, improper dihedrals are added based on the <code>impropers</code> section.</p>
<p>Not all values have to be written explicitly. Take the first bond in <a href="#lst-top" class="quarto-xref">Listing&nbsp;2.3</a> between atoms with ids 1 and 2. The topology only specifies the functional form of the interaction, where type 1 is a harmonic potential <span class="citation" data-cites="FileFormatsGROMACS">[<a href="#ref-FileFormatsGROMACS" role="doc-biblioref">44</a>]</span>. The values for the parameters are then taken from the <code>bondtypes</code> section based on the atomtypes of the atoms involved in the bond, where in this case the <code>bondtypes</code> section was included from the force field file as seen in <a href="#lst-bonded" class="quarto-xref">Listing&nbsp;2.4</a>. It is also possible to specify the parameters directly, which will be especially relevant for our method KIMMDY, <a href="#sec-main-kimmdy" class="quarto-xref">Chapter&nbsp;4</a>. The same is done for higher-order interactions such as angles and dihedrals. A special case is the <code>pairs</code> section, which indicates pairs of atoms that have scaled LJ and Coulomb terms, usually due to them being part of a 1–4 interaction. This section also allows one to specify the Coulomb and Van der Waals parameters directly.</p>
<div id="lst-top" class="ini listing quarto-float quarto-figure quarto-figure-left anchored">
<figure class="quarto-float quarto-float-lst figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-lst" id="lst-top-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Listing&nbsp;2.3: Select simplified lines from a .top file of a capped glycine dipeptide
</figcaption>
<div aria-describedby="lst-top-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="code-copy-outer-scaffold"><div class="sourceCode" id="cb3"><pre class="sourceCode numberSource ini number-lines code-with-copy"><code class="sourceCode ini"><span id="cb3-1"><a href="#cb3-1"></a><span class="co">#include "./amber99sb-star-ildnp.ff/forcefield.itp"</span></span>
<span id="cb3-2"><a href="#cb3-2"></a></span>
<span id="cb3-3"><a href="#cb3-3"></a><span class="kw">[ moleculetype ]</span></span>
<span id="cb3-4"><a href="#cb3-4"></a><span class="co">; Name            nrexcl</span></span>
<span id="cb3-5"><a href="#cb3-5"></a><span class="dt">Protein             3</span></span>
<span id="cb3-6"><a href="#cb3-6"></a></span>
<span id="cb3-7"><a href="#cb3-7"></a><span class="kw">[ atoms ]</span></span>
<span id="cb3-8"><a href="#cb3-8"></a><span class="co">; nr  type resnr residue  atom  cgnr   charge  mass  typeB ..</span></span>
<span id="cb3-9"><a href="#cb3-9"></a><span class="dt">  1    CT     1    ACE    CH3    1  -0.3662   12.01</span></span>
<span id="cb3-10"><a href="#cb3-10"></a><span class="dt">  2    HC     1    ACE   HH31    2   0.1123   1.008</span></span>
<span id="cb3-11"><a href="#cb3-11"></a></span>
<span id="cb3-12"><a href="#cb3-12"></a><span class="kw">[ bonds ]</span></span>
<span id="cb3-13"><a href="#cb3-13"></a><span class="co">;  ai    aj funct    c0    c1      c2    c3</span></span>
<span id="cb3-14"><a href="#cb3-14"></a><span class="dt">    1     2     1</span></span>
<span id="cb3-15"><a href="#cb3-15"></a><span class="dt">    1     3     1</span></span>
<span id="cb3-16"><a href="#cb3-16"></a></span>
<span id="cb3-17"><a href="#cb3-17"></a><span class="kw">[ angles ]</span></span>
<span id="cb3-18"><a href="#cb3-18"></a><span class="co">;  ai    aj    ak funct   c0   c1   c2    c3</span></span>
<span id="cb3-19"><a href="#cb3-19"></a><span class="dt">    2     1     3     1</span></span>
<span id="cb3-20"><a href="#cb3-20"></a></span>
<span id="cb3-21"><a href="#cb3-21"></a><span class="kw">[ molecules ]</span></span>
<span id="cb3-22"><a href="#cb3-22"></a><span class="co">; Compound        #mols</span></span>
<span id="cb3-23"><a href="#cb3-23"></a><span class="dt">Protein             1</span></span>
<span id="cb3-24"><a href="#cb3-24"></a><span class="dt">SOL         39427</span></span>
<span id="cb3-25"><a href="#cb3-25"></a><span class="dt">NA               110</span></span>
<span id="cb3-26"><a href="#cb3-26"></a><span class="dt">CL               110</span></span></code></pre></div><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></div>
</div>
</figure>
</div>
<div id="lst-bonded" class="ini listing quarto-float quarto-figure quarto-figure-left anchored">
<figure class="quarto-float quarto-float-lst figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-lst" id="lst-bonded-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Listing&nbsp;2.4: Select simplified lines from ffbonded.itp
</figcaption>
<div aria-describedby="lst-bonded-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="code-copy-outer-scaffold"><div class="sourceCode" id="cb4"><pre class="sourceCode numberSource ini number-lines code-with-copy"><code class="sourceCode ini"><span id="cb4-1"><a href="#cb4-1"></a><span class="kw">[ bondtypes ]</span></span>
<span id="cb4-2"><a href="#cb4-2"></a><span class="co">; i   j  func    b0        kb</span></span>
<span id="cb4-3"><a href="#cb4-3"></a><span class="dt">  CT  HC   1     0.10900   284512.0</span></span></code></pre></div><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></div>
</div>
</figure>
</div>
<div id="lst-rtp" class="ini listing quarto-float quarto-figure quarto-figure-left anchored">
<figure class="quarto-float quarto-float-lst figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-lst" id="lst-rtp-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Listing&nbsp;2.5: Select lines from aminoacids.rtp
</figcaption>
<div aria-describedby="lst-rtp-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="code-copy-outer-scaffold"><div class="sourceCode" id="cb5"><pre class="sourceCode numberSource ini number-lines code-with-copy"><code class="sourceCode ini"><span id="cb5-1"><a href="#cb5-1"></a><span class="kw">[ ACE ]</span></span>
<span id="cb5-2"><a href="#cb5-2"></a><span class="dt"> </span><span class="kw">[ atoms ]</span></span>
<span id="cb5-3"><a href="#cb5-3"></a><span class="dt">  HH31    HC           0.11230     1</span></span>
<span id="cb5-4"><a href="#cb5-4"></a><span class="dt">   CH3    CT          -0.36620     2</span></span>
<span id="cb5-5"><a href="#cb5-5"></a><span class="dt">  HH32    HC           0.11230     3</span></span>
<span id="cb5-6"><a href="#cb5-6"></a><span class="dt">  HH33    HC           0.11230     4</span></span>
<span id="cb5-7"><a href="#cb5-7"></a><span class="dt">     C    C            0.59720     5</span></span>
<span id="cb5-8"><a href="#cb5-8"></a><span class="dt">     O    O           -0.56790     6</span></span>
<span id="cb5-9"><a href="#cb5-9"></a><span class="dt"> </span><span class="kw">[ bonds ]</span></span>
<span id="cb5-10"><a href="#cb5-10"></a><span class="dt">  HH31   CH3</span></span>
<span id="cb5-11"><a href="#cb5-11"></a><span class="dt">   CH3  HH32</span></span>
<span id="cb5-12"><a href="#cb5-12"></a><span class="dt">   CH3  HH33</span></span>
<span id="cb5-13"><a href="#cb5-13"></a><span class="dt">   CH3     C</span></span>
<span id="cb5-14"><a href="#cb5-14"></a><span class="dt">     C     O</span></span>
<span id="cb5-15"><a href="#cb5-15"></a><span class="dt"> </span><span class="kw">[ impropers ]</span></span>
<span id="cb5-16"><a href="#cb5-16"></a><span class="dt">   CH3    +N     C     O</span></span></code></pre></div><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></div>
</div>
</figure>
</div>
</section>
</section>
<section id="periodic-boundary-conditions" class="level2 page-columns page-full" data-number="2.3">
<h2 data-number="2.3" class="anchored" data-anchor-id="periodic-boundary-conditions"><span class="header-section-number">2.3</span> Periodic boundary conditions</h2>
<p>When simulating systems such as those used in biology, like a protein solvated in water, we can extend our system at little cost by applying periodic boundary conditions (PBC) (<a href="#fig-periodic" class="quarto-xref">Figure&nbsp;2.9</a>). A central simulation cell is assumed to infinitely repeat in all directions. A particle exiting the cell enters at the opposite side. This allows one to mimic a large homogeneous solvation environment of the protein, though one has to be careful that the simulation box is sufficiently large to prevent the protein from interacting with its own periodic image.</p>

<div class="no-row-height column-margin column-container"><div id="fig-periodic" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="Periodic boundary conditions.">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-periodic-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./figures/periodic-annotated.png" class="lightbox" data-gallery="quarto-lightbox-gallery-25" title="Figure&nbsp;2.9: A unit cell surrounded by periodic images."><img src="./figures/periodic-annotated.png" class="img-fluid figure-img" data-fig-scap="Periodic boundary conditions."></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-periodic-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2.9: A unit cell surrounded by periodic images.
</figcaption>
</figure>
</div></div></section>
<section id="constraint-algorithms" class="level2" data-number="2.4">
<h2 data-number="2.4" class="anchored" data-anchor-id="constraint-algorithms"><span class="header-section-number">2.4</span> Constraint algorithms</h2>
<p>The higher we can set our timestep <span class="math inline">\(\Delta t\)</span>, the longer the total time we can simulate with our computational resources. However, higher timesteps also lead to more unstable simulations. Imagine a harmonic bond in a molecule between an oxygen and a hydrogen. The harmonic potential (<a href="#eq-energy-harmonic" class="quarto-xref">Equation&nbsp;2.8</a>) has a force constant <span class="math inline">\(k\)</span> of 462750&nbsp;kJ/mol/nm<sup>2</sup>. The masses of the atoms are 1.008&nbsp;u and 16&nbsp;u where 1&nbsp;u = 1.66×10<sup>-27</sup>&nbsp;kg. Together this is a reduced mass <span class="math inline">\(\mu\)</span> of 1.574×10<sup>-27</sup>&nbsp;kg.</p>
<p>The period <span class="math inline">\(T\)</span> of the harmonic oscillator is <span class="math inline">\(T=2 \pi \sqrt{\frac{\mu}{k}} \approx 1.5\)</span>&nbsp;fs. The fastest oscillation in a system sets an upper bound for the timestep, as taking a stride in time during which a bond should complete a full oscillation naturally leads to instabilities. As such we have an interest in constraining those high-frequency bonds to allow for larger timesteps.</p>
<p>The SHAKE algorithm <span class="citation" data-cites="ryckaertNumericalIntegrationCartesian1977">[<a href="#ref-ryckaertNumericalIntegrationCartesian1977" role="doc-biblioref">45</a>]</span> applies an update to the coordinates after the unconstrained position update to fulfill a list of distance constraints by applying constraint forces. The SETTLE algorithm <span class="citation" data-cites="miyamotoSettleAnalyticalVersion1992">[<a href="#ref-miyamotoSettleAnalyticalVersion1992" role="doc-biblioref">46</a>]</span> is a dedicated implementation for rigid water molecules, which provides a significant speedup (after all, a large part of most molecular dynamics simulations is the box of water molecules). The LINCS algorithm <span class="citation" data-cites="hessLINCSLinearConstraint1997">[<a href="#ref-hessLINCSLinearConstraint1997" role="doc-biblioref">47</a>]</span> is used in practice for its significant speed. It is non-iterative and sets bond lengths to their correct lengths after the unconstrained update. This allows for timesteps upwards of 1&nbsp;fs.</p>
</section>
<section id="initial-conditions-and-ensembles" class="level2" data-number="2.5">
<h2 data-number="2.5" class="anchored" data-anchor-id="initial-conditions-and-ensembles"><span class="header-section-number">2.5</span> Initial conditions and ensembles</h2>
<p>The initial conditions for molecular dynamics simulations typically come from a structure determined experimentally through x-ray crystallography or cryo-electron microscopy (cryo-EM), though with the rise of AlphaFold <span class="citation" data-cites="jumperHighlyAccurateProtein2021">[<a href="#ref-jumperHighlyAccurateProtein2021" role="doc-biblioref">48</a>]</span>, sometimes an amino acid sequence can be a good-enough starting point for protein simulations. The initial structure is then minimized, solvated, and equilibrated, where typically restraints are employed to ease the protein into the ensemble. Here, we differentiate between the microcanonical (NVE), canonical (NVT), and isothermal-isobaric (NPT) ensemble, where N stands for the fixed number of particles, V is a fixed volume, E is a fixed total energy, T is a fixed temperature, and P is a fixed pressure. This is to make sure that the system is stable and to reduce the bias due to the initial conditions.</p>
</section>
<section id="thermostats-and-barostats" class="level2" data-number="2.6">
<h2 data-number="2.6" class="anchored" data-anchor-id="thermostats-and-barostats"><span class="header-section-number">2.6</span> Thermostats and barostats</h2>
<p>In order to achieve the various ensembles, it is necessary to control the respective constants. Keeping the number of particles the same is trivial, the other observables require a bit more math.</p>
<p>From the equipartition theorem <span class="citation" data-cites="boltzmannEinigeAllgemeineSatze1871">[<a href="#ref-boltzmannEinigeAllgemeineSatze1871" role="doc-biblioref">49</a>]</span> it follows that the temperature <span class="math inline">\(T\)</span> of an ideal gas system with <span class="math inline">\(N\)</span> particles can be described by <a href="#eq-equipartition" class="quarto-xref">Equation&nbsp;2.13</a> as a function of the individual particle velocities.</p>
<p><span id="eq-equipartition"><span class="math display">\[
T = \frac{E_{kin}}{\frac{3}{2}N k_{\mathrm{B}}}
= \frac{
\langle \sum_{i=1}^{N} \frac{1}{2} m_i v_i^2  \rangle
}{\frac{3}{2}N k_{\mathrm{B}}},
\tag{2.13}\]</span></span></p>
<p>where <span class="math inline">\(m_i\)</span> is the mass of particle <span class="math inline">\(i\)</span> and <span class="math inline">\(v_i\)</span> is its velocity. Accordingly, the temperature of a system can be controlled by adjusting the velocities of the atoms in the system. Given a virtual external heat bath of temperature <span class="math inline">\(T_0\)</span>, the Berendsen algorithm <span class="citation" data-cites="berendsenMolecularDynamicsCoupling1984">[<a href="#ref-berendsenMolecularDynamicsCoupling1984" role="doc-biblioref">50</a>]</span> couples the temperature of the system to it by <a href="#eq-berendsen" class="quarto-xref">Equation&nbsp;2.14</a>.</p>
<p><span id="eq-berendsen"><span class="math display">\[
\frac{dT}{dt} = \frac{T_0-T}{\tau}
\tag{2.14}\]</span></span></p>
<p>The time constant <span class="math inline">\(\tau\)</span> of the exponentially decaying temperature deviation adjusts the strength of the coupling. (Stochastic) velocity-rescaling <span class="citation" data-cites="bussiCanonicalSamplingVelocity2007">[<a href="#ref-bussiCanonicalSamplingVelocity2007" role="doc-biblioref">51</a>]</span> is a newer method for temperature coupling based on the Berendsen thermostat that ensures correct kinetic energy distribution. It keeps the advantages of the Berendsen thermostat, the fast first-order decay of temperature deviations and the absence of oscillations, while producing a correct canonical ensemble.</p>
<p>Likewise, the Berendsen algorithm can couple the pressure to an external “pressure bath” by scaling the volume by a factor <span class="math inline">\(\lambda\)</span> via <a href="#eq-berendsen-p" class="quarto-xref">Equation&nbsp;2.15</a>.</p>
<p><span id="eq-berendsen-p"><span class="math display">\[
\lambda = 1 - \beta \frac{\Delta t}{\tau P} \left( P_0 - P \right),
\tag{2.15}\]</span></span></p>
<p>with the isothermal compressibility <span class="math inline">\(\beta\)</span>, the simulation timestep <span class="math inline">\(\Delta t\)</span> and the reference pressure <span class="math inline">\(P_0\)</span>. The Berendsen barostat does not result in the exact NPT ensemble and does not compute the correct fluctuations in pressure or volume, but it is fast for relaxation without oscillations. As such, it is typically only used briefly at the start of a simulation during equilibration, while for longer simulations the Parrinello-Rahman barostat <span class="citation" data-cites="parrinelloPolymorphicTransitionsSingle1981">[<a href="#ref-parrinelloPolymorphicTransitionsSingle1981" role="doc-biblioref">52</a>]</span> is used.</p>
<p>It is worth noting that pressure coupling does not play well with QM/MM simulations (<a href="#sec-meth-qmmm" class="quarto-xref">Section&nbsp;2.12</a>), because the MD engine does not account for the contribution of QM/MM forces to the system virial. As such, those simulations will use the canonical ensemble.</p>
</section>
<section id="long-range-interactions" class="level2" data-number="2.7">
<h2 data-number="2.7" class="anchored" data-anchor-id="long-range-interactions"><span class="header-section-number">2.7</span> Long-range interactions</h2>
<p>Explicitly computing all non-bonded interactions between all particles becomes exceedingly costly as the number of particles increases. It gets further complicated by the use of periodic boundary conditions. Coulomb interactions decrease with a rate of <span class="math inline">\(1/r^2\)</span> (see <a href="#eq-coulomb" class="quarto-xref">Equation&nbsp;2.11</a>), while the Lennard-Jones potential decreases with <span class="math inline">\(1/r^6\)</span> (see <a href="#eq-lennard-jones" class="quarto-xref">Equation&nbsp;2.10</a>). As such, the non-bonded van der Waals interactions can readily be treated with a cutoff, whereby no forces are calculated for interactions beyond a certain distance (typically around 1&nbsp;nm), as their contribution would be negligible. The interactions for the slowly decaying electrostatic interactions, on the other hand, require more finesse. Particle Mesh Ewald (PME) <span class="citation" data-cites="dardenParticleMeshEwald1993">[<a href="#ref-dardenParticleMeshEwald1993" role="doc-biblioref">53</a>]</span> is based on Ewald summation <span class="citation" data-cites="ewaldBerechnungOptischerUnd1921">[<a href="#ref-ewaldBerechnungOptischerUnd1921" role="doc-biblioref">54</a>]</span>, but operates on a grid. Charges are assigned to the nearest grid point and forces on grid points as the result of Coulomb interactions are redistributed to the atoms around the grid point. Having a grid and periodic boundary conditions allows the PME method to solve the problem in frequency space after a Fourier transform, simplifying the computation down to a single sum over the grid. With this, PME scales with <span class="math inline">\(N log(N)\)</span>.</p>
<p>If we had to compute which particle is within the cutoff distance of which other particle for all particles at every step, we would not have gained much from using a cutoff in the first place. To alleviate this, a buffered neighbor list is used. Particles keep track of their neighbors within a region that is slightly larger than the cutoff distance (the Verlet buffer <span class="citation" data-cites="verletComputerExperimentsClassical1967">[<a href="#ref-verletComputerExperimentsClassical1967" role="doc-biblioref">55</a>]</span>). The neighbor list can then be updated less frequently than at every timestep and use clusters of particles <span class="citation" data-cites="pallFlexibleAlgorithmCalculating2013">[<a href="#ref-pallFlexibleAlgorithmCalculating2013" role="doc-biblioref">56</a>]</span>.</p>
</section>
<section id="external-biasing-potentials" class="level2" data-number="2.8">
<h2 data-number="2.8" class="anchored" data-anchor-id="external-biasing-potentials"><span class="header-section-number">2.8</span> External biasing potentials</h2>
<p>Since we are investigating the influence of external forces on collagen, we also need a way of adding those in our molecular dynamics simulations. Force-probe MD simulations allow biasing the energy of a system by external forces <span class="citation" data-cites="GROMACS20253Documentation">[<a href="#ref-GROMACS20253Documentation" role="doc-biblioref">57</a>]</span>. The force can be added to individual atoms or the center of mass (COM) of groups of atoms. To this end, a virtual spring is attached and moved in the direction of a defined pulling direction. This can either be defined by absolute coordinates or through the distance between pull groups. The spring can either add a constant force (constant force pulling), increase the force over time (force-ramp) or the virtual attachment point of the spring itself can travel at a constant speed along the pulling coordinate (constant velocity pulling). In the latter case the force can vary over time and will depend on the pulling speed and the force constant <span class="math inline">\(k\)</span> of the harmonic potential of the virtual spring.</p>
</section>
<section id="sec-meth-us" class="level2 page-columns page-full" data-number="2.9">
<h2 data-number="2.9" class="anchored" data-anchor-id="sec-meth-us"><span class="header-section-number">2.9</span> Umbrella sampling and the weighted histogram analysis method</h2>
<p>By using a harmonic potential, sometimes also called umbrella potential, to trace out a reaction coordinate, we get an initial trajectory. From this initial trajectory, snapshots (umbrella windows) are extracted along the reaction coordinate. The reaction coordinate is a collective variable that can be made up from a combination of measurements such as positions, distances, and angles. From those snapshots, more simulations are started, with the umbrella potential this time applied to keep the collective variable around the same value as initially in the snapshot. Together, these simulations can be used to estimate the free energy profile along the reaction coordinate. This free energy profile is sometimes also called the potential of mean force (PMF), although this is not technically correct due to the inclusion of entropic effects in the sampling <span class="citation" data-cites="grossfieldWHAMWeightedHistogram">[<a href="#ref-grossfieldWHAMWeightedHistogram" role="doc-biblioref">58</a>]</span>.</p>

<div class="no-row-height column-margin column-container"><div id="fig-squirrel" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="Squirrel holding upside-down umbrella.">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-squirrel-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="img/umbrella-squirrel.png" class="lightbox" data-gallery="quarto-lightbox-gallery-26" title="Figure&nbsp;2.10: Because the harmonic potential holding each umbrella window in place along the reaction coordinate \xi is often also called an umbrella potential, I was initially quite confused: “That’s not how you hold an umbrella!” It starts to make more sense when referring to the histograms of the umbrella windows (lots of small umbrellas, Figure&nbsp;3.9 (d)), but I nonetheless made this squirrel holding an upside-down umbrella for a seminar using a combination of manual painting and StableDiffusion [@rombachHighResolutionImageSynthesis2022] running locally via InvokeAI [@InvokeaiInvokeAI2025]."><img src="img/umbrella-squirrel.png" class="img-fluid figure-img" data-fig-scap="Squirrel holding upside-down umbrella."></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-squirrel-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2.10: Because the harmonic potential holding each umbrella window in place along the reaction coordinate <span class="math inline">\(\xi\)</span> is often also called an umbrella potential, I was initially quite confused: “That’s not how you hold an umbrella!” It starts to make more sense when referring to the histograms of the umbrella windows (lots of small umbrellas, <a href="#fig-hist-us" class="quarto-xref">Figure&nbsp;3.9 (d)</a>), but I nonetheless made this squirrel holding an upside-down umbrella for a seminar using a combination of manual painting and StableDiffusion <span class="citation" data-cites="rombachHighResolutionImageSynthesis2022">[<a href="#ref-rombachHighResolutionImageSynthesis2022" role="doc-biblioref">59</a>]</span> running locally via InvokeAI <span class="citation" data-cites="InvokeaiInvokeAI2025">[<a href="#ref-InvokeaiInvokeAI2025" role="doc-biblioref">60</a>]</span>.
</figcaption>
</figure>
</div></div><p>The weighted histogram analysis method (WHAM) <span class="citation" data-cites="kumarWeightedHistogramAnalysis1992 souailleExtensionWeightedHistogram2001 hubG_whamAFreeWeighted2010">[<a href="#ref-kumarWeightedHistogramAnalysis1992" role="doc-biblioref">61</a>–<a href="#ref-hubG_whamAFreeWeighted2010" role="doc-biblioref">63</a>]</span> is then used to analyze the resulting forces and collective variable values. With a reaction coordinate <span class="math inline">\(\xi\)</span> the goal is to find the free energy profile <span class="math inline">\(A(\xi)\)</span>. Given the probability <span class="math inline">\(P(\xi)\)</span> for visiting a point in phase space along the reaction coordinate, the free energy is given by <a href="#eq-energy-profile" class="quarto-xref">Equation&nbsp;2.16</a>.</p>
<p><span id="eq-energy-profile"><span class="math display">\[
A(\xi) = -k_{\mathrm{B}} T \ln P(\xi),
\tag{2.16}\]</span></span></p>
<p>with the temperature <span class="math inline">\(T\)</span> and the Boltzmann constant <span class="math inline">\(k_\mathrm{B}\)</span>. In the limit of infinite sampling we could just run our simulations forever to obtain the true probability distribution <span class="math inline">\(P\)</span>, but with the umbrella windows along the reaction coordinate we can speed things up. Each umbrella window <span class="math inline">\(i\)</span> from 1 to <span class="math inline">\(n\)</span> produces a biased probability distribution <span class="math inline">\(P_i^{bias}(\xi)\)</span> with the biasing umbrella potential <span class="math inline">\(V_i(\xi)\)</span> and the free-energy contribution from the biasing potential <span class="math inline">\(F_i\)</span>. Together the free energy of one window is described in terms of the biased distributions by <a href="#eq-energy-profile-from-bias" class="quarto-xref">Equation&nbsp;2.17</a>.</p>
<p><span id="eq-energy-profile-from-bias"><span class="math display">\[
A_i(\xi) = −k_{\mathrm{B}} T \ln P_i^\mathrm{bias}(\xi) - V_i(\xi) + F_i
\tag{2.17}\]</span></span></p>
<p>Next, the weights <span class="math inline">\(w_i\)</span> of <a href="#eq-wham" class="quarto-xref">Equation&nbsp;2.18</a> are adjusted to minimize statistical error.</p>
<p><span id="eq-wham"><span class="math display">\[
P(\xi) = \sum_{i=1}^{n} w_i P_i^{\mathrm{bias}}(\xi)
\tag{2.18}\]</span></span></p>
<p>This makes it another self-consistent method, aided by the fact the umbrella windows, or rather their histograms along the reaction coordinate, <span class="math inline">\(\xi\)</span> overlap. The more the umbrella windows agree on the resulting unbiased probability distribution in a region <span class="math inline">\(\Delta\xi\)</span>, the higher the convergence. The result is a free energy profile and the histograms of the umbrella windows, which can be used to visualize overlap. Bootstrapping allows for an estimate of the uncertainty of the free energy profile by comparing the predictions with varying holdouts <span class="citation" data-cites="hubG_whamAFreeWeighted2010">[<a href="#ref-hubG_whamAFreeWeighted2010" role="doc-biblioref">63</a>]</span>.</p>
</section>
<section id="free-energy-perturbation-by-slow-growth" class="level2" data-number="2.10">
<h2 data-number="2.10" class="anchored" data-anchor-id="free-energy-perturbation-by-slow-growth"><span class="header-section-number">2.10</span> Free energy perturbation by slow-growth</h2>
<p>Slow-growth is another method of calculating free energy differences, although you will find that in this thesis it is used for a different purpose. The underlying idea is that the Hamiltonian is being made dependent on an additional term <span class="math inline">\(\lambda\)</span> next to the positions <span class="math inline">\(\vec{r}\)</span> and momenta <span class="math inline">\(\vec{p}\)</span> (<a href="#eq-lambda" class="quarto-xref">Equation&nbsp;2.19</a>):</p>
<p><span id="eq-lambda"><span class="math display">\[
\lambda: H=H(\vec{r},\vec{p},\lambda),
\tag{2.19}\]</span></span></p>
<p>where the coupling parameter <span class="math inline">\(\lambda\)</span> is used to transition from the Hamiltonian in state A at <span class="math inline">\(\lambda=0\)</span> to the Hamiltonian in state B at <span class="math inline">\(\lambda=1\)</span> in <a href="#eq-ab" class="quarto-xref">Equation&nbsp;2.20</a>.</p>
<p><span id="eq-ab"><span class="math display">\[
\begin{array}{rcl}
H(\vec{r},\vec{p},0) &amp;=&amp; H{^{\mathrm{A}}}(\vec{r},\vec{p}) \\
H(\vec{r},\vec{p},1) &amp;=&amp; H{^{\mathrm{B}}}(\vec{r},\vec{p})
\end{array}
\tag{2.20}\]</span></span></p>
<p>The computational engine provides a way of specifying different parameters for state A and state B for the energy terms. For example, a harmonic bond may start with a spring constant <span class="math inline">\(k_\mathrm{A}\)</span> and transition to a spring constant <span class="math inline">\(k_{\mathrm{B}}\)</span> <a href="#fn2" class="footnote-ref" id="fnref2" role="doc-noteref"><sup>2</sup></a>.</p>
</section>
<section id="quantum-mechanics-and-density-functional-theory" class="level2" data-number="2.11">
<h2 data-number="2.11" class="anchored" data-anchor-id="quantum-mechanics-and-density-functional-theory"><span class="header-section-number">2.11</span> Quantum mechanics and density functional theory</h2>
<p>A key limitation of MM is that bonds can not form or break, i.e., chemistry can not happen, because all connectivity and parameters thereof have to be defined ahead of the simulation. So, let us take a look at a more detailed but also computationally expensive technique. In Quantum mechanical (QM) simulations, we are no longer treating atoms as indivisible units and instead also simulate the electrons around the nuclei to some extent. To what extent depends on the concrete method and implementation. We will be looking at density functional theory (DFT) in particular, but first we need to lay some groundwork. The overarching idea of MD remains, but the form of our energy function will change quite a bit.</p>
<section id="the-schrödinger-equation" class="level3" data-number="2.11.1">
<h3 data-number="2.11.1" class="anchored" data-anchor-id="the-schrödinger-equation"><span class="header-section-number">2.11.1</span> The Schrödinger equation</h3>
<p>The general equation for the energy of a molecule (or system) is the Schrödinger equation (<a href="#eq-schroedinger" class="quarto-xref">Equation&nbsp;2.21</a>) <span class="citation" data-cites="schrodingerUndulatoryTheoryMechanics1926">[<a href="#ref-schrodingerUndulatoryTheoryMechanics1926" role="doc-biblioref">64</a>]</span>.</p>
<p><span id="eq-schroedinger"><span class="math display">\[
\hat{H}\Psi=E\Psi
\tag{2.21}\]</span></span></p>
<p>This is an eigenvalue equation where the eigenvalue of the Hamilton operator <span class="math inline">\(\hat{H}\)</span> acting on the wavefunction <span class="math inline">\(\Psi\)</span> (a set of eigenfunctions of the Hamiltonian) is the total energy <span class="math inline">\(E\)</span>. For each eigenfunction <span class="math inline">\(\Psi_n\)</span> we have an associated energy <span class="math inline">\(E_n\)</span>. The wavefunction describes the matter wave of each particle, where, according to de Broglie, the wavelength <span class="math inline">\(\lambda\)</span> of the wave is the Planck constant <span class="math inline">\(h\)</span> divided by the momentum <span class="math inline">\(p\)</span> (<a href="#eq-debroglie" class="quarto-xref">Equation&nbsp;2.22</a>) <span class="citation" data-cites="debroglieRecherchesTheorieQuanta1924">[<a href="#ref-debroglieRecherchesTheorieQuanta1924" role="doc-biblioref">65</a>]</span>.</p>
<p><span id="eq-debroglie"><span class="math display">\[
\lambda = {\frac{h}{p}}
\tag{2.22}\]</span></span></p>
<p>Furthermore, the Born rule <span class="citation" data-cites="bornZurQuantenmechanikStossvorgaenge1926">[<a href="#ref-bornZurQuantenmechanikStossvorgaenge1926" role="doc-biblioref">66</a>]</span> tells us that the wavefunction times its own complex conjugate results in a probability density. So the probability of finding a particle in a volume slice <span class="math inline">\(dr\)</span> is equal to the integral over <span class="math inline">\(dr\)</span>, normalized such that <a href="#eq-born-rule" class="quarto-xref">Equation&nbsp;2.23</a> holds:</p>
<p><span id="eq-born-rule"><span class="math display">\[
\int_{-\infty}^{\infty}{ \Psi^*(r)\Psi(r) dr}=P=1
\tag{2.23}\]</span></span></p>
<p>The Schrödinger equation is still incredibly hard to solve for, but we can get a step closer by invoking the Born-Oppenheimer approximation <span class="citation" data-cites="bornZurQuantentheorieMolekeln1927">[<a href="#ref-bornZurQuantentheorieMolekeln1927" role="doc-biblioref">67</a>]</span>: Due to the electrons being so much lighter and faster than the nuclei, we can solve for both systems independently of each other. As far as the electrons are concerned, the nuclei are pretty much stationary.</p>
</section>
<section id="the-electronic-schrödinger-equation" class="level3" data-number="2.11.2">
<h3 data-number="2.11.2" class="anchored" data-anchor-id="the-electronic-schrödinger-equation"><span class="header-section-number">2.11.2</span> The electronic Schrödinger equation</h3>
<p>This leads us to the electronic Schrödinger equation (<a href="#eq-el-schroedinger-small" class="quarto-xref">Equation&nbsp;2.24</a>):</p>
<p><span id="eq-el-schroedinger-small"><span class="math display">\[
\hat{H}_{el}(\mathbf {r} ,\mathbf {R} )\Psi_{el} (\mathbf {r} ,\mathbf {R} )=E_{el}\Psi_{el} (\mathbf {r} ,\mathbf {R} )
\tag{2.24}\]</span></span></p>
<p>In the following, <span class="math inline">\(\Psi\)</span> will refer to <span class="math inline">\(\Psi_{el}\)</span> and we can expand the equation to <a href="#eq-el-schroedinger" class="quarto-xref">Equation&nbsp;2.25</a></p>
<p><span id="eq-el-schroedinger"><span class="math display">\[
\begin{array}{rcl}
\hat {H}\Psi
&amp;=&amp; E\Psi \\
&amp;=&amp; \left[{\hat {T}}+{\hat {V}}+{\hat {U}}\right]\Psi \\
&amp;=&amp; \left[\sum _{i=1}^{N}\left(-{\frac {\hbar ^{2}}{2m_{i}}}\nabla_{i}^{2}\right)+\sum_{i=1}^{N}V(\mathbf{r} _{i})+\sum _{i&lt;j}^{N}U\left(\mathbf {r} _{i},\mathbf {r} _{j}\right)\right]\Psi,
\end{array}
\tag{2.25}\]</span></span></p>
<p>where <span class="math inline">\(T\)</span> is the kinetic energy of the electrons (or respectively <span class="math inline">\(\hat{T}\)</span> is the kinetic energy operator), <span class="math inline">\(V\)</span> is the potential energy of the electrons due to the interaction with the (now assumed static) nuclei and <span class="math inline">\(U\)</span> is the potential energy of electron-electron repulsion. <span class="math inline">\(N\)</span> is the number of electrons.</p>
<p>This equation, although looking deceptively simple, is still near impossible to solve for any real-world system. The next step is to find easier-to-solve-for formulations of the wavefunction.</p>
</section>
<section id="hartree-product" class="level3" data-number="2.11.3">
<h3 data-number="2.11.3" class="anchored" data-anchor-id="hartree-product"><span class="header-section-number">2.11.3</span> Hartree product</h3>
<p>The Hartree product <span class="citation" data-cites="hartreeCalculationAtomicStructures1957">[<a href="#ref-hartreeCalculationAtomicStructures1957" role="doc-biblioref">68</a>]</span> describes the electronic wavefunction as a product of individual electron wavefunctions (<a href="#eq-hartree-product" class="quarto-xref">Equation&nbsp;2.26</a>):</p>
<p><span id="eq-hartree-product"><span class="math display">\[
\Psi ({x}_{1}, {x}_{2}, {x}_{3},\ldots,{x}_{n})=\psi_1( {x}_{1})\psi_2({x} _{2})\psi_3( {x} _{3})\ldots\psi_n( {x} _{n}),
\tag{2.26}\]</span></span></p>
<p>where <span class="math inline">\(x\)</span> is the state of an electron given by its position <span class="math inline">\(r\)</span> and spin <span class="math inline">\(\omega\)</span>. This is not quite sufficient, because, as fermions, electrons are meant to be indistinguishable, and their wavefunction is meant to be antisymmetric, meaning it should change sign upon exchange of any two coordinates. We can enforce this by writing the wavefunction a bit differently. For a two-particle system this looks like <a href="#eq-two-particle-slater" class="quarto-xref">Equation&nbsp;2.27</a>:</p>
<p><span id="eq-two-particle-slater"><span class="math display">\[
\Psi(x_1,x_2) = \frac{1}{\sqrt{2}}\left[ \psi_1(x_1) \psi_2(x_2) - \psi_1(x_2) \psi_2(x_1) \right],
\tag{2.27}\]</span></span></p>
<p>which satisfies antisymmetry (<a href="#eq-antisym" class="quarto-xref">Equation&nbsp;2.28</a>).</p>
<p><span id="eq-antisym"><span class="math display">\[
\Psi(x_1,x_2)=-\Psi(x_2,x_1)
\tag{2.28}\]</span></span></p>
</section>
<section id="slater-determinant" class="level3" data-number="2.11.4">
<h3 data-number="2.11.4" class="anchored" data-anchor-id="slater-determinant"><span class="header-section-number">2.11.4</span> Slater determinant</h3>
<p>This generalizes to more particles in the form of a determinant, the Slater determinant <span class="citation" data-cites="slaterTheoryComplexSpectra1929">[<a href="#ref-slaterTheoryComplexSpectra1929" role="doc-biblioref">69</a>]</span> (<a href="#eq-slater" class="quarto-xref">Equation&nbsp;2.29</a>):</p>
<p><span id="eq-slater"><span class="math display">\[
\Psi(x_1,x_2){\frac {1}{\sqrt{2}}}=
\begin{vmatrix}
\psi_{1}(\mathbf{x} _{1}) &amp; \psi_{2}(\mathbf{x} _{1}) \\
\psi_{1}(\mathbf{x} _{2}) &amp; \psi _{2}(\mathbf {x} _{2})
\end{vmatrix},
\tag{2.29}\]</span></span></p>
<p>where taking a 2-electron system to an <span class="math inline">\(N\)</span>-electron system means replacing the <span class="math inline">\(2 \times 2\)</span> matrix with an <span class="math inline">\(N \times N\)</span> matrix and <span class="math inline">\(2\)</span> turning into <span class="math inline">\(N!\)</span>. We now have a way of constructing wavefunctions or molecular orbitals (MOs), in which electrons can live. But how do we find the correct functions?</p>
</section>
<section id="the-hartree-fock-method" class="level3" data-number="2.11.5">
<h3 data-number="2.11.5" class="anchored" data-anchor-id="the-hartree-fock-method"><span class="header-section-number">2.11.5</span> The Hartree-Fock method</h3>
<p>The Hartree-Fock method <span class="citation" data-cites="fockNaeherungsmethodeZurLoesung1930">[<a href="#ref-fockNaeherungsmethodeZurLoesung1930" role="doc-biblioref">70</a>]</span> now assumes that the electronic energy eigenfunctions can be described as a single Slater determinant made up of individual one-electron wavefunctions. Each such one-electron wavefunction (a molecular orbital) is built up from a finite set of basis functions and is thus a linear combination of atomic orbitals (LCAO). The orbitals are then varied such that the Hartree-Fock energy is minimized under the assumption that each electron only interacts with the average charge density of all other electrons (variational method and mean-field approximation). This is also called the self-consistent field method.</p>
</section>
<section id="hohenberg-kohn-theorems" class="level3" data-number="2.11.6">
<h3 data-number="2.11.6" class="anchored" data-anchor-id="hohenberg-kohn-theorems"><span class="header-section-number">2.11.6</span> Hohenberg-Kohn theorems</h3>
<p>With the Hohenberg-Kohn theorems <span class="citation" data-cites="hohenbergInhomogeneousElectronGas1964">[<a href="#ref-hohenbergInhomogeneousElectronGas1964" role="doc-biblioref">71</a>]</span>:</p>
<ol type="1">
<li><p>The ground state energy is a unique functional of the electron density:<br>
<span class="math inline">\(E=E_0[\eta_0(r)]\)</span></p></li>
<li><p>The electron density function that minimizes the energy is the correct electron density:<br>
<span class="math inline">\(\forall \eta : E[\eta(r)]\ge E_0[\eta_0(r)]\)</span>,</p></li>
</ol>
<p>we can apply the basic principles to larger systems, because we now “only” have to compute the electron density function as opposed to all individual electron wavefunctions.</p>
</section>
<section id="kohn-sham-density-functional-theory" class="level3" data-number="2.11.7">
<h3 data-number="2.11.7" class="anchored" data-anchor-id="kohn-sham-density-functional-theory"><span class="header-section-number">2.11.7</span> Kohn-Sham density functional theory</h3>
<p>In Kohn-Sham Density Functional Theory (KS DFT) <span class="citation" data-cites="kohnSelfConsistentEquationsIncluding1965">[<a href="#ref-kohnSelfConsistentEquationsIncluding1965" role="doc-biblioref">72</a>]</span> we compute the energy of electrons by assuming that the density consists of a single Slater determinant (non-interacting limit) (<a href="#eq-ks-dft" class="quarto-xref">Equation&nbsp;2.30</a>), which we can then treat like in the Hartree-Fock method (i.e., with a variational, self-consistent field approach).</p>
<p><span id="eq-ks-dft"><span class="math display">\[
E[\rho ]=T[\rho ]+\int d\mathbf {r} \,v_{\text{ext}}(\mathbf {r} )\rho (\mathbf {r} )+E_{\text{coulomb}}[\rho ]+E_{\text{xc}}[\rho],
\tag{2.30}\]</span></span></p>
<p>with electron density <span class="math inline">\(\rho\)</span>, kinetic energy <span class="math inline">\(T\)</span>, external potential <span class="math inline">\(v\)</span>, position in space <span class="math inline">\(\mathbf{r}\)</span>, and the errors from electron-electron exchange and correlation lumped into an exchange correlation energy <span class="math inline">\(E_{\text{xc}}\)</span>. The type of exchange correlation functional (XC) used has a large impact on computational cost and accuracy of the calculations. XC functionals can be ordered in terms of their accuracy. The local density approximation (LDA), where the functional only depends on the local electron density at a given point, is exceeded by the generalized gradient approximation (GGA), where the functional also depends on the local gradient of the density (e.g., LYP after Lee, Yang, and Parr <span class="citation" data-cites="leeDevelopmentColleSalvettiCorrelationenergy1988">[<a href="#ref-leeDevelopmentColleSalvettiCorrelationenergy1988" role="doc-biblioref">73</a>]</span>). Adding a second derivative leads to meta-GGA and hybrid DFT mixes in exact exchange (e.g., B3LYP after Becke 3-parameter and LYP <span class="citation" data-cites="beckeDensityfunctionalThermochemistryIII1993 leeDevelopmentColleSalvettiCorrelationenergy1988">[<a href="#ref-leeDevelopmentColleSalvettiCorrelationenergy1988" role="doc-biblioref">73</a>, <a href="#ref-beckeDensityfunctionalThermochemistryIII1993" role="doc-biblioref">74</a>]</span> or PBE0 after Perdew–Burke–Ernzerhof <span class="citation" data-cites="perdewRationaleMixingExact1996 adamoReliableDensityFunctional1999">[<a href="#ref-perdewRationaleMixingExact1996" role="doc-biblioref">75</a>, <a href="#ref-adamoReliableDensityFunctional1999" role="doc-biblioref">76</a>]</span>). Lastly, double-hybrid methods such as MP2 <span class="citation" data-cites="head-gordonMP2EnergyEvaluation1988">[<a href="#ref-head-gordonMP2EnergyEvaluation1988" role="doc-biblioref">77</a>]</span> add Møller–Plesset perturbation theory <span class="citation" data-cites="mollerNoteApproximationTreatment1934">[<a href="#ref-mollerNoteApproximationTreatment1934" role="doc-biblioref">78</a>]</span> electron correlation as well.</p>
</section>
<section id="basis-sets" class="level3" data-number="2.11.8">
<h3 data-number="2.11.8" class="anchored" data-anchor-id="basis-sets"><span class="header-section-number">2.11.8</span> Basis sets</h3>
<p>A collection of basis functions that we can use in DFT is called a basis set. A linear combination of basis functions makes up a molecular orbital (MO) for one electron, which is then combined with other MOs in one Slater determinant for the whole system. For the basis functions, we have the choice between Slater-type orbitals (STO), which are the typical atomic orbitals <span class="math inline">\(Nx^a y^b z^c e^{-\zeta r}\)</span> (where <span class="math inline">\(N\)</span> is a normalization constant, <span class="math inline">\(a,b,c\)</span> control the angular momentum, and <span class="math inline">\(\zeta\)</span> controls the width) and Gaussian-type orbitals (GTO) <span class="math inline">\(Nx^a y^b z^c e^{-\zeta r^2}\)</span>. Gaussians have the added benefit that the product of two gaussians is another Gaussian, which makes them mathematically favored to work with, even though they are less physical. Contracting multiple GTOs via a linear combination (CGTO) is used to mimic a single STO at lower computational cost.</p>
<p>A minimal basis set encompasses one basis function for each atomic orbital (AO). Using two basis functions per AO is called double-zeta (and so forth). This allows one to more accurately capture interactions between atoms. Doing so only for the valence AOs and keeping the core AOs at single-zeta is called a split valence basis set. Allowing AOs to be polarized by interactions with other atoms requires mixing each AO with an orbital of a higher angular momentum (l+1), e.g., mixing s orbitals with p orbitals. This is then called a polarized basis set.</p>
</section>
</section>
<section id="sec-meth-qmmm" class="level2" data-number="2.12">
<h2 data-number="2.12" class="anchored" data-anchor-id="sec-meth-qmmm"><span class="header-section-number">2.12</span> Quantum mechanics/molecular mechanics</h2>
<p>While it is feasible to simulate millions of atoms with molecular mechanics, DFT can only reach orders of thousands of atoms at great computational cost. We need a method that not only can simulate a large enough portion of a collagen fibril with sufficient sampling time but can also model the chemistry of hydrolysis. Quantum mechanics/molecular mechanics (QM/MM) combines both methods. A large system is simulated by molecular mechanics, while a small subsystem is treated quantum mechanically.</p>
<p>There are different ways of combining the two systems. The method used in this thesis is called Gaussian expansion of the electrostatic potential (GEEP) <span class="citation" data-cites="lainoEfficientRealSpace2005">[<a href="#ref-lainoEfficientRealSpace2005" role="doc-biblioref">79</a>]</span> and is a form of electrostatic embedding. While in a purely mechanical embedding, the electrostatic interactions between MM and QM atoms would only be treated by molecular mechanics terms, in an electrostatic (or electronic) embedding, the QM subsystem also sees the charges of the MM atoms and can be polarized by those. One step up from that would be a polarized embedding, where the MM subsystem can also be polarized by the QM region. The QM/MM simulations in this thesis use the electrostatic embedding (<a href="#fig-qmmm" class="quarto-xref">Figure&nbsp;2.11</a>).</p>
<div id="fig-qmmm" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="QM/MM electrostatic embedding.">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-qmmm-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="img/electrostatic-embedding.png" class="lightbox" data-gallery="quarto-lightbox-gallery-27" title="Figure&nbsp;2.11: Electrostatic embedding scheme for QM/MM. Figure by Dmitry Morozov [@morozovWebinarMultiscaleQM2020]."><img src="img/electrostatic-embedding.png" class="img-fluid figure-img" data-fig-scap="QM/MM electrostatic embedding."></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-qmmm-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2.11: Electrostatic embedding scheme for QM/MM. Figure by Dmitry Morozov <span class="citation" data-cites="morozovWebinarMultiscaleQM2020">[<a href="#ref-morozovWebinarMultiscaleQM2020" role="doc-biblioref">80</a>]</span>.
</figcaption>
</figure>
</div>
<p>The new Hamiltonian of the QM subsystem consists of the original Hamiltonian with the addition of the electrostatic interactions between the electrons of the QM region and the MM atoms and between the nuclei of the QM region and the MM atoms (<a href="#eq-qmmm" class="quarto-xref">Equation&nbsp;2.31</a>, from the GROMACS documentation <span class="citation" data-cites="HybridQuantumClassicalSimulations">[<a href="#ref-HybridQuantumClassicalSimulations" role="doc-biblioref">81</a>]</span>):</p>
<p><span id="eq-qmmm"><span class="math display">\[
H^{QM/MM} =
H^{QM}_{el}-\sum_i^n\sum_J^M\frac{e^2 Q_J}{4\pi\epsilon_0r_{iJ}}+\sum_A^N\sum_J^M\frac{e^2 Z_A Q_J}{e\pi\epsilon_0 R_{AJ}},
\tag{2.31}\]</span></span></p>
<p>with the number of electrons <span class="math inline">\(n\)</span> and nuclei <span class="math inline">\(N\)</span> in the QM region and the number of charged MM atoms <span class="math inline">\(M\)</span>. Here, charged also includes partial charges, not just ions.</p>
<p><span class="math inline">\(H^{QM}_{el}\)</span> is the original electronic Hamiltonian of an isolated QM system. The first double sum (n,M) is the electrostatic interaction between the QM electrons and the MM atoms. The second double sum (N,M) is the electrostatic interaction of the QM nuclei with the MM atoms. Lennard-Jones and Coulomb interactions between QM atoms are excluded in the MM Hamiltonian to avoid double counting, but Lennard-Jones interactions between QM and MM atoms are calculated. Bonded interactions at the QM-MM interface that contain at most one QM atom are described by MM. Bonds crossing the QM-MM interface are capped with a so-called link atom, a hydrogen which only exists in the QM region. The force on this atom is distributed over the two atoms of the bond.</p>
</section>
<section id="computational-molecular-dynamics-engines" class="level2" data-number="2.13">
<h2 data-number="2.13" class="anchored" data-anchor-id="computational-molecular-dynamics-engines"><span class="header-section-number">2.13</span> Computational molecular dynamics engines</h2>
<p>The molecular dynamics engine of choice for this thesis is GROMACS <span class="citation" data-cites="abrahamGROMACSHighPerformance2015">[<a href="#ref-abrahamGROMACSHighPerformance2015" role="doc-biblioref">40</a>]</span> with documentation available at <a href="https://manual.gromacs.org/">manual.gromacs.org</a> <span class="citation" data-cites="GROMACS20253Documentation">[<a href="#ref-GROMACS20253Documentation" role="doc-biblioref">57</a>]</span>. Our newly developed software method KIMMDY was tested with GROMACS versions ranging from v2021 to v2025.3. Early QM/MM simulations to test feasibility used the developer build of GROMACS v2022. The production QM/MM simulations in this thesis use GROMACS v2023.2. The QM engine used and interfaced with GROMACS is CP2K <span class="citation" data-cites="kuhneCP2KElectronicStructure2020">[<a href="#ref-kuhneCP2KElectronicStructure2020" role="doc-biblioref">82</a>]</span> v2023.2 available from <a href="https://www.cp2k.org/">cp2k.org</a>.</p>
</section>
<section id="sec-kmc" class="level2" data-number="2.14">
<h2 data-number="2.14" class="anchored" data-anchor-id="sec-kmc"><span class="header-section-number">2.14</span> Kinetic Monte Carlo</h2>
<p>Given that our method KIMMDY that will be introduced in <a href="#sec-main-kimmdy" class="quarto-xref">Chapter&nbsp;4</a>, stands for Kinetic Monte Carlo Molecular Dynamics, it makes sense to also explain what kinetic Monte Carlo (kMC) is. Monte Carlo methods in general are methods that use random sampling to solve a numerical problem. In kinetic Monte Carlo, we are modelling the time progression through a collection of states connected by processes with known transition rates. The simplest example is rejection-free kMC (rf-kMC) <span class="citation" data-cites="bortzNewAlgorithmMonte1975 gillespieGeneralMethodNumerically1976">[<a href="#ref-bortzNewAlgorithmMonte1975" role="doc-biblioref">83</a>, <a href="#ref-gillespieGeneralMethodNumerically1976" role="doc-biblioref">84</a>]</span>:</p>
<p>We start at time <span class="math inline">\(t=0\)</span> in a state <span class="math inline">\(S_k\)</span> and collect the rates <span class="math inline">\(r_{ki}\)</span> for all processes leading to new states <span class="math inline">\(S_l\)</span>. The list of rates has length <span class="math inline">\(N_k\)</span> and is indexed by <span class="math inline">\(i\)</span>. Stack the rates on top of each other and normalize to a total rate of 1, which results in the normalized cumulative function <span class="math inline">\(R^{\prime}_k(i)\)</span> in <a href="#eq-kmc-cum" class="quarto-xref">Equation&nbsp;2.32</a> with a rate sum of <span class="math inline">\(Q_k\)</span> (<a href="#fig-kmc-bar" class="quarto-xref">Figure&nbsp;2.12</a>):</p>
<p><span id="eq-kmc-cum"><span class="math display">\[
\begin{split}
R_k(i) &amp;= \sum_{j=1}^{i}r_{ki} \\
Q_k &amp;= R_k(N_k) \\
R^{\prime}_k(i) &amp;= \frac{R_k(i)}{Q_k}
\end{split}
\tag{2.32}\]</span></span></p>
<div id="fig-kmc-bar" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-kmc-bar-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./plots/plots/kmc-stacked-bar.png" class="lightbox" data-gallery="quarto-lightbox-gallery-28" title="Figure&nbsp;2.12: Stacked rates to sample from during kMC."><img src="./plots/plots/kmc-stacked-bar.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-kmc-bar-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2.12: Stacked rates to sample from during kMC.
</figcaption>
</figure>
</div>
<p>Then, draw a random number <span class="math inline">\(u\in (0,1]\)</span> from a uniform distribution. From the normalized cumulative function, find the event <span class="math inline">\(i\)</span> for which <span class="math inline">\(R^\prime_{k}(i-1)&lt;u\leq R^\prime_{k}(i)\)</span>. Execute the process tied to event <span class="math inline">\(i\)</span>. Draw a second uniform random number <span class="math inline">\(u^\prime \in (0,1]\)</span>. Determine <span class="math inline">\(\Delta t=\frac{\ln(1/u^\prime)}{Q_k}\)</span> and set the new time <span class="math inline">\(t=t+\Delta t\)</span>. Repeat all steps until sufficient sampling of the states has been achieved.</p>
<p>The event list of a state and the list of possible states is not always known beforehand. Adaptive kMC <span class="citation" data-cites="henkelmanLongTimeScale2001">[<a href="#ref-henkelmanLongTimeScale2001" role="doc-biblioref">85</a>]</span> characterizes a method in which the event list is calculated on the fly.</p>
</section>
<section id="sec-arrhenius" class="level2" data-number="2.15">
<h2 data-number="2.15" class="anchored" data-anchor-id="sec-arrhenius"><span class="header-section-number">2.15</span> Arrhenius equation and transition state theory</h2>
<p>In order to apply kinetic Monte Carlo, we need reaction rates.</p>
<p>The Arrhenius equation <a href="#eq-arrhenius" class="quarto-xref">Equation&nbsp;2.33</a> describes an empirical relationship between the reaction rate <span class="math inline">\(k\)</span> and the activation energy <span class="math inline">\(E_\mathrm{a}\)</span> of a reaction <span class="citation" data-cites="arrheniusUberReaktionsgeschwindigkeitBei1889">[<a href="#ref-arrheniusUberReaktionsgeschwindigkeitBei1889" role="doc-biblioref">86</a>]</span>.</p>
<p><span id="eq-arrhenius"><span class="math display">\[
k = A\,\mathrm{e}^{\left(\frac{-E_\mathrm{a}}{RT}\right)},
\tag{2.33}\]</span></span></p>
<p>with the pre-exponential factor <span class="math inline">\(A\)</span> and the gas constant <span class="math inline">\(R\)</span> as well as the temperature <span class="math inline">\(T\)</span>. If <span class="math inline">\(E_\mathrm{a}\)</span> is given per molecule and not per mole, <span class="math inline">\(R\)</span> is replaced by the Boltzmann constant <span class="math inline">\(k_\mathrm{B}\)</span>.</p>
<p>The Eyring equation <span class="citation" data-cites="eyringActivatedComplexChemical1935">[<a href="#ref-eyringActivatedComplexChemical1935" role="doc-biblioref">87</a>]</span> contains a way of calculating the pre-exponential factor, or attempt frequency via <a href="#eq-eyring" class="quarto-xref">Equation&nbsp;2.34</a>.</p>
<p><span id="eq-eyring"><span class="math display">\[
A = \kappa \frac{k_{\mathrm{B}}T}{h},
\tag{2.34}\]</span></span></p>
<p>with the transmission coefficient <span class="math inline">\(\kappa\)</span>, which represents the fraction of reaction paths through the transition state that end in the product state without passing the transition state again. A transition state is a saddle point in the potential energy surface between the educt and product state. <span class="math inline">\(\kappa\)</span> is often assumed to be 1, which equals accepting the fundamental assumption of transition state theory, in which the educt state and the transition state are in equilibrium, but once the transition state has been passed the reaction proceeds to the product state.</p>
<p>When experimental data for reaction rates at different temperatures is available, <span class="math inline">\(A\)</span> can be determined for the specific reaction based on an Arrhenius plot. Here, the logarithm of the reaction rate (<span class="math inline">\(\ln(k)\)</span>) is plotted against the reciprocal temperature <span class="math inline">\(1/T\)</span>. With the Arrhenius equation translated to the form in <a href="#eq-arrhenius-plot" class="quarto-xref">Equation&nbsp;2.35</a></p>
<p><span id="eq-arrhenius-plot"><span class="math display">\[
\ln(k)=\ln(A)-{\frac {E_{\text{a}}}{R}}\left({\frac{1}{T}}\right),
\tag{2.35}\]</span></span></p>
<p>the y-intercept of a log-log-linear fit corresponds to <span class="math inline">\(\ln(A)\)</span> and the slope of the fit equal <span class="math inline">\(-E_\mathrm{a}/R\)</span>.</p>
<div style="page-break-after: always;"></div>
</section>
</section>
<section id="sec-main-qm" class="level1 page-columns page-full" data-number="3">
<h1 data-number="3"><span class="header-section-number">3</span> Collagen hydrolysis by quantum mechanics/molecular mechanics</h1>
<p>As introduced earlier, collagen under mechanical stress can undergo two competing reactions: peptide bond hydrolysis (breaking the bond through reaction with water) and homolytic bond scission (direct mechanical rupture producing radicals). While experiments have detected radicals in tensed collagen <span class="citation" data-cites="zappMechanoradicalsTensedTendon2020">[<a href="#ref-zappMechanoradicalsTensedTendon2020" role="doc-biblioref">17</a>]</span>, and other studies showed that hydrolysis is accelerated by force <span class="citation" data-cites="pillMechanicalActivationDrastically2019">[<a href="#ref-pillMechanicalActivationDrastically2019" role="doc-biblioref">20</a>]</span>, it remained unclear how these two pathways compete under physiologically relevant conditions.</p>
<p>Simulating an entire collagen fibril at the quantum mechanical level of detail would be computationally infeasible. This is why I reduced the core question to a comparison between a single peptide, as was the case in the single molecule pulling experiments by Pill et al. <span class="citation" data-cites="pillMechanicalActivationDrastically2019">[<a href="#ref-pillMechanicalActivationDrastically2019" role="doc-biblioref">20</a>]</span>, and a collagen triple helix. In graphics and schematics these systems will be referred to as <code>single</code> and <code>triple</code>. This chapter addresses the hydrolysis pathway using hybrid quantum mechanics/molecular mechanics (QM/MM) simulations, the method of choice for reactive subsystems for large biomolecular systems (see <a href="#sec-meth-qmmm" class="quarto-xref">Section&nbsp;2.12</a>). The effect of the fibrillar assembly will come into play again in <a href="#sec-main-kimmdy" class="quarto-xref">Chapter&nbsp;4</a>, but this chapter will only be concerned with <em>how</em> the hydrolysis reaction happens.</p>
<p>It is worth pointing out that while it makes sense to tell the story in this order, QM/MM followed by KIMMDY (<a href="#sec-main-kimmdy" class="quarto-xref">Chapter&nbsp;4</a>), many discoveries are the culmination of things happening in parallel. Thus, a key reason as to why it makes sense to focus the QM/MM simulations on just a triple helix as opposed to a complete fibril is only explained later in <a href="#sec-sasa" class="quarto-xref">Section&nbsp;4.3.2</a>, where I talk about the solvent accessibility of the peptide bond.</p>
<p>Originally I had planned to simulate both homolytic and hydrolytic bond cleavage in the same QM/MM setup to get a direct comparison. However, it is a fundamental limitation of my chosen QM methodology, density functional theory (DFT), that a system has to be representable by a single Slater determinant – meaning the total spin of the system cannot change during the simulation. This renders a direct sampling approach to homolytic (and thus radical) cleavage reactions futile, as bond homolysis creates two unpaired electrons. As such, this thesis will not contain QM/MM simulations of homolysis and I will assume instead, as is commonly done in the field <span class="citation" data-cites="liangQuantumChemistryStudy2022 kosarBenchmarkDFTStudies2019 johnPredictionOrganicHomolytic2020">[<a href="#ref-liangQuantumChemistryStudy2022" role="doc-biblioref">88</a>–<a href="#ref-johnPredictionOrganicHomolytic2020" role="doc-biblioref">90</a>]</span>, that the transition state energy of the cleavage reaction is very close to the dissociation energy of the respective bond, wherever a direct comparison is required. In the following I will thus focus on base-catalyzed hydrolysis and my extensive sampling of the reaction in a QM/MM scheme.</p>
<section id="previous-work-on-base-catalyzed-hydrolysis" class="level2 page-columns page-full" data-number="3.1">
<h2 data-number="3.1" class="anchored" data-anchor-id="previous-work-on-base-catalyzed-hydrolysis"><span class="header-section-number">3.1</span> Previous work on base-catalyzed hydrolysis</h2>
<p>Base-catalyzed hydrolysis is a multi-step reaction that breaks a peptide bond through nucleophilic attack by a hydroxide ion (<span class="math inline">\(\ce{OH-}\)</span>) (<a href="#fig-hyd-reaction" class="quarto-xref">Figure&nbsp;3.1</a>). The mechanism proceeds as follows: First, the catalytic hydroxide (acting as a nucleophile) attacks the electrophilic carbon atom of the peptide bond’s carbonyl group. This attack goes through a transition state TS1, overcoming the associated energy barrier, to form a tetrahedral intermediate (TI), where the carbon is temporarily bonded to four substituents. The TI is stabilized by hydrogen bonds with surrounding solvent molecules or potentially with protons from the protein backbone – a stabilization mechanism worth investigating. Next, the protonation of the peptide bond nitrogen leads to a zwitterionic intermediate (ZI), which can then rupture at the peptide bond (passing transition state TS2), leaving separate C- and N-termini that can, depending on the pH value, also be protonated or deprotonated. Ultimately, since the proton to form the ZI comes from water, a hydroxide ion is regenerated, making it truly catalytic.</p>
<div id="fig-hyd-reaction" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="Hydrolysis reaction scheme.">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-hyd-reaction-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./img/hydrolysis-reaction.svg" class="lightbox" data-gallery="quarto-lightbox-gallery-29" title="Figure&nbsp;3.1: Reaction schema of base-catalyzed peptide bond hydrolysis. TS is a transition state, TI is the tetrahedral intermediate and ZI is the zwitterionic state."><img src="./img/hydrolysis-reaction.svg" class="img-fluid figure-img" data-fig-scap="Hydrolysis reaction scheme."></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-hyd-reaction-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.1: Reaction schema of base-catalyzed peptide bond hydrolysis. TS is a transition state, TI is the tetrahedral intermediate and ZI is the zwitterionic state.
</figcaption>
</figure>
</div>
<p>To put the results of this chapter into context, I compiled previous results by Pill et al. <span class="citation" data-cites="pillMechanicalActivationDrastically2019">[<a href="#ref-pillMechanicalActivationDrastically2019" role="doc-biblioref">20</a>]</span> into a single figure (<a href="#fig-pill-energies" class="quarto-xref">Figure&nbsp;3.3 (b)</a>). It is worth going into some detail about how these results were obtained to make the comparison to my results clearer. Broadly, they obtained reaction rates and energies by two distinct methods, designed to investigate how external force affects base-catalyzed hydrolysis:</p>
<p>Experimental approach: A single molecule pulling setup with an atomic force microscopy (AFM) cantilever measured bond lifetimes under varying applied forces and temperatures. These lifetimes can be translated to activation energies and attempt frequencies (pre-exponential factors) via an Arrhenius plot (see <a href="#sec-arrhenius" class="quarto-xref">Section&nbsp;2.15</a>).</p>
<p>Computational approach: They calculated <em>ab initio</em> single-point energies at the MP2/TZVP level of theory for B3LYP-optimized structures (using the 6-31+G(d) basis set) under external force (<a href="#fig-pill-pull" class="quarto-xref">Figure&nbsp;3.2</a>) for each of the states shown in <a href="#fig-hyd-reaction" class="quarto-xref">Figure&nbsp;3.1</a>. As shown in <a href="#fig-pill-systems" class="quarto-xref">Figure&nbsp;3.3 (a)</a>, those energies are comprised as a sum of three auxiliary systems that are minimized independently of each other and chosen in such a way that the total number of water molecules, hydrogen bridges and number of strained HOH bond angles stays the same. The first subsystem contains the actual peptide bond as a part of N-ethylethanamide, the second and third contain the initial hydroxide ion and more water molecules to balance out the quantities mentioned above that are to stay constant.</p>

<div class="no-row-height column-margin column-container"><div id="fig-pill-pull" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="Pill et al. pulling setup.">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-pill-pull-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="img/pill-pull.png" class="lightbox" data-gallery="quarto-lightbox-gallery-30" title="Figure&nbsp;3.2: Illustration of the pulling setup from the SI of @pillMechanicalActivationDrastically2019."><img src="img/pill-pull.png" class="img-fluid figure-img" data-fig-scap="Pill et al. pulling setup."></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-pill-pull-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.2: Illustration of the pulling setup from the SI of <span class="citation" data-cites="pillMechanicalActivationDrastically2019">[<a href="#ref-pillMechanicalActivationDrastically2019" role="doc-biblioref">20</a>]</span>.
</figcaption>
</figure>
</div></div><div id="fig-pill" class="quarto-layout-panel">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-pill-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="quarto-layout-row quarto-layout-valign-bottom">
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-pill" style="flex-basis: 40.0%;justify-content: flex-start;">
<div id="fig-pill-systems" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="Pill et al. QM setup.">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-pill-systems-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./img/pill-systems.png" class="lightbox" data-gallery="fig-pill" title="Figure&nbsp;3.3&nbsp;(a): Excerpt from figure S7 of the SI of @pillMechanicalActivationDrastically2019 showing the auxiliary subsystems for the first 3 reaction steps. From top to bottom: educt, TS1, TI."><img src="./img/pill-systems.png" class="img-fluid figure-img" data-fig-scap="Pill et al. QM setup." data-ref-parent="fig-pill"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-pill-systems-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(a) Excerpt from figure S7 of the SI of <span class="citation" data-cites="pillMechanicalActivationDrastically2019">[<a href="#ref-pillMechanicalActivationDrastically2019" role="doc-biblioref">20</a>]</span> showing the auxiliary subsystems for the first 3 reaction steps. From top to bottom: educt, TS1, TI.
</figcaption>
</figure>
</div>
</div>
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-pill" style="flex-basis: 60.0%;justify-content: flex-start;">
<div id="fig-pill-energies" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="Pill et al. energies.">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-pill-energies-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./plots/plots/pill-energies.png" class="lightbox" data-gallery="fig-pill" title="Figure&nbsp;3.3&nbsp;(b): (Force-dependent) energies of the reaction steps based on AFM (experimental) and ab initio QM (theoretical) by @pillMechanicalActivationDrastically2019. The energy of the educt state is used subtracted as a baseline."><img src="./plots/plots/pill-energies.png" class="img-fluid figure-img" data-fig-scap="Pill et al. energies." data-ref-parent="fig-pill"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-pill-energies-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(b) (Force-dependent) energies of the reaction steps based on AFM (experimental) and <em>ab initio</em> QM (theoretical) by <span class="citation" data-cites="pillMechanicalActivationDrastically2019">[<a href="#ref-pillMechanicalActivationDrastically2019" role="doc-biblioref">20</a>]</span>. The energy of the educt state is used subtracted as a baseline.
</figcaption>
</figure>
</div>
</div>
</div>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-pill-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.3: Visual summary of the setup and data points obtained by <span class="citation" data-cites="pillMechanicalActivationDrastically2019">[<a href="#ref-pillMechanicalActivationDrastically2019" role="doc-biblioref">20</a>]</span> that are relevant for this thesis.
</figcaption>
</figure>
</div>
<p>Using this setup, they found that the first energy barrier (TS1) is barely lowered by adding force, while the second energy barrier (TS2) is dramatically lowered even by adding small external forces. This makes intuitive sense: the attack of the hydroxide occurs perpendicular to the axis of applied force, while the breaking apart of the ZI happens along the pulling axis. Notably, the energy barriers for TS1 calculated from experimental bond lifetimes are significantly lower than the barriers from the <em>ab initio</em> QM calculations. For comparison, the reaction rate obtained from experiments at an external force of 1&nbsp;nN was <span class="math inline">\(k\)</span>=6.89&nbsp;±&nbsp;0.85&nbsp;s<sup>-1</sup> with an energy barrier of 58.1&nbsp;±&nbsp;2.8&nbsp;kJ/mol and an attempt frequency of 1.3×10<sup>11±0.5</sup>&nbsp;s<sup>-1</sup> (or an overall average attempt frequency <span class="math inline">\(A\)</span> of 10<sup>11</sup>&nbsp;s<sup>−1</sup>. The total rate constant from the QM calculations can be derived from the individual energies of TS1 and TS2 with <a href="#eq-total-k" class="quarto-xref">Equation&nbsp;3.1</a></p>
<p><span id="eq-total-k"><span class="math display">\[
\begin{array}{rcl}
k1 &amp;=&amp; A1 e^{-E_{TS1}/k_{\mathrm{B}}T} \\
k2 &amp;=&amp; A1 e^{-E_{TS2}/k_{\mathrm{B}}T} \\
k &amp;= &amp;\frac{k_1 k_2 [OH^{-}]}{k_1+k_2},
\end{array}
\tag{3.1}\]</span></span></p>
<p>with the concentration of hydroxide <span class="math inline">\([OH^{-}]\)</span> coming in as 10^-(14 - 7.4)^ at the experimental pH of 7.4 and a temperature of 21° C (294.15&nbsp;K). The auxiliary attempt frequency <span class="math inline">\(A1\)</span> was assumed fixed such that <span class="math inline">\(A1*[OH^{-}]=A\)</span> matches the average attempt frequency <span class="math inline">\(A\)</span> gathered from the AFM experiments. Together this results in a theoretical reaction rate of 10<sup>-3</sup>&nbsp;s<sup>-1</sup>.</p>
<p>These results serve as a reference for my calculations and on top of that will also play a role in <a href="#sec-main-kimmdy" class="quarto-xref">Chapter&nbsp;4</a>.</p>
</section>
<section id="goals-of-the-calculations" class="level2" data-number="3.2">
<h2 data-number="3.2" class="anchored" data-anchor-id="goals-of-the-calculations"><span class="header-section-number">3.2</span> Goals of the calculations</h2>
<p>The goal of my research into hydrolysis was to go beyond highly optimized and reduced model systems and explore how hydrolysis behaves in a complex, dynamical environment. Rather than using small, pre-optimized structures, the following will employ large-scale QM/MM simulations of peptides where the QM region encompasses two whole residues (and their connecting peptide bond) plus surrounding water molecules. This approach directly samples the reaction in a more realistic, fluctuating environment, providing the unique opportunity to observe reaction dynamics and the sequence of events to complement single-point energies.</p>
<p>Two caveats apply when comparing energies between different approaches. Firstly, my pipeline will measure free energy differences, which incorporates entropic contributions, as opposed to the pure electronic energies shown in <a href="#fig-pill-energies" class="quarto-xref">Figure&nbsp;3.3 (b)</a>. Secondly, the most favorable conformation, as is found by energy minimization, might not be covered by phase space sampling. Exploring the accessibility of said conformation is in itself a feature of the sampling approach, but it does also mean that values for energy barriers will differ between approaches. This is why the focus will be on trends between systems and not on the absolute values of the energies and energy barriers.</p>
<p>The overarching hypothesis is that the triple helical structure of collagen peptides in particular should hinder the approach of the hydroxide and energetically disfavor the tetrahedral intermediate.</p>
</section>
<section id="system-setup-and-simulation-pipeline" class="level2 page-columns page-full" data-number="3.3">
<h2 data-number="3.3" class="anchored" data-anchor-id="system-setup-and-simulation-pipeline"><span class="header-section-number">3.3</span> System setup and simulation pipeline</h2>
<p>In my early tests I found that a combination of 14 QM water molecules and the TZV2P-MOLOPT-GTH basis set <span class="citation" data-cites="vandevondeleGaussianBasisSets2007">[<a href="#ref-vandevondeleGaussianBasisSets2007" role="doc-biblioref">91</a>]</span> with the PBE functional <span class="citation" data-cites="perdewGeneralizedGradientApproximation1996">[<a href="#ref-perdewGeneralizedGradientApproximation1996" role="doc-biblioref">92</a>]</span> provides the most stable simulations. Due to its good polarizability and diffuse characteristics, this basis set can stabilize the negative charge in the QM region (originating from the hydroxide ion) and allows charge delocalization via proton transfer reactions. The dynamic nature of the protons in the QM region (meaning they can hop between water molecules and the reactants) is a key feature of this setup and will be explored further in the results section.</p>
<section id="initial-structure-preparation" class="level3" data-number="3.3.1">
<h3 data-number="3.3.1" class="anchored" data-anchor-id="initial-structure-preparation"><span class="header-section-number">3.3.1</span> Initial structure preparation</h3>
<p>The triple helix initial structure is a cut-out from a <em>Homo sapiens</em> collagen type I fibril without crosslinks that was obtained from ColBuilder <span class="citation" data-cites="obarska-kosinskaColBuilderServerBuild2021">[<a href="#ref-obarska-kosinskaColBuilderServerBuild2021" role="doc-biblioref">93</a>]</span>. It is composed of three chains (A, B, and C) with individual lengths of 45, 43, and 45 amino acids respectively (<a href="#fig-render-triple-single" class="quarto-xref">Figure&nbsp;3.5</a>). The N-terminus (displayed on the left of each molecular render, <a href="#fig-render-triple-single" class="quarto-xref">Figure&nbsp;3.5</a>, <a href="#fig-sites" class="quarto-xref">Figure&nbsp;3.6</a>, <a href="#fig-qm-system" class="quarto-xref">Figure&nbsp;3.8</a>) was acetyl-capped (ACE), while the C-terminus was capped with an N-methyl group (NME) in PyMOL <span class="citation" data-cites="PyMOL">[<a href="#ref-PyMOL" role="doc-biblioref">94</a>]</span> to account for the fact that the actual collagen protein is longer than the cut-out simulated here. From this point onwards, everything is handled by my custom automated simulation and analysis pipeline. The code for this is available at <a href="https://github.com/graeter-group/hydrolysis-qmmm-workflow">github.com/graeter-group/hydrolysis-qmmm-workflow</a>. This version does not include the outputs of the simulations, as those would be way too large. They are instead archived locally at the Heidelberg Institute for Theoretical Studies (HITS) and available upon request. The workflow is organized in Python modules and controlled from a central notebook <code>index.qmd</code> loading the modules. Simulations are computed locally or dispatched to the in-house Slurm <span class="citation" data-cites="jetteArchitectureSlurmWorkload2023">[<a href="#ref-jetteArchitectureSlurmWorkload2023" role="doc-biblioref">95</a>]</span> high performance computing (HPC) cluster. Interactions with GROMACS are solely based on the command line interface via files and system calls to shell commands from Python. A <code>settings.py</code> file creates a permutation matrix of input conditions and other parameters that were varied and iterated on during development of the pipeline. Templating allows sharing assets such as mdp files across systems and parameters. A file-based caching mechanism makes sure that work doesn’t have to be repeated unless explicitly required. Iterations of the code were tracked with <a href="https://git-scm.com/">git</a>, but the public repository is a freshly instanced one to avoid uploading large files that were only tracked during development to provide synchronization between a remote and local workstation. In addition to the simulation pipeline there is an analysis pipeline based on a directed acyclic graph (DAG) built with <a href="https://hamilton.dagworks.io/en/latest/">Apache Hamilton</a>. The cache of the DAG can also be synchronized between workstations with a custom script, which allows local analysis of otherwise unwieldy data.</p>
<p>Throughout my work I embrace reproducible research and open access and a lot of work went into crafting a robust and extensible workflow. But I like talking about workflows way too much, which is why I will stop myself here and refer to the code linked above or questions directed at future me instead. For now, what is more important than how the workflow is built, is what it does and the results thereof.</p>
</section>
<section id="equilibration-protocol" class="level3 page-columns page-full" data-number="3.3.2">
<h3 data-number="3.3.2" class="anchored" data-anchor-id="equilibration-protocol"><span class="header-section-number">3.3.2</span> Equilibration protocol</h3>
<p>Each system (the triple helix and each of the chains individually) starts as a PDB file. It is converted to GROMACS coordinates and a topology with <code>gmx pdb2gmx</code>. All simulations use the <code>amber99sb-star-ildnp</code> force field and partial charges for the hydroxide ion in its MM representation were ported from CHARMM36 <span class="citation" data-cites="brooksCHARMMBiomolecularSimulation2009">[<a href="#ref-brooksCHARMMBiomolecularSimulation2009" role="doc-biblioref">36</a>]</span>. A box is created with a minimum distance to the protein of 2&nbsp;nm on each side with each dimension then scaled by 130%. This ensures sufficient space, especially in the direction of pulling. The protein is then solvated in TIP3P water and neutralized with 0.15&nbsp;M NaCl to mimic physiological salt concentration. After energy minimization, the system is equilibrated first in the NVT ensemble (constant number of particles, volume, and temperature) and then in the NPT ensemble (constant number of particles, pressure, and temperature) while applying position restraints to the protein atoms for 100&nbsp;ps each. This staged equilibration allows the solvent to relax around the protein structure before applying external forces. Each system is then equilibrated under an external force to simulate mechanical stress. In theory, this external force is an input parameter, but in practice the sampling ended up so computationally expensive that only one value for it will be explored here. A force of 1&nbsp;nN per chain (603&nbsp;kJ/mol/nm in GROMACS units) is applied to the center of mass (COM) of the <span class="math inline">\(\alpha\)</span> carbons of the capping residues in opposing directions for 100&nbsp;ps (50000 steps at dt=0.002&nbsp;ps). The triple helix configuration gains additional rotational restraints to prevent unwinding. This mimics it being packed inside of a fibril and restrained by the surrounding triple helices. The length of the equilibration was found to be sufficient by observing the distances of the pull groups (<a href="#fig-eq-pull" class="quarto-xref">Figure&nbsp;3.4</a>).</p>

<div class="no-row-height column-margin column-container"><div id="fig-eq-pull" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="Pulling equilibration.">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-eq-pull-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./plots/qm/triple_equilibration.png" class="lightbox" data-gallery="quarto-lightbox-gallery-33" title="Figure&nbsp;3.4: The distance between the pull groups for the external force as a function of time for the equilibration of the triple helix under 1\,nN."><img src="./plots/qm/triple_equilibration.png" class="img-fluid figure-img" data-fig-scap="Pulling equilibration."></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-eq-pull-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.4: The distance between the pull groups for the external force as a function of time for the equilibration of the triple helix under <span class="math inline">\(1\,nN\)</span>.
</figcaption>
</figure>
</div></div><div id="fig-render-triple-single" class="quarto-layout-panel">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-render-triple-single-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="quarto-layout-row quarto-layout-valign-bottom">
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-render-triple-single" style="flex-basis: 43.8%;justify-content: flex-start;">
<div id="fig-render-triple" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-render-triple-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./figures/render-triple-tubes-cpk.png" class="lightbox" data-gallery="fig-render-triple-single" title="Figure&nbsp;3.5&nbsp;(a): The triple helix before equilibration under force."><img src="./figures/render-triple-tubes-cpk.png" class="img-fluid figure-img" data-ref-parent="fig-render-triple-single"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-render-triple-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(a) The triple helix before equilibration under force.
</figcaption>
</figure>
</div>
</div>
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-render-triple-single" style="flex-basis: 56.3%;justify-content: flex-start;">
<div id="fig-render-triple-pull" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-render-triple-pull-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./figures/render-triple-pull-annotations.png" class="lightbox" data-gallery="fig-render-triple-single" title="Figure&nbsp;3.5&nbsp;(b): The triple helix after equilibration under 1&nbsp;nN of external force."><img src="./figures/render-triple-pull-annotations.png" class="img-fluid figure-img" data-ref-parent="fig-render-triple-single"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-render-triple-pull-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(b) The triple helix after equilibration under 1&nbsp;nN of external force.
</figcaption>
</figure>
</div>
</div>
</div>
<div class="quarto-layout-row quarto-layout-valign-bottom">
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-render-triple-single" style="flex-basis: 43.8%;justify-content: flex-start;">
<div id="fig-render-single" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-render-single-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./figures/render-chain-c-tubes-cpk.png" class="lightbox" data-gallery="fig-render-triple-single" title="Figure&nbsp;3.5&nbsp;(c): A single peptide (chain C) before equilibration under force."><img src="./figures/render-chain-c-tubes-cpk.png" class="img-fluid figure-img" data-ref-parent="fig-render-triple-single"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-render-single-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(c) A single peptide (chain C) before equilibration under force.
</figcaption>
</figure>
</div>
</div>
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-render-triple-single" style="flex-basis: 56.3%;justify-content: flex-start;">
<div id="fig-render-single-pull" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-render-single-pull-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./figures/render-single-pull-annotations.png" class="lightbox" data-gallery="fig-render-triple-single" title="Figure&nbsp;3.5&nbsp;(d): A single peptide (chain C) after equilibration under 1&nbsp;nN of external force."><img src="./figures/render-single-pull-annotations.png" class="img-fluid figure-img" data-ref-parent="fig-render-triple-single"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-render-single-pull-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(d) A single peptide (chain C) after equilibration under 1&nbsp;nN of external force.
</figcaption>
</figure>
</div>
</div>
</div>
<div class="quarto-layout-row quarto-layout-valign-bottom">
<div class="cell quarto-layout-cell-subref quarto-layout-cell" data-fig-width="100%" data-fig-height="50%" data-ref-parent="fig-render-triple-single" style="flex-basis: 100.0%;justify-content: flex-start;">
<div class="cell-output-display quarto-layout-cell-subref" data-ref-parent="fig-render-triple-single">
<div id="fig-qmmm-pipeline" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-qmmm-pipeline-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div>
<svg width="NaN" height="NaN" viewbox="0.00 0.00 438.20 44.00" xmlns="http://www.w3.org/2000/svg" xlink="http://www.w3.org/1999/xlink" style="; max-width: none; max-height: none">
<g id="graph0" class="graph" transform="scale(1 1) rotate(0) translate(4 40)">
<title>g</title>
<polygon fill="white" stroke="transparent" points="-4,4 -4,-40 434.2,-40 434.2,4 -4,4"></polygon>
<!-- [system] -->
<g id="node1" class="node">
<title>[system]</title>
<polygon fill="white" stroke="black" points="54,-36 0,-36 0,0 54,0 54,-36"></polygon>
<text text-anchor="middle" x="27" y="-15.6" font-family="Roboto" font-size="8.00">[system]</text>
</g>
<!-- equilibrate [F] -->
<g id="node2" class="node">
<title>equilibrate [F]</title>
<polygon fill="white" stroke="black" points="149.98,-36 90.01,-36 90.01,0 149.98,0 149.98,-36"></polygon>
<text text-anchor="middle" x="119.99" y="-15.6" font-family="Roboto" font-size="8.00">equilibrate [F]</text>
</g>
<!-- [system]&#45;&gt;equilibrate [F] -->
<g id="edge1" class="edge">
<title>[system]-&gt;equilibrate [F]</title>
<path fill="none" stroke="black" d="M54.32,-18C62.23,-18 71.09,-18 79.65,-18"></path>
<polygon fill="black" stroke="black" points="79.86,-21.5 89.86,-18 79.86,-14.5 79.86,-21.5"></polygon>
</g>
<!-- QM setup [site] -->
<g id="node3" class="node">
<title>QM setup [site]</title>
<polygon fill="white" stroke="black" points="250.31,-36 185.88,-36 185.88,0 250.31,0 250.31,-36"></polygon>
<text text-anchor="middle" x="218.09" y="-15.6" font-family="Roboto" font-size="8.00">QM setup [site]</text>
</g>
<!-- equilibrate [F]&#45;&gt;QM setup [site] -->
<g id="edge2" class="edge">
<title>equilibrate [F]-&gt;QM setup [site]</title>
<path fill="none" stroke="black" d="M150.09,-18C158.1,-18 166.95,-18 175.52,-18"></path>
<polygon fill="black" stroke="black" points="175.76,-21.5 185.76,-18 175.76,-14.5 175.76,-21.5"></polygon>
</g>
<!-- TS1 -->
<g id="node4" class="node">
<title>TS1</title>
<polygon fill="white" stroke="black" points="340.2,-36 286.2,-36 286.2,0 340.2,0 340.2,-36"></polygon>
<text text-anchor="middle" x="313.2" y="-15.6" font-family="Roboto" font-size="8.00">TS1</text>
</g>
<!-- QM setup [site]&#45;&gt;TS1 -->
<g id="edge3" class="edge">
<title>QM setup [site]-&gt;TS1</title>
<path fill="none" stroke="black" d="M250.35,-18C258.62,-18 267.6,-18 276.1,-18"></path>
<polygon fill="black" stroke="black" points="276.14,-21.5 286.14,-18 276.14,-14.5 276.14,-21.5"></polygon>
</g>
<!-- TS2 -->
<g id="node5" class="node">
<title>TS2</title>
<polygon fill="white" stroke="black" points="430.2,-36 376.2,-36 376.2,0 430.2,0 430.2,-36"></polygon>
<text text-anchor="middle" x="403.2" y="-15.6" font-family="Roboto" font-size="8.00">TS2</text>
</g>
<!-- TS1&#45;&gt;TS2 -->
<g id="edge4" class="edge">
<title>TS1-&gt;TS2</title>
<path fill="none" stroke="black" d="M340.6,-18C348.59,-18 357.51,-18 366.02,-18"></path>
<polygon fill="black" stroke="black" points="366.12,-21.5 376.12,-18 366.12,-14.5 366.12,-21.5"></polygon>
</g>
</g>
</svg>
</div>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-qmmm-pipeline-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(e) QM/MM pipeline with choices from the input parameter space highlighted by square brackets <code>[]</code>.
</figcaption>
</figure>
</div>
</div>
</div>
</div>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-render-triple-single-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.5: The collagen triple helix and single peptide before and after equilibration under external force.
</figcaption>
</figure>
</div>
</section>
<section id="quantum-mechanics-subsystem-selection-and-setup" class="level3 page-columns page-full" data-number="3.3.3">
<h3 data-number="3.3.3" class="anchored" data-anchor-id="quantum-mechanics-subsystem-selection-and-setup"><span class="header-section-number">3.3.3</span> Quantum mechanics subsystem selection and setup</h3>
<p>From the force equilibration trajectory, the last 10 frames are extracted to provide multiple starting points for the actual hydrolysis reaction. This is done because QM/MM simulations can be unstable and finding an initial stable trajectory suitable for umbrella sampling is not always guaranteed. Additionally, using multiple starting configurations increases the sampling of phase space and improves statistical convergence.</p>
<p>In the triple helix, 8 peptide bonds were chosen at random, but close to the center to avoid anisotropies in the external force, which attaches at the ends of the peptides. Each site is defined by the indices of its carbon and nitrogen atom, where the <span class="math inline">\(\ce{C}\)</span> is contributed by the residue on the N-terminal side and the <span class="math inline">\(\ce{N}\)</span> is contributed by the residue from the C-terminal side. From each starting point and chosen peptide bond, the workflow now looks for water molecules that can execute the first step of the reaction: the attack of the hydroxide (TS1 in <a href="#fig-hyd-reaction" class="quarto-xref">Figure&nbsp;3.1</a>).</p>
<div id="fig-sites" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="Sampled hydrolysis sites.">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-sites-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./figures/sites-eq-close-annotations.png" class="lightbox" data-gallery="quarto-lightbox-gallery-38" title="Figure&nbsp;3.6: A close-up of the labeled sites that were sampled for hydrolysis. Each site exists as part of the triple helix and just in its own single chain. The QM region of each site, that is all atoms of both residues that are connected with the peptide bond that is to be hydrolyzed, is rendered with the ball-and-stick representation in color, while the rest of the protein is shown with black lines."><img src="./figures/sites-eq-close-annotations.png" class="img-fluid figure-img" data-fig-scap="Sampled hydrolysis sites."></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-sites-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.6: A close-up of the labeled sites that were sampled for hydrolysis. Each site exists as part of the triple helix and just in its own single chain. The QM region of each site, that is all atoms of both residues that are connected with the peptide bond that is to be hydrolyzed, is rendered with the ball-and-stick representation in color, while the rest of the protein is shown with black lines.
</figcaption>
</figure>
</div>
<p>As visualized in <a href="#fig-bd" class="quarto-xref">Figure&nbsp;3.7</a>, the Bürgi-Dunitz angle (BD) is the angle between the approach vector of a nucleophile (<span class="math inline">\(\ce{OH-}\)</span>) and an electrophilic center (the carbonyl carbon of the peptide bond), where the optimal angle is 107° <span class="citation" data-cites="burgiStereochemistryReactionPaths1974">[<a href="#ref-burgiStereochemistryReactionPaths1974" role="doc-biblioref">96</a>]</span>. Similarly, the Flippin-Lodge angle <span class="citation" data-cites="lodgeStericEffectsWell1987 flippinACYCLICSTEREOSELECTIONXVI1983">[<a href="#ref-lodgeStericEffectsWell1987" role="doc-biblioref">97</a>, <a href="#ref-flippinACYCLICSTEREOSELECTIONXVI1983" role="doc-biblioref">98</a>]</span> characterizes the displacement of the nucleophile at its elevation towards or away from the substituents attached to the electrophilic center. For the purpose of finding the best candidate, the optimal Flippin-Lodge angle is assumed to be the one where the distance to both substituents is maximized and defined as 0° at the angle bisector. This leads to the following penalty function <a href="#eq-bd-penalty" class="quarto-xref">Equation&nbsp;3.2</a>:</p>

<div class="no-row-height column-margin column-container"><div id="fig-bd" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="Schema of the Bürgi-Dunitz and Flippin-Lodge angles.">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-bd-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./figures/render-qm-bd-annotations.png" class="lightbox" data-gallery="quarto-lightbox-gallery-39" title="Figure&nbsp;3.7: Schema of the Bürgi-Dunitz and Flippin-Lodge angles."><img src="./figures/render-qm-bd-annotations.png" class="img-fluid figure-img" data-fig-scap="Schema of the Bürgi-Dunitz and Flippin-Lodge angles."></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-bd-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.7: Schema of the Bürgi-Dunitz and Flippin-Lodge angles.
</figcaption>
</figure>
</div></div><p><span id="eq-bd-penalty"><span class="math display">\[
p=\sqrt{(BD-107)^2 + (FL-0)^2},
\tag{3.2}\]</span></span></p>
<p>where <span class="math inline">\(BD\)</span> is the Bürgi-Dunitz angle and <span class="math inline">\(FL\)</span> is the angle between the projection of the <span class="math inline">\(\ce{OH-}\)</span> approach vector onto the electrophile plane and the angle bisector of the angle between the substituents and the electrophilic center.</p>
<p>Thus, first the 15 <a href="#fn3" class="footnote-ref" id="fnref3" role="doc-noteref"><sup>3</sup></a> solvent molecules closest to the carbonyl <span class="math inline">\(\ce{C}\)</span> are chosen, and out of those, the one with the lowest penalty is designated as the attack hydroxide. The remaining 14 solvent molecules are also designated as QM atoms in the QM/MM scheme. All atoms of both residues that make up the target peptide bond are marked as QM atoms. <a href="#fig-qm-system" class="quarto-xref">Figure&nbsp;3.8</a> shows a close-up render of the QM/MM setup for the site Gly842-Leu843 (i.e., the peptide bond between glycine with residue ID 842 and leucine with ID 843) in the single peptide configuration. Black atoms are MM atoms and non-QM solvent molecules are not shown. Added forces are signified with the symbol <span class="math inline">\(\text{F}\)</span>. It is worth noting that in addition to the constant external force applied to the ends of the peptide and the force from the harmonic potential that will pull the hydroxide onto the carbonyl C, there is an additional potential on the COM of the QM water molecules to keep them centered around the target peptide bond. This is to prevent them from being exchanged with MM water and I will later show in <a href="#fig-tidists" class="quarto-xref">Figure&nbsp;3.13</a> that this is indeed working as intended. Note that the hydroxide is also part of this pull group, as otherwise it would artificially increase the pressure in the QM region by being pulled towards the carbonyl carbon.</p>
<div id="fig-qm-system" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="The QM/MM subsystems, initial state.">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-qm-system-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./figures/render-qm-far-annotations.png" class="lightbox" data-gallery="quarto-lightbox-gallery-40" title="Figure&nbsp;3.8: The QM subsystem (in color) embedded in the MM system (in black) with arrows for the forces involved at the start of the attack of the hydroxide for the single peptide system at a Glycine-Leucine site. MM water molecules are not shown."><img src="./figures/render-qm-far-annotations.png" class="img-fluid figure-img" data-fig-scap="The QM/MM subsystems, initial state."></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-qm-system-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.8: The QM subsystem (in color) embedded in the MM system (in black) with arrows for the forces involved at the start of the attack of the hydroxide for the single peptide system at a Glycine-Leucine site. MM water molecules are not shown.
</figcaption>
</figure>
</div>
<p>Once the QM atoms and the hydroxide have been designated by the workflow, the QM/MM system is set up. To this end, it creates the necessary index files that GROMACS uses to label groups of atoms as, e.g., QM atoms or pull groups and fills templates for other input files such as <code>mdp</code> for GROMACS and <code>inp</code> for CP2K A short 100 step QM/MM warm-up simulation with a small delta t of just 0.2&nbsp;fs increases the stability of the system before the actual approach trajectory is run. During the warm-up the hydroxide is kept at its original distance by an umbrella potential. The workflow then checks for successful warm-ups and continues with the approach simulation. The approach is simulated at a timestep of 1&nbsp;fs and typically takes around 2&nbsp;ps. The force constant of the harmonic potential pulling the hydroxide closer to the carbonyl <span class="math inline">\(\ce{C}\)</span> is 40000&nbsp;kJ/mol/nm<sup>2</sup>. From this initial trajectory, 50 snapshots are extracted such that there are 30 equidistant snapshots (with regards to the <span class="math inline">\(\ce{OH- \bond{...}C}\)</span> distance) below a distance of 0.25&nbsp;nm and 20 snapshots above the cutoff distance. This is done do improve sampling in the critical transition state (TS) region. From these snapshots, also called umbrella sampling windows, additional simulations are started, where the distance between the hydroxide and peptide bond is kept near the initial distance of the snapshot. These simulations use a rather steep potential with a force constant of 50000&nbsp;kJ/mol/nm<sup>2</sup> to ensure that peak of the free energy surface (the transition state) can actually be sampled. This results in sufficient coverage of the reaction coordinate and adequate overlap of the umbrella windows (<a href="#fig-hist-us" class="quarto-xref">Figure&nbsp;3.9 (d)</a>). The umbrella sampling simulations are analyzed with WHAM (see <a href="#sec-meth-us" class="quarto-xref">Section&nbsp;2.9</a>) to obtain the free energy profile of the reaction step. After the umbrella sampling of TS1, the pipeline moves on to the second reaction step. Because the system contains enough QM solvent molecules that facilitate proton mobility it is not required to artificially separate establishing the zwitterionic state (ZI) from the breaking of the peptide bond (TS2). Instead, from each of stable TIs a snapshot is used as the starting point of a breaking simulation, whereby an additional umbrella potential pulls apart the <span class="math inline">\(\ce{C}\)</span> and <span class="math inline">\(\ce{N}\)</span> atom. This is then used as the initial trajectory for umbrella sampling of TS2.</p>
<p>Out of the 8 initially chosen hydrolysis sites, all were able to get at least 5 successful hydroxide attack trajectories (starting from the last 10 frames of the equilibration as detailed above). Many got 8 successful approaches and 2 had all 10 simulations succeed (<a href="#fig-hist-starting-frames" class="quarto-xref">Figure&nbsp;3.9 (a)</a>). The criterion for success in this case, is a simulation that ran without crashing at least until the distance between the carbonyl <span class="math inline">\(\ce{C}\)</span> and the hydroxide <span class="math inline">\(\ce{O}\)</span> is below 0.145&nbsp;nN, which would allow for sampling of the tetrahedral intermediate (whose optimal <span class="math inline">\(\ce{C-O}\)</span> distance for the free energy minimum will later be found to be around 0.15&nbsp;nN). Out of these trajectories most managed to start the target 50 umbrella sampling windows, although very few didn’t establish the sampling simulation due to instabilities during the QM/MM warm-up (<a href="#fig-hist-n-windows" class="quarto-xref">Figure&nbsp;3.9 (b)</a>). Most sampling simulations managed to run their entire 5&nbsp;ps, but some terminated earlier (<a href="#fig-hist-n-points" class="quarto-xref">Figure&nbsp;3.9 (c)</a>).</p>
<div id="fig-us-sampling-stats" class="quarto-layout-panel">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-us-sampling-stats-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="quarto-layout-row">
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-us-sampling-stats" style="flex-basis: 50.0%;justify-content: flex-start;">
<div id="fig-hist-starting-frames" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-hist-starting-frames-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./plots/qm/wethyd_us_n_starting_frames_hist.png" class="lightbox" data-gallery="fig-us-sampling-stats" title="Figure&nbsp;3.9&nbsp;(a): Histogram of the number of successful simulations of the hydroxide attacks on the peptide bond out of 10 starting frames."><img src="./plots/qm/wethyd_us_n_starting_frames_hist.png" class="img-fluid figure-img" data-ref-parent="fig-us-sampling-stats"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-hist-starting-frames-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(a) Histogram of the number of successful simulations of the hydroxide attacks on the peptide bond out of 10 starting frames.
</figcaption>
</figure>
</div>
</div>
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-us-sampling-stats" style="flex-basis: 50.0%;justify-content: flex-start;">
<div id="fig-hist-n-windows" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-hist-n-windows-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./plots/qm/wethyd_us_n_windows_hist.png" class="lightbox" data-gallery="fig-us-sampling-stats" title="Figure&nbsp;3.9&nbsp;(b): Histogram of the number of umbrella sampling windows per hydroxide attack trajectory. "><img src="./plots/qm/wethyd_us_n_windows_hist.png" class="img-fluid figure-img" data-ref-parent="fig-us-sampling-stats"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-hist-n-windows-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(b) Histogram of the number of umbrella sampling windows per hydroxide attack trajectory.
</figcaption>
</figure>
</div>
</div>
</div>
<div class="quarto-layout-row">
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-us-sampling-stats" style="flex-basis: 50.0%;justify-content: flex-start;">
<div id="fig-hist-n-points" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-hist-n-points-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./plots/qm/wethyd_us_n_time_points_hist.png" class="lightbox" data-gallery="fig-us-sampling-stats" title="Figure&nbsp;3.9&nbsp;(c): Histogram oft the achieved simulation lengths of the umbrella window simulations."><img src="./plots/qm/wethyd_us_n_time_points_hist.png" class="img-fluid figure-img" data-ref-parent="fig-us-sampling-stats"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-hist-n-points-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(c) Histogram oft the achieved simulation lengths of the umbrella window simulations.
</figcaption>
</figure>
</div>
</div>
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-us-sampling-stats" style="flex-basis: 50.0%;justify-content: flex-start;">
<div id="fig-hist-us" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-hist-us-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./plots/qm/wethyd_us_histogram_example.png" class="lightbox" data-gallery="fig-us-sampling-stats" title="Figure&nbsp;3.9&nbsp;(d): Histograms of the coverage of the umbrella windows over the reaction coordinate of TS1."><img src="./plots/qm/wethyd_us_histogram_example.png" class="img-fluid figure-img" data-ref-parent="fig-us-sampling-stats"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-hist-us-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(d) Histograms of the coverage of the umbrella windows over the reaction coordinate of TS1.
</figcaption>
</figure>
</div>
</div>
</div>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-us-sampling-stats-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.9: Statistics of the umbrella sampling simulations of TS1. All windows combined total a sampling time of 22.72&nbsp;ns.
</figcaption>
</figure>
</div>
</section>
</section>
<section id="results" class="level2 page-columns page-full" data-number="3.4">
<h2 data-number="3.4" class="anchored" data-anchor-id="results"><span class="header-section-number">3.4</span> Results</h2>
<section id="simulations-reveal-mechanistic-details-of-the-tetrahedral-intermediate-formation" class="level3 page-columns page-full" data-number="3.4.1">
<h3 data-number="3.4.1" class="anchored" data-anchor-id="simulations-reveal-mechanistic-details-of-the-tetrahedral-intermediate-formation"><span class="header-section-number">3.4.1</span> Simulations reveal mechanistic details of the tetrahedral intermediate formation</h3>
<p>With the simulation setup described above, initial reactive trajectories are obtained. Within the QM subsystem defined around one of the sites (see <a href="#fig-sites" class="quarto-xref">Figure&nbsp;3.6</a>), the hydroxide ion, surrounded by water, is pulled towards the peptide bond and in most cases stably forms the tetrahedral intermediate shown in <a href="#fig-ti" class="quarto-xref">Figure&nbsp;3.10</a>. The tetrahedral Intermediate (also see <a href="#fig-hyd-reaction-ts1" class="quarto-xref">Figure&nbsp;3.11</a>) and how it is formed warrant further investigation. For this we can look not only at the initial approach trajectory, but also at the sampling windows spawned from each initial trajectory.</p>
<div id="fig-ti" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="The QM/MM subsystems, TI.">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-ti-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./figures/render-qm-ti.png" class="lightbox" data-gallery="quarto-lightbox-gallery-45" title="Figure&nbsp;3.10: The Tetrahedral Intermediate (TI) state stably formed in a QM/MM simulation of the hydroxide attack in the triple helix system at a Glycine-Leucine site. The QM subsystem is shown in color embedded in the MM system in black. MM water molecules are not shown."><img src="./figures/render-qm-ti.png" class="img-fluid figure-img" data-fig-scap="The QM/MM subsystems, TI."></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-ti-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.10: The Tetrahedral Intermediate (TI) state stably formed in a QM/MM simulation of the hydroxide attack in the triple helix system at a Glycine-Leucine site. The QM subsystem is shown in color embedded in the MM system in black. MM water molecules are not shown.
</figcaption>
</figure>
</div>

<div class="no-row-height column-margin column-container"><div id="fig-hyd-reaction-ts1" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-hyd-reaction-ts1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./img/hydrolysis-reaction-vertical-ts1.svg" class="lightbox" data-gallery="quarto-lightbox-gallery-46" title="Figure&nbsp;3.11: Reminder of where we are in the hydrolysis reaction."><img src="./img/hydrolysis-reaction-vertical-ts1.svg" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-hyd-reaction-ts1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.11: Reminder of where we are in the hydrolysis reaction.
</figcaption>
</figure>
</div></div><p>Very few constraints are imposed on the system, so many different states with regard to protonation states are sampled. The attacking hydroxide is not forced to remain a hydroxide as it approaches the peptide bond in the initial trajectory and during the umbrella window simulations. It can be protonated by the surrounding QM water, delocalizing the negative charge over the QM region. Likewise, the carbonyl oxygen can be stabilized by hydrogen bridges or even fully protonated when the TI is formed. This led to an unexpected progression of the reaction, where instead of the addition reaction, forming the TI, a substitution reaction can take place whereby the original carbonyl <span class="math inline">\(\ce{O}\)</span> gets displaced by the attacking hydroxide. This effectively results in no reaction taking place. However, this was a rare event and only 62 out of 6359 umbrella windows had to be discarded for sampling the wrong reaction. The decision to discard a window was made based on a distance criterion as shown in <a href="#fig-hyd-filter" class="quarto-xref">Figure&nbsp;5.2 (a)</a>. Windows where the average distance of the carbonyl <span class="math inline">\(\ce{C}\)</span> to the carbonyl <span class="math inline">\(\ce{O}\)</span> exceeds 0.155&nbsp;nm were deemed accidental substitution reactions and those where the sum of the average distances of <span class="math inline">\(\ce{C}\)</span> to the carbonyl <span class="math inline">\(\ce{O}\)</span> and <span class="math inline">\(\ce{C}\)</span> to the hydroxide <span class="math inline">\(\ce{O}\)</span> was lower than 0.277&nbsp;nm were deemed as collapsing TIs and also excluded. The effect of this filtering, by example of the protonation plots (<a href="#fig-tiprot" class="quarto-xref">Figure&nbsp;3.15</a>), is shown in extended <a href="#fig-tiprot-filtering" class="quarto-xref">Figure&nbsp;5.3</a>.</p>
<p>Looking at <a href="#fig-tidists" class="quarto-xref">Figure&nbsp;3.13</a> we can see how the TI is stabilized. Note that <a href="#fig-tidists-annotation" class="quarto-xref">Figure&nbsp;3.12</a> provides a visual representation of the legend. As the hydroxide approaches, which is read from right to left as the distance decreases, the hydroxide reaches a critical point around 0.2&nbsp;nm where its negative charge influences the carbonyl O. This carbonyl consequently accumulates more charge density and is stabilized more by the surrounding QM water, which can be seen in the form of the distance to the closest proton of any QM water molecule to the carbonyl <span class="math inline">\(\ce{O}\)</span> decreasing. The QM water consistently contributes the closest proton, indicating that no MM water molecule has snuck into the QM region to replace it. Interestingly, protein H atoms rarely contribute the closest H. This is mostly due to the geometry of the peptide bonds in the sampled sites. In the triple helix formation, the carbonyl <span class="math inline">\(\ce{O}\)</span> of the peptide bond can be oriented in such a way that it points away from the other two chains, like in the exemplary TI in <a href="#fig-ti" class="quarto-xref">Figure&nbsp;3.10</a>. On average this is not more likely than the bond pointing the other way, into the helix, but the sampled sites presented in this thesis have a higher propensity of pointing outwards or being parallel to the center axis of the helix (<a href="#fig-carbonyl-orientation" class="quarto-xref">Figure&nbsp;5.2 (b)</a>). At the time of writing, computations for additional sites that feature the inverted orientation are running, but the results will be for another publication.</p>

<div class="no-row-height column-margin column-container"><div id="fig-tidists-annotation" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="Visual explanation of the proton distances.">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-tidists-annotation-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./figures/render-qm-ti-annotations-dist.png" class="lightbox" data-gallery="quarto-lightbox-gallery-47" title="Figure&nbsp;3.12: Visual explanation of the distances color code in Figure&nbsp;3.13."><img src="./figures/render-qm-ti-annotations-dist.png" class="img-fluid figure-img" data-fig-scap="Visual explanation of the proton distances."></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-tidists-annotation-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.12: Visual explanation of the distances color code in <a href="#fig-tidists" class="quarto-xref">Figure&nbsp;3.13</a>.
</figcaption>
</figure>
</div></div><div id="fig-tidists" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="Distances of the TI to the nearest proton.">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-tidists-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./plots/qm/wethyd_us_tidists.png" class="lightbox" data-gallery="quarto-lightbox-gallery-48" title="Figure&nbsp;3.13: Small multiples plot of the approach of the hydroxide in terms of distances of the carbonyl \ce{O} of the TI to nearest proton of either a MM solvent molecule (cyan), the protein (black) or the QM water (magenta) across all umbrella windows. Distances are average within each window and the windows belonging to the same approach simulation are connected. The thick lines are a LOESS curve across all windows. Figure&nbsp;3.12 provides a visual aid for the color coding."><img src="./plots/qm/wethyd_us_tidists.png" class="img-fluid figure-img" data-fig-scap="Distances of the TI to the nearest proton."></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-tidists-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.13: Small multiples plot of the approach of the hydroxide in terms of distances of the carbonyl <span class="math inline">\(\ce{O}\)</span> of the TI to nearest proton of either a MM solvent molecule (cyan), the protein (black) or the QM water (magenta) across all umbrella windows. Distances are average within each window and the windows belonging to the same approach simulation are connected. The thick lines are a LOESS curve across all windows. <a href="#fig-tidists-annotation" class="quarto-xref">Figure&nbsp;3.12</a> provides a visual aid for the color coding.
</figcaption>
</figure>
</div>
<div style="page-break-after: always;"></div>
<p>Going over all trajectories and assigning each proton of a QM water or the initial QM hydroxide to the closes QM oxygen out of the QM water molecules, the carbonyl <span class="math inline">\(\ce{O}\)</span> and the initial hydroxide results in <a href="#fig-tiprot" class="quarto-xref">Figure&nbsp;3.15</a> (with annotations in <a href="#fig-tiprot-annotation" class="quarto-xref">Figure&nbsp;3.14</a>). The number of protons assigned to the totality of QM water molecules is not shown, as this is implicit in the sum of protons assigned to the attacking hydroxide and the carbonyl O. As the hydroxide approaches the carbonyl group, it is embedded in QM water, so it can either be in its deprotonated form or be protonated to a simple water molecule. This is why it fluctuates between having two protons and just one. At higher <span class="math inline">\(\ce{OH-}\)</span>-distances the carbonyl <span class="math inline">\(\ce{O}\)</span> very rarely gets assigned a proton, but as the hydroxide approaches and the TI is formed, the higher negative charge density attracts more protons on average. For windows in which the TI is formed, the original carbonyl <span class="math inline">\(\ce{O}\)</span> and the original hydroxide <span class="math inline">\(\ce{O}\)</span> are indistinguishable, so the hydroxide can also lose its original proton and the carbonyl <span class="math inline">\(\ce{O}\)</span> can gain a proton. All this is facilitated by the proton mobility in the QM water. Interestingly, in the triple helix systems the carbonyl <span class="math inline">\(\ce{O}\)</span> gets protonated more in the TI than in the single chain system (<a href="#fig-tiprot-combined" class="quarto-xref">Figure&nbsp;3.16</a>).</p>

<div class="no-row-height column-margin column-container"><div id="fig-tiprot-annotation" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="Visual explanation of the proton counting.">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-tiprot-annotation-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./figures/render-qm-ti-annotations.png" class="lightbox" data-gallery="quarto-lightbox-gallery-49" title="Figure&nbsp;3.14: Visual explanation of the proton counting color code in Figure&nbsp;3.15."><img src="./figures/render-qm-ti-annotations.png" class="img-fluid figure-img" data-fig-scap="Visual explanation of the proton counting."></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-tiprot-annotation-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.14: Visual explanation of the proton counting color code in <a href="#fig-tiprot" class="quarto-xref">Figure&nbsp;3.15</a>.
</figcaption>
</figure>
</div></div><div id="fig-tiprot" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-tiprot-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./plots/qm/wethyd_us_tiprots_filtered.png" class="lightbox" data-gallery="quarto-lightbox-gallery-50" title="Figure&nbsp;3.15: Protonation states of the carbonyl \ce{O} of the TI and approaching hydroxide as the TI is formed. QM water and hydroxide protons were assigned uniquely to one oxygen (carbonyl O, hydroxide \ce{O} or QM water O) for each time point within each umbrella window. Proton counts are average within each window. The windows belonging to the same approach simulation are connected and the thick lines are LOESS curves across all windows Figure&nbsp;3.14 provides a visual aid for the color coding."><img src="./plots/qm/wethyd_us_tiprots_filtered.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-tiprot-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.15: Protonation states of the carbonyl <span class="math inline">\(\ce{O}\)</span> of the TI and approaching hydroxide as the TI is formed. QM water and hydroxide protons were assigned uniquely to one oxygen (carbonyl O, hydroxide <span class="math inline">\(\ce{O}\)</span> or QM water O) for each time point within each umbrella window. Proton counts are average within each window. The windows belonging to the same approach simulation are connected and the thick lines are LOESS curves across all windows <a href="#fig-tiprot-annotation" class="quarto-xref">Figure&nbsp;3.14</a> provides a visual aid for the color coding.
</figcaption>
</figure>
</div>
<p>In the region that will be used to determine the free energy barrier of the reaction, TS1, shaded cyan in <a href="#fig-tiprot-combined" class="quarto-xref">Figure&nbsp;3.16</a>, a number of data points can be seen, where the hydroxyl <span class="math inline">\(\ce{O}\)</span> initiating the reaction still has two protons assigned to it, meaning it is still a water molecule at this point. In fact, the average number of protons in the middle of the TS1 region is around 1.5. The availability of a proton acceptor around the attacking hydroxide to turn it into an electrophile should be crucial for overcoming the energy barrier towards the TI.</p>
<div id="fig-tiprot-combined" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-tiprot-combined-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./plots/qm/wethyd_us_tiprots_combined.png" class="lightbox" data-gallery="quarto-lightbox-gallery-51" title="Figure&nbsp;3.16: Further average of Figure&nbsp;3.15. The protonation states are shown as the average number of protons assigned to either the hydroxyl \ce{O} or the carbonyl \ce{O} with all sampling windows averaged for each system. The region for detection of TS1 is shaded cyan, while the region for detection of the TI is shaded gray."><img src="./plots/qm/wethyd_us_tiprots_combined.png" class="img-fluid figure-img" style="width:50.0%"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-tiprot-combined-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.16: Further average of <a href="#fig-tiprot" class="quarto-xref">Figure&nbsp;3.15</a>. The protonation states are shown as the average number of protons assigned to either the hydroxyl <span class="math inline">\(\ce{O}\)</span> or the carbonyl <span class="math inline">\(\ce{O}\)</span> with all sampling windows averaged for each system. The region for detection of TS1 is shaded cyan, while the region for detection of the TI is shaded gray.
</figcaption>
</figure>
</div>
</section>
<section id="the-triple-helix-marginally-impedes-hydroxide-attack-thermodynamically" class="level3" data-number="3.4.2">
<h3 data-number="3.4.2" class="anchored" data-anchor-id="the-triple-helix-marginally-impedes-hydroxide-attack-thermodynamically"><span class="header-section-number">3.4.2</span> The triple helix marginally impedes hydroxide attack thermodynamically</h3>
<p>Umbrella sampling (see <a href="#sec-meth-us" class="quarto-xref">Section&nbsp;2.9</a>) allows calculating free energies along a reaction coordinate, which in this case is the distance between the attacking hydroxide and the carbonyl center. To provide better sampling coverage, all sampling windows belonging to the same site (<a href="#fig-sites" class="quarto-xref">Figure&nbsp;3.6</a>) are pooled regardless of the initial approach trajectory (of which there are 5 to 10 per site, see <a href="#fig-hist-starting-frames" class="quarto-xref">Figure&nbsp;3.9 (a)</a>)</p>
<p>Applying WHAM to the windows, after filtering out the unwanted side-reactions, results in free energy profiles of the reaction coordinate (<a href="#fig-ts1-profs" class="quarto-xref">Figure&nbsp;3.17</a>). From these, the free energies for TS1 and the TI are computed. All energies (<span class="math inline">\(\Delta G\)</span>) are reported as differences from the baseline of the educt state, where the hydroxide ion is far away and solvated. This baseline energy is defined as the mean free energy at the reaction coordinate of 0.45 ± 0.02&nbsp;nm for each profile. After subtraction of the baseline, the TS1 free energy barrier is the highest point in the cyan shaded region, while the TI energy is the lowest point in the region shaded in gray.</p>
<div id="fig-ts1-profs" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="Free energy profiles of TS1 and the TI.">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-ts1-profs-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./plots/qm/wethyd_us_profiles.png" class="lightbox" data-gallery="quarto-lightbox-gallery-52" title="Figure&nbsp;3.17: Free energy profiles obtained via WHAM of the umbrella sampling of the attack of the hydroxide (via TS1). The region for detection of TS1 is shaded cyan, while the region for detection of the TI is shaded gray. The baseline energy of the educt state has been subtracted. The shaded region around the lines represents the error estimate from bootstrapping."><img src="./plots/qm/wethyd_us_profiles.png" class="img-fluid figure-img" data-fig-scap="Free energy profiles of TS1 and the TI."></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-ts1-profs-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.17: Free energy profiles obtained via WHAM of the umbrella sampling of the attack of the hydroxide (via TS1). The region for detection of TS1 is shaded cyan, while the region for detection of the TI is shaded gray. The baseline energy of the educt state has been subtracted. The shaded region around the lines represents the error estimate from bootstrapping.
</figcaption>
</figure>
</div>
<p>The profiles are converged and exhibit low bootstrapping errors. As the hydroxide approaches (from right to left), the free energy increases. While all profiles at least level off around the distance that is characteristic for the TI, not all show a clear dip for the TI. This indicates that in order to form a stable intermediate, the TI needs a high degree of coordination with the solvent molecules that can be entropically unfavored in a complex dynamical system.</p>
<p>Notably, no amino-acid-specific effects can be observed in the free energy profiles and the free energy barriers have a considerable spread. The standard deviation of the TS1 barrier is 7.3&nbsp;kJ/mol for the single peptide and 17.7 kJ/mol for the triple helix. According to the observations from <a href="#fig-tidists" class="quarto-xref">Figure&nbsp;3.13</a> and <a href="#fig-tiprot" class="quarto-xref">Figure&nbsp;3.15</a>, the stabilization of the TI is mostly provided by the QM water molecules and not hydrogen bridges with the rest of the protein (be it QM or MM atoms). This indicates that while the local chemical environment around the sampled site has a strong influence on the energetics of the reaction, this influence is mostly governed by the solvent arrangement around the reaction center. In turn, how well the highly dynamic solvent system can organize around the peptide bond is subject to fluctuations and entropic contributions that take a lot of computational time to sample.</p>
<p>A pairwise comparison between the single peptide and triple helix energies for each site shows no significant difference for the TI (<a href="#fig-ti-dg" class="quarto-xref">Figure&nbsp;3.18 (a)</a>) and a significant difference for TS1 (<a href="#fig-ts1-dg" class="quarto-xref">Figure&nbsp;3.18 (b)</a>). The average reaction free energy of TS1 for the single chain system across all sites was 102.5 ±&nbsp;2.6&nbsp;kJ/mol (±SEM) and 111.6&nbsp;±&nbsp;6.6&nbsp;kJ/mol for the triple helix. However, it should be noted that the most appropriate significance test here is a paired <em>t</em>-test, because each site is represented in its single chain and triple helix configuration and the residuals are roughly normally distributed, and the alternative hypothesis is the original hypothesis that the triple helix would lead to higher energies (protecting collagen from being hydrolyzed). All these assumptions in place result in a <em>p</em>-value of 0.04. This also means that due to the large spread, dropping any of the requirements to perform the significance test in the manner described, would result in a failure to reject the null hypothesis.</p>
<p>The average <span class="math inline">\(\Delta\Delta\)</span>G going from a single peptide to same site in the triple helix was 9.02&nbsp;±4.41 kJ/mol (±SEM). While this may not seem like a large difference in free energy, it should be noted that since the reaction barrier enters exponentially into reaction rate calculations (see <a href="#eq-arrhenius" class="quarto-xref">Equation&nbsp;2.33</a>), this difference still corresponds to a decrease in the reaction rate by a factor of 30 to 40.</p>
<div id="fig-ts1" class="quarto-layout-panel">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-ts1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-ts1" style="flex-basis: 50.0%;justify-content: flex-start;">
<div id="fig-ti-dg" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-ti-dg-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./plots/qm/wethyd_us_deltaE_comparison_ti.png" class="lightbox" data-gallery="fig-ts1" title="Figure&nbsp;3.18&nbsp;(a): Comparison of the reaction free energies of the TI after subtraction of the educt state energy as a baseline for the single peptide and triple helix systems. The same site in both systems is connected with a line, color coded by the difference. "><img src="./plots/qm/wethyd_us_deltaE_comparison_ti.png" class="img-fluid figure-img" data-ref-parent="fig-ts1"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-ti-dg-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(a) Comparison of the reaction free energies of the TI after subtraction of the educt state energy as a baseline for the single peptide and triple helix systems. The same site in both systems is connected with a line, color coded by the difference.
</figcaption>
</figure>
</div>
</div>
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-ts1" style="flex-basis: 50.0%;justify-content: flex-start;">
<div id="fig-ts1-dg" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-ts1-dg-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./plots/qm/wethyd_us_deltaE_comparison.png" class="lightbox" data-gallery="fig-ts1" title="Figure&nbsp;3.18&nbsp;(b): Comparison of the reaction free energy barriers of TS1 (baseline is the educt state) for the single peptide and triple helix systems. The same site in both systems is connected with a line, color coded by the difference. The star indicates statistical significance at \alpha=0.05. The p-value of a paired t-test with the alternative hypothesis \Delta\Delta G>0 is 0.04."><img src="./plots/qm/wethyd_us_deltaE_comparison.png" class="img-fluid figure-img" data-ref-parent="fig-ts1"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-ts1-dg-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(b) Comparison of the reaction free energy barriers of TS1 (baseline is the educt state) for the single peptide and triple helix systems. The same site in both systems is connected with a line, color coded by the difference. The star indicates statistical significance at <span class="math inline">\(\alpha\)</span>=0.05. The <em>p</em>-value of a paired <em>t</em>-test with the alternative hypothesis <span class="math inline">\(\Delta\Delta G&gt;0\)</span> is 0.04.
</figcaption>
</figure>
</div>
</div>
</div>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-ts1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.18: Results of the umbrella sampling of the attack of the hydroxide, reaching the TI via TS1.
</figcaption>
</figure>
</div>
</section>
<section id="proton-mobility-in-quantum-mechanically-described-solvent-molecules-allows-breaking-of-the-peptide-bond" class="level3 page-columns page-full" data-number="3.4.3">
<h3 data-number="3.4.3" class="anchored" data-anchor-id="proton-mobility-in-quantum-mechanically-described-solvent-molecules-allows-breaking-of-the-peptide-bond"><span class="header-section-number">3.4.3</span> Proton mobility in quantum mechanically described solvent molecules allows breaking of the peptide bond</h3>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="./figures/render-qm-break.png" class="lightbox" data-gallery="quarto-lightbox-gallery-55" title="The QM subsystem (in color) and the MM subsystem (in black) of a single chain system during the rupture of the peptide bond (\ce{C-N})."><img src="./figures/render-qm-break.png" class="img-fluid figure-img" alt="The QM subsystem (in color) and the MM subsystem (in black) of a single chain system during the rupture of the peptide bond (\ce{C-N})."></a></p>
<figcaption>The QM subsystem (in color) and the MM subsystem (in black) of a single chain system during the rupture of the peptide bond (<span class="math inline">\(\ce{C-N}\)</span>).</figcaption>
</figure>
</div>

<div class="no-row-height column-margin column-container"><div id="fig-hyd-reaction-ts2" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-hyd-reaction-ts2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./img/hydrolysis-reaction-vertical-ts2.svg" class="lightbox" data-gallery="quarto-lightbox-gallery-56" title="Figure&nbsp;3.19: Reminder of where we are in the hydrolysis reaction"><img src="./img/hydrolysis-reaction-vertical-ts2.svg" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-hyd-reaction-ts2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.19: Reminder of where we are in the hydrolysis reaction
</figcaption>
</figure>
</div></div><p>After the TI is formed, in a second step of this multi-step hydrolysis reaction, the <span class="math inline">\(\ce{C-N}\)</span> bond along the protein backbone is cleaved, after first being protonated at the <span class="math inline">\(\ce{N}\)</span> to form a zwitterionic intermediate (ZI) according to the reaction scheme (<a href="#fig-hyd-reaction-ts2" class="quarto-xref">Figure&nbsp;3.19</a>). The scission is again followed by protonation state changes to form a negatively charged C-terminus (carboxyl group) and a positively charged N-terminus (amino group). This was simulated as a single reaction coordinate, the elongation of the <span class="math inline">\(\ce{C-N}\)</span> bond. Again, initial reactive trajectories were obtained by pulling with a harmonic potential, from which 50 equally spaced snapshots along the reaction coordinate were extracted. Those snapshots then served as the starting points for umbrella sampling simulations during which an umbrella potential kept the pulling distance near the starting value of each snapshot. While TS1 and the TI are the focus of this study, comparisons of the free energy profile of the breaking of the peptide bond obtained via WHAM are shown in <a href="#fig-ts2" class="quarto-xref">Figure&nbsp;3.20 (a)</a> for three pairs of single chain and triple helix systems. As they only have limited sampling, they don’t allow conclusions about difference between the single chain and triple helix for breaking of the peptide bond, but they do show that it is possible to run end-to-end simulations of the complete base-catalyzed peptide bond hydrolysis reaction with minimal restraints.</p>
<div id="fig-break" class="quarto-layout-panel">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-break-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="quarto-layout-row">
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-break" style="flex-basis: 40.0%;justify-content: flex-start;">
<div id="fig-ts2" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="Free energy profiles of TS2.">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-ts2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./plots/qm/wetbreak_us_profiles_example.png" class="lightbox" data-gallery="fig-break" title="Figure&nbsp;3.20&nbsp;(a): Free energy profiles obtained via WHAM of the umbrella sampling of the breaking of the peptide bond (TS2). "><img src="./plots/qm/wetbreak_us_profiles_example.png" class="img-fluid figure-img" data-fig-scap="Free energy profiles of TS2." data-ref-parent="fig-break"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-ts2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(a) Free energy profiles obtained via WHAM of the umbrella sampling of the breaking of the peptide bond (TS2).
</figcaption>
</figure>
</div>
</div>
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-break" style="flex-basis: 60.0%;justify-content: flex-start;">
<div id="fig-break-prot" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="Protonation states during TS2.">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-break-prot-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./plots/qm/wetbreak_us_tiprots_example.png" class="lightbox" data-gallery="fig-break" title="Figure&nbsp;3.20&nbsp;(b): Protonation states of the carboxyl group (\ce{COO-}) and the amino group (\ce{NH3+}) as the \ce{C-N} bond is pulled apart. QM protons \ce{H+} were assigned uniquely to one oxygen or nitrogen (carboxyl \ce{O}, QM water \ce{O} or amino group \ce{N}) for each time point within each umbrella window. Proton counts are average within each window. The lines are LOESS curves across all sampling windows. Protons assigned to the QM water molecules are not shown, as they can be inferred from the sum of protons assigned to either of the groups. Figure&nbsp;3.21 provides a visual aid for the color coding."><img src="./plots/qm/wetbreak_us_tiprots_example.png" class="img-fluid figure-img" data-fig-scap="Protonation states during TS2." data-ref-parent="fig-break"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-break-prot-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(b) Protonation states of the carboxyl group (<span class="math inline">\(\ce{COO-}\)</span>) and the amino group (<span class="math inline">\(\ce{NH3+}\)</span>) as the <span class="math inline">\(\ce{C-N}\)</span> bond is pulled apart. QM protons <span class="math inline">\(\ce{H+}\)</span> were assigned uniquely to one oxygen or nitrogen (carboxyl <span class="math inline">\(\ce{O}\)</span>, QM water <span class="math inline">\(\ce{O}\)</span> or amino group <span class="math inline">\(\ce{N}\)</span>) for each time point within each umbrella window. Proton counts are average within each window. The lines are LOESS curves across all sampling windows. Protons assigned to the QM water molecules are not shown, as they can be inferred from the sum of protons assigned to either of the groups. <a href="#fig-break-annotation" class="quarto-xref">Figure&nbsp;3.21</a> provides a visual aid for the color coding.
</figcaption>
</figure>
</div>
</div>
</div>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-break-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.20: Results of the umbrella sampling of the breaking of the peptide bond.
</figcaption>
</figure>
</div>

<div class="no-row-height column-margin column-container"><div id="fig-break-annotation" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="Visual explanation of the distances during peptide bond breaking.">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-break-annotation-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./figures/render-qm-break-annotations-dist.png" class="lightbox" data-gallery="quarto-lightbox-gallery-59" title="Figure&nbsp;3.21: Visual explanation of the proton counting color code in Figure&nbsp;3.20 (b)"><img src="./figures/render-qm-break-annotations-dist.png" class="img-fluid figure-img" data-fig-scap="Visual explanation of the distances during peptide bond breaking."></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-break-annotation-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3.21: Visual explanation of the proton counting color code in <a href="#fig-break-prot" class="quarto-xref">Figure&nbsp;3.20 (b)</a>
</figcaption>
</figure>
</div></div><p>Furthermore, looking into the movement of the protons in the QM subsystem during all sampling windows reveals different possible sequences of events for the final steps of the reaction (<a href="#fig-break-prot" class="quarto-xref">Figure&nbsp;3.20 (b)</a>). As chemists, we like to simplify reactions into easy to manage reaction steps. While this is an incredibly useful model, reality is messier than that <a href="#fn4" class="footnote-ref" id="fnref4" role="doc-noteref"><sup>4</sup></a>. We see that the reaction can occur in the sequence shown in the reaction schema like in panel Gly840-Ala841-single in the top right or Ala854-Gly846-single in the top left, where the ZI is formed by first protonating the <span class="math inline">\(\ce{N}\)</span>, making it go from 1 to 2 protons, before the carboxyl group is deprotonated. But it can also happen in a concerted manner, whereby the amino group gets protonated at the same time as the carboxyl group gets deprotonated as seen in the top and bottom middle panels Ala845-Gly846-triple and Hyp851-Gly852-single. As of now there is not enough data to make inferences about the energy landscape of both pathways.</p>
</section>
</section>
<section id="discussion-and-outlook-of-the-hybrid-simulations" class="level2" data-number="3.5">
<h2 data-number="3.5" class="anchored" data-anchor-id="discussion-and-outlook-of-the-hybrid-simulations"><span class="header-section-number">3.5</span> Discussion and outlook of the hybrid simulations</h2>
<p>It should be noted that the absolute energies and activation energies of this study cannot directly be compared to the energies obtained by <span class="citation" data-cites="pillMechanicalActivationDrastically2019">[<a href="#ref-pillMechanicalActivationDrastically2019" role="doc-biblioref">20</a>]</span> shown in <a href="#fig-pill" class="quarto-xref">Figure&nbsp;3.3</a>. The former correspond to free energies (<span class="math inline">\(G\)</span>), which includes entropic contributions, while the latter represent the electronic energy of minimized structures. This is why it is not unusual that the average energy of the TI in this study is around 125&nbsp;kJ/mol over the baseline of the educt state, while the average electronic TI energy for the minimized structure is only around 20&nbsp;KJ/mol. Comparisons within each reference frame are still valid, but a direct comparison between the two approaches remains elusive.</p>
<p>The unconstrained nature of the chosen approach is a double-edged sword. It enabled sampling of interesting regions of phase space and direct observation of physically relevant behavior such as proton mobility in the QM region during the reaction. This also showed how the base-catalyzed hydrolysis reaction can proceed in a varying sequence of steps, with the ZI forming dynamically as the peptide bond is rupturing. However, the broad sampling makes it harder to attribute the results to any specific structural effect without even higher computational costs.</p>
<p>While a difference in free energy was found for the activation energy of the hydroxide attack between single chain and triple helix systems, the small effect size does not fully explain the prevalence of the competing reaction (homolytic backbone cleavage) that is observed experimentally in tissue samples.</p>
<p>Furthermore, there is a risk of survivorship bias. Finding the set of parameters that allow simulating the reaction without crashing and the initial selection of the hydrolysis sites is likely to favor easier to access and simulate systems. This is also apparent in <a href="#fig-carbonyl-orientation" class="quarto-xref">Figure&nbsp;5.2 (b)</a>, which shows that the current selection of peptide bonds tends to have the carbonyl oxygen point away from the helix, making the tetrahedral intermediate easier to access by the solvent molecules. It also highlights sites of future simulations to mitigate this bias. The trajectories that are harder to simulate are those with a rougher free energy landscape and those with higher energy states. The new simulations include more of those, such a peptide bond between a proline and a hydroxyproline residue. Interestingly, because the sites that are harder to stabilize in the triple helix due to their orientation were underrepresented in the initial dataset, intuition would predict that the true effect of the triple helix might be even greater than what was shown here already.</p>
<p>The fundamental limitation is computational cost: simulating a single umbrella window on one HPC node using 20 cores at 2.1 GHz for 5&nbsp;ps took on average 60 hours. With 50 windows per approach trajectory and 8 starting configurations on average, this sums up to 1000 CPU-days per site.</p>
<p>Going to even larger systems (full fibrils) is computationally unfeasible with the current QM/MM approach. Further sampling at additional sites would shed more light on the details of the reaction and eliminate potential bias in the selected attack sites, but it would not overcome the fundamental scaling limit. While the compute time could be reduced through further optimizations and targeted sampling of the most critical regions, comparing two competing reactions at scale in realistic system sizes with sufficient sampling requires a fundamentally different approach.</p>
<p>This limitation motivates the development of KIMMDY (<a href="#sec-main-kimmdy" class="quarto-xref">Chapter&nbsp;4</a>), which then incorporates insights from these QM/MM simulations into an efficient emulator of hydrolysis that can simulate many reactions even in large collagen fibrils.</p>
<div style="page-break-after: always;"></div>
</section>
</section>
<section id="sec-main-kimmdy" class="level1 page-columns page-full" data-number="4">
<h1 data-number="4"><span class="header-section-number">4</span> Kinetic Monte Carlo Molecular Dynamics: KIMMDY</h1>

<div class="no-row-height column-margin column-container"><div id="fig-kimmdy-logo" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="KIMMDY logo.">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-kimmdy-logo-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./img/kimmdy-logo.svg" class="lightbox" data-gallery="quarto-lightbox-gallery-60" title="Figure&nbsp;4.1: The logo of KIMMDY, designed by Denis Kieswetter and fine tuned by me."><img src="./img/kimmdy-logo.svg" class="img-fluid figure-img" data-fig-scap="KIMMDY logo."></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-kimmdy-logo-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4.1: The logo of KIMMDY, designed by Denis Kieswetter and fine tuned by me.
</figcaption>
</figure>
</div></div><div id="fig-kimmdy-schema" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="KIMMDY schema.">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-kimmdy-schema-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./figures/kimmdy-schema.png" class="lightbox" data-gallery="quarto-lightbox-gallery-61" title="Figure&nbsp;4.2: The KIMMDY schema that Denis Kieswetter made for @kimmdy2025. Given an initial state at t_0, KIMMDY generates an ensemble, typically via MD. Reaction plugins then provide rates for their potential reactions. KIMMDY then chooses a reaction and updates the global time. Next it effects the reaction, adapting the topology and coordinates of the system to the new state after the reaction. A new round of sampling can begin."><img src="./figures/kimmdy-schema.png" class="img-fluid figure-img" style="width:60.0%" data-fig-scap="KIMMDY schema."></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-kimmdy-schema-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4.2: The KIMMDY schema that Denis Kieswetter made for <span class="citation" data-cites="kimmdy2025">[<a href="#ref-kimmdy2025" role="doc-biblioref">99</a>]</span>. Given an initial state at <span class="math inline">\(t_0\)</span>, KIMMDY generates an ensemble, typically via MD. Reaction plugins then provide rates for their potential reactions. KIMMDY then chooses a reaction and updates the global time. Next it effects the reaction, adapting the topology and coordinates of the system to the new state after the reaction. A new round of sampling can begin.
</figcaption>
</figure>
</div>
<p>KIMMDY stands for Kinetic Monte Carlo Molecular Dynamics, a concept that was first devised by Benedikt Rennekamp to tackle the question of homolytic rupture in large molecular dynamics simulations of collagen fibrils <span class="citation" data-cites="rennekampHybridKineticMonte2020">[<a href="#ref-rennekampHybridKineticMonte2020" role="doc-biblioref">100</a>]</span>. The core idea is this: Molecular dynamics simulations, in force field based methods, allow simulating and sampling large systems over relatively long periods of time. However, they lack the key feature of life, chemistry. Bonds can not break or form in these simulations, so no reactions can occur. Furthermore, even new reactive simulation methods like reactive force fields <span class="citation" data-cites="vanduinReaxFFReactiveForce2001 senftleReaxFFReactiveForcefield2016">[<a href="#ref-vanduinReaxFFReactiveForce2001" role="doc-biblioref">101</a>, <a href="#ref-senftleReaxFFReactiveForcefield2016" role="doc-biblioref">102</a>]</span>, or machine-learned potentials <span class="citation" data-cites="schuttSchNetDeepLearning2018 behlerGeneralizedNeuralNetworkRepresentation2007 smithANI1ExtensibleNeural2017 NEURIPS2022_4a36c3c5 musaelianLearningLocalEquivariant2023 mazitovPETMADUniversalInteratomic2025">[<a href="#ref-schuttSchNetDeepLearning2018" role="doc-biblioref">103</a>–<a href="#ref-mazitovPETMADUniversalInteratomic2025" role="doc-biblioref">108</a>]</span>, reach a fundamental limit when it comes to reactions whose dwell-time exceeds the time that can reasonably be simulated by orders of magnitude. This was previously solved by my colleague Benedikt for force-dependent homolysis, by viewing a molecular dynamics simulation as a sampling step in a Monte Carlo scheme, during which the bond lengths are recorded and used in a physical model of homolysis that converts forces acting on bonds into activation energies for the homolysis. Those activation energies are then converted to reaction rates with the Arrhenius equation (<a href="#eq-arrhenius" class="quarto-xref">Equation&nbsp;2.33</a>). A set of python scripts then realized the scheme to sample with a MD simulation, wait for the simulation to complete and calculate the predicted rupture rates for all backbone bonds. A reaction was then chosen with the Monte Carlo scheme (see <a href="#sec-kmc" class="quarto-xref">Section&nbsp;2.14</a>) and the topology of the system was modified to accommodate the change. This then allows for the next sampling MD and looking for follow-up reactions.</p>
<p>When my colleagues Kai Riedmiller, Eric Hartmann and I heard of this, we decided that such a scheme should be applicable not just to homolysis, but to all possible chemical reactions. The name KIMMDY was kept, but it was now realized as a user-friendly, extensible and powerful framework written in python and accessible via the command line and configuration files. Some of the figures I will show in this chapter have already been published in our manuscript, which is available on BioRxiv <span class="citation" data-cites="kimmdy2025">[<a href="#ref-kimmdy2025" role="doc-biblioref">99</a>]</span> and at the time of writing under review at Nature Communications. As not only a joint first author paper, but also a collaborative software project with tens of thousands of lines of code added, removed and changed, it is not straightforward to pin any given feature on a single one of us. For this thesis, I will do my best to highlight especially the parts on which I had a particular focus, but nonetheless it remains the product of an incredibly fruitful collaboration.</p>
<p><a href="#fig-kimmdy-schema" class="quarto-xref">Figure&nbsp;4.2</a> shows an overview of how KIMMDY works. KIMMDY is based on a plugin architecture. Possible reactions, such as bond rupture, hydrolysis, or hydrogen atom transfer (HAT) are implemented as plugins that report back reaction rates. KIMMDY then takes care of executing the chosen reaction, stabilizing the system, and running the sampling simulations. In practice those are MD simulations, though KIMMDY is agnostic with regards to the type of simulation that generates the ensembles. The reaction plugins are also free to use any method necessary to obtain reaction rates based on the state of the system. This could be a heuristic fit to experimental data, a physical model or a machine learned one. All are present in the currently available plugins, such as the machine learned model predicting HAT by Kai, a plugin based on Benedikt’s original physical model for homolysis or my heuristic model for peptide bond hydrolysis. But most importantly, we provide extensive documentation to enable any user to write their own reaction plugin and get started with KIMMDY: <a href="https://graeter-group.github.io/kimmdy/">graeter-group.github.io/kimmdy/</a>.</p>
<p>The structure of our documentation is based on the Diátaxis framework developed by Daniele Procida, who explains it further on <a href="https://diataxis.fr/">diataxis.fr</a> <span class="citation" data-cites="Procida_Diataxis_documentation_framework">[<a href="#ref-Procida_Diataxis_documentation_framework" role="doc-biblioref">109</a>]</span>. Briefly, each piece of information and writing is categorized by whether its primary purpose is user-action or user-cognition oriented and whether the goal is to acquire knowledge or to solve an immediate application problem (<a href="#fig-diataxis" class="quarto-xref">Figure&nbsp;4.3</a>). This creates distinct types of documentation. Tutorials are oriented towards learning and require active participation by the user. How-To Guides are directed at a specific task, which is actively performed by the user. Explanations are for understanding and are consumed passively. A Reference serves to provide task-specific documentation.</p>

<div class="no-row-height column-margin column-container"><div id="fig-diataxis" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-diataxis-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./img/diataxis.svg" class="lightbox" data-gallery="quarto-lightbox-gallery-62" title="Figure&nbsp;4.3: Schema of the diataxis documentation framework, based on a figure by Daniele Procida [@Procida_Diataxis_documentation_framework]."><img src="./img/diataxis.svg" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-diataxis-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4.3: Schema of the diataxis documentation framework, based on a figure by Daniele Procida <span class="citation" data-cites="Procida_Diataxis_documentation_framework">[<a href="#ref-Procida_Diataxis_documentation_framework" role="doc-biblioref">109</a>]</span>.
</figcaption>
</figure>
</div></div><p>In this chapter, I will first go over how KIMMDY works and use the typical flow of a user setting up a KIMMDY run as the guiding thread to explain how they interact with KIMMDY and what goes on under the hood. Then I will apply KIMMDY to the biological question posed in the very beginning, integrating insights from the QM/MM simulations into an emulator for peptide bond hydrolysis to directly compare reaction rates of homolysis and hydrolysis in a large collagen fibril simulation.</p>
<section id="running-kimmdy-implementation-and-usage" class="level2 page-columns page-full" data-number="4.1">
<h2 data-number="4.1" class="anchored" data-anchor-id="running-kimmdy-implementation-and-usage"><span class="header-section-number">4.1</span> Running KIMMDY: implementation and usage</h2>
<p>Note: the implementation details of KIMMDY have already been described by me and edited by my co-authors in the supplementary materials of <span class="citation" data-cites="kimmdy2025">[<a href="#ref-kimmdy2025" role="doc-biblioref">99</a>]</span>. As such, sections of text will be similar and may only have minor adaptions for the context of this thesis.</p>
<section id="installation" class="level3" data-number="4.1.1">
<h3 data-number="4.1.1" class="anchored" data-anchor-id="installation"><span class="header-section-number">4.1.1</span> Installation</h3>
<p>Let us picture a user who has a new type of reaction that they want to simulate within MD. They are already familiar with their MD engine GROMACS and have the files to simulate their system without any reactions. After going to our documentation website they install KIMMDY. KIMMDY can just be install from the Python package index (PyPI) with <code>pip install</code>, but to speed up installing dependencies, especially those of our machine learning plugins, we recommend using <a href="https://docs.astral.sh/uv/"><code>uv</code></a>, e.g., with <code>uv tool install -p 3.11 kimmdy</code>. Now the <code>kimmdy</code> command is available on the command line. Typing <code>kimmdy --help</code> brings up more options:</p>
<div class="cell-output cell-output-stdout">
<pre><code>usage: kimmdy [-h] [--input INPUT] [--restart] [--loglevel LOGLEVEL]
              [--logfile LOGFILE] [--show-plugins] [--generate-jobscript]
              [--version] [--debug] [--callgraph]

Welcome to KIMMDY. `kimmdy` runs KIMMDY. Additional tools are available as
`kimmdy-...` commands. These are `-analysis`, `-modify-top` and `-build-
examples`. Access their help with `kimmdy-... -h.`Visit the documentation
online at &lt;https://graeter-group.github.io/kimmdy/&gt;

options:
  -h, --help            show this help message and exit
  --input INPUT, -i INPUT
                        Kimmdy input file. Defaults to `kimmdy.yml`. See
                        &lt;https://graeter-
                        group.github.io/kimmdy/guide/references/input.html&gt;
                        for all options. CLI flags (e.g. --restart or
                        --loglevel) have precedence over their counterparts in
                        the input file.
  --restart, -r         Restart or continue from a previous run instead of
                        incrementing the run number for the output directory.
                        If the output directory does not exist, it will be
                        like a regular fresh run.
  --loglevel LOGLEVEL, -l LOGLEVEL
                        Logging level (CRITICAL, ERROR, WARNING, INFO, DEBUG)
  --logfile LOGFILE, -f LOGFILE
                        Logfile
  --show-plugins        List available plugins
  --generate-jobscript  Instead of running KIMMDY directly, generate the
                        output directory and a jobscript `jobscript.sh` for
                        slurm HPC clusters. You can then run this jobscript
                        with sbatch jobscript.sh.
  --version             Show version and exit.
  --debug               On error, drop into debugger
  --callgraph           Generate a visualization of function calls for
                        debugging and documentation.</code></pre>
</div>
<p>Additional provided tools include <code>kimmdy-analysis</code> for post-simulation analysis, and <code>kimmdy-modify-top</code> for modifying or parametrizing a system outside of a KIMMDY run.</p>
</section>
<section id="configuration-and-inputoutput" class="level3" data-number="4.1.2">
<h3 data-number="4.1.2" class="anchored" data-anchor-id="configuration-and-inputoutput"><span class="header-section-number">4.1.2</span> Configuration and input/output</h3>
<p>KIMMDY is controlled via the command-line interface (CLI) and a <code>yaml</code> formatted configuration file. In there, every task KIMMDY performs has a corresponding configuration section. The sequence of tasks KIMMDY executes is also defined in this configuration, making KIMMDY highly flexible. The configuration file contains input files such as the topology and initial coordinates, high performance computing (HPC) options, logging and working directory options, and general settings for KIMMDY. For a complete documentation of all options, see the automatically updated list at <a href="https://graeter-group.github.io/kimmdy/guide/references/input.html">graeter-group.github.io/kimmdy/guide/references/input.html</a>.</p>
<p>When the <code>yaml</code> file, typically named <code>kimmdy.yml</code>, is opened in properly configured code editor supporting the <a href="https://microsoft.github.io/language-server-protocol/overviews/lsp/overview/">language server protocol</a>, such as the popular <a href="https://code.visualstudio.com/">Visual Studio Code</a> or <a href="https://neovim.io/">Neovim</a>, the user will automatically get features such as auto completion for keywords and options (<a href="#fig-kimmdy-completion" class="quarto-xref">Figure&nbsp;4.4 (b)</a>), hover documentation (<a href="#fig-kimmdy-hover" class="quarto-xref">Figure&nbsp;4.4 (a)</a>). They will also see warnings for missing, unknown keys or incompatible values (<a href="#fig-kimmdy-warnings" class="quarto-xref">Figure&nbsp;4.4 (c)</a>). This reduces the likelihood of disappointing application crashes, because the validity of the input can be checked before KIMMDY is even started. Not everything can be checked before runtime, such as the availability of some plugin dependencies or the existence of input files, but further consistency checks are front loaded at the start of KIMMDY, such that it stops earlier rather than later in a run. KIMMDY checks that all input files are present, which avoids running a whole MD simulation only to find out that the mdp file for the next step is missing.</p>
<div id="fig-kmdy-yml" class="quarto-layout-panel">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-kmdy-yml-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="quarto-layout-row">
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-kmdy-yml" style="flex-basis: 57.6%;justify-content: flex-start;">
<div id="fig-kimmdy-hover" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-kimmdy-hover-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="img/kimmdy-yml-hover.png" class="lightbox" data-gallery="fig-kmdy-yml" title="Figure&nbsp;4.4&nbsp;(a): kimmdy.yml hover documentation. "><img src="img/kimmdy-yml-hover.png" class="img-fluid figure-img" data-ref-parent="fig-kmdy-yml"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-kimmdy-hover-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(a) <code>kimmdy.yml</code> hover documentation.
</figcaption>
</figure>
</div>
</div>
<div class="quarto-figure-spacer quarto-layout-cell" style="flex-basis: 3.6%;justify-content: flex-start;">
<p>&nbsp;</p>
</div>
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-kmdy-yml" style="flex-basis: 38.8%;justify-content: flex-start;">
<div id="fig-kimmdy-completion" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-kimmdy-completion-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="img/kimmdy-yml-completion.png" class="lightbox" data-gallery="fig-kmdy-yml" title="Figure&nbsp;4.4&nbsp;(b): kimmdy.yml autocompletion."><img src="img/kimmdy-yml-completion.png" class="img-fluid figure-img" data-ref-parent="fig-kmdy-yml"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-kimmdy-completion-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(b) <code>kimmdy.yml</code> autocompletion.
</figcaption>
</figure>
</div>
</div>
</div>
<div class="quarto-layout-row">
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-kmdy-yml" style="flex-basis: 100.0%;justify-content: flex-start;">
<div id="fig-kimmdy-warnings" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-kimmdy-warnings-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="img/kimmdy-yml-warnings.png" class="lightbox" data-gallery="fig-kmdy-yml" title="Figure&nbsp;4.4&nbsp;(c): kimmdy.yml warnings."><img src="img/kimmdy-yml-warnings.png" class="img-fluid figure-img" data-ref-parent="fig-kmdy-yml"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-kimmdy-warnings-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(c) <code>kimmdy.yml</code> warnings.
</figcaption>
</figure>
</div>
</div>
</div>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-kmdy-yml-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4.4: Various features of modern text editors working with a <code>kimmdy.yml</code> file.
</figcaption>
</figure>
</div>
<p>The user then sets up the configuration file (the config) and decides on a <code>sequence</code>, a series of tasks that KIMMDY will run. This could be for example different MD simulations, such as an equilibration and a production run, followed by sampling for reactions, and a slow-growth simulation (see below) to equilibrate after each reaction. The sequence can include nested tasks and tasks with a multiplicity, i.e., tasks that are run multiple times. Each task gets its own output subdirectory in the KIMMDY run output directory and KIMMDY tracks the latest version of each type of file. This means that if one task runs an MD trajectory, the next task will continue after the trajectory (or in the middle of it, if a reaction should occur at a certain time point) and if a task modifies the topology, the next task will use the modified topology.</p>
<p>In the example shown in <a href="#fig-kmdy-yml" class="quarto-xref">Figure&nbsp;4.4</a>, the first task is an MD simulation.</p>
</section>
<section id="ensemble-generation" class="level3" data-number="4.1.3">
<h3 data-number="4.1.3" class="anchored" data-anchor-id="ensemble-generation"><span class="header-section-number">4.1.3</span> Ensemble generation</h3>
<p>In theory, KIMMDY can work with any ensemble generation method that results in atomistic trajectories. This includes <em>ab initio</em> MD, machine learned potential (MLP) simulations, classical MD, or other ensemble generation methods. In practice, the KIMMDY architecture is oriented towards GROMACS <span class="citation" data-cites="abrahamGROMACSHighPerformance2015 berendsenGROMACSMessagepassingParallel1995">[<a href="#ref-abrahamGROMACSHighPerformance2015" role="doc-biblioref">40</a>, <a href="#ref-berendsenGROMACSMessagepassingParallel1995" role="doc-biblioref">110</a>]</span> simulations, limiting the immediate uses to unbiased MD, biased MD, and in the near-future MLP simulations.</p>
<p>KIMMDY interfaces with GROMACS via its command-line interface, which makes KIMMDY robust to GROMACS version changes. In the KIMMDY config, the GROMACS binary name and flags for commands such as <code>gmx grompp</code> and <code>gmx mdrun</code> are exposed as options. The <code>.mdp</code> files for requested MD tasks and all files defining the molecular structure are taken as user input. It should be noted that albeit MD simulations are conducted, no continuous reactive trajectories are obtained. Rather, separate trajectories are generated for every state and the slow-growth simulations (see below) in between. The concatenated trajectories that KIMMDY analysis tools can create from all states are only for visualization and inspection purposes.</p>
<p>The next task in the example is checking for reactions.</p>
</section>
<section id="reactions-and-recipes" class="level3" data-number="4.1.4">
<h3 data-number="4.1.4" class="anchored" data-anchor-id="reactions-and-recipes"><span class="header-section-number">4.1.4</span> Reactions and recipes</h3>
<p>KIMMDY manages reactions via a plugin system, providing users with an easy method to expand KIMMDY’s existing functionalities. A plugin generates reaction rates alongside reaction recipes based on the latest conformational ensemble.</p>
<p>How the reaction rates are determined is up to the plugin. They can be calculated based on simple or complex physical models, predicted by a machine-learned model or be determined based on experimental heuristics. Plugins consume simulation outputs in various ways, but popular choices include using MDAnalysis <span class="citation" data-cites="gowersMDAnalysisPythonPackage2016 michaud-agrawalMDAnalysisToolkitAnalysis2011">[<a href="#ref-gowersMDAnalysisPythonPackage2016" role="doc-biblioref">111</a>, <a href="#ref-michaud-agrawalMDAnalysisToolkitAnalysis2011" role="doc-biblioref">112</a>]</span> to read trajectories or PLUMED <span class="citation" data-cites="tribelloPLUMED2New2014">[<a href="#ref-tribelloPLUMED2New2014" role="doc-biblioref">113</a>]</span> to monitor distances.</p>
<p>The reaction recipes describe all modifications to the topology and structure required to perform the reaction corresponding to the calculated rate. A recipe is composed of recipe steps, which can break a bond, create a bond, place an atom at some coordinates, modify the topology, or request a specific MD simulation. Importantly, plugins only have to think about those elemental modifications, while KIMMDY takes care of the complex implications of them (see <a href="#sec-top" class="quarto-xref">Section&nbsp;4.1.6</a>). Additionally, plugins can defer returning the actual steps of the recipe until a reaction has been chosen by KIMMDY to reduce computational cost. For example, the hydrolysis plugin returns rates for all peptide bonds in the system, but only chooses a specific water molecule to execute the attack once a bond has been chosen as the target. Reaction rates can be valid for the whole period of the sampling simulation or vary over time for each possible reaction, which is also reported back to KIMMDY.</p>
<p>The example only asks the hydrolysis plugin for rates, which are then passed on to the chosen kMC algorithm.</p>
</section>
<section id="kinetic-monte-carlo-algorithms" class="level3" data-number="4.1.5">
<h3 data-number="4.1.5" class="anchored" data-anchor-id="kinetic-monte-carlo-algorithms"><span class="header-section-number">4.1.5</span> Kinetic Monte Carlo algorithms</h3>
<p>KIMMDY makes use of Adaptive kMC. It collects all reaction rates from the plugins and uses a user-selected choice of kMC algorithm to sample a reaction from the distribution. For a more detailed theoretical background of kMC than <a href="#sec-kmc" class="quarto-xref">Section&nbsp;2.14</a>, please also refer to the SI of <span class="citation" data-cites="kimmdy2025">[<a href="#ref-kimmdy2025" role="doc-biblioref">99</a>]</span>. Choices for kMC algorithms currently include rejection free kMC (“rfkmc”), “extrande” <span class="citation" data-cites="voliotisStochasticSimulationBiomolecular2016">[<a href="#ref-voliotisStochasticSimulationBiomolecular2016" role="doc-biblioref">114</a>]</span>, a modified version of extrande (“extrande_mod”), rejection free kMC with multiple choices at the same time (“multi_rfkmc”), as well as a debugging option that always chooses the first (or last) reaction. Individual reactions can declare their preferred algorithm in addition to the global choice of kMC algorithm. In this thesis only rejection free kMC is used.</p>
</section>
<section id="sec-top" class="level3 page-columns page-full" data-number="4.1.6">
<h3 data-number="4.1.6" class="anchored" data-anchor-id="sec-top"><span class="header-section-number">4.1.6</span> Topology module</h3>
<p>From the initially user-supplied topology, an internal <code>Topology</code> object is build. The force field, as well as topology <em>includes</em>, are explicitly resolved, meaning every interaction has individual parameters. This enables the use of GRAPPA (see <a href="#sec-grappa" class="quarto-xref">Section&nbsp;4.1.7</a>) for parametrizing on-the-fly, but is still compatible with established force fields, like Amber <span class="citation" data-cites="caseAmberTools2023">[<a href="#ref-caseAmberTools2023" role="doc-biblioref">29</a>]</span> or Charmm <span class="citation" data-cites="brooksCHARMMBiomolecularSimulation2009">[<a href="#ref-brooksCHARMMBiomolecularSimulation2009" role="doc-biblioref">36</a>]</span>. The <code>Topology</code> supports breaking bonds, creating bonds, and deleting hydrogen atoms. For all of these actions, the interaction terms, like bonds, angles, dihedrals, pair exclusions etc., are updated accordingly. Furthermore, it identifies, and keeps track of possible radicals and whether it needs parametrization due to some modifications. Creating bonds, deleting bonds, and moving atoms do not change the index of the atoms stored in the topology. This ensures compatibility with standard analysis techniques, but residues are not necessarily continuous in the resulting topology afterwards. If continuous residues are needed, the <code>Topology</code> provides a function to re-index itself. After an atom is deleted, the topology is re-indexed by default. To use the <code>Topology</code> in an MD simulation, a new <code>top</code> file is generated from the internal topology object.</p>
<p>The example focuses on peptide bond hydrolysis. Assuming a bond has been chosen for to be hydrolyzed, KIMMDY then gets the recipe to be executed. For simplicity, only a small portion of the complete molecule is shown around the peptide bond in <a href="#fig-peptide-water" class="quarto-xref">Figure&nbsp;4.6</a> and <a href="#fig-top" class="quarto-xref">Figure&nbsp;4.5</a>. The plugin only had to specify bonds that are to be deleted and the new ones to create; KIMMDY takes care of higher-order interactions. For example, deleting the peptide bond <span class="math inline">\(\ce{4\ C - 6\ N}\)</span> will also remove the angle <span class="math inline">\(\ce{6\ N - 4\ C - 5\ O}\)</span> and the dihedral <span class="math inline">\(\ce{7\ H - 6\ N - 4\ C - 5\ O}\)</span> and so forth. Similarly, the atoms that where previously excluded due to being bound or part of a 1–3 or 1–4 interaction will now have nonbonded interactions. Newly formed bonds such as <span class="math inline">\(\ce{9\ OW - 4\ C}\)</span> will create the corresponding angles, dihedrals and pairs for 1–4 interactions. Even improper dihedrals that are defined in the residuetypes, such as those for peptide bonds between amino acids, are re-created based on the residuetype recorded in the topology. If the basic operations are not sufficient or non-standard changes are required, a <code>RecipeStep</code> can also bring a custom function that is free to modify the topology however necessary.</p>
<div id="fig-top" class="quarto-layout-panel">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-top-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-top" style="flex-basis: 43.5%;justify-content: flex-start;">
<div id="fig-top-before" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="Topology before hydrolysis.">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-top-before-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./assets/top-before.svg" class="lightbox" data-gallery="fig-top" title="Figure&nbsp;4.5&nbsp;(a): The peptide bond (magenta) between \ce{4\ C} and \ce{6\ N} is to be hydrolyzed. \ce{9\ OW} (O Water) will be bound to \ce{4\ C} and the two HW atoms will be bound to \ce{6\ N}."><img src="./assets/top-before.svg" class="img-fluid figure-img" data-fig-scap="Topology before hydrolysis." data-ref-parent="fig-top"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-top-before-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(a) The peptide bond (magenta) between <span class="math inline">\(\ce{4\ C}\)</span> and <span class="math inline">\(\ce{6\ N}\)</span> is to be hydrolyzed. <span class="math inline">\(\ce{9\ OW}\)</span> (O Water) will be bound to <span class="math inline">\(\ce{4\ C}\)</span> and the two HW atoms will be bound to <span class="math inline">\(\ce{6\ N}\)</span>.
</figcaption>
</figure>
</div>
</div>
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-top" style="flex-basis: 56.5%;justify-content: flex-start;">
<div id="fig-top-after" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="Topology after hydrolysis.">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-top-after-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./assets/top-after.svg" class="lightbox" data-gallery="fig-top" title="Figure&nbsp;4.5&nbsp;(b): The newly created bonds are colored in blue."><img src="./assets/top-after.svg" class="img-fluid figure-img" data-fig-scap="Topology after hydrolysis." data-ref-parent="fig-top"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-top-after-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(b) The newly created bonds are colored in blue.
</figcaption>
</figure>
</div>
</div>
</div>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-top-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4.5: Simplified graph representation of a peptide bond undergoing hydrolysis.
</figcaption>
</figure>
</div>

<div class="no-row-height column-margin column-container"><div id="fig-peptide-water" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="Water and a peptide bond.">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-peptide-water-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./img/peptide-water.png" class="lightbox" data-gallery="quarto-lightbox-gallery-68" title="Figure&nbsp;4.6: A water molecule next to a peptide bond, simplified."><img src="./img/peptide-water.png" class="img-fluid figure-img" data-fig-scap="Water and a peptide bond."></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-peptide-water-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4.6: A water molecule next to a peptide bond, simplified.
</figcaption>
</figure>
</div></div><p>All modifications to the topology are fast and don’t scale with system size through the use of hash maps to store interactions. Thus, if the key for an interaction is known, the lookup scales with O(1). Each atom in the topology atom list also keeps a list of atoms that it is connected to, which drastically reduces the search space for interactions. For example, if one wants to know if a bond exists between two particular atoms, one look into the <code>bound_to</code> list of either one suffices. And if angles or dihedrals are to be checked, a small local topology can be built up from the <code>bound_to</code> lists without having to iterate over all bonds, angles, or dihedrals of a large topology. This is how KIMMDY stays performant even for multi-million atom systems.</p>
<p>By default, only the first moleculetype of a system is available for reactions, as many reactions don’t involve the solvent molecules. However, one can easily configure includes and excludes for the <code>Reactive</code> moleculetype (see <a href="#fig-kmdy-yml" class="quarto-xref">Figure&nbsp;4.4</a>). When multiple molecules of the same moleculetype are included, such as 100 water molecules in addition to the protein (<a href="#lst-molecules" class="quarto-xref">Listing&nbsp;4.1</a>), KIMMDY creates new atoms, bonds etc. for each instance of the moleculetype with increasing atom- and residue-numbers. In doing so, the atom ids in the topology file (<code>.top</code>) and the line numbers of the coordinate file (<code>.gro</code>) always line up. This is important, as the atom indices written in a <code>gro</code> file may be wrong due to the fixed-width nature of the format (they wrap around at 1 million); only their order (i.e., line number) is guaranteed to be unique.</p>

<div class="no-row-height column-margin column-container"><div id="lst-molecules" class="ini listing quarto-float quarto-figure quarto-figure-left anchored">
<figure class="quarto-float quarto-float-lst figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-lst" id="lst-molecules-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Listing&nbsp;4.1: A topology molecules section.
</figcaption>
<div aria-describedby="lst-molecules-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="code-copy-outer-scaffold"><div class="sourceCode" id="cb7"><pre class="sourceCode numberSource ini number-lines code-with-copy"><code class="sourceCode ini"><span id="cb7-1"><a href="#cb7-1"></a><span class="kw">[ molecules ]</span></span>
<span id="cb7-2"><a href="#cb7-2"></a><span class="dt">Protein 1</span></span>
<span id="cb7-3"><a href="#cb7-3"></a><span class="dt">SOL 100</span></span></code></pre></div><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></div>
</div>
</figure>
</div></div><p>Topology and force field objects are easy to inspect and modify when working on a reaction plugin or interactively working with KIMMDY code in a notebook. Even the topology visualizations in <a href="#fig-top" class="quarto-xref">Figure&nbsp;4.5</a> were initially created with KIMMDY code. The order of operations when describing a reaction does not matter, because KIMMDY keeps a consistent state at each step. For example, <a href="#lst-topology" class="quarto-xref">Listing&nbsp;4.2</a> shows how <a href="#fig-top-after" class="quarto-xref">Figure&nbsp;4.5 (b)</a> is created from <a href="#fig-top-before" class="quarto-xref">Figure&nbsp;4.5 (a)</a> and what the output of inspecting the topology object looks like (<a href="#lst-topology-output" class="quarto-xref">Listing&nbsp;4.3</a>).</p>
<div class="quarto-layout-panel" data-layout="[6,-1,8]">
<div class="quarto-layout-row">
<div class="quarto-layout-cell" style="flex-basis: 40.0%;justify-content: flex-start;">
<div id="lst-topology" class="python listing quarto-float quarto-figure quarto-figure-left anchored" data-code-line-numbers="false">
<figure class="quarto-float quarto-float-lst figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-lst" id="lst-topology-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Listing&nbsp;4.2: Creating a topology and modifying it.
</figcaption>
<div aria-describedby="lst-topology-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="code-copy-outer-scaffold"><div class="sourceCode" id="cb8"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb8-1"><a href="#cb8-1" aria-hidden="true" tabindex="-1"></a>top <span class="op">=</span> Topology.from_path(</span>
<span id="cb8-2"><a href="#cb8-2" aria-hidden="true" tabindex="-1"></a>  <span class="st">"./assets/gly-hoh.top"</span>,</span>
<span id="cb8-3"><a href="#cb8-3" aria-hidden="true" tabindex="-1"></a>  <span class="co"># [...]</span></span>
<span id="cb8-4"><a href="#cb8-4" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb8-5"><a href="#cb8-5" aria-hidden="true" tabindex="-1"></a>top.break_bond((<span class="st">'4'</span>, <span class="st">'6'</span>))</span>
<span id="cb8-6"><a href="#cb8-6" aria-hidden="true" tabindex="-1"></a>top.break_bond((<span class="st">'9'</span>, <span class="st">'10'</span>))</span>
<span id="cb8-7"><a href="#cb8-7" aria-hidden="true" tabindex="-1"></a>top.break_bond((<span class="st">'9'</span>, <span class="st">'11'</span>))</span>
<span id="cb8-8"><a href="#cb8-8" aria-hidden="true" tabindex="-1"></a>top.bind_bond((<span class="st">'4'</span>, <span class="st">'9'</span>))</span>
<span id="cb8-9"><a href="#cb8-9" aria-hidden="true" tabindex="-1"></a>top.bind_bond((<span class="st">'10'</span>, <span class="st">'6'</span>))</span>
<span id="cb8-10"><a href="#cb8-10" aria-hidden="true" tabindex="-1"></a>top.bind_bond((<span class="st">'11'</span>, <span class="st">'6'</span>))</span></code></pre></div><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></div>
</div>
</figure>
</div>
</div>
<div class="quarto-figure-spacer quarto-layout-cell" style="flex-basis: 6.7%;justify-content: flex-start;">
<p>&nbsp;</p>
</div>
<div class="quarto-layout-cell" style="flex-basis: 53.3%;justify-content: flex-start;">
<div id="lst-topology-output" class="text listing quarto-float quarto-figure quarto-figure-left anchored" data-code-line-numbers="false">
<figure class="quarto-float quarto-float-lst figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-lst" id="lst-topology-output-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Listing&nbsp;4.3: Inspecting the topology object.
</figcaption>
<div aria-describedby="lst-topology-output-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<pre class="text"><code>Topology with the following
molecules:
Reactive: 1

With Moleculetype Reactive with:
11 atoms,
9 bonds,
13 angles,
6 pairs,
[...]

ForceField parameters with
66 atomtypes,
98 bondtypes,
257 angletypes,
134 proper dihedraltypes
58 improper dihedraltypes
153 residuetypes</code></pre>
</div>
</figure>
</div>
</div>
</div>
</div>
<p>Due to their strongly typed nature, KIMMDY objects are exceptionally easy to work with. Expressive type annotations and documentation strings (also see <a href="#lst-python" class="quarto-xref">Listing&nbsp;5.3</a> and <a href="#lst-python2" class="quarto-xref">Listing&nbsp;5.4</a>) provide a fast and efficient development experience and allow for interactive exploration with modern editor tooling.</p>
</section>
<section id="sec-grappa" class="level3 page-columns page-full" data-number="4.1.7">
<h3 data-number="4.1.7" class="anchored" data-anchor-id="sec-grappa"><span class="header-section-number">4.1.7</span> Parametrization interface</h3>
<p>After each reaction, the product state needs to be parametrized. Hydrolyzing a peptide bond effectively removes one water molecule and creates two terminal amino acids. Terminal amino acids are nothing unusual for most force fields, so KIMMDY can adjust the topology parameters for the state after the reaction based on the force field. This is called the basic parametrizer. Usage of the basic parametrizer assumes that the product molecule parameters are contained in the force field included in the topology file or that additional required changes are done explicitly with the custom topology modification <code>RecipeStep</code>.</p>
<p>Other reactions are not as straightforward, but KIMMDY comes equipped to handle all of them. Grappa <span class="citation" data-cites="seuteGrappaMachineLearned2025">[<a href="#ref-seuteGrappaMachineLearned2025" role="doc-biblioref">115</a>]</span> is a Graph Attentional Protein Parametrization model developed by Leif Seute and Eric that can generate new MM parameters for unseen structures, even for radicals such as those as the result of bond homolysis or HAT. Grappa parametrizes around the atoms involved in the reaction. Additionally, the whole <code>Reactive</code> moleculetype can be parametrized with Grappa either before a KIMMDY run or at the start, producing consistent parameters for any reaction.</p>

<div class="no-row-height column-margin column-container"><div id="fig-grappa-logo" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="GRAPPA logo.">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-grappa-logo-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./img/grappa.png" class="lightbox" data-gallery="quarto-lightbox-gallery-69" title="Figure&nbsp;4.7: The logo of GRAPPA, designed by Denis Kieswetter."><img src="./img/grappa.png" class="img-fluid figure-img" data-fig-scap="GRAPPA logo."></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-grappa-logo-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4.7: The logo of GRAPPA, designed by Denis Kieswetter.
</figcaption>
</figure>
</div></div><p>Applying correct parameters is important not only to maintain stable simulations, but also to ensure that physically accurate states are sampled. For example, the radical center in <a href="#fig-ala-rad" class="quarto-xref">Figure&nbsp;4.8</a> should have its substituents in the same plane as the carbon atom, as it is now best represented as being sp<sup>2</sup> instead of sp<sup>3</sup> hybridized. Please note that while you will see an interactive element to explore the molecule via a plugin I wrote for our documentation system in the web version of this thesis, the print version will have just a screenshot. The plugin is available at <a href="https://github.com/jmbuhr/quarto-molstar">github.com/jmbuhr/quarto-molstar</a>.</p>
<div id="fig-ala-rad" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-ala-rad-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
    <div id="app-./assets/npt-just-ala.gro" class="molstar-app"></div>
    <script type="text/javascript">
    molstar.Viewer.create("app-./assets/npt-just-ala.gro", {"emdbProvider":"rcsb","layoutIsExpanded":false,"layoutShowControls":false,"layoutShowLeftPanel":true,"layoutShowLog":false,"layoutShowRemoteState":false,"layoutShowSequence":false,"pdbProvider":"rcsb","viewportShowAnimation":true,"viewportShowExpand":true,"viewportShowSelectionMode":false}).then(viewer => {
    viewer.loadStructureFromUrl("./assets/npt-just-ala.gro", format="gro");    });
    </script>
    
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-ala-rad-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4.8: Interactive molecule display of an alanine dipeptide with a radical center.
</figcaption>
</figure>
</div>
<p>Now KIMMDY has a trajectory with a chosen reaction time and knows the reaction that should happen. It also has the topology as it should be before and after the reaction, including all required parameters. Swapping out the topology immediately would likely explode the simulation system, as the coordinates may be very far away from a minimum energy state with the new parameters. This is where slow-growth comes in.</p>
</section>
<section id="slow-growth" class="level3 page-columns page-full" data-number="4.1.8">
<h3 data-number="4.1.8" class="anchored" data-anchor-id="slow-growth"><span class="header-section-number">4.1.8</span> Slow-growth</h3>
<p>For creating an initial structure of the reaction product state, minimizing and equilibrating the molecular systems is viable. However, this may take a long time for large systems, resulting in inefficient simulations. We use the slow-growth method implemented in the GROMACS free-energy module to interpolate between product and reactant state for continuous parameter and coordinate changes. Subsequently, a short equilibration ensures the states are Markovian, i.e., sampling happens from the Boltzmann distribution of the new state.</p>
<p>To this end, KIMMDY generates a merged topology that transitions between state A (educt) and state B (product). KIMMDY supports two modes of slow-growth. In “morse-only” mode, breaking or forming bonds are modeled entirely with morse potentials, where the unbound is mimicked by a morse potential with parameters such that the shape matches the Van der Waals potential of the two now non-bonded atoms. In “full” slow-growth mode all parameters around the affected atoms are transitioned, where non-bonded parameters are modeled using pair potentials. Slow-growth of non-bonded parameters, especially for Van der Waals interactions, can quickly lead to high energies and forces due to the exponential nature of the potential. As such the best choice of KIMMDY options and mdp parameters to relax the structure between reactions and sampling depends on the system. An example trajectory of a reaction including slow-growth can be seen at <a href="https://youtu.be/tgS5uleN288" class="uri">https://youtu.be/tgS5uleN288</a> (or directly in the web version of my thesis at <a href="https://jmbuhr.de/phd-thesis">jmbuhr.de/phd-thesis</a>) and is visualized in <a href="#fig-slow-growth" class="quarto-xref">Figure&nbsp;4.9</a>.</p>
<div id="fig-slow-growth" class="quarto-layout-panel">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-slow-growth-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="quarto-layout-row quarto-layout-valign-center">
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-slow-growth" style="flex-basis: 50.0%;justify-content: flex-start;">
<div id="fig-sg" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-sg-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./plots/kimmdy/slow-growth.png" class="lightbox" data-gallery="fig-slow-growth" title="Figure&nbsp;4.9&nbsp;(a): A slow-growth simulation visualized by the distances between the atoms involved in the reaction."><img src="./plots/kimmdy/slow-growth.png" class="img-fluid figure-img" data-ref-parent="fig-slow-growth"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-sg-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(a) A slow-growth simulation visualized by the distances between the atoms involved in the reaction.
</figcaption>
</figure>
</div>
</div>
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-slow-growth" style="flex-basis: 50.0%;justify-content: flex-start;">
<div id="fig-sg-annotations" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-sg-annotations-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./img/slow-growth-annotation.png" class="lightbox" data-gallery="fig-slow-growth" title="Figure&nbsp;4.9&nbsp;(b): Visual explanation of the distances plotted in Figure&nbsp;4.9 (a). Coloured double-arrows correspond to the distances in the plot."><img src="./img/slow-growth-annotation.png" class="img-fluid figure-img" data-ref-parent="fig-slow-growth"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-sg-annotations-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(b) Visual explanation of the distances plotted in <a href="#fig-sg" class="quarto-xref">Figure&nbsp;4.9 (a)</a>. Coloured double-arrows correspond to the distances in the plot.
</figcaption>
</figure>
</div>
</div>
</div>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-slow-growth-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4.9: KIMMDY’s slow-growth applied to hydrolysis of a peptide bond.
</figcaption>
</figure>
</div>

<div class="no-row-height column-margin column-container"><div class="">
<p><a href="./www/gly-hydrolysis.mp4" class="lightbox" data-gallery="quarto-lightbox-gallery-72"><video src="./www/gly-hydrolysis.mp4" class="img-fluid" controls=""></video></a><a href="./www/gly-hydrolysis.mp4">Video</a></p>
</div></div><p>Currently, exact continuations of MD simulations in GROMACS are only possible from checkpoint files, which do not allow for topology modifications. We use coordinates and velocities to continue MD simulations, which leads to a short period where thermostat and barostat equilibrate again. Switching between “free-energy” on and off in GROMACS also currently has some artifacts, albeit non-breaking, which may be improved in the future.</p>
<p>After the slow-growth task the next set of MD simulations and sampling for reactions can begin. But what if a single KIMMDY run includes long-running simulations or expensive models to compute?</p>
</section>
<section id="checkpoints-and-restarting" class="level3" data-number="4.1.9">
<h3 data-number="4.1.9" class="anchored" data-anchor-id="checkpoints-and-restarting"><span class="header-section-number">4.1.9</span> Checkpoints and restarting</h3>
<p>When KIMMDY is started with an already existing output directory, it can either create a new output directory with incrementing run numbers or attempt to restart an existing run in the output directory. The behavior can be set by the <code>--restart</code> flag of the CLI or the <code>restart:</code> keyword in the <code>kimmdy.yml</code> file. When restarting, KIMMDY first checks if the existing run has already completed, in which case no additional work is done. It then scans the existing output subdirectories of the already executed tasks in the sequence for incomplete tasks and for tasks during or after which a continuation is possible. Conveniently, GROMACS simulations can be restarted from checkpoints and the end of a simulation the current system state is written out explicitly. Thus, KIMMDY can restart after GROMACS simulations or even continue halted simulations from the latest checkpoint. Files previously written by KIMMDY are parsed to faithfully reproduce the current state of the KIMMDY simulation. All tasks started after the last MD task cannot be used and are discarded. This is not typically an issue, because the simulations are the computationally expensive part, not the reaction rate queries. However, reaction plugins can implement their own caching mechanisms to speed up restarting after a known simulation. Examples for this are the hydrolysis and homolysis plugins, which both need the query bond distances and average them over the simulation. Because of this, they use a shared cache file of those average distances, which not only allows fast restarting after a completed simulation but also allows for a speed-up when both reactions are in a competition setting, in which case only one of the plugins has to compute the average distances and the other can use the same shared cache.</p>
</section>
<section id="high-performance-computing-interface" class="level3" data-number="4.1.10">
<h3 data-number="4.1.10" class="anchored" data-anchor-id="high-performance-computing-interface"><span class="header-section-number">4.1.10</span> High performance computing interface</h3>
<p>KIMMDY simulations are easily parallelizable and due to the MD or machine-learning reaction plugins they require extensive resources, typically available on high performance computing (HPC) systems. We added utilities to run KIMMDY on HPC systems using Slurm <span class="citation" data-cites="jetteArchitectureSlurmWorkload2023">[<a href="#ref-jetteArchitectureSlurmWorkload2023" role="doc-biblioref">95</a>]</span> and expose prefixes to the GROMACS commands necessary for running it on a system using a message passing interface (MPI) such as <a href="https://www.open-mpi.org/">OpenMPI</a>. Restarting as described above is used in conjunction with automated resubmission of Slurm jobs for HPC clusters that have a time limit on job runtime.</p>
</section>
<section id="analysis" class="level3" data-number="4.1.11">
<h3 data-number="4.1.11" class="anchored" data-anchor-id="analysis"><span class="header-section-number">4.1.11</span> Analysis</h3>
<p>KIMMDY also includes analysis tools via <code>kimmdy-analysis</code>. Currently this involves <code>trjcat</code> to concatenate the individual trajectories into a continuous one for visualization, <code>energy</code> to plot various energy terms during the simulations, <code>rates</code> to visualize reaction rates for all reactions, and tools specially designed for radical migration via HAT (see <a href="#fig-hat-analysis" class="quarto-xref">Figure&nbsp;4.10</a>).</p>
<div id="fig-hat-analysis" class="quarto-layout-panel">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-hat-analysis-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="quarto-layout-row">
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-hat-analysis" style="flex-basis: 50.0%;justify-content: flex-start;">
<div id="fig-hat-vmd" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-hat-vmd-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="img/kimmdy-radical-vis.png" class="lightbox" data-gallery="fig-hat-analysis" title="Figure&nbsp;4.10&nbsp;(a): "><img src="img/kimmdy-radical-vis.png" id="fig-hat-vmd" class="img-fluid figure-img" data-ref-parent="fig-hat-analysis"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-hat-vmd-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(a)
</figcaption>
</figure>
</div>
</div>
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-hat-analysis" style="flex-basis: 50.0%;justify-content: flex-start;">
<div id="fig-hat-bar" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-hat-bar-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="img/kimmdy-radical-bars.png" class="lightbox" data-gallery="fig-hat-analysis" title="Figure&nbsp;4.10&nbsp;(b): "><img src="img/kimmdy-radical-bars.png" id="fig-hat-bar" class="img-fluid figure-img" data-ref-parent="fig-hat-analysis"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-hat-bar-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(b)
</figcaption>
</figure>
</div>
</div>
</div>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-hat-analysis-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4.10: KIMMDY analysis tools visualizing the radical occupancy after many successive HAT reactions. (a) Radical occupancy is written to a pdb file as beta values and then visualized with VMD. (b) Radical occupancy as a bar plot.
</figcaption>
</figure>
</div>
</section>
<section id="code-quality-and-testing" class="level3" data-number="4.1.12">
<h3 data-number="4.1.12" class="anchored" data-anchor-id="code-quality-and-testing"><span class="header-section-number">4.1.12</span> Code quality and testing</h3>
<p>Academic software developers often change rapidly due to the nature of short-term projects, such as Master’s and doctoral degrees. Hence, writing robust and maintainable code was made a top priority in the development of KIMMDY. To minimize friction when working with multiple people on the same code base, we adhered to best practices of research software development and established clear conventions, outlined in our online documentation. KIMMDY code is strongly typed and extensively tested. Further, KIMMDY has infrastructure in place to automate running unit- and integration tests before a change is merged. New KIMMDY versions are automatically created and released based on the <a href="https://www.conventionalcommits.org/">Conventional Commits</a> specification <span class="citation" data-cites="ConventionalCommits">[<a href="#ref-ConventionalCommits" role="doc-biblioref">116</a>]</span> to clearly identify new features and label backwards-incompatible changes. The version used for the results in this thesis is <code>v8.0.0</code>.</p>
</section>
</section>
<section id="applying-kimmdy-to-collagen-hydrolysis-and-homolysis" class="level2 page-columns page-full" data-number="4.2">
<h2 data-number="4.2" class="anchored" data-anchor-id="applying-kimmdy-to-collagen-hydrolysis-and-homolysis"><span class="header-section-number">4.2</span> Applying KIMMDY to collagen hydrolysis and homolysis</h2>
<p>Next, KIMMDY will be applied to directly compare peptide hydrolysis to homolysis of the backbone, to answer why we see radicals in experiments on collagen <span class="citation" data-cites="zappMechanoradicalsTensedTendon2020">[<a href="#ref-zappMechanoradicalsTensedTendon2020" role="doc-biblioref">17</a>]</span>, even though peptide hydrolysis should be significantly sped up under force <span class="citation" data-cites="pillMechanicalActivationDrastically2019">[<a href="#ref-pillMechanicalActivationDrastically2019" role="doc-biblioref">20</a>]</span>, which would mean the backbone would hydrolyse before it had a chance to break homolytically. To compare two reactions in KIMMDY, reaction rates are required.</p>
<section id="sec-hyd-hom-plugins" class="level3 page-columns page-full" data-number="4.2.1">
<h3 data-number="4.2.1" class="anchored" data-anchor-id="sec-hyd-hom-plugins"><span class="header-section-number">4.2.1</span> The hydrolysis and homolysis plugins</h3>
<p>Rates for homolysis use the physical model developed by Benedikt Rennekamp <span class="citation" data-cites="rennekampHybridKineticMonte2020">[<a href="#ref-rennekampHybridKineticMonte2020" role="doc-biblioref">100</a>]</span>, which follows a Bell-Evans model <span class="citation" data-cites="bellModelsSpecificAdhesion1978 evansDynamicStrengthMolecular1997">[<a href="#ref-bellModelsSpecificAdhesion1978" role="doc-biblioref">117</a>, <a href="#ref-evansDynamicStrengthMolecular1997" role="doc-biblioref">118</a>]</span>. That is as follows: Bonds are modeled using a Morse potential (<a href="#eq-energy-morse" class="quarto-xref">Equation&nbsp;2.7</a> and <a href="#fig-morse-potential" class="quarto-xref">Figure&nbsp;2.5</a>) in the MD simulation. The potential is shifted by the work already performed on the bond by the external force, which is calculated as the bond elongation multiplied with the force on the bond at the average bond distance. This assumes that the system is well equilibrated under the external force. If this is the case, the force in a bond is the derivative of the potential at the average bond distance <span class="math inline">\(\overline{r}\)</span> and computed by <a href="#eq-force-morse" class="quarto-xref">Equation&nbsp;4.1</a>, which is also shown in <a href="#fig-morse-force" class="quarto-xref">Figure&nbsp;4.11</a>.</p>
<p><span id="eq-force-morse"><span class="math display">\[
F(\overline{r}) = 2 \beta D^{-\beta (\overline{r} - r_0)} \left(1 - e^{-\beta (\overline{r} - r_0)}\right),
\tag{4.1}\]</span></span></p>
<p>with <span class="math inline">\(\beta=\sqrt{k / (2~D)}\)</span>,the spring constant <span class="math inline">\(k\)</span>, and the dissociation energy <span class="math inline">\(D\)</span>.</p>

<div class="no-row-height column-margin column-container"><div id="fig-morse-force" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="Derivative of the Morse potential.">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-morse-force-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./img/morse-force.png" class="lightbox" data-gallery="quarto-lightbox-gallery-75" title="Figure&nbsp;4.11: The derivative of the Morse potential, i.e., the force as a function of the bond distance r."><img src="./img/morse-force.png" class="img-fluid figure-img" data-fig-scap="Derivative of the Morse potential."></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-morse-force-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4.11: The derivative of the Morse potential, i.e., the force as a function of the bond distance r.
</figcaption>
</figure>
</div></div><p>The effective potential (<a href="#eq-v-eff" class="quarto-xref">Equation&nbsp;4.2</a>)</p>
<p><span id="eq-v-eff"><span class="math display">\[
V_{eff}(r)=V_{morse}(r)-F(\overline{r})~r,
\tag{4.2}\]</span></span></p>
<p>has a minimum and a maximum given by <a href="#eq-r-min" class="quarto-xref">Equation&nbsp;4.3</a></p>
<p><span id="eq-r-min"><span class="math display">\[
r_{min/max}=r_0 - \frac{1}{\beta} \left( \beta~D \pm \sqrt{\frac{\beta^2D^{2}-2D\beta F(\overline{r})} {2\beta D} } \right),
\tag{4.3}\]</span></span></p>
<p>and consequently provides the energy barrier <span class="math inline">\(\Delta E\)</span> for force-dependent homolysis via <a href="#eq-hom-barrier" class="quarto-xref">Equation&nbsp;4.4</a>:</p>
<p><span id="eq-hom-barrier"><span class="math display">\[
\Delta E = V_{eff}(r_{max}) - V_{eff}(r_{min}),
\tag{4.4}\]</span></span></p>
<p>which is then used in the Arrhenius equation (<a href="#eq-arrhenius" class="quarto-xref">Equation&nbsp;2.33</a>) to obtain a reaction rate (<a href="#eq-hom-rate" class="quarto-xref">Equation&nbsp;4.5</a>).</p>
<p><span id="eq-hom-rate"><span class="math display">\[
k_{homolysis} = A\,\text{e}^{\left(\frac{-\Delta E}{RT}\right)}
\tag{4.5}\]</span></span></p>
<p>Rates for hydrolysis are also force-dependent. The experiments by Pill et al. <span class="citation" data-cites="pillMechanicalActivationDrastically2019">[<a href="#ref-pillMechanicalActivationDrastically2019" role="doc-biblioref">20</a>]</span> already allow fitting a model to match external forces applied to a single peptide bond to bond lifetimes and thus reaction rates (<a href="#fig-force-response" class="quarto-xref">Figure&nbsp;4.12</a>). Because they observed a bi-exponential decay at 0.7&nbsp;nN and thus fit two Bell models, one for the region below 0.7 and one for above 0.7, in order to obtain continuous rates to use in KIMMDY, the two models were interpolated around 0.7&nbsp;nN in a log-linear fashion. The rates as one would obtain them if the theoretical energy barriers from their <em>ab initio</em> QM calculations were used, are also shown. In the KIMMDY plugin for hydrolysis, the choice between experimental and theoretical rates is a configuration option, but here I will use just the experimental rates from AFM to be as close to the experimental setting as possible.</p>
<div id="fig-force-response" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-scap="Hydrolysis reaction rates fit.">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-force-response-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./plots/kimmdy/force-response.png" class="lightbox" data-gallery="quarto-lightbox-gallery-76" title="Figure&nbsp;4.12: Experimental (AFM) and theoretical (QM) reaction rates used as an input to the hydrolysis plugin as a function of force. Points represent the averages of the AFM experiments [@pillMechanicalActivationDrastically2019]."><img src="./plots/kimmdy/force-response.png" class="img-fluid figure-img" style="width:50.0%" data-fig-scap="Hydrolysis reaction rates fit."></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-force-response-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4.12: Experimental (AFM) and theoretical (QM) reaction rates used as an input to the hydrolysis plugin as a function of force. Points represent the averages of the AFM experiments <span class="citation" data-cites="pillMechanicalActivationDrastically2019">[<a href="#ref-pillMechanicalActivationDrastically2019" role="doc-biblioref">20</a>]</span>.
</figcaption>
</figure>
</div>
<p>Crucially, the hydrolysis plugin can not simply use the difference of the bond extension to the equilibrium distance defined in the force field to relate to externally applied forces. When forces are applied just like in <a href="#fig-render-triple-single" class="quarto-xref">Figure&nbsp;3.5</a>, an externally applied force of 0,nN should relate to a force of 0,nN in the model from the AFM experiments. However, when no force is applied, the average equilibrium bond distances of all bonds are not equal to the equilibrium distance of the morse potential from the force field. This is due to the entropic contributions in the simulation and statistical mechanics: The probability <span class="math inline">\(P_i\)</span> of finding a system in a state <span class="math inline">\(i\)</span> is given by the Boltzmann distribution as a function of the states energy <span class="math inline">\(E_i\)</span> by <a href="#eq-boltzman" class="quarto-xref">Equation&nbsp;4.6</a>.</p>
<p><span id="eq-boltzman"><span class="math display">\[
P_{i}\propto e^{(-{\frac {E_{i}}{k_{\mathrm{B}} T}})}
\tag{4.6}\]</span></span></p>
<p>For an asymmetric potential energy function like the Morse potential, this leads to the aforementioned effect. The solution is straightforward. The internal bond forces calculated from bond distances of each peptide bond are recorded during the equilibrium simulations at 0 applied force and used as a baseline for all other forces. This way, the force that is nominally applied to the simulation matches the force that was nominally applied during the AFM experiment.</p>
<p>The force responses for both reactions can now be directly compared (<a href="#fig-force-response" class="quarto-xref">Figure&nbsp;4.12</a>). As the external force acts on the system, the average bond distances increase, which leads to an apparent force acting directly on the bond. The values for one such bond are plotted in <a href="#fig-stretch-force" class="quarto-xref">Figure&nbsp;4.13 (a)</a> and the reaction rates for both reactions, homolysis and hydrolysis, as a function of those forces are shown in <a href="#fig-force-hyd-hom" class="quarto-xref">Figure&nbsp;4.13 (b)</a>.</p>
<div id="fig-force-responses" class="quarto-layout-panel">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-force-responses-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="quarto-layout-row">
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-force-responses" style="flex-basis: 50.0%;justify-content: flex-start;">
<div id="fig-stretch-force" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-stretch-force-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./plots/kimmdy/force-extension.png" class="lightbox" data-gallery="fig-force-responses" title="Figure&nbsp;4.13&nbsp;(a): "><img src="./plots/kimmdy/force-extension.png" id="fig-stretch-force" class="img-fluid figure-img" data-ref-parent="fig-force-responses"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-stretch-force-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(a)
</figcaption>
</figure>
</div>
</div>
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-force-responses" style="flex-basis: 50.0%;justify-content: flex-start;">
<div id="fig-force-hyd-hom" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-force-hyd-hom-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./plots/kimmdy/force-response-homolysis.png" class="lightbox" data-gallery="fig-force-responses" title="Figure&nbsp;4.13&nbsp;(b): "><img src="./plots/kimmdy/force-response-homolysis.png" id="fig-force-hyd-hom" class="img-fluid figure-img" data-ref-parent="fig-force-responses"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-force-hyd-hom-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(b)
</figcaption>
</figure>
</div>
</div>
</div>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-force-responses-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4.13: Force response of the homolysis and hydrolysis reaction rates. (a) The force that has performed work on a bond as a function of bond stretching, i.e., <span class="math inline">\(r-r_0\)</span>. (b) Reaction rates for hydrolysis (experimental model) and homolysis (physical model) as a function of force. For homolysis, the rates depends on the BDE, so two exemplary lines are shown. The lower line for a C-C bond in a protein backbone with a BDE of 341 kJ/mol and the upper line for a C-C bond in a Pyridinoline (PYD) cross-link with a BDE of 282 kJ/mol.
</figcaption>
</figure>
</div>
<p>A few more modifications to the hydrolysis reaction rate apply. Firstly, the experiment used a single, freely accessible peptide bond. The hydrolysis plugin captures that a peptide bond may be less accessible by taking into account the solvent accessible surface are (SASA). SASA is calculated with the Lee-Richards algorithm <span class="citation" data-cites="leeInterpretationProteinStructures1971">[<a href="#ref-leeInterpretationProteinStructures1971" role="doc-biblioref">119</a>]</span> via a small module within the hydrolysis plugin that I coined <code>minisasa</code> by interfacing with the freesasa C-library <span class="citation" data-cites="mitternachtFreeSASAOpenSource2016">[<a href="#ref-mitternachtFreeSASAOpenSource2016" role="doc-biblioref">120</a>]</span>. SASA acts as a scaling factor for the attempt frequency (<span class="math inline">\(A\)</span>) in the form of the ratio of the SASA of the bond to the SASA of a freely accessible peptide bond in a glycine dipeptide.</p>
<p>Secondly, the AFM experiments happened at a fixed pH of 7.4, while the hydrolysis plugin can take this into account again as a scaling factor to the attempt frequency. The concentration of hydroxide ions, which are required catalytically for the reaction, is given by <span class="math inline">\(c_{\ce{OH-}} = 10^{-(14 - \mathrm{pH})}\)</span> and consequently <span class="math inline">\(c_{\ce{OH-}_{\mathrm{exp}}} = 10^{-(14 - \mathrm{pH}_\mathrm{exp})}\)</span>. Since the concentration of <span class="math inline">\(\ce{OH-}\)</span> is proportional to the probability that any given solvent molecule is a catalytically active <span class="math inline">\(\ce{OH-}\)</span>, the scaling factor is the ratio of concentrations, which leads to the final equation for the reaction rate of a peptide bond in a sampled state, <a href="#eq-hydrolysis-rate" class="quarto-xref">Equation&nbsp;4.7</a>.</p>
<p><span id="eq-hydrolysis-rate"><span class="math display">\[
\mathrm{Rate}(bond, t) = \
\frac{\mathrm{SASA}(bond, t)}{\mathrm{SASA}_{max}} \cdot \
\frac{c_{\ce{OH-}}}{c_{\ce{OH-}_{\mathrm{exp}}}} \
\cdot \mathrm{rate}_{\mathrm{exp}}(F_{bond,t})
\tag{4.7}\]</span></span></p>
<p>Both the force and the SASA can be time-dependent or averaged over the entire state. Once a peptide bond has been chosen to react by the kMC step, the hydrolysis plugin calculates the optimal solvent molecule to execute the attack based on distance and angles (Bürgi-Dunitz and Flippin-Lodge, see <a href="#fig-bd" class="quarto-xref">Figure&nbsp;3.7</a>).</p>
</section>
<section id="system-setup" class="level3 page-columns page-full" data-number="4.2.2">
<h3 data-number="4.2.2" class="anchored" data-anchor-id="system-setup"><span class="header-section-number">4.2.2</span> System setup</h3>
<p>In addition to the single chain and triple helix systems as introduced for <a href="#sec-main-qm" class="quarto-xref">Chapter&nbsp;3</a> (just without the QM/MM), a complete collagen fibril is introduced. It is an all-atom model of a <em>Rattus norvegicus</em> collagen fibril (PDB ID: 3HR2) comprised of 41 triple helices spanning one central overlap and one gap region for a total of roughly 320 000 protein atoms. ColBuilder <span class="citation" data-cites="obarska-kosinskaColBuilderServerBuild2021">[<a href="#ref-obarska-kosinskaColBuilderServerBuild2021" role="doc-biblioref">93</a>]</span> was used to generate a model with N- and C-terminal Pyridinoline (PYD) crosslinks with a connectivity of 9.C-5.B-944.B and 1047.C-1047.A-98.B, respectively. Furthermore, all Phe and Tyr residues were randomly mutated to either dihydroxyphenylalanine (DOPA) deprotonated at the C<span class="math inline">\(\epsilon\)</span> (DO1) or C<span class="math inline">\(\zeta\)</span> (DO2) hydroxy group.</p>

<div class="no-row-height column-margin column-container"><div id="fig-fibril2" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-fibril2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./figures/render-fibril-black-cropped.png" class="lightbox" data-gallery="quarto-lightbox-gallery-79" title="Figure&nbsp;4.14: A collagen fibril."><img src="./figures/render-fibril-black-cropped.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-fibril2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4.14: A collagen fibril.
</figcaption>
</figure>
</div></div><p>The fibril was solvated and neutralized with 0.15&nbsp;M NaCl. After an energy minimization using the steepest descent method, a 10&nbsp;ns NVT equilibration and then a 10&nbsp;ns NPT equilibration were performed. It was then equilibrated under the different external forces ranging from 0 to 3&nbsp;nN per chain for 2&nbsp;ns and the convergence of the pulling simulation was verified based on the distances between the pull groups. The equilibration was followed by 2&nbsp;ns of sampling simulations, during which bond distances where recorded with PLUMED <span class="citation" data-cites="tribelloPLUMED2New2014">[<a href="#ref-tribelloPLUMED2New2014" role="doc-biblioref">113</a>]</span>. Likewise, a single triple helix out of a collagen fibril and a single peptide out of the same triple helix where solvated in TIP3 water, neutralized with 0.15&nbsp;M NaCl, energy minimized and equilibrated under NVT and NPT ensembles at a temperature of 300&nbsp;K. The same temperature is used in the rate calculations for all reactions. The external forces applied range from no external force to 3&nbsp;nN. In the case of the complete collagen fibril, an additional, more physiological, sheared pulling was introduced by sampling the force on each triple helix from a Gaussian distribution with a standard deviation of 30&nbsp;% of the mean force centered around the nominal external force. This mimics the effect of unequal loading under strain in a real tendon or ligament.</p>
<div style="page-break-after: always;"></div>
</section>
</section>
<section id="results-1" class="level2 page-columns page-full" data-number="4.3">
<h2 data-number="4.3" class="anchored" data-anchor-id="results-1"><span class="header-section-number">4.3</span> Results</h2>
<p>Please note that some of the results of this section have also already been discussed in <span class="citation" data-cites="kimmdy2025">[<a href="#ref-kimmdy2025" role="doc-biblioref">99</a>]</span> and thus will have a high text similarity.</p>
<section id="physiologically-expected-ph-changes-have-only-a-minor-contribution-to-hydrolysis-rates" class="level3" data-number="4.3.1">
<h3 data-number="4.3.1" class="anchored" data-anchor-id="physiologically-expected-ph-changes-have-only-a-minor-contribution-to-hydrolysis-rates"><span class="header-section-number">4.3.1</span> Physiologically expected pH changes have only a minor contribution to hydrolysis rates</h3>
<p>With KIMMDY it is now possible to rapidly sample variations of various parameters, one of which is the pH value. This is visualized in extended data <a href="#fig-ext-hyd-ph" class="quarto-xref">Figure&nbsp;5.4 (a)</a>, which explores the impact of external force and the pH value on the two systems already introduced for QM/MM in <a href="#sec-main-qm" class="quarto-xref">Chapter&nbsp;3</a>, the single peptide chain and the triple helix. At this time, the pH value only enters the rate calculation as a global input parameter and is not affected by the outcome of the molecular dynamics sampling step. So by just looking at <a href="#eq-hydrolysis-rate" class="quarto-xref">Equation&nbsp;4.7</a>, we can calculate that a drop in pH from 7.4 to pH 7.0, as one would expect to see physiologically, e.g., in muscle tissue during exercise <span class="citation" data-cites="streetInterstitialPHHuman2001">[<a href="#ref-streetInterstitialPHHuman2001" role="doc-biblioref">121</a>]</span>, is expected to reduce the rate by a factor of 2.5. This is because it only contributes linearly to the pre-exponential factor.</p>
</section>
<section id="sec-sasa" class="level3" data-number="4.3.2">
<h3 data-number="4.3.2" class="anchored" data-anchor-id="sec-sasa"><span class="header-section-number">4.3.2</span> Solvent accessibility of peptide bonds in the fibril is diverse</h3>
<p>The solvent accessible surface area (SASA) warrants a deeper look.</p>
<p><a href="#fig-sasa" class="quarto-xref">Figure&nbsp;4.15</a> explores the SASA of the peptide bonds in the three systems under no force and 1&nbsp;nN of pulling (with shear pulling for the fibril). The SASA encompasses 7 orders of magnitude, but plenty of peptide bonds remain relatively accessible even in the fibril system compared to the reference SASA of the peptide bond in a Glycine-Glycine dipeptide. This is especially true under force, as the backbones of the chains get stretched out, making the peptide bonds even more accessible. Also note how the shape of the distribution for the triple helix matches the shape for the fibril.</p>
<div id="fig-sasa" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-sasa-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./plots/kimmdy/sasa.png" class="lightbox" data-gallery="quarto-lightbox-gallery-80" title="Figure&nbsp;4.15: Average Solvent Accessible Surface Area (SASA, log10-scale) per peptide bond in the collagen fibril, the single chain and the triple helix system while no or 1 nN of force is applied. The 1&nbsp;nN case for the fibril represents the shear pulling setup. The SASA of a freely accessible peptide bond in a Glycine-Glycine dipeptide is shown as a reference in red."><img src="./plots/kimmdy/sasa.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-sasa-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4.15: Average Solvent Accessible Surface Area (SASA, log10-scale) per peptide bond in the collagen fibril, the single chain and the triple helix system while no or 1 nN of force is applied. The 1&nbsp;nN case for the fibril represents the shear pulling setup. The SASA of a freely accessible peptide bond in a Glycine-Glycine dipeptide is shown as a reference in red.
</figcaption>
</figure>
</div>
<p>Because of this I wondered specifically about the effect of the fibril. Is a triple helix in the middle of the fibril less accessible than a fibril near the edge of the packing? This would lead to boundary effects that should be excluded. However, looking at slices through the fibril system (<a href="#fig-sasa-y-slice" class="quarto-xref">Figure&nbsp;4.16</a> and <a href="#fig-sasa-x-slice" class="quarto-xref">Figure&nbsp;4.17</a>) reveals that even the inner triple helices in the overlap region contain bonds with high SASA.</p>
<div id="fig-sasa-y-slice" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-sasa-y-slice-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./plots/kimmdy/sasa_y_slice.png" class="lightbox" data-gallery="quarto-lightbox-gallery-81" title="Figure&nbsp;4.16: A slice through the collagen fibril in the x-z-plane with y-coordinates filtered to be between 7.9 and 9.6&nbsp;nm. Each peptide bond is colored by SASA for the 0&nbsp;nN and 1&nbsp;nN shear pulling setup."><img src="./plots/kimmdy/sasa_y_slice.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-sasa-y-slice-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4.16: A slice through the collagen fibril in the x-z-plane with y-coordinates filtered to be between 7.9 and 9.6&nbsp;nm. Each peptide bond is colored by SASA for the 0&nbsp;nN and 1&nbsp;nN shear pulling setup.
</figcaption>
</figure>
</div>
<div id="fig-sasa-x-slice" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-sasa-x-slice-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./plots/kimmdy/sasa_z_slice.png" class="lightbox" data-gallery="quarto-lightbox-gallery-82" title="Figure&nbsp;4.17: A slice through the collagen fibril in the x-y-plane with the z-coordinates filtered to be between 40 and 50&nbsp;nm. Each peptide bond is colored by SASA for the 0&nbsp;nN and 1&nbsp;nN shear pulling setup."><img src="./plots/kimmdy/sasa_z_slice.png" class="img-fluid figure-img" style="width:50.0%"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-sasa-x-slice-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4.17: A slice through the collagen fibril in the x-y-plane with the z-coordinates filtered to be between 40 and 50&nbsp;nm. Each peptide bond is colored by SASA for the 0&nbsp;nN and 1&nbsp;nN shear pulling setup.
</figcaption>
</figure>
</div>
<p>More quantitatively, plotting the SASA against the radial distance from the center in the x-y-plane for peptide bonds in the overlap region reveals no significant increase of the SASA for the outer triple helices (<a href="#fig-sasa-r" class="quarto-xref">Figure&nbsp;4.18</a>).</p>
<div id="fig-sasa-r" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-sasa-r-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./plots/kimmdy/sasa_radial_distance.png" class="lightbox" data-gallery="quarto-lightbox-gallery-83" title="Figure&nbsp;4.18: SASA of the peptide bonds in the overlap region of the collagen fibril (z-coordinate between 30 and 60&nbsp;nN) plotted against their radial distance from the center in the x-y plane. Black lines are linear models. 1&nbsp;nN refers to the 1&nbsp;nN shear pulling setup."><img src="./plots/kimmdy/sasa_radial_distance.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-sasa-r-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4.18: SASA of the peptide bonds in the overlap region of the collagen fibril (z-coordinate between 30 and 60&nbsp;nN) plotted against their radial distance from the center in the x-y plane. Black lines are linear models. 1&nbsp;nN refers to the 1&nbsp;nN shear pulling setup.
</figcaption>
</figure>
</div>
<p>I discarded the hypothesis that pulling would “squeeze out” water from the fibril, as instead the extension of the chains creates more surface area. Thus the SASA remains high and the influence of nearby triple helices can be assumed negligible for a large fraction of the peptide bonds. Still it can be explored in more detail with regards to the unique properties of collagen in the next section, <a href="#sec-prolines" class="quarto-xref">Section&nbsp;4.3.3</a>.</p>
<div style="page-break-after: always;"></div>
</section>
<section id="sec-prolines" class="level3 page-columns page-full" data-number="4.3.3">
<h3 data-number="4.3.3" class="anchored" data-anchor-id="sec-prolines"><span class="header-section-number">4.3.3</span> Prolines in collagen may offer protection against hydrolysis</h3>
<p>Given the unusually high content of proline in collagen, this special amino acid deserves a closer look.</p>

<div class="no-row-height column-margin column-container"><div id="fig-top-bond-f" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-top-bond-f-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./img/gly-gro.svg" class="lightbox" data-gallery="quarto-lightbox-gallery-84" title="Figure&nbsp;4.19: A Glycine-Proline peptide highlighting the three bond types of the protein backbone. Blue: amide \ce{C}-\ce{N} (the peptide bond with atom types C-N); Green: \ce{N}-\ce{C_{\alpha}} (with atom types N-CT); Gray: \ce{C_{\alpha}}-\ce{C} (with atom types CT-C)."><img src="./img/gly-gro.svg" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-top-bond-f-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4.19: A Glycine-Proline peptide highlighting the three bond types of the protein backbone. Blue: amide <span class="math inline">\(\ce{C}-\ce{N}\)</span> (the peptide bond with atom types C-N); Green: <span class="math inline">\(\ce{N}-\ce{C_{\alpha}}\)</span> (with atom types N-CT); Gray: <span class="math inline">\(\ce{C_{\alpha}}-\ce{C}\)</span> (with atom types CT-C).
</figcaption>
</figure>
</div></div><div id="fig-sasa-pro" class="quarto-layout-panel">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-sasa-pro-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="quarto-layout-row">
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-sasa-pro" style="flex-basis: 100.0%;justify-content: flex-start;">
<div id="fig-mean-bond-f" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-mean-bond-f-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./plots/kimmdy/bond-forces.png" class="lightbox" data-gallery="fig-sasa-pro" title="Figure&nbsp;4.20&nbsp;(a): Distribution of the average force as calculated from bond extension from its nominal equilibrium distance in the force field via Equation&nbsp;4.1 for the backbone bond of the collagen fibril system under no external pulling and 1&nbsp;nN of shear pulling. Figure&nbsp;4.19 provides a visual explanation of the atoms and bonds involved. Special consideration is given to bonds that involve proline."><img src="./plots/kimmdy/bond-forces.png" class="img-fluid figure-img" data-ref-parent="fig-sasa-pro"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-mean-bond-f-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(a) Distribution of the average force as calculated from bond extension from its nominal equilibrium distance in the force field via <a href="#eq-force-morse" class="quarto-xref">Equation&nbsp;4.1</a> for the backbone bond of the collagen fibril system under no external pulling and 1&nbsp;nN of shear pulling. <a href="#fig-top-bond-f" class="quarto-xref">Figure&nbsp;4.19</a> provides a visual explanation of the atoms and bonds involved. Special consideration is given to bonds that involve proline.
</figcaption>
</figure>
</div>
</div>
</div>
<div class="quarto-layout-row">
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-sasa-pro" style="flex-basis: 50.0%;justify-content: flex-start;">
<div id="fig-sasa-pro-density" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-sasa-pro-density-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./plots/kimmdy/sasa_pro_density.png" class="lightbox" data-gallery="fig-sasa-pro" title="Figure&nbsp;4.20&nbsp;(b): Distribution of the SASA of the peptide bonds by proline participation."><img src="./plots/kimmdy/sasa_pro_density.png" class="img-fluid figure-img" data-ref-parent="fig-sasa-pro"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-sasa-pro-density-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(b) Distribution of the SASA of the peptide bonds by proline participation.
</figcaption>
</figure>
</div>
</div>
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-sasa-pro" style="flex-basis: 50.0%;justify-content: flex-start;">
<div id="fig-sasa-pro-ratio" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-sasa-pro-ratio-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./plots/kimmdy/sasa_pro_boxplot_ratio.png" class="lightbox" data-gallery="fig-sasa-pro" title="Figure&nbsp;4.20&nbsp;(c): Ratio of SASA with and without pulling split by proline participation."><img src="./plots/kimmdy/sasa_pro_boxplot_ratio.png" class="img-fluid figure-img" data-ref-parent="fig-sasa-pro"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-sasa-pro-ratio-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(c) Ratio of SASA with and without pulling split by proline participation.
</figcaption>
</figure>
</div>
</div>
</div>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-sasa-pro-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4.20: The effect of pulling on bond strain and solvent accessibility in the collagen fibril.
</figcaption>
</figure>
</div>
<p><a href="#fig-sasa-pro" class="quarto-xref">Figure&nbsp;4.20</a> explores the impact of shear pulling on the internal bond forces as calculated via <a href="#eq-force-morse" class="quarto-xref">Equation&nbsp;4.1</a> and the solvent accessibility of the peptide bond. The average force is a function of how much the average bond distance in the simulation differs from the equilibrium bond distance in the force field. As such it is expected to have positive values due to entropic contributions in the simulation and the asymmetric nature of the Morse potential (<a href="#fig-morse-potential" class="quarto-xref">Figure&nbsp;2.5</a>). In <a href="#fig-mean-bond-f" class="quarto-xref">Figure&nbsp;4.20 (a)</a>, the distributions of the average force are classified by the type of bond and whether one or both of the participating atoms is part of a proline residue. The bond types and atoms involved are visually explained in <a href="#fig-top-bond-f" class="quarto-xref">Figure&nbsp;4.19</a>.</p>
<p>In equilibrium (0&nbsp;nN external pulling), the peptide bond (bond type C-N) tends have a higher extension compared to its equilibrium distance (<span class="math inline">\(r_0\)</span>) when it is between two prolines than if no prolines are involved. The same is true if the <span class="math inline">\(\ce{N}\)</span> of the peptide bond is contributed by a proline and the <span class="math inline">\(\ce{C}\)</span> by some other amino acid (cyan curve) for part of the bimodal distribution. If just the C-terminal end is from a proline (yellow curve) the situation is the same as if no proline is part of the bond (black curve). Under external force (1&nbsp;nN), the peptide bonds with two prolines or the proline <span class="math inline">\(\ce{N}\)</span> are under the highest strain. The <span class="math inline">\(\ce{C_{\alpha}}-\ce{C}\)</span> bond (with atom types CT-C) is under more strain in non-proline residues, but the <span class="math inline">\(\ce{N}-\ce{C_{\alpha}}\)</span> bond (with atom types N-CT) is more strained in prolines due to the ring structure.</p>
<p>Notably, despite the higher strain, peptide bonds between proline residues have a lower solvent accessibility than those between other residues (<a href="#fig-sasa-pro-density" class="quarto-xref">Figure&nbsp;4.20 (b)</a>) and the accessibility also increases less under external pulling (<a href="#fig-sasa-pro-ratio" class="quarto-xref">Figure&nbsp;4.20 (c)</a>). The effect is attenuated when just the N-terminal side of the peptide bond is a proline, because these cases then have a high chance of the smallest amino acid Glycine on the C-terminal side. In nature, the metallopeptidases that cleave collagen peptides (collagenase) are also less likely to cleave between proline residues according to the MEROPS database <span class="citation" data-cites="rawlingsMEROPSDatabaseProteolytic2018">[<a href="#ref-rawlingsMEROPSDatabaseProteolytic2018" role="doc-biblioref">122</a>]</span> (<a href="https://www.ebi.ac.uk/merops/cgi-bin/pepsum?id=M10.001">ebi.ac.uk/merops/cgi-bin/pepsum?id=M10.001</a>, see ID M10.001 and up)</p>
<p>However, because in the current model the SASA ratio only goes into the hydrolysis rate calculation linearly, its influence on the final rate is small compared to the influence of the bond stretching by external force (<a href="#fig-rates-proline" class="quarto-xref">Figure&nbsp;4.21</a>).</p>
<div id="fig-rates-proline" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-rates-proline-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./plots/kimmdy/rates-proline.png" class="lightbox" data-gallery="quarto-lightbox-gallery-88" title="Figure&nbsp;4.21: Distribution of hydrolysis rates by proline participation for the fibril setup with increasing shear pulling forces."><img src="./plots/kimmdy/rates-proline.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-rates-proline-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4.21: Distribution of hydrolysis rates by proline participation for the fibril setup with increasing shear pulling forces.
</figcaption>
</figure>
</div>
</section>
<section id="sec-kmdy-rates" class="level3" data-number="4.3.4">
<h3 data-number="4.3.4" class="anchored" data-anchor-id="sec-kmdy-rates"><span class="header-section-number">4.3.4</span> Homolysis becomes competitive with hydrolysis by force concentration</h3>
<p>The externally applied force and in turn the stretching of the bonds enters exponentially into the rate calculations for both reactions (see <a href="#fig-force-hyd-hom" class="quarto-xref">Figure&nbsp;4.13 (b)</a>). Furthermore, a small divergence in bond length from the equilibrium bond length has a large influence on the measured force in the bond (see <a href="#fig-stretch-force" class="quarto-xref">Figure&nbsp;4.13 (a)</a>).</p>
<p>It follows from <a href="#fig-hyd-vs-hom" class="quarto-xref">Figure&nbsp;4.22</a> that in the absence of an external pulling force, both hydrolysis and homolysis yield low rates. This is unsurprising, given that spontaneous fragmentation of peptides is highly unlikely in ambient conditions. When the single chain protein systems are pulled at 1&nbsp;nN, hydrolysis outcompetes homolysis for a single peptide chain (<a href="#fig-hyd-vs-hom" class="quarto-xref">Figure&nbsp;4.22</a>, right). However, for a densely packed and cross-linked collagen system pulled with shear stress and the same average external force of 1&nbsp;nN, with all else being equal, homolysis is accelerated drastically (<a href="#fig-hyd-vs-hom" class="quarto-xref">Figure&nbsp;4.22</a>, left). It reaches rates of similar orders of magnitude as hydrolysis and outcompetes it when comparing the highest rates.</p>
<div id="fig-hyd-vs-hom" class="quarto-layout-panel">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-hyd-vs-hom-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="quarto-layout-row">
<div class="quarto-figure-spacer quarto-layout-cell" style="flex-basis: 5.9%;justify-content: flex-start;">
<p>&nbsp;</p>
</div>
<div class="quarto-layout-cell" style="flex-basis: 82.4%;justify-content: center;">
<p><a href="./figures/hyd/fibril-single.png" class="lightbox" data-gallery="quarto-lightbox-gallery-89" title="Figure&nbsp;4.22: Reaction rates (log10-scale) for homolysis and hydrolysis in the collagen fibril system (left) and the single chain (right), with the systems visualized above the plot. The 1&nbsp;nN case for the fibril system represents the shear pulling setup. The value of the highest rate for each setup is shown above the respective distribution. The white points in the hydrolysis rate distributions of the single peptide denote the experimental reference value from AFM experiments [@pillMechanicalActivationDrastically2019] for the respective force. Rates lower than 10-25&nbsp;1/s are not shown."><img src="./figures/hyd/fibril-single.png" class="img-fluid figure-img"></a></p>
</div>
<div class="quarto-figure-spacer quarto-layout-cell" style="flex-basis: 11.8%;justify-content: flex-start;">
<p>&nbsp;</p>
</div>
</div>
<div class="quarto-layout-row">
<div class="quarto-layout-cell" style="flex-basis: 100.0%;justify-content: center;">
<p><a href="./plots/kimmdy/rates.png" class="lightbox" data-gallery="quarto-lightbox-gallery-90" title="Figure&nbsp;4.22: Reaction rates (log10-scale) for homolysis and hydrolysis in the collagen fibril system (left) and the single chain (right), with the systems visualized above the plot. The 1&nbsp;nN case for the fibril system represents the shear pulling setup. The value of the highest rate for each setup is shown above the respective distribution. The white points in the hydrolysis rate distributions of the single peptide denote the experimental reference value from AFM experiments [@pillMechanicalActivationDrastically2019] for the respective force. Rates lower than 10-25&nbsp;1/s are not shown."><img src="./plots/kimmdy/rates.png" class="img-fluid figure-img"></a></p>
</div>
</div>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-hyd-vs-hom-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4.22: Reaction rates (log10-scale) for homolysis and hydrolysis in the collagen fibril system (left) and the single chain (right), with the systems visualized above the plot. The 1&nbsp;nN case for the fibril system represents the shear pulling setup. The value of the highest rate for each setup is shown above the respective distribution. The white points in the hydrolysis rate distributions of the single peptide denote the experimental reference value from AFM experiments <span class="citation" data-cites="pillMechanicalActivationDrastically2019">[<a href="#ref-pillMechanicalActivationDrastically2019" role="doc-biblioref">20</a>]</span> for the respective force. Rates lower than 10<sup>-25</sup>&nbsp;1/s are not shown.
</figcaption>
</figure>
</div>
<p>This is due to areas of high stress concentration in the large collagen fibril system that push the reaction rates into a region of the force response curve where hydrolysis rates already taper off and homolysis rates still continue to rise with force (see <a href="#fig-force-responses" class="quarto-xref">Figure&nbsp;4.13</a> and <span class="citation" data-cites="rennekampHybridKineticMonte2020">[<a href="#ref-rennekampHybridKineticMonte2020" role="doc-biblioref">100</a>]</span>). This force concentration is the result of the shear pulling in combination with the cross-links. Ultimately, these high rates are what we are concerned with the most, because once a reaction has happened during instantaneous loading, the stress should subside. This puts more focus on the fastest rates. The competition would be even more in favor of homolysis if the hydrolysis model used the theoretical reaction rates based on the <em>ab initio</em> activation energy calculations by <span class="citation" data-cites="pillMechanicalActivationDrastically2019">[<a href="#ref-pillMechanicalActivationDrastically2019" role="doc-biblioref">20</a>]</span> as they can be seen in the extended data <a href="#fig-rates-hydrolysis" class="quarto-xref">Figure&nbsp;5.6</a>. Extended data <a href="#fig-rates-fibril-shear" class="quarto-xref">Figure&nbsp;5.7</a> shows a more fine-grained force profile between 0 and 1&nbsp;nN of shear pulling.</p>
<div style="page-break-after: always;"></div>
</section>
<section id="discussion-and-outlook-of-applying-kimmdy-to-hydrolysis-and-homolysis" class="level3" data-number="4.3.5">
<h3 data-number="4.3.5" class="anchored" data-anchor-id="discussion-and-outlook-of-applying-kimmdy-to-hydrolysis-and-homolysis"><span class="header-section-number">4.3.5</span> Discussion and outlook of applying KIMMDY to hydrolysis and homolysis</h3>
<p>We were able to simulate direct competition of the two reactions and provide qualitative answers as to how hydrolysis and homolysis compete. But directly connecting quantitative values from experiments to simulated reaction rates remains a challenge. KIMMDY bridges the gap between AFM experiments on single peptides and large scale biomolecular simulations of collagen fibrils, but we are still missing a concrete link to experimental data of real tissues. Zapp et al.&nbsp;pulled with stresses of 2-40&nbsp;MPa on tissue samples of tendons and saw signals of radicals in EPR measurements <span class="citation" data-cites="zappMechanoradicalsTensedTendon2020">[<a href="#ref-zappMechanoradicalsTensedTendon2020" role="doc-biblioref">17</a>]</span>, but it is unclear how the number of radicals could be quantified in a way that would map down to reaction rates at the molecular level. The external pulling force (with shear pulling) at which we see the first homolysis reaction rates at the second timescale in the KIMMDY simulations is greater than 0.6&nbsp;nN per chain (see <a href="#fig-rates-fibril-shear" class="quarto-xref">Figure&nbsp;5.7</a>) and hydrolysis would be predicted to occur before that according to the model based on AFM experiments. This force equates to 1.8&nbsp;nN per triple helix, which has a cross-sectional area of 1.77&nbsp;nm<sup>2</sup> (1.5&nbsp;nm diameter) and thus results in a stress of 1.02 GPa, which is two to three orders of magnitude higher than what is used in experiments and what is measured physiologically (<a href="#sec-intro-mech" class="quarto-xref">Section&nbsp;1.2</a>), even with taking into account that only 60% of the tendon weight would be collagen and subtracting the water content to get an effective CSA. Together this suggests that the forces within complex biological systems can momentarily and locally confined exceed computationally determined thresholds due to irregularities as they can only be modeled to a limited extent even with the large collagen fibril simulation system. Thus, a model for the missing scale between complete tendons and sections of fibrils would be highly interesting for studying force propagation.</p>
<p>The current model for hydrolysis only takes the pH value into account in the form of a global scaling factor. Combining KIMMDY with advances in constant pH simulations in GROMACS <span class="citation" data-cites="ahoScalableConstantPH2022">[<a href="#ref-ahoScalableConstantPH2022" role="doc-biblioref">123</a>]</span> could allow for a more fine-grained influence and lead to more accurate predictions of the hydrolysis rate.</p>
<p>Likewise, the influence of the solvent accessibility of the peptide bond is highly likely to be underestimated in the current model for multiple reasons. Firstly, as shown in <a href="#sec-main-qm" class="quarto-xref">Chapter&nbsp;3</a>, the mobility of protons, the formation of hydrogen bonds and in general the arrangement of water and hydroxide molecules around the peptide bond is crucial for the hydrolysis reaction. So is the angle of attack of the hydroxide towards the electrophilic center. It is a current limitation of the simple model that SASA only influences the rate linearly, which becomes clear especially when looking at the range of possible values in the collagen fibril system. The SASA only entering the rate calculation linearly is a very conservative assumption. It is expected that a more accurate model that takes into account the likelihood of water molecules arranging favorably around the reaction site will produce lower reaction rates, which would further swing the reaction rates in favor of homolysis. Secondly, the collagen fibril model, and the way external pulling is applied, best describes a slice through an infinite collagen fiber. But nature is not infinite. Thus, even with <a href="#sec-sasa" class="quarto-xref">Section&nbsp;4.3.2</a> showing that large portions of peptide bonds remain accessible to solvent even near the center of the fibril, it remains to be seen if this is actually the case in even larger scale simulations. Those could potentially better take into account the thinning of the system as it is stretched and answer how this affects the water content and the packing of the triple helices in the fibril.</p>
<div style="page-break-after: always;"></div>
</section>
</section>
<section id="beyond-hydrolysis-kimmdy-can-be-applied-to-a-variety-of-systems-and-reactions" class="level2 page-columns page-full" data-number="4.4">
<h2 data-number="4.4" class="anchored" data-anchor-id="beyond-hydrolysis-kimmdy-can-be-applied-to-a-variety-of-systems-and-reactions"><span class="header-section-number">4.4</span> Beyond hydrolysis: KIMMDY can be applied to a variety of systems and reactions</h2>
<p>KIMMDY is designed as a framework that can be easily extended to different chemistries. As such, we included examples for various systems and reactions in our manuscript <span class="citation" data-cites="kimmdy2025">[<a href="#ref-kimmdy2025" role="doc-biblioref">99</a>]</span>. I would like to especially shout out my colleagues Eric Hartmann, Kai Riedmiller, Evgeni Ulanov, Boris Schüpp, Denis Kiesewetter, Daniel Sucerquia and Camilo Aponte-Santamaría. The examples I am about to show were set up by them, while I only provided implementation help for the reaction plugins as well as features and specialized fixes on the KIMMDY side.</p>
<p>We showed that KIMMDY can be applied to a very diverse set of systems. Boris Schüpp applied KIMMDY to cross-linking reactions in DNA origami and Camilo Aponte-Santamaría showed that KIMMDY can even work with coarse-grained polymer systems. Both of these systems are interesting in their own right, but don’t quite fit into the core story of this thesis. They provide a great way of showing the versatility of KIMMDY, but there is one result I would like to focus on in particular, because it complements the rest of the collagen discoveries very well.</p>
<section id="kimmdy-finds-a-new-radical-scavenging-candidate-in-collagen" class="level3 page-columns page-full" data-number="4.4.1">
<h3 data-number="4.4.1" class="anchored" data-anchor-id="kimmdy-finds-a-new-radical-scavenging-candidate-in-collagen"><span class="header-section-number">4.4.1</span> KIMMDY finds a new radical scavenging candidate in collagen</h3>
<p>With the power of KIMMDY we can simulate the time evolution of reactions through large biomolecular systems. First, with KIMMDY we simulated homolytic bond breaks in the collagen fibril system (see <a href="#fig-hyd-vs-hom" class="quarto-xref">Figure&nbsp;4.22</a> and <a href="#fig-fibril2" class="quarto-xref">Figure&nbsp;4.14</a>). Next, we used it to simulate what happens to the radicals that form as the result of those bond scissions (<a href="#fig-hat-schema" class="quarto-xref">Figure&nbsp;4.23</a>). A radical can move through a molecule by a process known as hydrogen atom transfer (HAT), whereby a nearby hydrogen atom forms a covalent bond with the radical, leaving its original binding partner as the new radical. Kai Riedmiller trained a graph neural network to emulate the reaction energy barrier of HAT with high accuracy <span class="citation" data-cites="riedmillerSubstitutingDensityFunctional2024">[<a href="#ref-riedmillerSubstitutingDensityFunctional2024" role="doc-biblioref">124</a>]</span>, which in turn provides reaction rates for KIMMDY in the form of the HAT-plugin. Evgeni Ulanov showed in simulations of small n-alkyl radicals that KIMMDY robustly reproduces the overall reaction dynamics compared to experiments, with only slight deviations for the most prominent shifts relative to each other. Eric Hartmann and Denis Kiesewetter then applied this to the large collagen system.</p>
<div id="fig-hat-schema" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-hat-schema-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./figures/kimmdy/kimmdy-hat-schema.png" class="lightbox" data-gallery="quarto-lightbox-gallery-91" title="Figure&nbsp;4.23: Schema of successive HAT reactions after homolytic bond scission simulated in KIMMDY."><img src="./figures/kimmdy/kimmdy-hat-schema.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-hat-schema-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4.23: Schema of successive HAT reactions after homolytic bond scission simulated in KIMMDY.
</figcaption>
</figure>
</div>
<p>After many consecutive HAT reactions, the dwell-time for the residues was analyzed and two compounds stood out. The first is the familiar dihydroxyphenylalanine or DOPA (<a href="#fig-dopa" class="quarto-xref">Figure&nbsp;4.25</a>), which forms a stable radical at one of the hydroxides. The second is a new face, the pyridinoline cross-link (PYD, <a href="#fig-pyd" class="quarto-xref">Figure&nbsp;4.26</a>). It can also form a stable radical at its hydroxide, as BDE calculations then revealed (<a href="#fig-hat-bdes" class="quarto-xref">Figure&nbsp;4.27 (a)</a>). Furthermore, when Daniel Sucerquia calculated the expected EPR spectrum of the PYD radical it makes up for part of the missing intensity that is seen in experimental spectra compared to just using DOPA. This is an exciting discovery, because it suggests that not only is collagen more likely to break at or near the crosslinks <span class="citation" data-cites="rennekampHybridKineticMonte2020">[<a href="#ref-rennekampHybridKineticMonte2020" role="doc-biblioref">100</a>]</span>, nature also came prepared with a radical scavenger in the very vicinity.</p>

<div class="no-row-height column-margin column-container"><div id="fig-phe" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-phe-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./img/phenylalanine.svg" class="lightbox" data-gallery="quarto-lightbox-gallery-92" title="Figure&nbsp;4.24: phenylalanine"><img src="./img/phenylalanine.svg" class="img-fluid figure-img" style="width:50.0%"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-phe-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4.24: phenylalanine
</figcaption>
</figure>
</div><div id="fig-dopa" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-dopa-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./img/dopa.svg" class="lightbox" data-gallery="quarto-lightbox-gallery-93" title="Figure&nbsp;4.25: Dihydroxyphenylalanine, (DOPA)"><img src="./img/dopa.svg" class="img-fluid figure-img" style="width:60.0%"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-dopa-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4.25: Dihydroxyphenylalanine, (DOPA)
</figcaption>
</figure>
</div><div id="fig-pyd" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-pyd-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./img/pyridinoline-center.svg" class="lightbox" data-gallery="quarto-lightbox-gallery-94" title="Figure&nbsp;4.26: Center of a pyridinoline crosslink"><img src="./img/pyridinoline-center.svg" class="img-fluid figure-img" style="width:60.0%"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-pyd-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4.26: Center of a pyridinoline crosslink
</figcaption>
</figure>
</div></div>

<div id="fig-dopa-pyd" class="quarto-layout-panel">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-dopa-pyd-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="quarto-layout-row">
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-dopa-pyd" style="flex-basis: 50.0%;justify-content: flex-start;">
<div id="fig-hat-bdes" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-hat-bdes-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./figures/kimmdy/kimmdy-hat-bdes.svg" class="lightbox" data-gallery="fig-dopa-pyd" title="Figure&nbsp;4.27&nbsp;(a): "><img src="./figures/kimmdy/kimmdy-hat-bdes.svg" id="fig-hat-bdes" class="img-fluid figure-img" data-ref-parent="fig-dopa-pyd"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-hat-bdes-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(a)
</figcaption>
</figure>
</div>
</div>
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-dopa-pyd" style="flex-basis: 50.0%;justify-content: flex-start;">
<div id="fig-hat-epr" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-hat-epr-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./figures/kimmdy/kimmdy-hat-epr.svg" class="lightbox" data-gallery="fig-dopa-pyd" title="Figure&nbsp;4.27&nbsp;(b): "><img src="./figures/kimmdy/kimmdy-hat-epr.svg" id="fig-hat-epr" class="img-fluid figure-img" data-ref-parent="fig-dopa-pyd"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-hat-epr-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(b)
</figcaption>
</figure>
</div>
</div>
</div>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-dopa-pyd-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4.27: Analysis of the DOPA and PYD radicals. (a) Bond dissociation energy BDE of PYD (blue line) and DOPA (orange line) compared to a distribution of BDE of abstractions processes in different amino acids from <span class="citation" data-cites="treydeBondDissociationEnergies2022">[<a href="#ref-treydeBondDissociationEnergies2022" role="doc-biblioref">125</a>]</span> (gray bars). (b) Experimental EPR spectrum compared to theoretical EPR spectrum for a DOPA and PYD radical.
</figcaption>
</figure>
</div>
<div style="page-break-after: always;"></div>
</section>
</section>
</section>
<section id="concluding-thoughts-and-next-steps" class="level1 page-columns page-full" data-number="5">
<h1 data-number="5"><span class="header-section-number">5</span> Concluding thoughts and next steps</h1>
<p>I set out to answer how two reactions, peptide bond hydrolysis and homolytic bond scission, compete under external force in complex biological systems, specifically in our most important structural protein, collagen, with its unique multi scale structure (<a href="#fig-collagen-structure" class="quarto-xref">Figure&nbsp;1.9</a>). To address this question, I used and developed two complementary methods.</p>
<p>Hybrid quantum mechanics/molecular mechanics simulations provided atomistic insights into the reaction mechanisms of base-catalyzed hydrolysis. They highlighted the dynamic nature of the hydrolysis reaction, showing how proton mobility within the solvent is crucial. The free energy profiles of the reaction for the same sites in both a single peptide and a triple helix revealed that the triple helix does marginally impede peptide hydrolysis thermodynamically. Since the reaction barrier enters exponentially into reaction rate calculations (see <a href="#eq-arrhenius" class="quarto-xref">Equation&nbsp;2.33</a>), this seemingly small difference of 9.02&nbsp;kJ/mol still corresponds to a decrease in the reaction rate by a factor of 30 to 40. However, the computational costs of QM/MM simulations in combination with the excessive sampling requirements of a reaction as complex as hydrolysis, limit the scope when looking into the two competing reactions at scale. This limitation motivated the development of KIMMDY (Kinetic Monte Carlo Molecular Dynamics), a generalized framework for bridging timescales in reactive molecular dynamics simulations.</p>
<p>Built as a collaborative effort with Eric Hartmann and Kai Riedmiller, KIMMDY combines classical molecular dynamics simulations with a plugin architecture for incorporating diverse reaction types based on physical models, machine learning, or experimental heuristics. KIMMDY allowed us to directly compare competing reactions at scale and revealed the critical role of force concentration in cross-linked fibrils. Localized stress concentrations lead to a regime where homolysis becomes competitive with or even outcompetes hydrolysis. Additional analysis uncovered that solvent accessibility of peptide bonds remains surprisingly high even in the fibril interior, particularly under tension. Proline residues, which are part of the characteristic Gly-X-Y motif in collagen, appear to offer some protection against hydrolysis by reducing solvent accessibility, even though they experience higher mechanical strain. The findings from the QM/MM simulations strongly suggest that the influence of solvent accessibility should be greater than how it is currently modeled, but more data is required to answer this question quantitatively. This provides an opportunity to build a more accurate model for hydrolysis in the future. One approach could be a machine learned emulator trained on QM/MM data that predicts reaction barriers based on the local chemical environment of each peptide bond, similar to the HAT model by Kai Riedmiller <span class="citation" data-cites="riedmillerSubstitutingDensityFunctional2024">[<a href="#ref-riedmillerSubstitutingDensityFunctional2024" role="doc-biblioref">124</a>]</span>. However, due to the complexity of the reaction mechanism, this still poses a considerable challenge. The homolysis reaction on the other hand could also be modeled more accurately, by using more information about the chemical environment than just the current bond stretching and the BDE.</p>
<p>Beyond the specific question of hydrolysis versus homolysis, KIMMDY enabled the simulation of radical dynamics through hydrogen atom transfer reactions. This led to the discovery that pyridinoline crosslinks can act as radical scavengers, complementing DOPA, which helped explaining previously unaccounted features in experimental EPR spectra. The finding that nature has placed radical scavengers in the very vicinity where bonds are most likely to break <span class="citation" data-cites="rennekampCollagenBreaksWeak2023">[<a href="#ref-rennekampCollagenBreaksWeak2023" role="doc-biblioref">126</a>]</span> reveals an elegant protective mechanism.</p>
<p>The forces at which homolysis reactions become prevalent in simulations remain orders of magnitude higher than experimentally measured tissue stresses, suggesting that additional factors beyond what can be captured even in large-scale fibril simulations contribute to force amplification in biological systems. Thus it would be a great opportunity in the future to build a model that bridges the missing scale between all-atom molecular dynamics simulations and real tissue samples. Coarse grained force fields such as Martini3 <span class="citation" data-cites="souzaMartini3General2021">[<a href="#ref-souzaMartini3General2021" role="doc-biblioref">127</a>]</span> provide a way of simulating larger systems by merging multiple atoms into a combined representation and KIMMDY is already prepared to work with such topologies. This leaves only the reaction plugins to be adapted to the coarse grained representation. An even more ambitious approach would be to couple KIMMDY with finite element models <span class="citation" data-cites="hrennikoffSolutionProblemsElasticity2021 courantVariationalMethodsSolution1943">[<a href="#ref-hrennikoffSolutionProblemsElasticity2021" role="doc-biblioref">128</a>, <a href="#ref-courantVariationalMethodsSolution1943" role="doc-biblioref">129</a>]</span> of entire tissues <span class="citation" data-cites="freutelFiniteElementModeling2014">[<a href="#ref-freutelFiniteElementModeling2014" role="doc-biblioref">130</a>]</span>, which would allow for simulating tendons or ligaments, albeit at a lower resolution.</p>
<p>Crossing the experimental to computational divide from the other side with more single-molecule experiments should also be fruitful. While we now have experimental lifetimes of peptide bonds <span class="citation" data-cites="pillMechanicalActivationDrastically2019">[<a href="#ref-pillMechanicalActivationDrastically2019" role="doc-biblioref">20</a>]</span>, seeing the same for homolysis would be incredibly exciting. This could be achieved with an AFM setup with a medium containing a probe for radicals or the products of their follow-up reactions with the solvent (reactive oxygen species) such as 2’,7’-Dichlorodihydrofluorescein (DCFH2, <a href="#fig-dcfh2" class="quarto-xref">Figure&nbsp;5.1</a>) <span class="citation" data-cites="chen27DichlorodihydrofluoresceinFluorescentProbe2010">[<a href="#ref-chen27DichlorodihydrofluoresceinFluorescentProbe2010" role="doc-biblioref">131</a>]</span>. Combining this with the experimental setup of Pill et al. <span class="citation" data-cites="pillMechanicalActivationDrastically2019">[<a href="#ref-pillMechanicalActivationDrastically2019" role="doc-biblioref">20</a>]</span>, could directly test the predictions made by KIMMDY about the competition of hydrolysis and homolysis under force.</p>

<div class="no-row-height column-margin column-container"><div id="fig-dcfh2" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-dcfh2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./img/dcfh2.svg" class="lightbox" data-gallery="quarto-lightbox-gallery-97" title="Figure&nbsp;5.1: 2’,7’-Dichlorodihydrofluorescein (DCFH2)"><img src="./img/dcfh2.svg" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-dcfh2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;5.1: 2’,7’-Dichlorodihydrofluorescein (DCFH2)
</figcaption>
</figure>
</div></div><p>Quite a while ago I was working on my Bachelor’s thesis in the lab of Matthias Mayer at the Center for Molecular Biology of Heidelberg University (ZMBH). During a group meeting I presented my findings and came across two seemingly contradictory data points, about which I then stated: “… and now I’m just confused.” Matthias immediately jumped in to say: “That’s brilliant! Being confused is always the first step.” Since then I cherish those moments of confusion, those drivers of discovery. What I didn’t fully grasp at that time is that every next step after the first, simultaneously has the potential to be the first step of something new. I sincerely hope that in this thesis, I provided some answers and built a tool to find more in the future, but more importantly, also opened up many new questions, so that we can all revel in confusion together.</p>

<div class="no-row-height column-margin column-container"><div class="">
<p><em>«&nbsp;Being confused is always the first step!&nbsp;»</em></p>
</div></div><div style="page-break-after: always;"></div>
</section>
<section id="appendix" class="level1 unnumbered">
<h1 class="unnumbered">Appendix</h1>
<section id="extended-data-figures" class="level2 unnumbered">
<h2 class="unnumbered anchored" data-anchor-id="extended-data-figures">Extended data figures</h2>
<div id="fig-add-qm" class="quarto-layout-panel">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-add-qm-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="quarto-layout-row">
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-add-qm" style="flex-basis: 50.0%;justify-content: flex-start;">
<div id="fig-hyd-filter" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-hyd-filter-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./plots/qm/wethyd_us_ti_filter.png" class="lightbox" data-gallery="fig-add-qm" title="Figure&nbsp;5.2&nbsp;(a): "><img src="./plots/qm/wethyd_us_ti_filter.png" id="fig-hyd-filter" class="img-fluid figure-img" data-ref-parent="fig-add-qm"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-hyd-filter-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(a)
</figcaption>
</figure>
</div>
</div>
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-add-qm" style="flex-basis: 50.0%;justify-content: flex-start;">
<div id="fig-carbonyl-orientation" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-carbonyl-orientation-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./plots/qm/carbonyl_orientation_example.png" class="lightbox" data-gallery="fig-add-qm" title="Figure&nbsp;5.2&nbsp;(b): "><img src="./plots/qm/carbonyl_orientation_example.png" id="fig-carbonyl-orientation" class="img-fluid figure-img" data-ref-parent="fig-add-qm"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-carbonyl-orientation-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(b)
</figcaption>
</figure>
</div>
</div>
</div>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-add-qm-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;5.2: Extended QM/MM sampling statistics. (a) Bond length of the original carbonyl C=O bond plotted against the distance of the attacking hydroxide to the carbonyl C. Each point represents the average within one umbrella sampling window. Windows colored red were discarded due to being side reactions.<br>
(b) The distance of the carbonyl <span class="math inline">\(\ce{O}\)</span> minus the distance of the carbonyl <span class="math inline">\(\ce{C}\)</span> of the peptide bond from the central axis of the force-equilibrated collagen triple helix structure. This represents the orientation of the carbonyl C=O bond, where a number greater than 0 means that the bond is pointing outwards from the backbone, away from the respective other two chains of the helix and a value smaller than 0 means it is pointing into the helix. The sampled sites are highlighted in red. The blue points are additional sites that are being sampled for the next publication.
</figcaption>
</figure>
</div>
<div id="fig-tiprot-filtering" class="quarto-layout-panel">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-tiprot-filtering-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="quarto-layout-row">
<div class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="./plots/qm/wethyd_us_tiprots_no_filter_example.png" class="lightbox" data-gallery="quarto-lightbox-gallery-100" title="Protonation states before filtering."><img src="./plots/qm/wethyd_us_tiprots_no_filter_example.png" class="img-fluid figure-img"></a></p>
<figcaption>Protonation states before filtering.</figcaption>
</figure>
</div>
</div>
<div class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="./plots/qm/wethyd_us_tiprots_filtered_example.png" class="lightbox" data-gallery="quarto-lightbox-gallery-101" title="Protonation states after filtering."><img src="./plots/qm/wethyd_us_tiprots_filtered_example.png" class="img-fluid figure-img"></a></p>
<figcaption>Protonation states after filtering.</figcaption>
</figure>
</div>
</div>
</div>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-tiprot-filtering-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;5.3: The effect of filtering out side reactions for one affected example.
</figcaption>
</figure>
</div>
<div id="fig-add-hyd" class="quarto-layout-panel">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-add-hyd-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="quarto-layout-row">
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-add-hyd" style="flex-basis: 50.0%;justify-content: flex-start;">
<div id="fig-ext-hyd-ph" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-ext-hyd-ph-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./plots/kimmdy/rates-triple-vs-single.png" class="lightbox" data-gallery="fig-add-hyd" title="Figure&nbsp;5.4&nbsp;(a): Hydrolysis reaction rates for increasing forces and varying pH values in the single peptide chain and triple helix systems as sampled with KIMMDY. "><img src="./plots/kimmdy/rates-triple-vs-single.png" class="img-fluid figure-img" data-ref-parent="fig-add-hyd"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-ext-hyd-ph-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(a) Hydrolysis reaction rates for increasing forces and varying pH values in the single peptide chain and triple helix systems as sampled with KIMMDY.
</figcaption>
</figure>
</div>
</div>
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-add-hyd" style="flex-basis: 50.0%;justify-content: flex-start;">
<div id="fig-sasa-f" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-sasa-f-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./plots/kimmdy/sasa_vs_mean_force.png" class="lightbox" data-gallery="fig-add-hyd" title="Figure&nbsp;5.4&nbsp;(b): SASA in 1&nbsp;nN shear pulling fibril simulation plotted against the Mean Excess Force, i.e., the force derived from bond extension in each peptide bond after accounting for average bond forces in the baseline, 0&nbsp;nN simulations."><img src="./plots/kimmdy/sasa_vs_mean_force.png" class="img-fluid figure-img" data-ref-parent="fig-add-hyd"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-sasa-f-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(b) SASA in 1&nbsp;nN shear pulling fibril simulation plotted against the Mean Excess Force, i.e., the force derived from bond extension in each peptide bond after accounting for average bond forces in the baseline, 0&nbsp;nN simulations.
</figcaption>
</figure>
</div>
</div>
</div>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-add-hyd-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;5.4: Extended hydrolysis results from KIMMDY runs.
</figcaption>
</figure>
</div>
<div id="fig-mean-bond-f-pep" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-mean-bond-f-pep-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./plots/kimmdy/bond-forces-peptide.png" class="lightbox" data-gallery="quarto-lightbox-gallery-104" title="Figure&nbsp;5.5: A closer look at the mean bond force in the peptide bonds of the collagen fibril system under shear pulling."><img src="./plots/kimmdy/bond-forces-peptide.png" class="img-fluid figure-img" style="width:50.0%"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-mean-bond-f-pep-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;5.5: A closer look at the mean bond force in the peptide bonds of the collagen fibril system under shear pulling.
</figcaption>
</figure>
</div>
<div id="fig-rates-hydrolysis" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-rates-hydrolysis-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./plots/kimmdy/rates-hydrolysis.png" class="lightbox" data-gallery="quarto-lightbox-gallery-105" title="Figure&nbsp;5.6: Hydrolysis reaction rates (log10-scale) for a single chain under force increasing from 0 to 3&nbsp;nN using the experimental (exp) model compared to the physical model based on single-point energies. Rates lower than 10-13&nbsp;1/s are not shown."><img src="./plots/kimmdy/rates-hydrolysis.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-rates-hydrolysis-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;5.6: Hydrolysis reaction rates (log10-scale) for a single chain under force increasing from 0 to 3&nbsp;nN using the experimental (exp) model compared to the physical model based on single-point energies. Rates lower than 10<sup>-13</sup>&nbsp;1/s are not shown.
</figcaption>
</figure>
</div>
<div id="fig-rates-fibril-shear" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-rates-fibril-shear-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="./plots/kimmdy/rates-fibril-shear.png" class="lightbox" data-gallery="quarto-lightbox-gallery-106" title="Figure&nbsp;5.7: Reaction rates (log10-scale) for hydrolysis and homolysis in the collagen fibril system under shear stress for increasing external force with the 1&nbsp;nN no-shear setup shown for comparison. Rates lower than 10-25&nbsp;1/s are not shown."><img src="./plots/kimmdy/rates-fibril-shear.png" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-rates-fibril-shear-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;5.7: Reaction rates (log10-scale) for hydrolysis and homolysis in the collagen fibril system under shear stress for increasing external force with the 1&nbsp;nN no-shear setup shown for comparison. Rates lower than 10<sup>-25</sup>&nbsp;1/s are not shown.
</figcaption>
</figure>
</div>
<div style="page-break-after: always;"></div>
</section>
<section id="extended-data-listings" class="level2 unnumbered">
<h2 class="unnumbered anchored" data-anchor-id="extended-data-listings">Extended data listings</h2>
<div id="lst-gro" class="text listing quarto-float quarto-figure quarto-figure-left anchored">
<figure class="quarto-float quarto-float-lst figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-lst" id="lst-gro-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Listing&nbsp;5.1: Select lines from a .gro file of a capped glycine dipeptide
</figcaption>
<div aria-describedby="lst-gro-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="code-copy-outer-scaffold"><div class="sourceCode" id="cb10"><pre class="sourceCode numberSource text number-lines code-with-copy"><code class="sourceCode"><span id="cb10-1"><a href="#cb10-1"></a>   26</span>
<span id="cb10-2"><a href="#cb10-2"></a>    1ACE    CH3    1  -0.351  -0.014  -0.085</span>
<span id="cb10-3"><a href="#cb10-3"></a>    1ACE   HH31    2  -0.365  -0.002   0.013</span>
<span id="cb10-4"><a href="#cb10-4"></a>[...]</span>
<span id="cb10-5"><a href="#cb10-5"></a>   1.11109   0.40658   0.49846</span></code></pre></div><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></div>
</div>
</figure>
</div>
<div id="lst-pdb" class="text listing quarto-float quarto-figure quarto-figure-left anchored">
<figure class="quarto-float quarto-float-lst figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-lst" id="lst-pdb-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Listing&nbsp;5.2: Select lines from a .pdb file of a capped glycine dipeptide, shortened.
</figcaption>
<div aria-describedby="lst-pdb-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="code-copy-outer-scaffold"><div class="sourceCode" id="cb11"><pre class="sourceCode numberSource text number-lines code-with-copy"><code class="sourceCode"><span id="cb11-1"><a href="#cb11-1"></a>ATOM  1  C  ACE  1  -2.031  -0.018  -1.163  1.00  0.00  C</span>
<span id="cb11-2"><a href="#cb11-2"></a>ATOM  2  O  ACE  1  -1.615  -0.128  -2.324  1.00  0.00  O</span></code></pre></div><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></div>
</div>
</figure>
</div>
<div id="lst-python" class="python listing quarto-float quarto-figure quarto-figure-left anchored">
<figure class="quarto-float quarto-float-lst figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-lst" id="lst-python-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Listing&nbsp;5.3: Excerpts from the code of KIMMDY.
</figcaption>
<div aria-describedby="lst-python-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="code-copy-outer-scaffold"><div class="sourceCode" id="cb12"><pre class="sourceCode numberSource python number-lines code-with-copy"><code class="sourceCode python"><span id="cb12-1"><a href="#cb12-1"></a><span class="kw">class</span> MoleculeType:</span>
<span id="cb12-2"><a href="#cb12-2"></a>    <span class="co">"""One moleculetype in the topology</span></span>
<span id="cb12-3"><a href="#cb12-3"></a></span>
<span id="cb12-4"><a href="#cb12-4"></a><span class="co">    Attributes</span></span>
<span id="cb12-5"><a href="#cb12-5"></a><span class="co">    ----------</span></span>
<span id="cb12-6"><a href="#cb12-6"></a><span class="co">    atoms : dict[str, Atom]</span></span>
<span id="cb12-7"><a href="#cb12-7"></a><span class="co">    bonds : dict[tuple[str, str], Bond]</span></span>
<span id="cb12-8"><a href="#cb12-8"></a><span class="co">    pairs : dict[tuple[str, str], Pair]</span></span>
<span id="cb12-9"><a href="#cb12-9"></a><span class="co">    angles : dict[tuple[str, str, str], Angle]</span></span>
<span id="cb12-10"><a href="#cb12-10"></a><span class="co">    proper_dihedrals : dict[tuple[str, str, str, str], MultipleDihedrals]</span></span>
<span id="cb12-11"><a href="#cb12-11"></a><span class="co">    improper_dihedrals : dict[tuple[str, str, str, str], Dihedral]</span></span>
<span id="cb12-12"><a href="#cb12-12"></a><span class="co">    position_restraints : dict[str, PositionRestraint]</span></span>
<span id="cb12-13"><a href="#cb12-13"></a><span class="co">    dihedral_restraints : dict[tuple[str, str, str, str], DihedralRestraint]</span></span>
<span id="cb12-14"><a href="#cb12-14"></a><span class="co">    radicals : dict[str, Atom]</span></span>
<span id="cb12-15"><a href="#cb12-15"></a><span class="co">        dict mapping atom indices to atom objects for storing all radical atoms</span></span>
<span id="cb12-16"><a href="#cb12-16"></a></span>
<span id="cb12-17"><a href="#cb12-17"></a><span class="co">    """</span></span></code></pre></div><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></div>
</div>
</figure>
</div>
<div id="lst-python2" class="python listing quarto-float quarto-figure quarto-figure-left anchored">
<figure class="quarto-float quarto-float-lst figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-lst" id="lst-python2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Listing&nbsp;5.4: Excerpts from the code of KIMMDY.
</figcaption>
<div aria-describedby="lst-python2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="code-copy-outer-scaffold"><div class="sourceCode" id="cb13"><pre class="sourceCode numberSource python number-lines code-with-copy"><code class="sourceCode python"><span id="cb13-1"><a href="#cb13-1"></a><span class="at">@dataclass</span>()</span>
<span id="cb13-2"><a href="#cb13-2"></a><span class="kw">class</span> Atom:</span>
<span id="cb13-3"><a href="#cb13-3"></a>    <span class="co">"""Information about one atom</span></span>
<span id="cb13-4"><a href="#cb13-4"></a></span>
<span id="cb13-5"><a href="#cb13-5"></a><span class="co">    A class containing atom information as in the atoms section of the topology.</span></span>
<span id="cb13-6"><a href="#cb13-6"></a><span class="co">    An atom keeps a list of which atoms it is bound to and its radical state.</span></span>
<span id="cb13-7"><a href="#cb13-7"></a></span>
<span id="cb13-8"><a href="#cb13-8"></a><span class="co">    From gromacs topology:</span></span>
<span id="cb13-9"><a href="#cb13-9"></a><span class="co">    ; nr type resnr residue atom cgnr charge mass typeB chargeB massB</span></span>
<span id="cb13-10"><a href="#cb13-10"></a><span class="co">    """</span></span>
<span id="cb13-11"><a href="#cb13-11"></a></span>
<span id="cb13-12"><a href="#cb13-12"></a>    nr: <span class="bu">str</span></span>
<span id="cb13-13"><a href="#cb13-13"></a>    <span class="bu">type</span>: <span class="bu">str</span></span>
<span id="cb13-14"><a href="#cb13-14"></a>    resnr: <span class="bu">str</span></span>
<span id="cb13-15"><a href="#cb13-15"></a>    residue: <span class="bu">str</span></span>
<span id="cb13-16"><a href="#cb13-16"></a>    atom: <span class="bu">str</span></span>
<span id="cb13-17"><a href="#cb13-17"></a>    cgnr: <span class="bu">str</span></span>
<span id="cb13-18"><a href="#cb13-18"></a>    charge: <span class="bu">str</span></span>
<span id="cb13-19"><a href="#cb13-19"></a>    mass: Optional[<span class="bu">str</span>] <span class="op">=</span> <span class="va">None</span></span>
<span id="cb13-20"><a href="#cb13-20"></a>    typeB: Optional[<span class="bu">str</span>] <span class="op">=</span> <span class="va">None</span></span>
<span id="cb13-21"><a href="#cb13-21"></a>    chargeB: Optional[<span class="bu">str</span>] <span class="op">=</span> <span class="va">None</span></span>
<span id="cb13-22"><a href="#cb13-22"></a>    massB: Optional[<span class="bu">str</span>] <span class="op">=</span> <span class="va">None</span></span>
<span id="cb13-23"><a href="#cb13-23"></a>    bound_to_nrs: <span class="bu">list</span>[<span class="bu">str</span>] <span class="op">=</span> field(default_factory<span class="op">=</span><span class="bu">list</span>, compare<span class="op">=</span><span class="va">False</span>)</span>
<span id="cb13-24"><a href="#cb13-24"></a>    is_radical: <span class="bu">bool</span> <span class="op">=</span> field(default<span class="op">=</span><span class="va">False</span>, compare<span class="op">=</span><span class="va">False</span>)</span>
<span id="cb13-25"><a href="#cb13-25"></a>    comment: Optional[<span class="bu">str</span>] <span class="op">=</span> field(default<span class="op">=</span><span class="va">None</span>, compare<span class="op">=</span><span class="va">False</span>)</span>
<span id="cb13-26"><a href="#cb13-26"></a></span>
<span id="cb13-27"><a href="#cb13-27"></a>    <span class="at">@classmethod</span></span>
<span id="cb13-28"><a href="#cb13-28"></a>    <span class="kw">def</span> from_top_line(cls, l: <span class="bu">list</span>[<span class="bu">str</span>]):</span>
<span id="cb13-29"><a href="#cb13-29"></a>        <span class="cf">return</span> cls(</span>
<span id="cb13-30"><a href="#cb13-30"></a>            nr<span class="op">=</span>l[<span class="dv">0</span>],</span>
<span id="cb13-31"><a href="#cb13-31"></a>            <span class="bu">type</span><span class="op">=</span>l[<span class="dv">1</span>],</span>
<span id="cb13-32"><a href="#cb13-32"></a>            resnr<span class="op">=</span>l[<span class="dv">2</span>],</span>
<span id="cb13-33"><a href="#cb13-33"></a>            residue<span class="op">=</span>l[<span class="dv">3</span>],</span>
<span id="cb13-34"><a href="#cb13-34"></a>            atom<span class="op">=</span>l[<span class="dv">4</span>],</span>
<span id="cb13-35"><a href="#cb13-35"></a>            cgnr<span class="op">=</span>l[<span class="dv">5</span>],</span>
<span id="cb13-36"><a href="#cb13-36"></a>            charge<span class="op">=</span>l[<span class="dv">6</span>],</span>
<span id="cb13-37"><a href="#cb13-37"></a>            mass<span class="op">=</span>field_or_none(l, <span class="dv">7</span>),</span>
<span id="cb13-38"><a href="#cb13-38"></a>            typeB<span class="op">=</span>field_or_none(l, <span class="dv">8</span>),</span>
<span id="cb13-39"><a href="#cb13-39"></a>            chargeB<span class="op">=</span>field_or_none(l, <span class="dv">9</span>),</span>
<span id="cb13-40"><a href="#cb13-40"></a>            massB<span class="op">=</span>field_or_none(l, <span class="dv">10</span>),</span>
<span id="cb13-41"><a href="#cb13-41"></a>        )</span></code></pre></div><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></div>
</div>
</figure>
</div>
<div style="page-break-after: always;"></div>
</section>
<section id="additional-software-and-resources" class="level2 unnumbered">
<h2 class="unnumbered anchored" data-anchor-id="additional-software-and-resources">Additional software and resources</h2>
<p>I would like to highlight a couple of additional resources that were instrumental not only during the research for my PhD, but also for the writing of this thesis. This is important to me, because I strongly believe in Open Source and Open Science. Many of these projects are maintained by volunteers and researchers that put in a lot of effort to make them available to the community without expecting anything in return. Especially research software rarely gets the recognition it deserves, despite being a crucial part of modern science.</p>
<section id="knowledge-resources" class="level3 unnumbered">
<h3 class="unnumbered anchored" data-anchor-id="knowledge-resources">Knowledge resources</h3>
<ul>
<li>David Sherrill of Georgia Tech University, who has excellent lectures online for brushing up on QM and MM simulations: <a href="https://www.youtube.com/@DavidSherrill1/">youtube.com/@DavidSherrill1</a></li>
</ul>
</section>
<section id="software" class="level3 unnumbered">
<h3 class="unnumbered anchored" data-anchor-id="software">Software</h3>
<ul>
<li>GROMACS: <a href="https://www.gromacs.org/">gromacs.org</a> <span class="citation" data-cites="abrahamGROMACSHighPerformance2015">[<a href="#ref-abrahamGROMACSHighPerformance2015" role="doc-biblioref">40</a>]</span></li>
<li>PLUMED: <a href="https://www.plumed.org/">plumed.org</a> <span class="citation" data-cites="tribelloPLUMED2New2014">[<a href="#ref-tribelloPLUMED2New2014" role="doc-biblioref">113</a>]</span></li>
<li>VMD: <a href="https://www.ks.uiuc.edu/Research/vmd/">ks.uiuc.edu/Research/vmd/</a> <span class="citation" data-cites="vmd">[<a href="#ref-vmd" role="doc-biblioref">132</a>]</span></li>
<li>R: <a href="https://www.r-project.org/">r-project.org</a> <span class="citation" data-cites="rcoreteamLanguageEnvironmentStatistical2018">[<a href="#ref-rcoreteamLanguageEnvironmentStatistical2018" role="doc-biblioref">133</a>]</span>
<ul>
<li>tidyverse: <a href="https://www.tidyverse.org/">tidyverse.org</a> <span class="citation" data-cites="wickhamWelcomeTidyverse2019">[<a href="#ref-wickhamWelcomeTidyverse2019" role="doc-biblioref">134</a>]</span></li>
</ul></li>
<li>Python: <a href="https://www.python.org/">python.org</a>
<ul>
<li>pandas: <a href="https://pandas.pydata.org/">pandas.pydata.org</a> <span class="citation" data-cites="teamPandasdevPandasPandas2025 mckinney-proc-scipy-2010">[<a href="#ref-teamPandasdevPandasPandas2025" role="doc-biblioref">135</a>, <a href="#ref-mckinney-proc-scipy-2010" role="doc-biblioref">136</a>]</span></li>
<li>polars: <a href="https://pola-rs.github.io/polars/">pola-rs.github.io/polars/</a></li>
<li>numpy: <a href="https://numpy.org/">numpy.org</a> <span class="citation" data-cites="harris2020array">[<a href="#ref-harris2020array" role="doc-biblioref">137</a>]</span></li>
<li>matplotlib: <a href="https://matplotlib.org/">matplotlib.org/</a> <span class="citation" data-cites="Hunter:2007">[<a href="#ref-Hunter:2007" role="doc-biblioref">138</a>]</span></li>
<li>plotnine: <a href="https://plotnine.org/">plotnine.org/</a> <span class="citation" data-cites="plotnine">[<a href="#ref-plotnine" role="doc-biblioref">139</a>]</span></li>
<li>MDAnalysis: <a href="https://www.mdanalysis.org/">mdanalysis.org</a> <span class="citation" data-cites="michaud-agrawalMDAnalysisToolkitAnalysis2011 gowersMDAnalysisPythonPackage2016">[<a href="#ref-gowersMDAnalysisPythonPackage2016" role="doc-biblioref">111</a>, <a href="#ref-michaud-agrawalMDAnalysisToolkitAnalysis2011" role="doc-biblioref">112</a>]</span></li>
<li>uv: <a href="https://docs.astral.sh/uv/">docs.astral.sh/uv/</a></li>
</ul></li>
<li>moldraw (ketcher): <a href="https://moldraw.com">moldraw.com</a></li>
<li>Neovim: <a href="https://neovim.io/">neovim.io</a></li>
<li>Zotero: <a href="https://zotero.org/">zotero.org</a></li>
<li>Quarto: <a href="https://quarto.org/">quarto.org</a> <span class="citation" data-cites="allaireQuarto2022">[<a href="#ref-allaireQuarto2022" role="doc-biblioref">140</a>]</span></li>
<li>Apache Hamilton: <a href="https://hamilton.apache.org/">hamilton.apache.org</a></li>
<li>git: <a href="https://git-scm.com/">git-scm.com</a></li>
<li>mol*: <a href="https://molstar.org/">molstar.org</a> <span class="citation" data-cites="Sehnal_Mol_Viewer_modern_2021">[<a href="#ref-Sehnal_Mol_Viewer_modern_2021" role="doc-biblioref">141</a>]</span></li>
</ul>
<div style="page-break-after: always;"></div>
</section>
</section>
<section id="list-of-abbreviations-and-constants" class="level2 unnumbered">
<h2 class="unnumbered anchored" data-anchor-id="list-of-abbreviations-and-constants">List of abbreviations and constants</h2>
<p>AA: Amino Acid<br>
ACE: acetyl-capped<br>
AFM: atomic force microscopy<br>
Ala: alanine<br>
AO: atomic orbital<br>
BD: Bürgi-Dunitz angle<br>
BDE: bond dissociation energy<br>
CGTO: contracted Gaussian-type orbitals<br>
CLI: command-line interface<br>
COM: Center of Mass<br>
cryo-EM: cryo-electron microscopy<br>
CSA: cross-sectional area<br>
DAG: directed acyclic graph<br>
DFT: density functional theory<br>
DOPA: dihydroxyphenylalanine<br>
e.g.: exempli gratia<br>
EM: electron microscopy<br>
EPR: electron paramagnetic resonance<br>
ER: endoplasmic reticulum<br>
GEEP: Gaussian expansion of the electrostatic potential<br>
GGA: generalized gradient approximation<br>
Gly: glycine<br>
GTO: Gaussian-type orbitals<br>
HAT: hydrogen atom transfer<br>
HITS: Heidelberg Institute for Theoretical Studies<br>
HPC: high performance computing<br>
Hyp: hydroxyproline<br>
ID: identifier<br>
i.e.: id est<br>
kMC: kinetic Monte Carlo<br>
KS: Kohn-Sham<br>
LCAO: linear combination of atomic orbitals<br>
LDA: local density approximation<br>
LJ: Lennard Jones<br>
LLMs: large language models<br>
MD: molecular dynamics<br>
ML: machine learning<br>
MLP: machine learned potential<br>
MM: molecular mechanics<br>
MO: molecular orbital<br>
MPI: message passing interface<br>
MPI-P: Max-Planck-Institute for Polymer Research<br>
NME: N-methyl group<br>
NPT: isothermal-isobaric ensemble<br>
NVE: microcanonical ensemble<br>
NVT: canonical ensemble<br>
PBC: periodic boundary conditions<br>
PME: particle mesh Ewald<br>
PMF: potential of mean force<br>
Pro: proline<br>
PYD: pyridinoline<br>
PyPI: Python package index<br>
QM/MM: quantum mechanics/molecular mechanics<br>
QM: quantum mechanics<br>
QQ: Coulomb<br>
RESP: restrained electrostatic potential<br>
ROS: reactive oxygen species<br>
SASA: solvent accessible surface area<br>
SEM: scanning electron microscopy (also: standard error of the mean)<br>
Slurm: simple linux utility for resource management<br>
STO: Slater-type orbitals<br>
TI: tetrahedral intermediate<br>
TS: transition state<br>
WHAM: weighted histogram analysis method<br>
XC: exchange correlation functional<br>
ZI: zwitterionic intermediate<br>
ZMBH: Center for Molecular Biology of Heidelberg University</p>
<p><span class="math inline">\(N_\mathrm{A}\)</span>: Avogadro’s constant (6.022E-23)<br>
<span class="math inline">\(k_{\mathrm{B}}\)</span>: Boltzmann constant<br>
<span class="math inline">\(h\)</span>: Planck constant<br>
<span class="math inline">\(\hbar\)</span>: reduced Planck constant<br>
</p>
<div style="page-break-after: always;"></div>
<div style="page-break-after: always;"></div>
</section>
</section>
<section id="references" class="level1 unnumbered">
<h1 class="unnumbered">References</h1>
<div id="refs" class="references csl-bib-body" data-entry-spacing="0" data-line-spacing="2" role="list">
<div id="ref-priceAgeScurvy2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">1. </div><div class="csl-right-inline">Price, C. <a href="https://www.sciencehistory.org/stories/magazine/the-age-of-scurvy/">The <span>Age</span> of <span>Scurvy</span></a>. (2017).</div>
</div>
<div id="ref-lindTreatiseScurvyThree1772" class="csl-entry" role="listitem">
<div class="csl-left-margin">2. </div><div class="csl-right-inline">Lind, J. <em>A <span>Treatise</span> on the <span>Scurvy</span>: <span>In Three Parts</span>. <span>Containing</span> an <span>Inquiry Into</span> the <span>Nature</span>, <span>Causes</span>, and <span>Cure</span>, of That <span>Disease</span>. <span>Together</span> with a <span>Critical</span> and <span>Chronological View</span> of What <span>Has Been Published</span> on the <span>Subject</span></em>. <em>S. Crowder</em>. (S. Crowder, 1772).</div>
</div>
<div id="ref-baronSailorsScurvyJames2009" class="csl-entry" role="listitem">
<div class="csl-left-margin">3. </div><div class="csl-right-inline">Baron, J. H. <a href="https://doi.org/10.1111/j.1753-4887.2009.00205.x">Sailors’ scurvy before and after <span>James Lind</span> – a reassessment</a>. <em>Nutrition Reviews</em> <strong>67</strong>, 315–332 (2009).</div>
</div>
<div id="ref-verzarAgingCollagenFiber1964" class="csl-entry" role="listitem">
<div class="csl-left-margin">4. </div><div class="csl-right-inline">Verzár, F. <a href="https://doi.org/10.1016/B978-1-4831-6751-0.50012-4">Aging of the <span>Collagen Fiber</span></a>. in <em>Elsevier</em>. vol. 2 243–300 (Elsevier, 1964).</div>
</div>
<div id="ref-ricard-blumCollagenFamily2011" class="csl-entry" role="listitem">
<div class="csl-left-margin">5. </div><div class="csl-right-inline">Ricard-Blum, S. <a href="https://doi.org/10.1101/cshperspect.a004978">The <span>Collagen Family</span></a>. <em>Cold Spring Harbor Perspectives in Biology</em> <strong>3</strong>, a004978 (2011).</div>
</div>
<div id="ref-collagenSEM2005" class="csl-entry" role="listitem">
<div class="csl-left-margin">6. </div><div class="csl-right-inline">The University of Wales Bioimaging laboratory, Institute of Biological Sciences, The University of Wales, Aberystwyth, Wales, UK, Hughes, L., Archer, C. &amp; Ap Gwynn, I. <a href="https://doi.org/10.22203/eCM.v009a09">The ultrastructure of mouse articular cartilage: <span>Collagen</span> orientation and implications for tissue functionality. <span>A</span> polarised light and scanning electron microscope study and review.</a> <em>European Cells and Materials</em> <strong>9</strong>, 68–84 (2005).</div>
</div>
<div id="ref-fangExperimentalEvaluationMultiscale2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">7. </div><div class="csl-right-inline">Fang, F. &amp; Lake, S. P. <a href="https://doi.org/10.1002/jor.23488">Experimental evaluation of multiscale tendon mechanics</a>. <em>Journal of Orthopaedic Research</em> <strong>35</strong>, 1353–1365 (2017).</div>
</div>
<div id="ref-sahinisReliabilityDistalHamstring2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">8. </div><div class="csl-right-inline">Sahinis, C., Kellis, E., Dafkou, K. &amp; Ellinoudis, A. <a href="https://doi.org/10.1016/j.ultrasmedbio.2021.05.025">Reliability of <span>Distal Hamstring Tendon Length</span> and <span class="nocase">Cross-sectional Area Using</span> 3-<span>D Freehand Ultrasound</span></a>. <em>Ultrasound in Medicine and Biology</em> <strong>47</strong>, 2579–2588 (2021).</div>
</div>
<div id="ref-besierKneeMuscleForces2009" class="csl-entry" role="listitem">
<div class="csl-left-margin">9. </div><div class="csl-right-inline">Besier, T. F., Fredericson, M., Gold, G. E., Beaupré, G. S. &amp; Delp, S. L. <a href="https://doi.org/10.1016/j.jbiomech.2009.01.032">Knee muscle forces during walking and running in patellofemoral pain patients and pain-free controls</a>. <em>Journal of Biomechanics</em> <strong>42</strong>, 898–905 (2009).</div>
</div>
<div id="ref-fukashiroComparisonDirectlyMeasured1993" class="csl-entry" role="listitem">
<div class="csl-left-margin">10. </div><div class="csl-right-inline">Fukashiro, S., Komi, P. V., Järvinen, M. &amp; Miyashita, M. <a href="https://doi.org/10.1016/S0268-0033(05)80006-3">Comparison between the directly measured achilles tendon force and the tendon force calculated from the ankle joint moment during vertical jumps</a>. <em>Clinical Biomechanics</em> <strong>8</strong>, 25–30 (1993).</div>
</div>
<div id="ref-sponbeckAchillesTendonCross2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">11. </div><div class="csl-right-inline">Sponbeck, J. K., Perkins, C. L., Berg, M. J. &amp; Rigby, J. H. <a href="https://doi.org/10.70252/NOKT1275">Achilles <span>Tendon Cross Sectional Area Changes Over</span> a <span>Division I NCAA Cross Country Season</span></a>. <em>International Journal of Exercise Science</em> <strong>10</strong>, 1226–1234 (2017).</div>
</div>
<div id="ref-finniVivoHumanTriceps2000" class="csl-entry" role="listitem">
<div class="csl-left-margin">12. </div><div class="csl-right-inline">Finni, T., Komi, P. V. &amp; Lepola, V. <a href="https://doi.org/10.1007/s004210000289">In vivo human triceps surae and quadriceps femoris muscle function in a squat jump and counter movement jump</a>. <em>European Journal of Applied Physiology</em> <strong>83</strong>, 416–426 (2000).</div>
</div>
<div id="ref-svenssonMechanicalPropertiesHuman2012" class="csl-entry" role="listitem">
<div class="csl-left-margin">13. </div><div class="csl-right-inline">Svensson, R. B., Hansen, P., Hassenkam, T., <em>et al.</em> <a href="https://doi.org/10.1152/japplphysiol.01172.2011">Mechanical properties of human patellar tendon at the hierarchical levels of tendon and fibril</a>. <em>Journal of Applied Physiology</em> <strong>112</strong>, 419–426 (2012).</div>
</div>
<div id="ref-chimichWaterContentAlters1992" class="csl-entry" role="listitem">
<div class="csl-left-margin">14. </div><div class="csl-right-inline">Chimich, D., Shrive, N., Frank, C., Marchuk, L. &amp; Bray, R. <a href="https://doi.org/10.1016/0021-9290(92)90223-N">Water content alters viscoelastic behaviour of the normal adolescent rabbit medial collateral ligament</a>. <em>Journal of Biomechanics</em> <strong>25</strong>, 831–837 (1992).</div>
</div>
<div id="ref-kauzmannViscousFlowLarge1940" class="csl-entry" role="listitem">
<div class="csl-left-margin">15. </div><div class="csl-right-inline">Kauzmann, W. &amp; Eyring, H. <a href="https://doi.org/10.1021/ja01868a059">The <span>Viscous Flow</span> of <span>Large Molecules</span></a>. <em>Journal of the American Chemical Society</em> <strong>62</strong>, 3113–3125 (1940).</div>
</div>
<div id="ref-beyerMechanochemistryMechanicalActivation2005" class="csl-entry" role="listitem">
<div class="csl-left-margin">16. </div><div class="csl-right-inline">Beyer, M. K. &amp; Clausen-Schaumann, H. <a href="https://doi.org/10.1021/cr030697h">Mechanochemistry:  <span>The Mechanical Activation</span> of <span>Covalent Bonds</span></a>. <em>Chemical Reviews</em> <strong>105</strong>, 2921–2948 (2005).</div>
</div>
<div id="ref-zappMechanoradicalsTensedTendon2020" class="csl-entry" role="listitem">
<div class="csl-left-margin">17. </div><div class="csl-right-inline">Zapp, C., Obarska-Kosinska, A., Rennekamp, B., <em>et al.</em> <a href="https://doi.org/10.1038/s41467-020-15567-4">Mechanoradicals in tensed tendon collagen as a source of oxidative stress</a>. <em>Nature Communications</em> <strong>11</strong>, 2315 (2020).</div>
</div>
<div id="ref-fitchMechanochemicalReactionCascade2014" class="csl-entry" role="listitem">
<div class="csl-left-margin">18. </div><div class="csl-right-inline">Fitch, K. R. &amp; Goodwin, A. P. <a href="https://doi.org/10.1021/cm503253n">Mechanochemical <span>Reaction Cascade</span> for <span>Sensitive Detection</span> of <span>Covalent Bond Breakage</span> in <span>Hydrogels</span></a>. <em>Chemistry of Materials</em> <strong>26</strong>, 6771–6776 (2014).</div>
</div>
<div id="ref-xiaBaseCatalyzedPeptideHydrolysis2011" class="csl-entry" role="listitem">
<div class="csl-left-margin">19. </div><div class="csl-right-inline">Xia, F., Bronowska, A. K., Cheng, S. &amp; Gräter, F. <a href="https://doi.org/10.1021/jp202162r">Base-<span>Catalyzed Peptide Hydrolysis Is Insensitive</span> to <span>Mechanical Stress</span></a>. <em>The Journal of Physical Chemistry B</em> <strong>115</strong>, 10126–10132 (2011).</div>
</div>
<div id="ref-pillMechanicalActivationDrastically2019" class="csl-entry" role="listitem">
<div class="csl-left-margin">20. </div><div class="csl-right-inline">Pill, M. F., East, A. L. L., Marx, D., Beyer, M. K. &amp; Clausen‐Schaumann, H. <a href="https://doi.org/10.1002/anie.201902752">Mechanical <span>Activation Drastically Accelerates Amide Bond Hydrolysis</span>, <span>Matching Enzyme Activity</span></a>. <em>Angewandte Chemie International Edition</em> <strong>58</strong>, 9787–9790 (2019).</div>
</div>
<div id="ref-hockneyQuietHighresolutionComputer1974" class="csl-entry" role="listitem">
<div class="csl-left-margin">21. </div><div class="csl-right-inline">Hockney, R. W., Goel, S. P. &amp; Eastwood, J. W. <a href="https://doi.org/10.1016/0021-9991(74)90010-2">Quiet high-resolution computer models of a plasma</a>. <em>Journal of Computational Physics</em> <strong>14</strong>, 148–158 (1974).</div>
</div>
<div id="ref-boltzmannVorlesungenUeberGastheorie1896" class="csl-entry" role="listitem">
<div class="csl-left-margin">22. </div><div class="csl-right-inline">Boltzmann, L. <em><span>Vorlesungen über Gastheorie</span></em>. <em>J.A. Barth</em>. (J.A. Barth, Leipzig, 1896).</div>
</div>
<div id="ref-taylorMethodusIncrementorumDirecta1715" class="csl-entry" role="listitem">
<div class="csl-left-margin">23. </div><div class="csl-right-inline">Taylor, B. <em><a href="http://archive.org/details/UFIE003454_TO0324_PNI-2529_000000"><span>Methodus incrementorum directa &amp; inversa. Auctore Brook Taylor ...</span></a></em> <em>Londini : Typis Pearsonianis prostant apud Gul. Innys ad Insignia Principis in Coemeterio Paulino, 1715</em>. (Londini : Typis Pearsonianis prostant apud Gul. Innys ad Insignia Principis in Coemeterio Paulino, 1715, 1715).</div>
</div>
<div id="ref-morseDiatomicMoleculesAccording1929" class="csl-entry" role="listitem">
<div class="csl-left-margin">24. </div><div class="csl-right-inline">Morse, P. M. <a href="https://doi.org/10.1103/PhysRev.34.57">Diatomic <span>Molecules According</span> to the <span>Wave Mechanics</span>. <span>II</span>. <span>Vibrational Levels</span></a>. <em>Physical Review</em> <strong>34</strong>, 57–64 (1929).</div>
</div>
<div id="ref-fourierMemoirePropagationChaleur2013" class="csl-entry" role="listitem">
<div class="csl-left-margin">25. </div><div class="csl-right-inline"><a href="https://doi.org/10.1017/CBO9781139568159.009">Mémoire sur la propagation de la chaleur dans les corps solides</a>. in <em>Cambridge University Press</em>. (eds. Fourier, J. B. J. &amp; Darboux, J. G.) vol. 2 213–222 (Cambridge University Press, Cambridge, 2013 (1890)).</div>
</div>
<div id="ref-lennard-jonesCohesion1931" class="csl-entry" role="listitem">
<div class="csl-left-margin">26. </div><div class="csl-right-inline">Lennard-Jones, J. E. <a href="https://doi.org/10.1088/0959-5309/43/5/301">Cohesion</a>. <em>Proceedings of the Physical Society</em> <strong>43</strong>, 461 (1931).</div>
</div>
<div id="ref-coulombFirstMemoirElectricity1785" class="csl-entry" role="listitem">
<div class="csl-left-margin">27. </div><div class="csl-right-inline">Coulomb, A. <a href="http://www.ampere.cnrs.fr/i-corpuspic/tab/Sources/coulomb/Coulomb_El_1785.pdf">First memoir on electricity and magnetism</a>. <em>A Source Book in Physics</em> 408–413 (1785).</div>
</div>
<div id="ref-baylyWellbehavedElectrostaticPotential2002" class="csl-entry" role="listitem">
<div class="csl-left-margin">28. </div><div class="csl-right-inline">Bayly, C. I., Cieplak, P., Cornell, W. &amp; Kollman, P. A. A well-behaved electrostatic potential based method using charge restraints for deriving atomic charges: The <span>RESP</span> model. <em>American Chemical Society</em>. (2002) doi:<a href="https://doi.org/10.1021/j100142a004">10.1021/j100142a004</a>.</div>
</div>
<div id="ref-caseAmberTools2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">29. </div><div class="csl-right-inline">Case, D. A., Aktulga, H. M., Belfon, K., <em>et al.</em> <a href="https://doi.org/10.1021/acs.jcim.3c01153"><span>AmberTools</span></a>. <em>Journal of Chemical Information and Modeling</em> <strong>63</strong>, 6183–6191 (2023).</div>
</div>
<div id="ref-cornellSecondGenerationForce1995" class="csl-entry" role="listitem">
<div class="csl-left-margin">30. </div><div class="csl-right-inline">Cornell, W. D., Cieplak, P., Bayly, C. I., <em>et al.</em> <a href="https://doi.org/10.1021/ja00124a002">A <span>Second Generation Force Field</span> for the <span>Simulation</span> of <span>Proteins</span>, <span>Nucleic Acids</span>, and <span>Organic Molecules</span></a>. <em>Journal of the American Chemical Society</em> <strong>117</strong>, 5179–5197 (1995).</div>
</div>
<div id="ref-wangHowWellDoes2000" class="csl-entry" role="listitem">
<div class="csl-left-margin">31. </div><div class="csl-right-inline">Wang, J., Cieplak, P. &amp; Kollman, P. A. <a href="https://doi.org/10.1002/1096-987X(200009)21:12<1049::AID-JCC3>3.0.CO;2-F"><span>How well does a restrained electrostatic potential (RESP) model perform in calculating conformational energies of organic and biological molecules?</span></a> <em>Journal of Computational Chemistry</em> <strong>21</strong>, 1049–1074 (2000).</div>
</div>
<div id="ref-ramachandranStereochemistryPolypeptideChain1963" class="csl-entry" role="listitem">
<div class="csl-left-margin">32. </div><div class="csl-right-inline">Ramachandran, G. N., Ramakrishnan, C. &amp; Sasisekharan, V. <a href="https://doi.org/10.1016/S0022-2836(63)80023-6">Stereochemistry of polypeptide chain configurations</a>. <em>Journal of Molecular Biology</em> <strong>7</strong>, 95–99 (1963).</div>
</div>
<div id="ref-bestOptimizedMolecularDynamics2009" class="csl-entry" role="listitem">
<div class="csl-left-margin">33. </div><div class="csl-right-inline">Best, R. B. &amp; Hummer, G. <a href="https://doi.org/10.1021/jp901540t">Optimized <span>Molecular Dynamics Force Fields Applied</span> to the <span>Helix</span>−<span>Coil Transition</span> of <span>Polypeptides</span></a>. <em>The Journal of Physical Chemistry B</em> <strong>113</strong>, 9004–9015 (2009).</div>
</div>
<div id="ref-lindahlGROMACS2020Source2020" class="csl-entry" role="listitem">
<div class="csl-left-margin">34. </div><div class="csl-right-inline">Lindahl, Abraham, Hess &amp; van der Spoel. <span>GROMACS</span> 2020 <span>Source</span> code. <em>Zenodo</em>. (2020) doi:<a href="https://doi.org/10.5281/zenodo.3562495">10.5281/zenodo.3562495</a>.</div>
</div>
<div id="ref-alievMotionalTimescalePredictions2014" class="csl-entry" role="listitem">
<div class="csl-left-margin">35. </div><div class="csl-right-inline">Aliev, A. E., Kulke, M., Khaneja, H. S., <em>et al.</em> <a href="https://doi.org/10.1002/prot.24350">Motional timescale predictions by molecular dynamics simulations: <span>Case</span> study using proline and hydroxyproline sidechain dynamics: <span>Proline Force Field Parameters</span></a>. <em>Proteins: Structure, Function, and Bioinformatics</em> <strong>82</strong>, 195–215 (2014).</div>
</div>
<div id="ref-brooksCHARMMBiomolecularSimulation2009" class="csl-entry" role="listitem">
<div class="csl-left-margin">36. </div><div class="csl-right-inline">Brooks, B. R., Brooks, C. L., Mackerell, A. D., <em>et al.</em> <a href="https://doi.org/10.1002/jcc.21287"><span>CHARMM</span>: <span>The</span> biomolecular simulation program</a>. <em>Journal of Computational Chemistry</em> <strong>30</strong>, 1545–1614 (2009).</div>
</div>
<div id="ref-schulerImprovedGROMOS96Force2001" class="csl-entry" role="listitem">
<div class="csl-left-margin">37. </div><div class="csl-right-inline">Schuler, L. D., Daura, X. &amp; van Gunsteren, W. F. <a href="https://doi.org/10.1002/jcc.1078">An improved <span>GROMOS96</span> force field for aliphatic hydrocarbons in the condensed phase</a>. <em>Journal of Computational Chemistry</em> <strong>22</strong>, 1205–1218 (2001).</div>
</div>
<div id="ref-robertsonImprovedPeptideProtein2015" class="csl-entry" role="listitem">
<div class="csl-left-margin">38. </div><div class="csl-right-inline">Robertson, M. J., Tirado-Rives, J. &amp; Jorgensen, W. L. <a href="https://doi.org/10.1021/acs.jctc.5b00356">Improved <span>Peptide</span> and <span>Protein Torsional Energetics</span> with the <span>OPLS-AA Force Field</span></a>. <em>Journal of Chemical Theory and Computation</em> <strong>11</strong>, 3499–3509 (2015).</div>
</div>
<div id="ref-sorinExploringHelixCoilTransition2005" class="csl-entry" role="listitem">
<div class="csl-left-margin">39. </div><div class="csl-right-inline">Sorin, E. J. &amp; Pande, V. S. <a href="https://doi.org/10.1529/biophysj.104.051938">Exploring the <span>Helix-Coil Transition</span> via <span>All-Atom Equilibrium Ensemble Simulations</span></a>. <em>Biophysical Journal</em> <strong>88</strong>, 2472–2493 (2005).</div>
</div>
<div id="ref-abrahamGROMACSHighPerformance2015" class="csl-entry" role="listitem">
<div class="csl-left-margin">40. </div><div class="csl-right-inline">Abraham, M. J., Murtola, T., Schulz, R., <em>et al.</em> <a href="https://doi.org/10.1016/j.softx.2015.06.001"><span>GROMACS</span>: <span>High</span> performance molecular simulations through multi-level parallelism from laptops to supercomputers</a>. <em>SoftwareX</em> <strong>1–2</strong>, 19–25 (2015).</div>
</div>
<div id="ref-lorentzUeberAnwendungSatzes1881" class="csl-entry" role="listitem">
<div class="csl-left-margin">41. </div><div class="csl-right-inline">Lorentz, H. A. Ueber die <span>Anwendung</span> des <span>Satzes</span> vom <span>Virial</span> in der kinetischen <span>Theorie</span> der <span>Gase</span>. (1881) doi:<a href="https://doi.org/10.1002/andp.18812480110">10.1002/andp.18812480110</a>.</div>
</div>
<div id="ref-berthelotMelangeGaz1898" class="csl-entry" role="listitem">
<div class="csl-left-margin">42. </div><div class="csl-right-inline">Berthelot, D. Sur le mélange des gaz. <em>Compt. Rendus</em> <strong>126</strong>, 15 (1898).</div>
</div>
<div id="ref-bermanProteinDataBank2014" class="csl-entry" role="listitem">
<div class="csl-left-margin">43. </div><div class="csl-right-inline">Berman, H. M., Kleywegt, G. J., Nakamura, H. &amp; Markley, J. L. <a href="https://doi.org/10.1007/s10822-014-9770-y">The <span>Protein Data Bank</span> archive as an open data resource</a>. <em>Journal of Computer-Aided Molecular Design</em> <strong>28</strong>, 1009–1014 (2014).</div>
</div>
<div id="ref-FileFormatsGROMACS" class="csl-entry" role="listitem">
<div class="csl-left-margin">44. </div><div class="csl-right-inline"><a href="https://manual.gromacs.org/current/reference-manual/topologies/topology-file-formats.html#topology-file">File formats - <span>GROMACS</span> 2025.3 documentation</a>.</div>
</div>
<div id="ref-ryckaertNumericalIntegrationCartesian1977" class="csl-entry" role="listitem">
<div class="csl-left-margin">45. </div><div class="csl-right-inline">Ryckaert, J.-P., Ciccotti, G. &amp; Berendsen, H. J. C. <a href="https://doi.org/10.1016/0021-9991(77)90098-5">Numerical integration of the cartesian equations of motion of a system with constraints: Molecular dynamics of <span><em>n</em></span>-alkanes</a>. <em>Journal of Computational Physics</em> <strong>23</strong>, 327–341 (1977).</div>
</div>
<div id="ref-miyamotoSettleAnalyticalVersion1992" class="csl-entry" role="listitem">
<div class="csl-left-margin">46. </div><div class="csl-right-inline">Miyamoto, S. &amp; Kollman, P. A. <a href="https://doi.org/10.1002/jcc.540130805">Settle: <span>An</span> analytical version of the <span>SHAKE</span> and <span>RATTLE</span> algorithm for rigid water models</a>. <em>Journal of Computational Chemistry</em> <strong>13</strong>, 952–962 (1992).</div>
</div>
<div id="ref-hessLINCSLinearConstraint1997" class="csl-entry" role="listitem">
<div class="csl-left-margin">47. </div><div class="csl-right-inline">Hess, B., Bekker, H., Berendsen, H. J. C. &amp; Fraaije, J. G. E. M. <a href="https://doi.org/10.1002/(SICI)1096-987X(199709)18:12<1463::AID-JCC4>3.0.CO;2-H"><span>LINCS</span>: <span>A</span> linear constraint solver for molecular simulations</a>. <em>Journal of Computational Chemistry</em> <strong>18</strong>, 1463–1472 (1997).</div>
</div>
<div id="ref-jumperHighlyAccurateProtein2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">48. </div><div class="csl-right-inline">Jumper, J., Evans, R., Pritzel, A., <em>et al.</em> <a href="https://doi.org/10.1038/s41586-021-03819-2">Highly accurate protein structure prediction with <span>AlphaFold</span></a>. <em>Nature</em> <strong>596</strong>, 583–589 (2021).</div>
</div>
<div id="ref-boltzmannEinigeAllgemeineSatze1871" class="csl-entry" role="listitem">
<div class="csl-left-margin">49. </div><div class="csl-right-inline">Boltzmann, L. E. <em>Einige Allgemeine Sätze Über <span>Wärmegleichgewicht</span></em>. <em>K. Akad. der Wissensch.</em> (K. Akad. der Wissensch., 1871).</div>
</div>
<div id="ref-berendsenMolecularDynamicsCoupling1984" class="csl-entry" role="listitem">
<div class="csl-left-margin">50. </div><div class="csl-right-inline">Berendsen, H. J. C., Postma, J. P. M., van Gunsteren, W. F., DiNola, A. &amp; Haak, J. R. <a href="https://doi.org/10.1063/1.448118">Molecular dynamics with coupling to an external bath</a>. <em>The Journal of Chemical Physics</em> <strong>81</strong>, 3684–3690 (1984).</div>
</div>
<div id="ref-bussiCanonicalSamplingVelocity2007" class="csl-entry" role="listitem">
<div class="csl-left-margin">51. </div><div class="csl-right-inline">Bussi, G., Donadio, D. &amp; Parrinello, M. <a href="https://doi.org/10.1063/1.2408420">Canonical sampling through velocity rescaling</a>. <em>The Journal of Chemical Physics</em> <strong>126</strong>, 014101 (2007).</div>
</div>
<div id="ref-parrinelloPolymorphicTransitionsSingle1981" class="csl-entry" role="listitem">
<div class="csl-left-margin">52. </div><div class="csl-right-inline">Parrinello, M. &amp; Rahman, A. <a href="https://doi.org/10.1063/1.328693">Polymorphic transitions in single crystals: <span>A</span> new molecular dynamics method</a>. <em>Journal of Applied Physics</em> <strong>52</strong>, 7182–7190 (1981).</div>
</div>
<div id="ref-dardenParticleMeshEwald1993" class="csl-entry" role="listitem">
<div class="csl-left-margin">53. </div><div class="csl-right-inline">Darden, T., York, D. &amp; Pedersen, L. <a href="https://www.academia.edu/download/93518571/article.pdf">Particle mesh <span>Ewald</span>: <span>An N</span> log (<span>N</span>) method for <span>Ewald</span> sums in large systems</a>. <em>Journal of chemical physics</em> <strong>98</strong>, 10089–10089 (1993).</div>
</div>
<div id="ref-ewaldBerechnungOptischerUnd1921" class="csl-entry" role="listitem">
<div class="csl-left-margin">54. </div><div class="csl-right-inline">Ewald, P. P. <a href="https://doi.org/10.1002/andp.19213690304">Die <span>Berechnung</span> optischer und elektrostatischer <span>Gitterpotentiale</span></a>. <em>Annalen der Physik</em> <strong>369</strong>, 253–287 (1921).</div>
</div>
<div id="ref-verletComputerExperimentsClassical1967" class="csl-entry" role="listitem">
<div class="csl-left-margin">55. </div><div class="csl-right-inline">Verlet, L. <a href="https://doi.org/10.1103/PhysRev.159.98">Computer "<span>Experiments</span>" on <span>Classical Fluids</span>. <span>I</span>. <span>Thermodynamical Properties</span> of <span>Lennard-Jones Molecules</span></a>. <em>Physical Review</em> <strong>159</strong>, 98–103 (1967).</div>
</div>
<div id="ref-pallFlexibleAlgorithmCalculating2013" class="csl-entry" role="listitem">
<div class="csl-left-margin">56. </div><div class="csl-right-inline">Páll, S. &amp; Hess, B. <a href="https://doi.org/10.1016/j.cpc.2013.06.003">A flexible algorithm for calculating pair interactions on <span>SIMD</span> architectures</a>. <em>Computer Physics Communications</em> <strong>184</strong>, 2641–2650 (2013).</div>
</div>
<div id="ref-GROMACS20253Documentation" class="csl-entry" role="listitem">
<div class="csl-left-margin">57. </div><div class="csl-right-inline"><a href="https://manual.gromacs.org/current/index.html"><span>GROMACS</span> 2025.3 documentation</a>.</div>
</div>
<div id="ref-grossfieldWHAMWeightedHistogram" class="csl-entry" role="listitem">
<div class="csl-left-margin">58. </div><div class="csl-right-inline">Grossfield, A. <a href="http://membrane.urmc.rochester.edu/?page_id=126"><span>WHAM</span>: The weighted histogram analysis method</a>.</div>
</div>
<div id="ref-rombachHighResolutionImageSynthesis2022" class="csl-entry" role="listitem">
<div class="csl-left-margin">59. </div><div class="csl-right-inline">Rombach, R., Blattmann, A., Lorenz, D., Esser, P. &amp; Ommer, B. High-<span>Resolution Image Synthesis</span> with <span>Latent Diffusion Models</span>. <em>arXiv</em>. (2022) doi:<a href="https://doi.org/10.48550/arXiv.2112.10752">10.48550/arXiv.2112.10752</a>.</div>
</div>
<div id="ref-InvokeaiInvokeAI2025" class="csl-entry" role="listitem">
<div class="csl-left-margin">60. </div><div class="csl-right-inline"><a href="https://github.com/invoke-ai/InvokeAI">Invoke-ai/<span>InvokeAI</span></a>. <em>InvokeAI</em>. (2025).</div>
</div>
<div id="ref-kumarWeightedHistogramAnalysis1992" class="csl-entry" role="listitem">
<div class="csl-left-margin">61. </div><div class="csl-right-inline">Kumar, S., Rosenberg, J. M., Bouzida, D., Swendsen, R. H. &amp; Kollman, P. A. <a href="https://doi.org/10.1002/jcc.540130812"><span>THE</span> weighted histogram analysis method for free-energy calculations on biomolecules. <span>I</span>. <span>The</span> method</a>. <em>Journal of Computational Chemistry</em> <strong>13</strong>, 1011–1021 (1992).</div>
</div>
<div id="ref-souailleExtensionWeightedHistogram2001" class="csl-entry" role="listitem">
<div class="csl-left-margin">62. </div><div class="csl-right-inline">Souaille, M. &amp; Roux, B. <a href="https://doi.org/10.1016/S0010-4655(00)00215-0">Extension to the weighted histogram analysis method: Combining umbrella sampling with free energy calculations</a>. <em>Computer Physics Communications</em> <strong>135</strong>, 40–57 (2001).</div>
</div>
<div id="ref-hubG_whamAFreeWeighted2010" class="csl-entry" role="listitem">
<div class="csl-left-margin">63. </div><div class="csl-right-inline">Hub, J. S., de Groot, B. L. &amp; van der Spoel, D. <a href="https://doi.org/10.1021/ct100494z">G_wham—<span>A Free Weighted Histogram Analysis Implementation Including Robust Error</span> and <span>Autocorrelation Estimates</span></a>. <em>Journal of Chemical Theory and Computation</em> <strong>6</strong>, 3713–3720 (2010).</div>
</div>
<div id="ref-schrodingerUndulatoryTheoryMechanics1926" class="csl-entry" role="listitem">
<div class="csl-left-margin">64. </div><div class="csl-right-inline">Schrödinger, E. <a href="https://doi.org/10.1103/PhysRev.28.1049">An <span>Undulatory Theory</span> of the <span>Mechanics</span> of <span>Atoms</span> and <span>Molecules</span></a>. <em>Physical Review</em> <strong>28</strong>, 1049–1070 (1926).</div>
</div>
<div id="ref-debroglieRecherchesTheorieQuanta1924" class="csl-entry" role="listitem">
<div class="csl-left-margin">65. </div><div class="csl-right-inline">De Broglie, L. <a href="https://theses.hal.science/tel-00006807/document">Recherches sur la théorie des quanta</a>. <em>Migration-université en cours d’affectation</em>. (Migration-université en cours d’affectation, 1924).</div>
</div>
<div id="ref-bornZurQuantenmechanikStossvorgaenge1926" class="csl-entry" role="listitem">
<div class="csl-left-margin">66. </div><div class="csl-right-inline">Born, M. <a href="https://doi.org/10.1007/BF01397477"><span>Zur Quantenmechanik der Stoßvorgänge</span></a>. <em>Zeitschrift für Physik</em> <strong>37</strong>, 863–867 (1926).</div>
</div>
<div id="ref-bornZurQuantentheorieMolekeln1927" class="csl-entry" role="listitem">
<div class="csl-left-margin">67. </div><div class="csl-right-inline">Born, M. &amp; Oppenheimer, R. <a href="https://doi.org/10.1002/andp.19273892002">Zur <span>Quantentheorie</span> der <span>Molekeln</span></a>. <em>Annalen der Physik</em> <strong>389</strong>, 457–484 (1927).</div>
</div>
<div id="ref-hartreeCalculationAtomicStructures1957" class="csl-entry" role="listitem">
<div class="csl-left-margin">68. </div><div class="csl-right-inline">Hartree, D. R. <em>The Calculation of Atomic Structures</em>. <em>J. Wiley</em>. (J. Wiley, 1957).</div>
</div>
<div id="ref-slaterTheoryComplexSpectra1929" class="csl-entry" role="listitem">
<div class="csl-left-margin">69. </div><div class="csl-right-inline">Slater, J. C. <a href="https://doi.org/10.1103/PhysRev.34.1293">The <span>Theory</span> of <span>Complex Spectra</span></a>. <em>Physical Review</em> <strong>34</strong>, 1293–1322 (1929).</div>
</div>
<div id="ref-fockNaeherungsmethodeZurLoesung1930" class="csl-entry" role="listitem">
<div class="csl-left-margin">70. </div><div class="csl-right-inline">Fock, V. <a href="https://doi.org/10.1007/BF01340294"><span>Näherungsmethode zur Lösung des quantenmechanischen Mehrkörperproblems</span></a>. <em>Zeitschrift für Physik</em> <strong>61</strong>, 126–148 (1930).</div>
</div>
<div id="ref-hohenbergInhomogeneousElectronGas1964" class="csl-entry" role="listitem">
<div class="csl-left-margin">71. </div><div class="csl-right-inline">Hohenberg, P. &amp; Kohn, W. <a href="https://doi.org/10.1103/PhysRev.136.B864">Inhomogeneous <span>Electron Gas</span></a>. <em>Physical Review</em> <strong>136</strong>, B864–B871 (1964).</div>
</div>
<div id="ref-kohnSelfConsistentEquationsIncluding1965" class="csl-entry" role="listitem">
<div class="csl-left-margin">72. </div><div class="csl-right-inline">Kohn, W. &amp; Sham, L. J. <a href="https://doi.org/10.1103/PhysRev.140.A1133">Self-<span>Consistent Equations Including Exchange</span> and <span>Correlation Effects</span></a>. <em>Physical Review</em> <strong>140</strong>, A1133–A1138 (1965).</div>
</div>
<div id="ref-leeDevelopmentColleSalvettiCorrelationenergy1988" class="csl-entry" role="listitem">
<div class="csl-left-margin">73. </div><div class="csl-right-inline">Lee, C., Yang, W. &amp; Parr, R. G. <a href="https://doi.org/10.1103/PhysRevB.37.785">Development of the <span>Colle-Salvetti</span> correlation-energy formula into a functional of the electron density</a>. <em>Physical Review B</em> <strong>37</strong>, 785–789 (1988).</div>
</div>
<div id="ref-beckeDensityfunctionalThermochemistryIII1993" class="csl-entry" role="listitem">
<div class="csl-left-margin">74. </div><div class="csl-right-inline">Becke, A. D. <a href="https://doi.org/10.1063/1.464913">Density‐functional thermochemistry. <span>III</span>. <span>The</span> role of exact exchange</a>. <em>The Journal of Chemical Physics</em> <strong>98</strong>, 5648–5652 (1993).</div>
</div>
<div id="ref-perdewRationaleMixingExact1996" class="csl-entry" role="listitem">
<div class="csl-left-margin">75. </div><div class="csl-right-inline">Perdew, J. P., Ernzerhof, M. &amp; Burke, K. <a href="https://doi.org/10.1063/1.472933">Rationale for mixing exact exchange with density functional approximations</a>. <em>The Journal of Chemical Physics</em> <strong>105</strong>, 9982–9985 (1996).</div>
</div>
<div id="ref-adamoReliableDensityFunctional1999" class="csl-entry" role="listitem">
<div class="csl-left-margin">76. </div><div class="csl-right-inline">Adamo, C. &amp; Barone, V. <a href="https://doi.org/10.1063/1.478522">Toward reliable density functional methods without adjustable parameters: <span>The PBE0</span> model</a>. <em>The Journal of Chemical Physics</em> <strong>110</strong>, 6158–6170 (1999).</div>
</div>
<div id="ref-head-gordonMP2EnergyEvaluation1988" class="csl-entry" role="listitem">
<div class="csl-left-margin">77. </div><div class="csl-right-inline">Head-Gordon, M., Pople, J. A. &amp; Frisch, M. J. <a href="https://doi.org/10.1016/0009-2614(88)85250-3"><span>MP2</span> energy evaluation by direct methods</a>. <em>Chemical Physics Letters</em> <strong>153</strong>, 503–506 (1988).</div>
</div>
<div id="ref-mollerNoteApproximationTreatment1934" class="csl-entry" role="listitem">
<div class="csl-left-margin">78. </div><div class="csl-right-inline">Møller, Chr. &amp; Plesset, M. S. <a href="https://doi.org/10.1103/PhysRev.46.618">Note on an <span>Approximation Treatment</span> for <span>Many-Electron Systems</span></a>. <em>Physical Review</em> <strong>46</strong>, 618–622 (1934).</div>
</div>
<div id="ref-lainoEfficientRealSpace2005" class="csl-entry" role="listitem">
<div class="csl-left-margin">79. </div><div class="csl-right-inline">Laino, T., Mohamed, F., Laio, A. &amp; Parrinello, M. <a href="https://doi.org/10.1021/ct050123f">An <span>Efficient Real Space Multigrid QM</span>/<span>MM Electrostatic Coupling</span></a>. <em>Journal of Chemical Theory and Computation</em> <strong>1</strong>, 1176–1184 (2005).</div>
</div>
<div id="ref-morozovWebinarMultiscaleQM2020" class="csl-entry" role="listitem">
<div class="csl-left-margin">80. </div><div class="csl-right-inline">Morozov, D. <a href="https://bioexcel.eu/webinar-multiscale-qm-mm-simulations-exploring-chemical-reactions-using-novel-gromacs-cp2k-interface-2020-12-08/">Webinar: <span>Multiscale QM</span>/<span>MM</span> simulations: Exploring chemical reactions using novel <span>GROMACS</span>/<span>CP2K</span> interface (2020-12-08)</a>. (2020).</div>
</div>
<div id="ref-HybridQuantumClassicalSimulations" class="csl-entry" role="listitem">
<div class="csl-left-margin">81. </div><div class="csl-right-inline"><a href="https://manual.gromacs.org/current/reference-manual/special/qmmm.html">Hybrid <span>Quantum-Classical</span> simulations (<span>QM</span>/<span>MM</span>) with <span>CP2K</span> interface - <span>GROMACS</span> 2025.3 documentation</a>.</div>
</div>
<div id="ref-kuhneCP2KElectronicStructure2020" class="csl-entry" role="listitem">
<div class="csl-left-margin">82. </div><div class="csl-right-inline">Kühne, T. D., Iannuzzi, M., Del Ben, M., <em>et al.</em> <a href="https://doi.org/10.1063/5.0007045"><span>CP2K</span>: <span>An</span> electronic structure and molecular dynamics software package - <span>Quickstep</span>: <span>Efficient</span> and accurate electronic structure calculations</a>. <em>The Journal of Chemical Physics</em> <strong>152</strong>, 194103 (2020).</div>
</div>
<div id="ref-bortzNewAlgorithmMonte1975" class="csl-entry" role="listitem">
<div class="csl-left-margin">83. </div><div class="csl-right-inline">Bortz, A. B., Kalos, M. H. &amp; Lebowitz, J. L. <a href="https://doi.org/10.1016/0021-9991(75)90060-1">A new algorithm for <span>Monte Carlo</span> simulation of <span>Ising</span> spin systems</a>. <em>Journal of Computational Physics</em> <strong>17</strong>, 10–18 (1975).</div>
</div>
<div id="ref-gillespieGeneralMethodNumerically1976" class="csl-entry" role="listitem">
<div class="csl-left-margin">84. </div><div class="csl-right-inline">Gillespie, D. T. <a href="https://doi.org/10.1016/0021-9991(76)90041-3">A general method for numerically simulating the stochastic time evolution of coupled chemical reactions</a>. <em>Journal of Computational Physics</em> <strong>22</strong>, 403–434 (1976).</div>
</div>
<div id="ref-henkelmanLongTimeScale2001" class="csl-entry" role="listitem">
<div class="csl-left-margin">85. </div><div class="csl-right-inline">Henkelman, G. &amp; Jónsson, H. <a href="https://doi.org/10.1063/1.1415500">Long time scale kinetic <span>Monte Carlo</span> simulations without lattice approximation and predefined event table</a>. <em>The Journal of Chemical Physics</em> <strong>115</strong>, 9657–9666 (2001).</div>
</div>
<div id="ref-arrheniusUberReaktionsgeschwindigkeitBei1889" class="csl-entry" role="listitem">
<div class="csl-left-margin">86. </div><div class="csl-right-inline">Arrhenius, S. <a href="https://doi.org/10.1515/zpch-1889-0416">Über die <span>Reaktionsgeschwindigkeit</span> bei der <span>Inversion</span> von <span>Rohrzucker</span> durch <span>Säuren</span></a>. <em>Zeitschrift für Physikalische Chemie</em> <strong>4U</strong>, 226–248 (1889).</div>
</div>
<div id="ref-eyringActivatedComplexChemical1935" class="csl-entry" role="listitem">
<div class="csl-left-margin">87. </div><div class="csl-right-inline">Eyring, H. <a href="https://doi.org/10.1063/1.1749604">The <span>Activated Complex</span> in <span>Chemical Reactions</span></a>. <em>The Journal of Chemical Physics</em> <strong>3</strong>, 107–115 (1935).</div>
</div>
<div id="ref-liangQuantumChemistryStudy2022" class="csl-entry" role="listitem">
<div class="csl-left-margin">88. </div><div class="csl-right-inline">Liang, Y., Zheng, W., Xie, H., Zha, X. &amp; Wang, T. <a href="https://doi.org/10.1016/j.jics.2022.100527">A quantum chemistry study on <span>C</span>–<span>H</span> homolytic bond dissociation enthalpies of five-membered and six-membered heterocyclic compounds</a>. <em>Journal of the Indian Chemical Society</em> <strong>99</strong>, 100527 (2022).</div>
</div>
<div id="ref-kosarBenchmarkDFTStudies2019" class="csl-entry" role="listitem">
<div class="csl-left-margin">89. </div><div class="csl-right-inline">Kosar, N., Ayub, K., Gilani, M. A. &amp; Mahmood, T. <a href="https://doi.org/10.1007/s00894-019-3930-x">Benchmark <span>DFT</span> studies on <span>C</span>–<span>CN</span> homolytic cleavage and screening the substitution effect on bond dissociation energy</a>. <em>Journal of Molecular Modeling</em> <strong>25</strong>, 47 (2019).</div>
</div>
<div id="ref-johnPredictionOrganicHomolytic2020" class="csl-entry" role="listitem">
<div class="csl-left-margin">90. </div><div class="csl-right-inline">John, P. C. S., Guan, Y., Kim, Y., Kim, S. &amp; Paton, R. S. <a href="https://doi.org/10.1038/s41467-020-16201-z">Prediction of organic homolytic bond dissociation enthalpies at near chemical accuracy with sub-second computational cost</a>. <em>Nature Communications</em> <strong>11</strong>, 2328 (2020).</div>
</div>
<div id="ref-vandevondeleGaussianBasisSets2007" class="csl-entry" role="listitem">
<div class="csl-left-margin">91. </div><div class="csl-right-inline">VandeVondele, J. &amp; Hutter, J. <a href="https://doi.org/10.1063/1.2770708">Gaussian basis sets for accurate calculations on molecular systems in gas and condensed phases</a>. <em>The Journal of Chemical Physics</em> <strong>127</strong>, 114105 (2007).</div>
</div>
<div id="ref-perdewGeneralizedGradientApproximation1996" class="csl-entry" role="listitem">
<div class="csl-left-margin">92. </div><div class="csl-right-inline">Perdew, J. P., Burke, K. &amp; Ernzerhof, M. <a href="https://doi.org/10.1103/PhysRevLett.77.3865">Generalized <span>Gradient Approximation Made Simple</span></a>. <em>Physical Review Letters</em> <strong>77</strong>, 3865–3868 (1996).</div>
</div>
<div id="ref-obarska-kosinskaColBuilderServerBuild2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">93. </div><div class="csl-right-inline">Obarska-Kosinska, A., Rennekamp, B., Ünal, A. &amp; Gräter, F. <span>ColBuilder</span>: A server to build collagen fibril models. <em>Biophysical journal</em> (2021) doi:<a href="https://doi.org/10.1016/j.bpj.2021.07.009">10.1016/j.bpj.2021.07.009</a>.</div>
</div>
<div id="ref-PyMOL" class="csl-entry" role="listitem">
<div class="csl-left-margin">94. </div><div class="csl-right-inline">Schrödinger, LLC. The <span>PyMOL</span> molecular graphics system, version 1.8. (2015).</div>
</div>
<div id="ref-jetteArchitectureSlurmWorkload2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">95. </div><div class="csl-right-inline">Jette, M. A. &amp; Wickberg, T. Architecture of&nbsp;the&nbsp;<span>Slurm Workload Manager</span>. in <em>Springer Nature Switzerland</em>. (eds. Klusáček, D., Corbalán, J. &amp; Rodrigo, G. P.) 3–23 (Springer Nature Switzerland, Cham, 2023). doi:<a href="https://doi.org/10.1007/978-3-031-43943-8_1">10.1007/978-3-031-43943-8_1</a>.</div>
</div>
<div id="ref-burgiStereochemistryReactionPaths1974" class="csl-entry" role="listitem">
<div class="csl-left-margin">96. </div><div class="csl-right-inline">Bürgi, H. B., Dunitz, J. D., Lehn, J. M. &amp; Wipff, G. <a href="https://doi.org/10.1016/S0040-4020(01)90678-7">Stereochemistry of reaction paths at carbonyl centres</a>. <em>Tetrahedron</em> <strong>30</strong>, 1563–1572 (1974).</div>
</div>
<div id="ref-lodgeStericEffectsWell1987" class="csl-entry" role="listitem">
<div class="csl-left-margin">97. </div><div class="csl-right-inline">Lodge, E. P. &amp; Heathcock, C. H. Steric effects, as well as σ*-orbital energies, are important in diastereoface differentiation in additions to chiral aldehydes. <em>Steric effects, as well as σ*-orbital energies, are important in diastereoface differentiation in additions to chiral aldehydes</em> <strong>109</strong>, 3353–3361 (1987).</div>
</div>
<div id="ref-flippinACYCLICSTEREOSELECTIONXVI1983" class="csl-entry" role="listitem">
<div class="csl-left-margin">98. </div><div class="csl-right-inline">Flippin, L. A. &amp; Heathcock, C. H. <span>ACYCLIC STEREOSELECTION</span>. <span>XVI</span>: <span>HIGH DIASTEREOFACIAL SELECTIVITY IN LEWIS ACID MEDIATED ADDITIONS OF ENOLSILANES TO CHIRAL ALDEHYDES</span>. <em>ACYCLIC STEREOSELECTION. XVI: HIGH DIASTEREOFACIAL SELECTIVITY IN LEWIS ACID MEDIATED ADDITIONS OF ENOLSILANES TO CHIRAL ALDEHYDES</em> (1983).</div>
</div>
<div id="ref-kimmdy2025" class="csl-entry" role="listitem">
<div class="csl-left-margin">99. </div><div class="csl-right-inline">Buhr*, J., Hartmann*, E., Riedmiller*, K., <em>et al.</em> <span>KIMMDY</span>: A biomolecular reaction emulator. <em>bioRxiv</em>. (2025) doi:<a href="https://doi.org/10.1101/2025.07.02.662624">10.1101/2025.07.02.662624</a> *These authors contributed equally to this work.</div>
</div>
<div id="ref-rennekampHybridKineticMonte2020" class="csl-entry" role="listitem">
<div class="csl-left-margin">100. </div><div class="csl-right-inline">Rennekamp, B., Kutzki, F., Obarska-Kosinska, A., Zapp, C. &amp; Gräter, F. <a href="https://doi.org/10.1021/acs.jctc.9b00786">Hybrid <span>Kinetic Monte Carlo</span>/<span>Molecular Dynamics Simulations</span> of <span>Bond Scissions</span> in <span>Proteins</span></a>. <em>Journal of Chemical Theory and Computation</em> <strong>16</strong>, 553–563 (2020).</div>
</div>
<div id="ref-vanduinReaxFFReactiveForce2001" class="csl-entry" role="listitem">
<div class="csl-left-margin">101. </div><div class="csl-right-inline">van Duin, A. C. T., Dasgupta, S., Lorant, F. &amp; Goddard, W. A. <a href="https://doi.org/10.1021/jp004368u"><span>ReaxFF</span>:  <span>A Reactive Force Field</span> for <span>Hydrocarbons</span></a>. <em>The Journal of Physical Chemistry A</em> <strong>105</strong>, 9396–9409 (2001).</div>
</div>
<div id="ref-senftleReaxFFReactiveForcefield2016" class="csl-entry" role="listitem">
<div class="csl-left-margin">102. </div><div class="csl-right-inline">Senftle, T. P., Hong, S., Islam, M. M., <em>et al.</em> <a href="https://doi.org/10.1038/npjcompumats.2015.11">The <span>ReaxFF</span> reactive force-field: Development, applications and future directions</a>. <em>npj Computational Materials</em> <strong>2</strong>, 1–14 (2016).</div>
</div>
<div id="ref-schuttSchNetDeepLearning2018" class="csl-entry" role="listitem">
<div class="csl-left-margin">103. </div><div class="csl-right-inline">Schütt, K. T., Sauceda, H. E., Kindermans, P.-J., Tkatchenko, A. &amp; Müller, K.-R. <a href="https://doi.org/10.1063/1.5019779"><span>SchNet</span> – <span>A</span> deep learning architecture for molecules and materials</a>. <em>The Journal of Chemical Physics</em> <strong>148</strong>, 241722 (2018).</div>
</div>
<div id="ref-behlerGeneralizedNeuralNetworkRepresentation2007" class="csl-entry" role="listitem">
<div class="csl-left-margin">104. </div><div class="csl-right-inline">Behler, J. &amp; Parrinello, M. <a href="https://doi.org/10.1103/PhysRevLett.98.146401">Generalized <span>Neural-Network Representation</span> of <span>High-Dimensional Potential-Energy Surfaces</span></a>. <em>Physical Review Letters</em> <strong>98</strong>, 146401 (2007).</div>
</div>
<div id="ref-smithANI1ExtensibleNeural2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">105. </div><div class="csl-right-inline">Smith, J., Isayev, O. &amp; E. Roitberg, A. <a href="https://doi.org/10.1039/C6SC05720A"><span>ANI-1</span>: An extensible neural network potential with <span>DFT</span> accuracy at force field computational cost</a>. <em>Chemical Science</em> <strong>8</strong>, 3192–3203 (2017).</div>
</div>
<div id="ref-NEURIPS2022_4a36c3c5" class="csl-entry" role="listitem">
<div class="csl-left-margin">106. </div><div class="csl-right-inline">Batatia, I., Kovacs, D. P., Simm, G., Ortner, C. &amp; Csanyi, G. <a href="https://proceedings.neurips.cc/paper_files/paper/2022/file/4a36c3c51af11ed9f34615b81edb5bbc-Paper-Conference.pdf"><span>MACE</span>: <span>Higher</span> order equivariant message passing neural networks for fast and accurate force fields</a>. in <em>Curran Associates, Inc.</em> (eds. Koyejo, S., Mohamed, S., Agarwal, A., et al.) vol. 35 11423–11436 (Curran Associates, Inc., 2022).</div>
</div>
<div id="ref-musaelianLearningLocalEquivariant2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">107. </div><div class="csl-right-inline">Musaelian, A., Batzner, S., Johansson, A., <em>et al.</em> <a href="https://doi.org/10.1038/s41467-023-36329-y">Learning local equivariant representations for large-scale atomistic dynamics</a>. <em>Nature Communications</em> <strong>14</strong>, 579 (2023).</div>
</div>
<div id="ref-mazitovPETMADUniversalInteratomic2025" class="csl-entry" role="listitem">
<div class="csl-left-margin">108. </div><div class="csl-right-inline">Mazitov, A., Bigi, F., Kellner, M., <em>et al.</em> <span>PET-MAD</span>, a universal interatomic potential for advanced materials modeling. <em>arXiv</em>. (2025) doi:<a href="https://doi.org/10.48550/arXiv.2503.14118">10.48550/arXiv.2503.14118</a>.</div>
</div>
<div id="ref-Procida_Diataxis_documentation_framework" class="csl-entry" role="listitem">
<div class="csl-left-margin">109. </div><div class="csl-right-inline">Procida, D. <a href="https://diataxis.fr/">Diátaxis documentation framework</a>.</div>
</div>
<div id="ref-berendsenGROMACSMessagepassingParallel1995" class="csl-entry" role="listitem">
<div class="csl-left-margin">110. </div><div class="csl-right-inline">Berendsen, H. J. C., van der Spoel, D. &amp; van Drunen, R. <a href="https://doi.org/10.1016/0010-4655(95)00042-E"><span>GROMACS</span>: <span>A</span> message-passing parallel molecular dynamics implementation</a>. <em>Computer Physics Communications</em> <strong>91</strong>, 43–56 (1995).</div>
</div>
<div id="ref-gowersMDAnalysisPythonPackage2016" class="csl-entry" role="listitem">
<div class="csl-left-margin">111. </div><div class="csl-right-inline">Gowers, R. J., Linke, M., Barnoud, J., <em>et al.</em> <span>MDAnalysis</span>: <span>A Python Package</span> for the <span>Rapid Analysis</span> of <span>Molecular Dynamics Simulations</span>. <em>scipy</em> (2016) doi:<a href="https://doi.org/10.25080/Majora-629e541a-00e">10.25080/Majora-629e541a-00e</a>.</div>
</div>
<div id="ref-michaud-agrawalMDAnalysisToolkitAnalysis2011" class="csl-entry" role="listitem">
<div class="csl-left-margin">112. </div><div class="csl-right-inline">Michaud-Agrawal, N., Denning, E. J., Woolf, T. B. &amp; Beckstein, O. <a href="https://doi.org/10.1002/jcc.21787"><span>MDAnalysis</span>: <span>A</span> toolkit for the analysis of molecular dynamics simulations</a>. <em>Journal of Computational Chemistry</em> <strong>32</strong>, 2319–2327 (2011).</div>
</div>
<div id="ref-tribelloPLUMED2New2014" class="csl-entry" role="listitem">
<div class="csl-left-margin">113. </div><div class="csl-right-inline">Tribello, G. A., Bonomi, M., Branduardi, D., Camilloni, C. &amp; Bussi, G. <a href="https://doi.org/10.1016/j.cpc.2013.09.018"><span>PLUMED</span> 2: <span>New</span> feathers for an old bird</a>. <em>Computer Physics Communications</em> <strong>185</strong>, 604–613 (2014).</div>
</div>
<div id="ref-voliotisStochasticSimulationBiomolecular2016" class="csl-entry" role="listitem">
<div class="csl-left-margin">114. </div><div class="csl-right-inline">Voliotis, M., Thomas, P., Grima, R. &amp; Bowsher, C. G. <a href="https://doi.org/10.1371/journal.pcbi.1004923">Stochastic <span>Simulation</span> of <span>Biomolecular Networks</span> in <span>Dynamic Environments</span></a>. <em>PLOS Computational Biology</em> <strong>12</strong>, e1004923 (2016).</div>
</div>
<div id="ref-seuteGrappaMachineLearned2025" class="csl-entry" role="listitem">
<div class="csl-left-margin">115. </div><div class="csl-right-inline">Seute, L., Hartmann, E., Stühmer, J. &amp; Gräter, F. <a href="https://doi.org/10.1039/D4SC05465B">Grappa – a machine learned molecular mechanics force field</a>. <em>Chemical Science</em> <strong>16</strong>, 2907–2930 (2025).</div>
</div>
<div id="ref-ConventionalCommits" class="csl-entry" role="listitem">
<div class="csl-left-margin">116. </div><div class="csl-right-inline"><a href="https://www.conventionalcommits.org/en/v1.0.0/">Conventional <span>Commits</span></a>.</div>
</div>
<div id="ref-bellModelsSpecificAdhesion1978" class="csl-entry" role="listitem">
<div class="csl-left-margin">117. </div><div class="csl-right-inline">Bell, G. I. <a href="https://doi.org/10.1126/science.347575">Models for the <span>Specific Adhesion</span> of <span>Cells</span> to <span>Cells</span></a>. <em>Science</em> <strong>200</strong>, 618–627 (1978).</div>
</div>
<div id="ref-evansDynamicStrengthMolecular1997" class="csl-entry" role="listitem">
<div class="csl-left-margin">118. </div><div class="csl-right-inline">Evans, E. &amp; Ritchie, K. <a href="https://doi.org/10.1016/S0006-3495(97)78802-7">Dynamic strength of molecular adhesion bonds</a>. <em>Biophysical Journal</em> <strong>72</strong>, 1541–1555 (1997).</div>
</div>
<div id="ref-leeInterpretationProteinStructures1971" class="csl-entry" role="listitem">
<div class="csl-left-margin">119. </div><div class="csl-right-inline">Lee, B. &amp; Richards, F. M. <a href="https://doi.org/10.1016/0022-2836(71)90324-x">The interpretation of protein structures: Estimation of static accessibility</a>. <em>Journal of Molecular Biology</em> <strong>55</strong>, 379–400 (1971).</div>
</div>
<div id="ref-mitternachtFreeSASAOpenSource2016" class="csl-entry" role="listitem">
<div class="csl-left-margin">120. </div><div class="csl-right-inline">Mitternacht, S. <span>FreeSASA</span>: <span>An</span> open source <span>C</span> library for solvent accessible surface area calculations. <em>F1000Research</em>. (2016) doi:<a href="https://doi.org/10.12688/f1000research.7931.1">10.12688/f1000research.7931.1</a>.</div>
</div>
<div id="ref-streetInterstitialPHHuman2001" class="csl-entry" role="listitem">
<div class="csl-left-margin">121. </div><div class="csl-right-inline">Street, D., Bangsbo, J. &amp; Juel, C. <a href="https://doi.org/10.1111/j.1469-7793.2001.00993.x">Interstitial <span class="nocase">pH</span> in human skeletal muscle during and after dynamic graded exercise</a>. <em>The Journal of Physiology</em> <strong>537</strong>, 993 (2001).</div>
</div>
<div id="ref-rawlingsMEROPSDatabaseProteolytic2018" class="csl-entry" role="listitem">
<div class="csl-left-margin">122. </div><div class="csl-right-inline">Rawlings, N. D., Barrett, A. J., Thomas, P. D., <em>et al.</em> <a href="https://doi.org/10.1093/nar/gkx1134">The <span>MEROPS</span> database of proteolytic enzymes, their substrates and inhibitors in 2017 and a comparison with peptidases in the <span>PANTHER</span> database</a>. <em>Nucleic Acids Research</em> <strong>46</strong>, D624–D632 (2018).</div>
</div>
<div id="ref-ahoScalableConstantPH2022" class="csl-entry" role="listitem">
<div class="csl-left-margin">123. </div><div class="csl-right-inline">Aho, N., Buslaev, P., Jansen, A., <em>et al.</em> <a href="https://doi.org/10.1021/acs.jctc.2c00516">Scalable <span class="nocase">Constant pH Molecular Dynamics</span> in <span>GROMACS</span></a>. <em>Journal of Chemical Theory and Computation</em> <strong>18</strong>, 6148–6160 (2022).</div>
</div>
<div id="ref-riedmillerSubstitutingDensityFunctional2024" class="csl-entry" role="listitem">
<div class="csl-left-margin">124. </div><div class="csl-right-inline">Riedmiller, K., Reiser, P., Bobkova, E., <em>et al.</em> <a href="https://doi.org/10.1039/D3SC03922F">Substituting density functional theory in reaction barrier calculations for hydrogen atom transfer in proteins</a>. <em>Chemical Science</em> <strong>15</strong>, 2518–2527 (2024).</div>
</div>
<div id="ref-treydeBondDissociationEnergies2022" class="csl-entry" role="listitem">
<div class="csl-left-margin">125. </div><div class="csl-right-inline">Treyde, W., Riedmiller, K. &amp; Gräter, F. <a href="https://doi.org/10.1039/D2RA04002F">Bond dissociation energies of <span>X</span>–<span>H</span> bonds in proteins</a>. <em>RSC Advances</em> <strong>12</strong>, 34557–34564 (2022).</div>
</div>
<div id="ref-rennekampCollagenBreaksWeak2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">126. </div><div class="csl-right-inline">Rennekamp, B., Karfusehr, C., Kurth, M., <em>et al.</em> <a href="https://doi.org/10.1038/s41467-023-37726-z">Collagen breaks at weak sacrificial bonds taming its mechanoradicals</a>. <em>Nature Communications</em> <strong>14</strong>, 2075 (2023).</div>
</div>
<div id="ref-souzaMartini3General2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">127. </div><div class="csl-right-inline">Souza, P. C. T., Alessandri, R., Barnoud, J., <em>et al.</em> <a href="https://doi.org/10.1038/s41592-021-01098-3">Martini 3: A general purpose force field for coarse-grained molecular dynamics</a>. <em>Nature Methods</em> <strong>18</strong>, 382–388 (2021).</div>
</div>
<div id="ref-hrennikoffSolutionProblemsElasticity2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">128. </div><div class="csl-right-inline">Hrennikoff, A. <a href="https://doi.org/10.1115/1.4009129">Solution of <span>Problems</span> of <span>Elasticity</span> by the <span>Framework Method</span></a>. <em>Journal of Applied Mechanics</em> <strong>8</strong>, A169–A175 (2021).</div>
</div>
<div id="ref-courantVariationalMethodsSolution1943" class="csl-entry" role="listitem">
<div class="csl-left-margin">129. </div><div class="csl-right-inline">Courant, R. <a href="https://doi.org/10.1090/S0002-9904-1943-07818-4">Variational methods for the solution of problems of equilibrium and vibrations</a>. <em>Bulletin of the American Mathematical Society</em> <strong>49</strong>, 1–23 (1943).</div>
</div>
<div id="ref-freutelFiniteElementModeling2014" class="csl-entry" role="listitem">
<div class="csl-left-margin">130. </div><div class="csl-right-inline">Freutel, M., Schmidt, H., Dürselen, L., Ignatius, A. &amp; Galbusera, F. <a href="https://doi.org/10.1016/j.clinbiomech.2014.01.006">Finite element modeling of soft tissues: <span>Material</span> models, tissue interaction and challenges</a>. <em>Clinical Biomechanics</em> <strong>29</strong>, 363–372 (2014).</div>
</div>
<div id="ref-chen27DichlorodihydrofluoresceinFluorescentProbe2010" class="csl-entry" role="listitem">
<div class="csl-left-margin">131. </div><div class="csl-right-inline">Chen, X., Zhong, Z., Xu, Z., Chen, L. &amp; Wang, Y. <a href="https://doi.org/10.3109/10715761003709802">2′,7′-<span>Dichlorodihydrofluorescein</span> as a fluorescent probe for reactive oxygen species measurement: <span>Forty</span> years of application and controversy</a>. <em>Free Radical Research</em> <strong>44</strong>, 587–604 (2010).</div>
</div>
<div id="ref-vmd" class="csl-entry" role="listitem">
<div class="csl-left-margin">132. </div><div class="csl-right-inline">Humphrey, W., Dalke, A. &amp; Schulten, K. <span>VMD</span> – <span>Visual Molecular Dynamics</span>. <em>Journal of Molecular Graphics</em> <strong>14</strong>, 33–38 (1996).</div>
</div>
<div id="ref-rcoreteamLanguageEnvironmentStatistical2018" class="csl-entry" role="listitem">
<div class="csl-left-margin">133. </div><div class="csl-right-inline">R Core Team. <em><a href="https://www.R-project.org/">R: <span>A Language</span> and <span>Environment</span> for <span>Statistical Computing</span></a></em>. <em>R Foundation for Statistical Computing</em>. (R Foundation for Statistical Computing, Vienna, Austria, 2018).</div>
</div>
<div id="ref-wickhamWelcomeTidyverse2019" class="csl-entry" role="listitem">
<div class="csl-left-margin">134. </div><div class="csl-right-inline">Wickham, H., Averick, M., Bryan, J., <em>et al.</em> <a href="https://doi.org/10.21105/joss.01686">Welcome to the <span class="nocase">tidyverse</span></a>. <em>Journal of Open Source Software</em> <strong>4</strong>, 1686 (2019).</div>
</div>
<div id="ref-teamPandasdevPandasPandas2025" class="csl-entry" role="listitem">
<div class="csl-left-margin">135. </div><div class="csl-right-inline">team, T. pandas development. Pandas-dev/pandas: <span>Pandas</span>. <em>Zenodo</em>. (2025) doi:<a href="https://doi.org/10.5281/zenodo.17229934">10.5281/zenodo.17229934</a>.</div>
</div>
<div id="ref-mckinney-proc-scipy-2010" class="csl-entry" role="listitem">
<div class="csl-left-margin">136. </div><div class="csl-right-inline">McKinney, W. Data <span>Structures</span> for <span>Statistical Computing</span> in <span>Python</span>. in (eds. van der Walt, S. &amp; Millman, J.) 56–61 (2010). doi:<a href="https://doi.org/10.25080/Majora-92bf1922-00a">10.25080/Majora-92bf1922-00a</a>.</div>
</div>
<div id="ref-harris2020array" class="csl-entry" role="listitem">
<div class="csl-left-margin">137. </div><div class="csl-right-inline">Harris, C. R., Millman, K. J., van der Walt, S. J., <em>et al.</em> <a href="https://doi.org/10.1038/s41586-020-2649-2">Array programming with <span>NumPy</span></a>. <em>Nature</em> <strong>585</strong>, 357–362 (2020).</div>
</div>
<div id="ref-Hunter:2007" class="csl-entry" role="listitem">
<div class="csl-left-margin">138. </div><div class="csl-right-inline">Hunter, J. D. <a href="https://doi.org/10.1109/MCSE.2007.55">Matplotlib: <span>A 2D</span> graphics environment</a>. <em>Computing in Science &amp; Engineering</em> <strong>9</strong>, 90–95 (2007).</div>
</div>
<div id="ref-plotnine" class="csl-entry" role="listitem">
<div class="csl-left-margin">139. </div><div class="csl-right-inline">team, T. plotnine development. Plotnine: <span>A</span> grammar of graphics for <span>Python</span>. doi:<a href="https://doi.org/10.5281/zenodo.1325308">10.5281/zenodo.1325308</a>.</div>
</div>
<div id="ref-allaireQuarto2022" class="csl-entry" role="listitem">
<div class="csl-left-margin">140. </div><div class="csl-right-inline">Allaire, J. J., Teague, C., Scheidegger, C., Xie, Y. &amp; Dervieux, C. Quarto. (2022) doi:<a href="https://doi.org/10.5281/zenodo.5960048">10.5281/zenodo.5960048</a>.</div>
</div>
<div id="ref-Sehnal_Mol_Viewer_modern_2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">141. </div><div class="csl-right-inline">Sehnal, D., Bittrich, S., Deshpande, M., <em>et al.</em> <a href="https://doi.org/10.1093/nar/gkab314">Mol* <span>Viewer</span>: Modern web app for <span>3D</span> visualization and analysis of large biomolecular structures</a>. <em>Nucleic Acids Research</em> <strong>49</strong>, (2021).</div>
</div>
</div>
<div style="page-break-after: always;"></div>
</section>


<div id="quarto-appendix" class="default"><section id="footnotes" class="footnotes footnotes-end-of-document" role="doc-endnotes"><h2 class="anchored quarto-appendix-heading">Footnotes</h2>

<ol>
<li id="fn1"><p>Yes, the naming scheme got a little out of hand.<a href="#fnref1" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn2"><p>though at this point it is a bit ironic to call it a spring <em>constant</em>.<a href="#fnref2" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn3"><p>technically this is also an input parameter.<a href="#fnref3" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn4"><p>Though in this case it is also just a different kind of model, a computational model, a simulation.<a href="#fnref4" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
</ol>
</section><section class="quarto-appendix-contents" id="quarto-citation"><h2 class="anchored quarto-appendix-heading">Citation</h2><div><div class="quarto-appendix-secondary-label">BibTeX citation:</div><pre class="sourceCode code-with-copy quarto-appendix-bibtex"><code class="sourceCode bibtex">@phdthesis{buhr2026,
  author = {Buhr, Jannik},
  publisher = {Heidelberg University},
  title = {Dissecting Cleavage Mechanisms\textbackslash{} in Tensed
    Collagen Fibrils\textbackslash{} using Reactive Molecular Dynamics
    Simulations},
  date = {2026-02-12},
  url = {https://jmbuhr.de/phd-thesis},
  langid = {en}
}
</code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre><div class="quarto-appendix-secondary-label">For attribution, please cite this work as:</div><div id="ref-buhr2026" class="csl-entry quarto-appendix-citeas" role="listitem">
<div class="">Buhr, J. <a href="https://jmbuhr.de/phd-thesis">Dissecting cleavage mechanisms\ in
tensed collagen fibrils\ using reactive molecular dynamics simulations
</a>. <em>Heidelberg University</em>. (Heidelberg University,
2026).</div>
</div></div></section></div></main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
  window.document.addEventListener("DOMContentLoaded", function (event) {
    // Ensure there is a toggle, if there isn't float one in the top right
    if (window.document.querySelector('.quarto-color-scheme-toggle') === null) {
      const a = window.document.createElement('a');
      a.classList.add('top-right');
      a.classList.add('quarto-color-scheme-toggle');
      a.href = "";
      a.onclick = function() { try { window.quartoToggleColorScheme(); } catch {} return false; };
      const i = window.document.createElement("i");
      i.classList.add('bi');
      a.appendChild(i);
      window.document.body.appendChild(a);
    }
    setColorSchemeToggle(hasAlternateSentinel())
    const icon = "";
    const anchorJS = new window.AnchorJS();
    anchorJS.options = {
      placement: 'right',
      icon: icon
    };
    anchorJS.add('.anchored');
    const isCodeAnnotation = (el) => {
      for (const clz of el.classList) {
        if (clz.startsWith('code-annotation-')) {                     
          return true;
        }
      }
      return false;
    }
    const onCopySuccess = function(e) {
      // button target
      const button = e.trigger;
      // don't keep focus
      button.blur();
      // flash "checked"
      button.classList.add('code-copy-button-checked');
      var currentTitle = button.getAttribute("title");
      button.setAttribute("title", "Copied!");
      let tooltip;
      if (window.bootstrap) {
        button.setAttribute("data-bs-toggle", "tooltip");
        button.setAttribute("data-bs-placement", "left");
        button.setAttribute("data-bs-title", "Copied!");
        tooltip = new bootstrap.Tooltip(button, 
          { trigger: "manual", 
            customClass: "code-copy-button-tooltip",
            offset: [0, -8]});
        tooltip.show();    
      }
      setTimeout(function() {
        if (tooltip) {
          tooltip.hide();
          button.removeAttribute("data-bs-title");
          button.removeAttribute("data-bs-toggle");
          button.removeAttribute("data-bs-placement");
        }
        button.setAttribute("title", currentTitle);
        button.classList.remove('code-copy-button-checked');
      }, 1000);
      // clear code selection
      e.clearSelection();
    }
    const getTextToCopy = function(trigger) {
      const outerScaffold = trigger.parentElement.cloneNode(true);
      const codeEl = outerScaffold.querySelector('code');
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
    const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
      text: getTextToCopy
    });
    clipboard.on('success', onCopySuccess);
    if (window.document.getElementById('quarto-embedded-source-code-modal')) {
      const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
        text: getTextToCopy,
        container: window.document.getElementById('quarto-embedded-source-code-modal')
      });
      clipboardModal.on('success', onCopySuccess);
    }
      var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
      var mailtoRegex = new RegExp(/^mailto:/);
        var filterRegex = new RegExp('/' + window.location.host + '/');
      var isInternal = (href) => {
          return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
      }
      // Inspect non-navigation links and adorn them if external
     var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
      for (var i=0; i<links.length; i++) {
        const link = links[i];
        if (!isInternal(link.href)) {
          // undo the damage that might have been done by quarto-nav.js in the case of
          // links that we want to consider external
          if (link.dataset.originalHref !== undefined) {
            link.href = link.dataset.originalHref;
          }
        }
      }
    function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
      const config = {
        allowHTML: true,
        maxWidth: 500,
        delay: 100,
        arrow: false,
        appendTo: function(el) {
            return el.parentElement;
        },
        interactive: true,
        interactiveBorder: 10,
        theme: 'quarto',
        placement: 'bottom-start',
      };
      if (contentFn) {
        config.content = contentFn;
      }
      if (onTriggerFn) {
        config.onTrigger = onTriggerFn;
      }
      if (onUntriggerFn) {
        config.onUntrigger = onUntriggerFn;
      }
      window.tippy(el, config); 
    }
    const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
    for (var i=0; i<noterefs.length; i++) {
      const ref = noterefs[i];
      tippyHover(ref, function() {
        // use id or data attribute instead here
        let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
        try { href = new URL(href).hash; } catch {}
        const id = href.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note) {
          return note.innerHTML;
        } else {
          return "";
        }
      });
    }
    const xrefs = window.document.querySelectorAll('a.quarto-xref');
    const processXRef = (id, note) => {
      // Strip column container classes
      const stripColumnClz = (el) => {
        el.classList.remove("page-full", "page-columns");
        if (el.children) {
          for (const child of el.children) {
            stripColumnClz(child);
          }
        }
      }
      stripColumnClz(note)
      if (id === null || id.startsWith('sec-')) {
        // Special case sections, only their first couple elements
        const container = document.createElement("div");
        if (note.children && note.children.length > 2) {
          container.appendChild(note.children[0].cloneNode(true));
          for (let i = 1; i < note.children.length; i++) {
            const child = note.children[i];
            if (child.tagName === "P" && child.innerText === "") {
              continue;
            } else {
              container.appendChild(child.cloneNode(true));
              break;
            }
          }
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(container);
          }
          return container.innerHTML
        } else {
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(note);
          }
          return note.innerHTML;
        }
      } else {
        // Remove any anchor links if they are present
        const anchorLink = note.querySelector('a.anchorjs-link');
        if (anchorLink) {
          anchorLink.remove();
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        if (note.classList.contains("callout")) {
          return note.outerHTML;
        } else {
          return note.innerHTML;
        }
      }
    }
    for (var i=0; i<xrefs.length; i++) {
      const xref = xrefs[i];
      tippyHover(xref, undefined, function(instance) {
        instance.disable();
        let url = xref.getAttribute('href');
        let hash = undefined; 
        if (url.startsWith('#')) {
          hash = url;
        } else {
          try { hash = new URL(url).hash; } catch {}
        }
        if (hash) {
          const id = hash.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note !== null) {
            try {
              const html = processXRef(id, note.cloneNode(true));
              instance.setContent(html);
            } finally {
              instance.enable();
              instance.show();
            }
          } else {
            // See if we can fetch this
            fetch(url.split('#')[0])
            .then(res => res.text())
            .then(html => {
              const parser = new DOMParser();
              const htmlDoc = parser.parseFromString(html, "text/html");
              const note = htmlDoc.getElementById(id);
              if (note !== null) {
                const html = processXRef(id, note);
                instance.setContent(html);
              } 
            }).finally(() => {
              instance.enable();
              instance.show();
            });
          }
        } else {
          // See if we can fetch a full url (with no hash to target)
          // This is a special case and we should probably do some content thinning / targeting
          fetch(url)
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.querySelector('main.content');
            if (note !== null) {
              // This should only happen for chapter cross references
              // (since there is no id in the URL)
              // remove the first header
              if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
                note.children[0].remove();
              }
              const html = processXRef(null, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      }, function(instance) {
      });
    }
        let selectedAnnoteEl;
        const selectorForAnnotation = ( cell, annotation) => {
          let cellAttr = 'data-code-cell="' + cell + '"';
          let lineAttr = 'data-code-annotation="' +  annotation + '"';
          const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
          return selector;
        }
        const selectCodeLines = (annoteEl) => {
          const doc = window.document;
          const targetCell = annoteEl.getAttribute("data-target-cell");
          const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
          const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
          const lines = annoteSpan.getAttribute("data-code-lines").split(",");
          const lineIds = lines.map((line) => {
            return targetCell + "-" + line;
          })
          let top = null;
          let height = null;
          let parent = null;
          if (lineIds.length > 0) {
              //compute the position of the single el (top and bottom and make a div)
              const el = window.document.getElementById(lineIds[0]);
              top = el.offsetTop;
              height = el.offsetHeight;
              parent = el.parentElement.parentElement;
            if (lineIds.length > 1) {
              const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
              const bottom = lastEl.offsetTop + lastEl.offsetHeight;
              height = bottom - top;
            }
            if (top !== null && height !== null && parent !== null) {
              // cook up a div (if necessary) and position it 
              let div = window.document.getElementById("code-annotation-line-highlight");
              if (div === null) {
                div = window.document.createElement("div");
                div.setAttribute("id", "code-annotation-line-highlight");
                div.style.position = 'absolute';
                parent.appendChild(div);
              }
              div.style.top = top - 2 + "px";
              div.style.height = height + 4 + "px";
              div.style.left = 0;
              let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
              if (gutterDiv === null) {
                gutterDiv = window.document.createElement("div");
                gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
                gutterDiv.style.position = 'absolute';
                const codeCell = window.document.getElementById(targetCell);
                const gutter = codeCell.querySelector('.code-annotation-gutter');
                gutter.appendChild(gutterDiv);
              }
              gutterDiv.style.top = top - 2 + "px";
              gutterDiv.style.height = height + 4 + "px";
            }
            selectedAnnoteEl = annoteEl;
          }
        };
        const unselectCodeLines = () => {
          const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
          elementsIds.forEach((elId) => {
            const div = window.document.getElementById(elId);
            if (div) {
              div.remove();
            }
          });
          selectedAnnoteEl = undefined;
        };
          // Handle positioning of the toggle
      window.addEventListener(
        "resize",
        throttle(() => {
          elRect = undefined;
          if (selectedAnnoteEl) {
            selectCodeLines(selectedAnnoteEl);
          }
        }, 10)
      );
      function throttle(fn, ms) {
      let throttle = false;
      let timer;
        return (...args) => {
          if(!throttle) { // first call gets through
              fn.apply(this, args);
              throttle = true;
          } else { // all the others get throttled
              if(timer) clearTimeout(timer); // cancel #2
              timer = setTimeout(() => {
                fn.apply(this, args);
                timer = throttle = false;
              }, ms);
          }
        };
      }
        // Attach click handler to the DT
        const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
        for (const annoteDlNode of annoteDls) {
          annoteDlNode.addEventListener('click', (event) => {
            const clickedEl = event.target;
            if (clickedEl !== selectedAnnoteEl) {
              unselectCodeLines();
              const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
              if (activeEl) {
                activeEl.classList.remove('code-annotation-active');
              }
              selectCodeLines(clickedEl);
              clickedEl.classList.add('code-annotation-active');
            } else {
              // Unselect the line
              unselectCodeLines();
              clickedEl.classList.remove('code-annotation-active');
            }
          });
        }
    const findCites = (el) => {
      const parentEl = el.parentElement;
      if (parentEl) {
        const cites = parentEl.dataset.cites;
        if (cites) {
          return {
            el,
            cites: cites.split(' ')
          };
        } else {
          return findCites(el.parentElement)
        }
      } else {
        return undefined;
      }
    };
    var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
    for (var i=0; i<bibliorefs.length; i++) {
      const ref = bibliorefs[i];
      const citeInfo = findCites(ref);
      if (citeInfo) {
        tippyHover(citeInfo.el, function() {
          var popup = window.document.createElement('div');
          citeInfo.cites.forEach(function(cite) {
            var citeDiv = window.document.createElement('div');
            citeDiv.classList.add('hanging-indent');
            citeDiv.classList.add('csl-entry');
            var biblioDiv = window.document.getElementById('ref-' + cite);
            if (biblioDiv) {
              citeDiv.innerHTML = biblioDiv.innerHTML;
            }
            popup.appendChild(citeDiv);
          });
          return popup.innerHTML;
        });
      }
    }
  });
  </script>
</div> <!-- /content -->
<script>var lightboxQuarto = GLightbox({"closeEffect":"zoom","descPosition":"bottom","loop":false,"openEffect":"zoom","selector":".lightbox"});
(function() {
  let previousOnload = window.onload;
  window.onload = () => {
    if (previousOnload) {
      previousOnload();
    }
    lightboxQuarto.on('slide_before_load', (data) => {
      const { slideIndex, slideNode, slideConfig, player, trigger } = data;
      const href = trigger.getAttribute('href');
      if (href !== null) {
        const imgEl = window.document.querySelector(`a[href="${href}"] img`);
        if (imgEl !== null) {
          const srcAttr = imgEl.getAttribute("src");
          if (srcAttr && srcAttr.startsWith("data:")) {
            slideConfig.href = srcAttr;
          }
        }
      } 
    });
  
    lightboxQuarto.on('slide_after_load', (data) => {
      const { slideIndex, slideNode, slideConfig, player, trigger } = data;
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(slideNode);
      }
    });
  
  };
  
})();
          </script>




</body></html>